{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Playground\n",
    "For visualization, refer to `./04-Visualize_Data-Portals.ipynb` and `./04-Visualize_Journal-Portals.ipynb` for more information."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Visualize\n",
    "### Import Libraries"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import altair as alt\n",
    "import os\n",
    "alt.data_transformers.disable_max_rows(); # Allow using rows more than 5000"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Journal Portals\n",
    "#### Basic Data Wrangling"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "\"\"\"\n",
    "Files available in GIST.\n",
    "You can find the following files in the GitHub repository as well.\n",
    "\"\"\"\n",
    "# Contains the accessibility issues for each page\n",
    "URL_EVALUATION_RESULTS = 'https://gist.githubusercontent.com/sehilyi/45ec1eb9115f969279d30fd1734b50e8/raw/68e91ff90b66df05ce4396acefb664c0c1272916/data-portal_evaluation.csv'\n",
    "\n",
    "# Contains the page types and URLs for each resource\n",
    "URL_PAGES = 'https://gist.githubusercontent.com/sehilyi/45ec1eb9115f969279d30fd1734b50e8/raw/68e91ff90b66df05ce4396acefb664c0c1272916/data-portal_pages.csv'\n",
    "\n",
    "# Contains the metadata for each resource\n",
    "URL_METADATA = 'https://gist.githubusercontent.com/sehilyi/45ec1eb9115f969279d30fd1734b50e8/raw/68e91ff90b66df05ce4396acefb664c0c1272916/data-portal_metadata.csv'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>page_id</th>\n",
       "      <th>is_success</th>\n",
       "      <th>error_count</th>\n",
       "      <th>label_missing</th>\n",
       "      <th>language_missing</th>\n",
       "      <th>button_empty</th>\n",
       "      <th>alt_missing</th>\n",
       "      <th>link_empty</th>\n",
       "      <th>aria_menu_broken</th>\n",
       "      <th>alt_link_missing</th>\n",
       "      <th>...</th>\n",
       "      <th>category_list</th>\n",
       "      <th>keywords_list</th>\n",
       "      <th>data_object_list</th>\n",
       "      <th>organism_list</th>\n",
       "      <th>theme_list</th>\n",
       "      <th>zindex</th>\n",
       "      <th>first_publication_year</th>\n",
       "      <th>search_example</th>\n",
       "      <th>cited_date</th>\n",
       "      <th>ess</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1_home</td>\n",
       "      <td>True</td>\n",
       "      <td>3.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>Pathway, Standard ontology and nomenclature</td>\n",
       "      <td>functional annotation</td>\n",
       "      <td>Animal, Plant, Fungi, Bacteria, Archaea</td>\n",
       "      <td>Homo sapiens, Mus musculus</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1847.80</td>\n",
       "      <td>2007</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2022-12-04 00:01:00</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>2_home</td>\n",
       "      <td>True</td>\n",
       "      <td>2.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>Pathway, Standard ontology and nomenclature</td>\n",
       "      <td>functional annotation</td>\n",
       "      <td>Animal, Plant, Fungi, Bacteria, Archaea, Virus</td>\n",
       "      <td>Escherichia coli, Homo sapiens</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1419.12</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2022-12-04 00:00:07</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>3_home</td>\n",
       "      <td>True</td>\n",
       "      <td>835.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>39.0</td>\n",
       "      <td>792.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>Genotype phenotype and variation, Expression, ...</td>\n",
       "      <td>tumor, cancer genome, TCGA</td>\n",
       "      <td>Animal</td>\n",
       "      <td>Homo sapiens</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1320.18</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2022-12-04 00:02:30</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>3 rows Ã— 63 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "  page_id  is_success  error_count  label_missing  language_missing  \\\n",
       "0  1_home        True          3.0            1.0               1.0   \n",
       "1  2_home        True          2.0            2.0               NaN   \n",
       "2  3_home        True        835.0            1.0               1.0   \n",
       "\n",
       "   button_empty  alt_missing  link_empty  aria_menu_broken  alt_link_missing  \\\n",
       "0           1.0          NaN         NaN               NaN               NaN   \n",
       "1           NaN          NaN         NaN               NaN               NaN   \n",
       "2           1.0         39.0       792.0               1.0               NaN   \n",
       "\n",
       "   ...                                      category_list  \\\n",
       "0  ...        Pathway, Standard ontology and nomenclature   \n",
       "1  ...        Pathway, Standard ontology and nomenclature   \n",
       "2  ...  Genotype phenotype and variation, Expression, ...   \n",
       "\n",
       "                keywords_list                                data_object_list  \\\n",
       "0       functional annotation         Animal, Plant, Fungi, Bacteria, Archaea   \n",
       "1       functional annotation  Animal, Plant, Fungi, Bacteria, Archaea, Virus   \n",
       "2  tumor, cancer genome, TCGA                                          Animal   \n",
       "\n",
       "                    organism_list  theme_list   zindex  \\\n",
       "0      Homo sapiens, Mus musculus         NaN  1847.80   \n",
       "1  Escherichia coli, Homo sapiens         NaN  1419.12   \n",
       "2                    Homo sapiens         NaN  1320.18   \n",
       "\n",
       "   first_publication_year  search_example           cited_date  ess  \n",
       "0                    2007             NaN  2022-12-04 00:01:00    0  \n",
       "1                       0             NaN  2022-12-04 00:00:07    0  \n",
       "2                       0             NaN  2022-12-04 00:02:30    0  \n",
       "\n",
       "[3 rows x 63 columns]"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\"\"\"\n",
    "Merge data for visualization\n",
    "\"\"\"\n",
    "df = pd.read_csv(URL_EVALUATION_RESULTS)\n",
    "\n",
    "# Can be useful for visualization\n",
    "ERROR_TYPES = df.columns.tolist()\n",
    "ERROR_TYPES.remove('page_id')\n",
    "ERROR_TYPES.remove('is_success')\n",
    "ERROR_TYPES.remove('error_count')\n",
    "\n",
    "# Add `id` of resources and `page_type` of pages\n",
    "df_pages = pd.read_csv(URL_PAGES)\n",
    "df = df.merge(df_pages[['id', 'page_id', 'page_type']], left_on='page_id', right_on='page_id', how='left')\n",
    "\n",
    "# Add metadata of resources\n",
    "df_meta = pd.read_csv(URL_METADATA)\n",
    "df_meta.drop(columns=['url'], inplace=True)\n",
    "df = df.merge(df_meta, left_on='id', right_on='id', how='left')\n",
    "\n",
    "df.head(3)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Let's add some useful columns for visualization\n",
    "df['has_error'] = df.error_count.apply(lambda x: x > 0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Index(['page_id', 'is_success', 'error_count', 'label_missing',\n",
       "       'language_missing', 'button_empty', 'alt_missing', 'link_empty',\n",
       "       'aria_menu_broken', 'alt_link_missing', 'th_empty', 'heading_empty',\n",
       "       'link_skip_broken', 'alt_spacer_missing', 'title_invalid',\n",
       "       'aria_reference_broken', 'alt_input_missing', 'alt_area_missing',\n",
       "       'alt_map_missing', 'label_empty', 'label_multiple', 'id', 'page_type',\n",
       "       'source_id', 'short_name', 'accession', 'full_name', 'description',\n",
       "       'founded_year', 'host_institution', 'address', 'city', 'province',\n",
       "       'country', 'current_version', 'contact_name', 'contact_email',\n",
       "       'available_protocol', 'last_update', 'age', 'citation', 'token',\n",
       "       'expire_date', 'ins', 'group_email', 'is_partner', 'funding',\n",
       "       'submitter', 'is_new', 'bigsearch_id', 'record_created',\n",
       "       'last_modified', 'data_type_list', 'category_list', 'keywords_list',\n",
       "       'data_object_list', 'organism_list', 'theme_list', 'zindex',\n",
       "       'first_publication_year', 'search_example', 'cited_date', 'ess'],\n",
       "      dtype='object')"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Available columns\n",
    "df.columns"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Create Altair Plots!\n",
    "Reference: https://altair-viz.github.io/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "<style>\n",
       "  #altair-viz-5b09532e7fe844c9b8b53ead3edc2124.vega-embed {\n",
       "    width: 100%;\n",
       "    display: flex;\n",
       "  }\n",
       "\n",
       "  #altair-viz-5b09532e7fe844c9b8b53ead3edc2124.vega-embed details,\n",
       "  #altair-viz-5b09532e7fe844c9b8b53ead3edc2124.vega-embed details summary {\n",
       "    position: relative;\n",
       "  }\n",
       "</style>\n",
       "<div id=\"altair-viz-5b09532e7fe844c9b8b53ead3edc2124\"></div>\n",
       "<script type=\"text/javascript\">\n",
       "  var VEGA_DEBUG = (typeof VEGA_DEBUG == \"undefined\") ? {} : VEGA_DEBUG;\n",
       "  (function(spec, embedOpt){\n",
       "    let outputDiv = document.currentScript.previousElementSibling;\n",
       "    if (outputDiv.id !== \"altair-viz-5b09532e7fe844c9b8b53ead3edc2124\") {\n",
       "      outputDiv = document.getElementById(\"altair-viz-5b09532e7fe844c9b8b53ead3edc2124\");\n",
       "    }\n",
       "    const paths = {\n",
       "      \"vega\": \"https://cdn.jsdelivr.net/npm/vega@5?noext\",\n",
       "      \"vega-lib\": \"https://cdn.jsdelivr.net/npm/vega-lib?noext\",\n",
       "      \"vega-lite\": \"https://cdn.jsdelivr.net/npm/vega-lite@5.15.1?noext\",\n",
       "      \"vega-embed\": \"https://cdn.jsdelivr.net/npm/vega-embed@6?noext\",\n",
       "    };\n",
       "\n",
       "    function maybeLoadScript(lib, version) {\n",
       "      var key = `${lib.replace(\"-\", \"\")}_version`;\n",
       "      return (VEGA_DEBUG[key] == version) ?\n",
       "        Promise.resolve(paths[lib]) :\n",
       "        new Promise(function(resolve, reject) {\n",
       "          var s = document.createElement('script');\n",
       "          document.getElementsByTagName(\"head\")[0].appendChild(s);\n",
       "          s.async = true;\n",
       "          s.onload = () => {\n",
       "            VEGA_DEBUG[key] = version;\n",
       "            return resolve(paths[lib]);\n",
       "          };\n",
       "          s.onerror = () => reject(`Error loading script: ${paths[lib]}`);\n",
       "          s.src = paths[lib];\n",
       "        });\n",
       "    }\n",
       "\n",
       "    function showError(err) {\n",
       "      outputDiv.innerHTML = `<div class=\"error\" style=\"color:red;\">${err}</div>`;\n",
       "      throw err;\n",
       "    }\n",
       "\n",
       "    function displayChart(vegaEmbed) {\n",
       "      vegaEmbed(outputDiv, spec, embedOpt)\n",
       "        .catch(err => showError(`Javascript Error: ${err.message}<br>This usually means there's a typo in your chart specification. See the javascript console for the full traceback.`));\n",
       "    }\n",
       "\n",
       "    if(typeof define === \"function\" && define.amd) {\n",
       "      requirejs.config({paths});\n",
       "      require([\"vega-embed\"], displayChart, err => showError(`Error loading script: ${err.message}`));\n",
       "    } else {\n",
       "      maybeLoadScript(\"vega\", \"5\")\n",
       "        .then(() => maybeLoadScript(\"vega-lite\", \"5.15.1\"))\n",
       "        .then(() => maybeLoadScript(\"vega-embed\", \"6\"))\n",
       "        .catch(showError)\n",
       "        .then(() => displayChart(vegaEmbed));\n",
       "    }\n",
       "  })({\"config\": {\"view\": {\"continuousWidth\": 300, \"continuousHeight\": 300}}, \"hconcat\": [{\"mark\": {\"type\": \"bar\"}, \"encoding\": {\"x\": {\"field\": \"page_type\", \"title\": \"Page Type\", \"type\": \"nominal\"}, \"y\": {\"aggregate\": \"sum\", \"field\": \"error_count\", \"title\": \"A Total of Error Counts\", \"type\": \"quantitative\"}}}, {\"mark\": {\"type\": \"point\"}, \"encoding\": {\"x\": {\"field\": \"citation\", \"type\": \"quantitative\"}, \"y\": {\"field\": \"error_count\", \"type\": \"quantitative\"}}}, {\"mark\": {\"type\": \"bar\"}, \"encoding\": {\"x\": {\"field\": \"country\", \"type\": \"nominal\"}, \"y\": {\"aggregate\": \"mean\", \"field\": \"error_count\", \"title\": \"The Average of Error Counts\", \"type\": \"quantitative\"}}}, {\"mark\": {\"type\": \"bar\"}, \"encoding\": {\"x\": {\"field\": \"country\", \"type\": \"nominal\"}, \"y\": {\"aggregate\": \"count\", \"field\": \"error_count\", \"title\": \"The Number of Data Portals\", \"type\": \"quantitative\"}}}, {\"mark\": {\"type\": \"bar\"}, \"encoding\": {\"x\": {\"field\": \"founded_year\", \"type\": \"ordinal\"}, \"y\": {\"aggregate\": \"mean\", \"field\": \"error_count\", \"title\": \"The Average of Error Counts\", \"type\": \"quantitative\"}}}], \"data\": {\"name\": \"data-c53e1ec6ed680e90a5eb33a1d31d139c\"}, \"$schema\": \"https://vega.github.io/schema/vega-lite/v5.15.1.json\", \"datasets\": {\"data-c53e1ec6ed680e90a5eb33a1d31d139c\": [{\"page_id\": \"1_home\", \"is_success\": true, \"error_count\": 3.0, \"label_missing\": 1.0, \"language_missing\": 1.0, \"button_empty\": 1.0, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 1, \"page_type\": \"home\", \"source_id\": \"dc_3061\", \"short_name\": \"DAVID\", \"accession\": \"DBC003061\", \"full_name\": \"Database for Annotation, Visualization and Integrated Discovery\", \"description\": \"The Database for Annotation, Visualization and Integrated Discovery (DAVID) comprises a full knowledgebase update to the sixth version of our original web-accessible programs. DAVID now provides a comprehensive set of functional annotation tools for investigators to understand biological meaning behind large list of genes. \", \"founded_year\": 2003, \"host_institution\": \"National Cancer Institute\", \"address\": \" Laboratory of Immunopathogenesis and Bioinformatics, National Cancer Institute, Frederick, MD 21702, USA\", \"city\": \"Frederick\", \"province\": \"Maryland\", \"country\": \"United States\", \"current_version\": \"version6.8\", \"contact_name\": \"DAVID Bioinformatic Team\", \"contact_email\": \"weizhong.chang@nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2022-2-19\", \"age\": 20, \"citation\": 36956, \"token\": \"$2a$10$zwt9PsSE9s4OW93qMVcFwe3j0lOsvk1m.vZgQ/V4/plC7WQ30GO0G\", \"expire_date\": 1543995762746.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1516919168000, \"last_modified\": \"2023-03-03 10:12:16\", \"data_type_list\": \"DNA\", \"category_list\": \"Pathway, Standard ontology and nomenclature\", \"keywords_list\": \"functional annotation\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 1847.8, \"first_publication_year\": 2007, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:01:00\", \"ess\": 0}, {\"page_id\": \"2_home\", \"is_success\": true, \"error_count\": 2.0, \"label_missing\": 2.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 2, \"page_type\": \"home\", \"source_id\": \"dc_184\", \"short_name\": \"KEGG\", \"accession\": \"DBC000184\", \"full_name\": \"Kyoto Encyclopedia of Genes and Genomes\", \"description\": \"KEGG is a database resource for understanding high-level functions and utilities of the biological system.\", \"founded_year\": 1997, \"host_institution\": \"Kyoto University\", \"address\": \"Uji,Kyoto 611-0011,Japan\", \"city\": \"Kyoto\", \"province\": null, \"country\": \"Japan\", \"current_version\": \"v86.1\", \"contact_name\": \"Minoru Kanehisa\", \"contact_email\": \"kanehisa@kuicr.kyoto-u.ac.jp\", \"available_protocol\": 1.0, \"last_update\": \"2018-05-10\", \"age\": 26, \"citation\": 36897, \"token\": \"$2a$10$PoC/2ktUbVoHS7Jjdb1R9.1vMe93zm3TG7TsKiN9MvpCeZ1aiLro.\", \"expire_date\": 1543898963375.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:53\", \"data_type_list\": \"Other\", \"category_list\": \"Pathway, Standard ontology and nomenclature\", \"keywords_list\": \"functional annotation\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Escherichia coli, Homo sapiens\", \"theme_list\": null, \"zindex\": 1419.12, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:07\", \"ess\": 0}, {\"page_id\": \"3_home\", \"is_success\": true, \"error_count\": 835.0, \"label_missing\": 1.0, \"language_missing\": 1.0, \"button_empty\": 1.0, \"alt_missing\": 39.0, \"link_empty\": 792.0, \"aria_menu_broken\": 1.0, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 3, \"page_type\": \"home\", \"source_id\": \"dc_6569\", \"short_name\": \"cBioPortal\", \"accession\": \"DBC006569\", \"full_name\": \"cBio Cancer Genomics Portal\", \"description\": \"cBio Cancer Genomics Portal is an open-access resource for interactive exploration of multidimensional cancer genomics data sets, currently providing access to data from more than 5,000 tumor samples from 20 cancer studies.\", \"founded_year\": 2012, \"host_institution\": \"Memorial Sloan Kettering Cancer Center\", \"address\": \"Computational Biology Center, Memorial Sloan-Kettering Cancer Center, 1275 York Avenue, New York, NY 10021.\", \"city\": \"New York\", \"province\": null, \"country\": \"United States\", \"current_version\": \"3.7.9\", \"contact_name\": \"Ethan Cerami\", \"contact_email\": \"cancergenomics@cbio.mskcc.org\", \"available_protocol\": 1.0, \"last_update\": \"2021-10-05\", \"age\": 11, \"citation\": 14522, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1571795759000, \"last_modified\": \"2023-03-03 10:16:31\", \"data_type_list\": \"DNA, RNA, Protein\", \"category_list\": \"Genotype phenotype and variation, Expression, Modification, Structure, Health and medicine\", \"keywords_list\": \"tumor, cancer genome, TCGA\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 1320.18, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:30\", \"ess\": 0}, {\"page_id\": \"4_home\", \"is_success\": true, \"error_count\": 48.0, \"label_missing\": 46.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 2.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 4, \"page_type\": \"home\", \"source_id\": \"dc_62\", \"short_name\": \"STRING\", \"accession\": \"DBC000062\", \"full_name\": \"Known and Predicted Protein-Protein Interactions\", \"description\": \"STRING is a database of known and predicted protein interactions. The interactions include direct (physical) and indirect (functional) associations\", \"founded_year\": 2000, \"host_institution\": \"Swiss Institute of Bioinformatics\", \"address\": \"Quartier Sorge - Batiment Amphipole\\n1015 Lausanne / Switzerland\", \"city\": null, \"province\": null, \"country\": \"Switzerland\", \"current_version\": \"v11.5\", \"contact_name\": \"Peer Bork\", \"contact_email\": \"bork@embl.de\", \"available_protocol\": 1.0, \"last_update\": null, \"age\": 23, \"citation\": 22950, \"token\": \"$2a$10$qCP054uAASyeYX0FoxplH.cHAIM4TlBWV1lqo/nPXM69GAAGR1HyS\", \"expire_date\": 1543898876141.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-08-23 10:25:04\", \"data_type_list\": \"Protein\", \"category_list\": \"Interaction\", \"keywords_list\": \"protein-protein interaction (PPI)\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea\", \"organism_list\": \"Homo sapiens, Methanococcus voltae, Arabidopsis thaliana\", \"theme_list\": null, \"zindex\": 997.826, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:04\", \"ess\": 0}, {\"page_id\": \"5_home\", \"is_success\": true, \"error_count\": 4.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": 3.0, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 5, \"page_type\": \"home\", \"source_id\": \"dc_317\", \"short_name\": \"ENCODE\", \"accession\": \"DBC000317\", \"full_name\": \"Encyclopedia of DNA Elements\", \"description\": \"The ENCODE Consortium is an international collaboration of research groups funded by the National Human Genome Research Institute (NHGRI). The goal of ENCODE is to build a comprehensive parts list of functional elements in the human genome, including elements that act at the protein and RNA levels,and regulatory elements that control cells and circumstances in which a gene is active.\", \"founded_year\": 2004, \"host_institution\": \"Stanford University\", \"address\": \"Center for Biomolecular Science and Engineering, School of Engineering, University of California Santa Cruz (UCSC), Santa Cruz, CA 95064, USA.\", \"city\": \"Stanford\", \"province\": \"CA\", \"country\": \"United States\", \"current_version\": \"v128\", \"contact_name\": \"ENCODE team\", \"contact_email\": \"encode-help@lists.stanford.edu\", \"available_protocol\": 1.0, \"last_update\": null, \"age\": 19, \"citation\": 17908, \"token\": \"$2a$10$jHL5wkABrxhdzzz065t9I..ybMHhGl2yVaf2jFfx.bmUX8d4z5euu\", \"expire_date\": 1543899046887.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-24 22:11:42\", \"data_type_list\": \"DNA, Protein, RNA\", \"category_list\": \"Gene genome and annotation, Raw bio-data\", \"keywords_list\": \"functional element, regulatory element\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 942.526, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:11\", \"ess\": 0}, {\"page_id\": \"6_home\", \"is_success\": true, \"error_count\": 11.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": 10.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 6, \"page_type\": \"home\", \"source_id\": \"dc_29\", \"short_name\": \"UniProt\", \"accession\": \"DBC000029\", \"full_name\": \"Universal Protein Resource\", \"description\": \"UniProt provides the scientific community with a comprehensive,high-quality and freely accessible resource of protein sequence and functional information.\", \"founded_year\": 2004, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SD, UK\", \"city\": \"Cambridge\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"2022_05\", \"contact_name\": \"Alex Bateman\", \"contact_email\": \"agb@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2022-12-14\", \"age\": 19, \"citation\": 17081, \"token\": \"$2a$10$KXuraWjdOou1ypsILkffj.prKtTSD8gKiDnv6u5FxZcsBrvur0oxC\", \"expire_date\": 1543898855446.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-06 11:06:30\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation, Structure, Literature\", \"keywords_list\": \"protein sequence, protein structure, protein function\", \"data_object_list\": \"Fungi, Bacteria, Archaea, Virus, Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 899.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:03\", \"ess\": 0}, {\"page_id\": \"7_home\", \"is_success\": true, \"error_count\": 3.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": 1.0, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 7, \"page_type\": \"home\", \"source_id\": \"dc_1861\", \"short_name\": \"IGSR\", \"accession\": \"DBC001861\", \"full_name\": \"International Genome Sample Resource\", \"description\": \"IGSR expands in data type and population diversity the resources from the 1000 Genomes Project. IGSR represents the largest open collection of human variation data and provides easy access to these resources.\", \"founded_year\": 2015, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"Wellcome Genome Campus, Hinxton, Cambridge CB10 1SD\", \"city\": \"Hinxton\", \"province\": \"Cambridge\", \"country\": \"United Kingdom\", \"current_version\": null, \"contact_name\": \"IGSR team\", \"contact_email\": \"info@1000genomes.org\", \"available_protocol\": 1.0, \"last_update\": \"2021-09-30\", \"age\": 8, \"citation\": 6498, \"token\": \"$2a$10$cMNRhQ8RWxiMGUSGneIVD.n73MeMYb.kPThr8uI9mU5CAhA0k6NAq\", \"expire_date\": 1543995395760.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1487560853000, \"last_modified\": \"2023-03-03 10:11:56\", \"data_type_list\": \"DNA\", \"category_list\": \"Genotype phenotype and variation\", \"keywords_list\": \"1000 Genome\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 812.25, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:53\", \"ess\": 0}, {\"page_id\": \"8_home\", \"is_success\": true, \"error_count\": 7.0, \"label_missing\": 7.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 8, \"page_type\": \"home\", \"source_id\": \"dc_186\", \"short_name\": \"Pfam\", \"accession\": \"DBC000186\", \"full_name\": \"Protein Families Database\", \"description\": \" A database of conserved protein families and domains. Pfam is a member database of InterPro. Pfam data is now accessible via the InterPro website.\", \"founded_year\": 1997, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI), Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SD, UK\", \"city\": \"Cambridge\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"v35\", \"contact_name\": \"Pfam team\", \"contact_email\": \"pfam-help@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2021-11-01\", \"age\": 26, \"citation\": 20861, \"token\": \"$2a$10$iaZh/Uo3kRz3BQkSgni3z.K4IIWtGzwDS9ZSCCzIhb6Ex13Rc3PLC\", \"expire_date\": 1543898964393.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-10-09 11:21:36\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation, Raw bio-data, Structure\", \"keywords_list\": \"protein domain, protein family, SARS-CoV-2\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 802.346, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:07\", \"ess\": 0}, {\"page_id\": \"9_home\", \"is_success\": true, \"error_count\": 2.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": 2.0, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 9, \"page_type\": \"home\", \"source_id\": \"dc_238\", \"short_name\": \"SILVA\", \"accession\": \"DBC000238\", \"full_name\": \"High quality ribosomal RNA databases\", \"description\": \"SILVA is a comprehensive, quality-controlled web resource for up-to-date aligned ribosomal RNA (rRNA) gene sequences from the Bacteria, Archaea and Eukaryota domains alongside supplementary online services. In addition to data products, SILVA provides various online tools such as alignment and classification, phylogenetic tree calculation and viewer, probe/primer matching, and an amplicon analysis pipeline. With every full release a curated guide tree is provided that contains the latest taxonomy and nomenclature based on multiple references. SILVA is an ELIXIR Core Data Resource.\", \"founded_year\": 2007, \"host_institution\": \"Max Planck Institute for Marine Microbiology\", \"address\": \"D-28359 Bremen, Germany\", \"city\": \"Bremen\", \"province\": null, \"country\": \"Germany\", \"current_version\": \"v138.1\", \"contact_name\": \"Frank Oliver Gl\\u00f6ckner\", \"contact_email\": \"contact@arb-silva.de\", \"available_protocol\": 1.0, \"last_update\": \"2020-08-27\", \"age\": 16, \"citation\": 12280, \"token\": \"$2a$10$s2ECbbpHh9YQkSCW0xlMjeAxCT03qE6IHk3l35SVoX1nuf669nvte\", \"expire_date\": 1543898998134.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:57\", \"data_type_list\": \"RNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"rRNA\", \"data_object_list\": \"Animal, Bacteria, Archaea\", \"organism_list\": null, \"theme_list\": null, \"zindex\": 767.5, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:08\", \"ess\": 0}, {\"page_id\": \"10_home\", \"is_success\": true, \"error_count\": 2.0, \"label_missing\": 2.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 10, \"page_type\": \"home\", \"source_id\": \"dc_6934\", \"short_name\": \"gnomAD\", \"accession\": \"DBC006934\", \"full_name\": \"Genome Aggregation Database\", \"description\": \"The Genome Aggregation Database (gnomAD) is a resource developed by an international coalition of investigators, with the goal of aggregating and harmonizing both exome and genome sequencing data from a wide variety of large-scale sequencing projects, and making summary data available for the wider scientific community.\", \"founded_year\": 2020, \"host_institution\": \"Broad Institute\", \"address\": null, \"city\": null, \"province\": null, \"country\": \"United States\", \"current_version\": \"v2.1.1 & v3\", \"contact_name\": \"Genome Aggregation Database Consortium\", \"contact_email\": \"exomeconsortium@gmail.com\", \"available_protocol\": 1.0, \"last_update\": null, \"age\": 3, \"citation\": 2300, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1594014179000, \"last_modified\": \"2023-03-03 10:16:54\", \"data_type_list\": \"DNA\", \"category_list\": \"Raw bio-data, Gene genome and annotation, Genotype phenotype and variation\", \"keywords_list\": \"human, genome, variation\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 766.667, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:37\", \"ess\": 0}, {\"page_id\": \"11_home\", \"is_success\": true, \"error_count\": 8.0, \"label_missing\": 5.0, \"language_missing\": null, \"button_empty\": 2.0, \"alt_missing\": 1.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 11, \"page_type\": \"home\", \"source_id\": \"dc_165\", \"short_name\": \"miRBase\", \"accession\": \"DBC000165\", \"full_name\": \"microRNA database\", \"description\": \"miRBase is the central repository for microRNA (miRNA) sequence information. miRBase has a role in defining the nomenclature for miRNA genes and assigning names to novel miRNAs for publication in peer-reviewed journals. The online miRBase database is a resource containing all published miRNA sequences, together with textual annotation and links to the primary literature and to other secondary databases. The database provides a variety of methods to query the data, by specific searches of sequences and associated text and literature.\", \"founded_year\": 2004, \"host_institution\": \"University of Manchester\", \"address\": \"Manchester, M13 9PT, UK\", \"city\": \"Manchester\", \"province\": null, \"country\": \"United Kingdom\", \"current_version\": \"v22.1\", \"contact_name\": \"Sam Griffiths-Jones\", \"contact_email\": \"sam.griffiths-jones@manchester.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2018-10\", \"age\": 19, \"citation\": 13219, \"token\": \"$2a$10$RXHT0aNg5L9bRBUSCAtcP.Mr.74LjE/7M2IHyx.dWBgnGYQmhBvjG\", \"expire_date\": 1543898952074.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:51\", \"data_type_list\": \"RNA\", \"category_list\": \"Standard ontology and nomenclature, Gene genome and annotation, Literature\", \"keywords_list\": \"miRNA, microRNA\", \"data_object_list\": \"Animal, Plant, Virus\", \"organism_list\": \"Arabidopsis thaliana, Drosophila melanogaster, Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 695.737, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:07\", \"ess\": 0}, {\"page_id\": \"12_home\", \"is_success\": true, \"error_count\": 6.0, \"label_missing\": null, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": 1.0, \"link_empty\": 3.0, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 12, \"page_type\": \"home\", \"source_id\": \"dc_93\", \"short_name\": \"UCSC Genome Browser\", \"accession\": \"DBC000093\", \"full_name\": \"University of California Santa Cruz Genome Browser Database\", \"description\": \"Launched in 2001 to showcase the draft human genome assembly, the UCSC Genome Browser database (http://genome.ucsc.edu) and associated tools continue to grow, providing a comprehensive resource of genome assemblies and annotations to scientists and students worldwide.\", \"founded_year\": 2002, \"host_institution\": \"University of California Santa Cruz\", \"address\": \"Santa Cruz, CA 95064, USA\", \"city\": \"Santa Cruz\", \"province\": \"CA\", \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Cath Tyner\", \"contact_email\": \"cath@ucsc.edu\", \"available_protocol\": 1.0, \"last_update\": \"2022-04-02\", \"age\": 21, \"citation\": 14446, \"token\": \"$2a$10$lzBoRmzKBCSq1H2T9dZXgucqlni3xTPk7bUs5ckBWSdn16W3Ae5ya\", \"expire_date\": 1543898901943.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-10-09 14:43:15\", \"data_type_list\": \"DNA, Protein, RNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"genome assembly, genome browser, genome alignment, SARS-CoV-2\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 687.905, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:04\", \"ess\": 0}, {\"page_id\": \"13_home\", \"is_success\": true, \"error_count\": 1.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 13, \"page_type\": \"home\", \"source_id\": \"dc_424\", \"short_name\": \"GEO\", \"accession\": \"DBC000424\", \"full_name\": \"Gene Expression Omnibus\", \"description\": \"The Gene Expression Omnibus is an international public repository for high-throughput microarray and next-generation sequence functional genomic data sets submitted by the research community.\", \"founded_year\": 2002, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"Bethesda,MD 20892,USA\", \"city\": \"Bethesda\", \"province\": \"MD\", \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Tanya Barrett\", \"contact_email\": \"barrett@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2015-12-30\", \"age\": 21, \"citation\": 14444, \"token\": \"$2a$10$CTgiIN2tFI9mB6cj8NXWNuLXdbqZ6TUnJ3PpmV/6mCmuKG04tmWju\", \"expire_date\": 1543906755125.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:10:13\", \"data_type_list\": \"RNA, DNA, Protein\", \"category_list\": \"Expression, Raw bio-data, Metadata\", \"keywords_list\": \"NGS, microarray, RNA-Seq, omics data\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 687.81, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:14\", \"ess\": 0}, {\"page_id\": \"14_home\", \"is_success\": true, \"error_count\": 14.0, \"label_missing\": 4.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 6.0, \"link_empty\": 2.0, \"aria_menu_broken\": null, \"alt_link_missing\": 2.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 14, \"page_type\": \"home\", \"source_id\": \"dc_6329\", \"short_name\": \"GTEx\", \"accession\": \"DBC006329\", \"full_name\": \"Genotype-Tissue Expression \", \"description\": \"GTEx established a data resource and tissue bank to study the relationship between genetic variation and gene expression in multiple human tissues. This release includes genotype data from approximately 714 donors and approximately 11688 RNA-seq samples across 53 tissue sites and 2 cell lines, with adequate power to detect Expression Quantitative Trait Loci in 48 tissues.\\n\", \"founded_year\": 2013, \"host_institution\": \"Broad Institute\", \"address\": \"9000 Rockville Pike, Bethesda, Maryland 20892\", \"city\": \"Bethesda\", \"province\": \"Maryland\", \"country\": \"United States\", \"current_version\": \"v8\", \"contact_name\": \"GTEx consortium\", \"contact_email\": \"volpis@mail.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2019-7-24\", \"age\": 10, \"citation\": 6832, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1564463149000, \"last_modified\": \"2023-03-03 10:16:14\", \"data_type_list\": \"DNA, RNA\", \"category_list\": \"Genotype phenotype and variation, Expression\", \"keywords_list\": \"normal tissue, tissue site, eQTL, RNA-seq\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 683.2, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:25\", \"ess\": 0}, {\"page_id\": \"15_home\", \"is_success\": true, \"error_count\": 11.0, \"label_missing\": 3.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 6.0, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": 1.0, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 15, \"page_type\": \"home\", \"source_id\": \"dc_92\", \"short_name\": \"Ensembl\", \"accession\": \"DBC000092\", \"full_name\": \"genome browser for vertebrate genomes\", \"description\": \"Ensembl is a genome browser for vertebrate genomes that supports research in comparative genomics, evolution, sequence variation and transcriptional regulation. Ensembl annotate genes, computes multiple alignments, predicts regulatory function and collects disease data. Ensembl tools include BLAST, BLAT, BioMart and the Variant Effect Predictor (VEP) for all supported species.\", \"founded_year\": 2002, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"European Molecular Biology Laboratory, European Bioinformatics Institute, Wellcome Genome Campus, Hinxton, CB10 1SD, Cambridge, UK.\", \"city\": \"Cambridge\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"release 110\", \"contact_name\": \"Fergal J Martin\", \"contact_email\": \"fergal@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2023-07\", \"age\": 21, \"citation\": 11979, \"token\": \"$2a$10$vQ..9kDsX7/gRQh/fnLlT.JxiBicHQFIusL22pepdl4VvfvmoWfnW\", \"expire_date\": 1543898901295.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-09-13 22:33:18\", \"data_type_list\": \"DNA, Protein, RNA\", \"category_list\": \"Gene genome and annotation, Genotype phenotype and variation, Phylogeny and homology, Interaction\", \"keywords_list\": \"protein-coding gene, pseudogene, lncRNA\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus, Danio rerio, Sus scrofa\", \"theme_list\": \"lncRNA\", \"zindex\": 570.429, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:04\", \"ess\": 0}, {\"page_id\": \"16_home\", \"is_success\": true, \"error_count\": 0.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 16, \"page_type\": \"home\", \"source_id\": \"dc_6331\", \"short_name\": \"TCGA\", \"accession\": \"DBC006331\", \"full_name\": \"The Cancer Genome Atlas\", \"description\": \"A landmark cancer genomics program, molecularly characterized over 20,000 primary cancer and matched normal samples spanning 33 cancer types. \", \"founded_year\": 2013, \"host_institution\": \"National Cancer Institute\", \"address\": \"Center for Cancer Genomics\\n31 Center Drive, Room 10A11\\nBethesda, MD 20892\", \"city\": \"Bethesda\", \"province\": null, \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Harold Varmus\", \"contact_email\": \"tcga@mail.nih.gov\", \"available_protocol\": 1.0, \"last_update\": null, \"age\": 10, \"citation\": 5085, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1564636398000, \"last_modified\": \"2023-03-03 10:16:14\", \"data_type_list\": \"DNA, RNA, Protein\", \"category_list\": \"Genotype phenotype and variation, Expression, Structure, Health and medicine, Modification\", \"keywords_list\": \"cancer, human\", \"data_object_list\": null, \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 508.5, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:25\", \"ess\": 0}, {\"page_id\": \"17_home\", \"is_success\": true, \"error_count\": 4.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 2.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 2.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 17, \"page_type\": \"home\", \"source_id\": \"dc_7433\", \"short_name\": \"AlphaFold DB\", \"accession\": \"DBC007433\", \"full_name\": \"AlphaFold Protein Structure Database\", \"description\": \"AlphaFold DB provides open access to AlphaFold protein structure predictions for the human proteome and other key organisms to accelerate scientific research.\", \"founded_year\": 2022, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"EMBL-EBI, Wellcome Genome Campus, Hinxton, Cambridgeshire, CB10 1SD, UK. +44 (0)1223 49 44 44\", \"city\": \"Hinxton\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": null, \"contact_name\": \"Sameer Velankar\", \"contact_email\": \"afdbhelp@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2021\", \"age\": 1, \"citation\": 476, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1634611017000, \"last_modified\": \"2023-03-03 10:17:30\", \"data_type_list\": \"Protein\", \"category_list\": \"Structure\", \"keywords_list\": \"AI, DeepMind, 3D structure\", \"data_object_list\": \"Animal, Plant, Bacteria, Archaea\", \"organism_list\": \"Homo sapiens, Arabidopsis thaliana, Caenorhabditis elegans, Candida albicans\", \"theme_list\": null, \"zindex\": 476.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:48\", \"ess\": 0}, {\"page_id\": \"18_home\", \"is_success\": true, \"error_count\": 7.0, \"label_missing\": 2.0, \"language_missing\": null, \"button_empty\": 2.0, \"alt_missing\": 1.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 2.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 18, \"page_type\": \"home\", \"source_id\": \"dc_1865\", \"short_name\": \"GWAS Catalog\", \"accession\": \"DBC001865\", \"full_name\": null, \"description\": \"The GWAS Catalog is a publicly available, manually curated resource of all published GWAS and association results, collaboratively produced and developed by the NHGRI and EMBL-EBI.\", \"founded_year\": 2009, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"Wellcome Genome Campus, Hinxton, Cambridge, CB10 1SD\", \"city\": \"Hinxton\", \"province\": \"Cambridge\", \"country\": \"United Kingdom\", \"current_version\": null, \"contact_name\": \"GWAS Catalog team\", \"contact_email\": \"gwas-info@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2017-01-01\", \"age\": 14, \"citation\": 6367, \"token\": \"$2a$10$aGMaTxqVlB0PDuY5ROMgK.4EG9PhdGTLnvWHj4zcp6FFJHm4IiII6\", \"expire_date\": 1543995396646.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1487571954000, \"last_modified\": \"2023-03-03 10:11:56\", \"data_type_list\": \"DNA\", \"category_list\": \"Genotype phenotype and variation\", \"keywords_list\": \"Genome-wide association studies (GWAS), SNP\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 454.786, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:54\", \"ess\": 0}, {\"page_id\": \"19_home\", \"is_success\": true, \"error_count\": 9.0, \"label_missing\": 2.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 3.0, \"link_empty\": 2.0, \"aria_menu_broken\": null, \"alt_link_missing\": 2.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 19, \"page_type\": \"home\", \"source_id\": \"dc_5009\", \"short_name\": \"GtoPdb\", \"accession\": \"DBC005009\", \"full_name\": \"IUPHAR/BPS Guide to PHARMACOLOGY\", \"description\": \"GtoPdb provides a searchable database with quantitative information on drug targets and the prescription medicines and experimental drugs that act on them.\", \"founded_year\": 2011, \"host_institution\": \"University of Edinburgh\", \"address\": \"Centre for Integrative Physiology, University of Edinburgh, Edinburgh, UK\", \"city\": null, \"province\": null, \"country\": \"United Kingdom\", \"current_version\": \"Release 2019.4\", \"contact_name\": \"Jamie A Davies\", \"contact_email\": \"j.davies@trinitycollegebristol.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2019-09-18\", \"age\": 12, \"citation\": 5208, \"token\": \"$2a$10$BRpnffTZzR9ulTOcpQeqoeTGqE73D2ZZ9e4TlbAkI4V/WhMgchRFq\", \"expire_date\": 1544148156020.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1517168584000, \"last_modified\": \"2023-10-10 10:17:41\", \"data_type_list\": \"Protein\", \"category_list\": \"Interaction, Health and medicine\", \"keywords_list\": \"pharmacology, ligand and target, SARS-CoV-2\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 434.0, \"first_publication_year\": 2015, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:01:55\", \"ess\": 1}, {\"page_id\": \"20_home\", \"is_success\": true, \"error_count\": 3.0, \"label_missing\": null, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 20, \"page_type\": \"home\", \"source_id\": \"dc_6158\", \"short_name\": \"GENCODE\", \"accession\": \"DBC006158\", \"full_name\": \"reference annotation for the human and mouse genomes\", \"description\": \"The goal of the GENCODE project is to identify and classify all gene features in the human and mouse genomes with high accuracy based on biological evidence, and to release these annotations for the benefit of biomedical research and genome interpretation.\", \"founded_year\": 2006, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"European Bioinformatics Institute, Wellcome Genome Campus, Hinxton, Cambridge CB10 1SD, UK\", \"city\": \"Cambridge\", \"province\": null, \"country\": \"United Kingdom\", \"current_version\": \"Release 44\", \"contact_name\": \"Paul Flicek\", \"contact_email\": \"flicek@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2023-07-17\", \"age\": 17, \"citation\": 7278, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1546534020000, \"last_modified\": \"2023-09-13 20:27:38\", \"data_type_list\": \"DNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"lncRNA, pseudogene, alternative splicing\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": \"lncRNA\", \"zindex\": 428.118, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:23\", \"ess\": 0}, {\"page_id\": \"21_home\", \"is_success\": true, \"error_count\": 4.0, \"label_missing\": 2.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": 1.0, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 21, \"page_type\": \"home\", \"source_id\": \"dc_433\", \"short_name\": \"PRIDE\", \"accession\": \"DBC000433\", \"full_name\": \"PRoteomics IDEntifications\", \"description\": \"An archive of protein expression data determined by mass spectrometry. The PRIDE database is a centralized, standards compliant, public data repository for proteomics data, including protein and peptide identifications, post-translational modifications and supporting spectral evidence.\", \"founded_year\": 2005, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"Wellcome Trust Genome Campus, Hinxton, Cambridge, UK\", \"city\": \"Cambridge\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"v1.0\", \"contact_name\": \"PRIDE team\", \"contact_email\": \"pride-support@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2018-12\", \"age\": 18, \"citation\": 7425, \"token\": \"$2a$10$.guYHfwqkY.yZteP02bDC.2Esa/0uB.uyKxgGlSGYdVp13BBSCAh.\", \"expire_date\": 1543906757995.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:10:13\", \"data_type_list\": \"Protein\", \"category_list\": \"Raw bio-data, Expression\", \"keywords_list\": \"protein and peptide identification, post-translational modification, mass spectrometry\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 412.5, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:15\", \"ess\": 0}, {\"page_id\": \"22_home\", \"is_success\": true, \"error_count\": 9.0, \"label_missing\": 4.0, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 3.0, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 22, \"page_type\": \"home\", \"source_id\": \"dc_148\", \"short_name\": \"GO\", \"accession\": \"DBC000148\", \"full_name\": \"Gene Ontology Consortium\", \"description\": \"GO is a community-based bioinformatics resource that supplies information about gene product function using ontologies to represent biological knowledge.\", \"founded_year\": 2001, \"host_institution\": \"na\", \"address\": \"Geneva,Switzerland\", \"city\": \"Geneva\", \"province\": null, \"country\": \"Switzerland\", \"current_version\": null, \"contact_name\": \"The Gene Ontology Consortium\", \"contact_email\": \"help@geneontology.org\", \"available_protocol\": 1.0, \"last_update\": \"2021-2-1\", \"age\": 22, \"citation\": 8684, \"token\": \"$2a$10$5GPfIdKhDVdOyJmfT77CeeiDIpTWb/9mHvLxN8YlbisjFmJ/jFWf.\", \"expire_date\": 1543898942953.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-08-28 20:20:52\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation, Standard ontology and nomenclature\", \"keywords_list\": \"biological process, cellular component, molecular function, functional annotation\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria\", \"organism_list\": \"Escherichia coli, Homo sapiens, Oryza sativa, Saccharomyces cerevisiae, Schizosaccharomyces pombe, Mus musculus\", \"theme_list\": null, \"zindex\": 394.727, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:06\", \"ess\": 0}, {\"page_id\": \"23_home\", \"is_success\": true, \"error_count\": 0.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 23, \"page_type\": \"home\", \"source_id\": \"dc_282\", \"short_name\": \"ClinVar\", \"accession\": \"DBC000282\", \"full_name\": \"ClinVar\", \"description\": \"ClinVar aggregates information about genomic variation and its relationship to human health.\", \"founded_year\": 2014, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"8600 Rockville Pike,Bethesda,MD 20894,USA\", \"city\": \"Bethesda\", \"province\": \"MD\", \"country\": \"United States\", \"current_version\": \"v1.0\", \"contact_name\": \"ClinVar team\", \"contact_email\": \"clinvar@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": null, \"age\": 9, \"citation\": 3539, \"token\": \"$2a$10$RicS5H80EqfuJxKAuyc5f.bmIRlzXa2CAzRPv6RAdl.HD2S8vYp96\", \"expire_date\": 1543899027028.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:10:00\", \"data_type_list\": \"DNA\", \"category_list\": \"Genotype phenotype and variation, Health and medicine\", \"keywords_list\": \"genomic variation\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 393.222, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:10\", \"ess\": 0}, {\"page_id\": \"24_home\", \"is_success\": true, \"error_count\": 12.0, \"label_missing\": null, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": 6.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 5.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 24, \"page_type\": \"home\", \"source_id\": \"dc_5051\", \"short_name\": \"UALCAN\", \"accession\": \"DBC005051\", \"full_name\": null, \"description\": \"UALCAN is a user-friendly, interactive web resource for analyzing cancer transcriptome data. It is built on PERL-CGI with high quality graphics using javascript and CSS. UALCAN is designed to, a) provide easy access to publicly available cancer transcriptome data (TCGA and MET500 transcriptome sequencing), b) allow users to identify biomarkers or to perform in silico validation of potential genes of interest, c) provide publication quality graphs and plots depicting gene expression and patient survival information based on gene expression, d) evaluate gene expression in molecular subtypes of breast and prostate cancer, e) provide additional information about the selected genes using links to HPRD, GeneCards, Pubmed, TargetScan and The human protein atlas.\", \"founded_year\": 2017, \"host_institution\": \"University of Alabama at Birmingham\", \"address\": \"Comprehensive Cancer Center, University of Alabama at Birmingham, Birmingham, AL 35233, USA\", \"city\": null, \"province\": null, \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Varambally S\", \"contact_email\": \"soorya@uab.edu.\", \"available_protocol\": 1.0, \"last_update\": \"2018\", \"age\": 6, \"citation\": 2336, \"token\": \"$2a$10$CP9lpu8gOvRa04WizS/GYepY8oA27YCE0nY9LFNWbIUJNyD.aybi6\", \"expire_date\": 1544148172454.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1517170437000, \"last_modified\": \"2023-03-03 10:14:52\", \"data_type_list\": \"DNA, Other\", \"category_list\": \"Health and medicine, Literature\", \"keywords_list\": \"cancer transcriptome data, biomarkers, gene expression, cancer\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 389.333, \"first_publication_year\": 2017, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:01:57\", \"ess\": 0}, {\"page_id\": \"25_home\", \"is_success\": true, \"error_count\": 0.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 25, \"page_type\": \"home\", \"source_id\": \"dc_7039\", \"short_name\": \"ALLEN BRAIN MAP\", \"accession\": \"DBC007039\", \"full_name\": null, \"description\": \"The Allen Institute for Brain Science uses a unique approach to generate data, tools and knowledge for researchers to explore the biological complexity of the mammalian brain. This portal provides access to high quality data and web-based applications created for the benefit of the global research community.\", \"founded_year\": 2007, \"host_institution\": \"Allen Institute for Brain Science\", \"address\": \"Seattle, WA, USA\", \"city\": \"Seattle\", \"province\": null, \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Edward S. Lein\", \"contact_email\": \"edl@alleninstitute.org\", \"available_protocol\": 1.0, \"last_update\": \"2020-06-25\", \"age\": 16, \"citation\": 6216, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1604676810000, \"last_modified\": \"2023-03-03 10:17:02\", \"data_type_list\": \"DNA, RNA, Other\", \"category_list\": \"Raw bio-data, Expression, Interaction, Health and medicine, Metadata, Literature\", \"keywords_list\": \"brain\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 388.5, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:40\", \"ess\": 0}, {\"page_id\": \"26_home\", \"is_success\": true, \"error_count\": 5.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": 3.0, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 26, \"page_type\": \"home\", \"source_id\": \"dc_220\", \"short_name\": \"CAZy\", \"accession\": \"DBC000220\", \"full_name\": \"Carbohydrate-active Enzymes Database\", \"description\": \"The CAZy database describes the families of structurally-related catalytic and carbohydrate-binding modules (or functional domains) of enzymes that degrade,modify,or create glycosidic bonds.\", \"founded_year\": 2009, \"host_institution\": \"Architecture et Fonction des Macromol\\u00e9cules Biologiques\", \"address\": \"13288 Marseille,France\", \"city\": \"Marseille\", \"province\": null, \"country\": \"France\", \"current_version\": \"v1.0\", \"contact_name\": \"Bernard Henrissat\", \"contact_email\": \"Bernard.Henrissat@afmb.univ-mrs.fr\", \"available_protocol\": 1.0, \"last_update\": \"5/22/2015\", \"age\": 14, \"citation\": 5353, \"token\": \"$2a$10$QMR8FYYqeWecfwNuZ6IiouV8VjkYhxCVjwnVlGCu.HV3oWA37yfHy\", \"expire_date\": 1543898988405.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:56\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"carbohydrate-active enzyme, glycoside hydrolase, glycosyltransferase, polysaccharide lyase, carbohydrate esterase\", \"data_object_list\": null, \"organism_list\": null, \"theme_list\": null, \"zindex\": 382.357, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:08\", \"ess\": 0}, {\"page_id\": \"27_home\", \"is_success\": true, \"error_count\": 4.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 4.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 27, \"page_type\": \"home\", \"source_id\": \"dc_30\", \"short_name\": \"InterPro\", \"accession\": \"DBC000030\", \"full_name\": \"Protein sequence analysis & classification\", \"description\": \"InterPro provides functional analysis of proteins by classifying them into families and predicting domains and important sites.\", \"founded_year\": 2000, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI), Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SD, UK\", \"city\": \"Cambridge\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"v92\", \"contact_name\": \"Alex Bateman\", \"contact_email\": \"agb@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2022-12-15\", \"age\": 23, \"citation\": 8643, \"token\": \"$2a$10$XXRDMgyw57XbhaL1X2YaweUs0ywar53fpZagpvFzpbDOtCIG8TGd6\", \"expire_date\": 1543898855889.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-08-23 10:41:57\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation, Standard ontology and nomenclature, Literature\", \"keywords_list\": \"protein classification, protein domain, protein function, protein family\", \"data_object_list\": \"Bacteria, Archaea, Animal, Plant, Fungi, Virus\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 375.783, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:03\", \"ess\": 0}, {\"page_id\": \"28_home\", \"is_success\": true, \"error_count\": 2.0, \"label_missing\": null, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 28, \"page_type\": \"home\", \"source_id\": \"dc_31\", \"short_name\": \"CDD\", \"accession\": \"DBC000031\", \"full_name\": \"Conserved Domain Database\", \"description\": \"CDD is a protein annotation resource that consists of a collection of well-annotated multiple sequence alignment models for ancient domains and full-length proteins. CDD content includes NCBI-curated domains, which use 3D-structure information to explicitly define domain boundaries and provide insights into sequence/structure/function relationships, as well as domain models imported from a number of external source databases (Pfam, SMART, COG, PRK, TIGRFAMs).\", \"founded_year\": 2002, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"Room 8N805, 8600 Rockville Pike, Bethesda, MD 20894, USA\", \"city\": \"Bethesda\", \"province\": \"MD\", \"country\": \"United States\", \"current_version\": \"v3.20\", \"contact_name\": \"Aron Marchler-Bauer\", \"contact_email\": \"bauer@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2022-09-20\", \"age\": 21, \"citation\": 7729, \"token\": \"$2a$10$xohVuj0TgShEJjb8arf36uD0BbPPMQuvT3brFGjBp.aA96PeOtv4m\", \"expire_date\": 1543898856440.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-09-06 20:43:28\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation, Phylogeny and homology, Structure\", \"keywords_list\": \"conserved domain, Protein Family Database\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 368.048, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:03\", \"ess\": 0}, {\"page_id\": \"29_home\", \"is_success\": true, \"error_count\": 8.0, \"label_missing\": 2.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 6.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 29, \"page_type\": \"home\", \"source_id\": \"dc_296\", \"short_name\": \"DrugBank\", \"accession\": \"DBC000296\", \"full_name\": \"DrugBank - Drug and Drug Target Database\", \"description\": \"DrugBank is a comprehensive online database containing extensive biochemical and pharmacological information about drugs,their mechanisms and their targets.\", \"founded_year\": 2006, \"host_institution\": \"University of Alberta\", \"address\": null, \"city\": \"Edmonton\", \"province\": \"AB\", \"country\": \"Canada\", \"current_version\": \"v5.1.1\", \"contact_name\": \"DrugBank Team\", \"contact_email\": \"info@drugbank.ca\", \"available_protocol\": 1.0, \"last_update\": \"2018-07-03\", \"age\": 17, \"citation\": 6237, \"token\": \"$2a$10$Q1/nA.V/xDJkWQFRWhUSJuDF/9GUMSpiiXs7ksVPeqSSRrrgrAqb.\", \"expire_date\": 1543899035439.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:10:01\", \"data_type_list\": \"Protein\", \"category_list\": \"Health and medicine\", \"keywords_list\": \"biochemical and pharmacological information, drug mechanism, drug target, pathway, drug metabolism, drug repurposing, drug interactions, drug products\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 366.882, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:10\", \"ess\": 0}, {\"page_id\": \"30_home\", \"is_success\": true, \"error_count\": 18.0, \"label_missing\": 7.0, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": 3.0, \"link_empty\": 3.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": 4.0, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 30, \"page_type\": \"home\", \"source_id\": \"dc_111\", \"short_name\": \"COSMIC\", \"accession\": \"DBC000111\", \"full_name\": \"Catalogue of Somatic Mutations in Cancer\", \"description\": \"COSMIC explores the world's knowledge of somatic mutations in human cancer.\", \"founded_year\": 2004, \"host_institution\": \"Wellcome Sanger Institute\", \"address\": \"Hinxton,Cambridge,UK,CB10 1SA\", \"city\": \"Cambridge\", \"province\": null, \"country\": \"United Kingdom\", \"current_version\": \"v80.0\", \"contact_name\": \"Simon A. Forbes\", \"contact_email\": \"saf@sanger.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2017-02-13\", \"age\": 19, \"citation\": 6827, \"token\": \"$2a$10$YV0UmxomkHVLEBwnfCg5A./VPad00LzCIyG6A2d4kEw2OVY0aLtJq\", \"expire_date\": 1543898917337.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:45\", \"data_type_list\": \"DNA\", \"category_list\": \"Health and medicine, Genotype phenotype and variation\", \"keywords_list\": \"cancer, somatic mutation\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 359.316, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:05\", \"ess\": 0}, {\"page_id\": \"31_home\", \"is_success\": true, \"error_count\": 23.0, \"label_missing\": 2.0, \"language_missing\": null, \"button_empty\": 5.0, \"alt_missing\": null, \"link_empty\": 16.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 31, \"page_type\": \"home\", \"source_id\": \"dc_6779\", \"short_name\": \"GTDB\", \"accession\": \"DBC006779\", \"full_name\": \"The Genome Taxonomy Database\", \"description\": \"GTDB is an initiative to establish a standardised microbial taxonomy based on genome phylogeny.\", \"founded_year\": 2018, \"host_institution\": \"University of Queensland\", \"address\": \"Australian Centre for Ecogenomics, School of Chemistry and Molecular Biosciences, University of Queensland, Queensland, Australia.\", \"city\": \"Queensland\", \"province\": null, \"country\": \"Australia\", \"current_version\": \"Release 04-RS89\", \"contact_name\": \"Philip Hugenholtz\", \"contact_email\": \"p.hugenholtz@uq.edu.au\", \"available_protocol\": 1.0, \"last_update\": \"2019-6-19\", \"age\": 5, \"citation\": 1730, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1572149519000, \"last_modified\": \"2023-03-03 10:16:47\", \"data_type_list\": \"DNA\", \"category_list\": \"Phylogeny and homology, Standard ontology and nomenclature\", \"keywords_list\": \"multiple sequence alignment, species cluster, prokaryote nomenclature, genome-based taxonomy\", \"data_object_list\": \"Bacteria, Archaea\", \"organism_list\": null, \"theme_list\": null, \"zindex\": 346.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:35\", \"ess\": 0}, {\"page_id\": \"32_home\", \"is_success\": true, \"error_count\": 38.0, \"label_missing\": 16.0, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": 3.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 4.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": 13.0, \"title_invalid\": 1.0, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 32, \"page_type\": \"home\", \"source_id\": \"dc_5935\", \"short_name\": \"PANTHER\", \"accession\": \"DBC005935\", \"full_name\": \"Protein ANalysis THrough Evolutionary Relationships\", \"description\": \"The PANTHER (Protein ANalysis THrough Evolutionary Relationships) Classification System was designed to classify proteins (and their genes) in order to facilitate high-throughput analysis.\", \"founded_year\": 2003, \"host_institution\": \"University of Southern California\", \"address\": \"Computational Biology, Applied Biosystems, 850 Lincoln Center Drive, Foster City, CA 94404, USA.\", \"city\": \"Foster City\", \"province\": null, \"country\": \"United States\", \"current_version\": \"v14\", \"contact_name\": \"Paul D. Thomas\", \"contact_email\": \"paul.thomas@appliedbiosystems.com\", \"available_protocol\": 1.0, \"last_update\": \"2021\", \"age\": 20, \"citation\": 6895, \"token\": \"$2a$10$lIm/OeUCy6/fPydtIhGUuuru/uUq8.Kym.we2W2DgNZC/OAjXa0MW\", \"expire_date\": 1544267110867.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1518205751000, \"last_modified\": \"2023-03-03 10:15:51\", \"data_type_list\": \"Protein\", \"category_list\": \"Phylogeny and homology\", \"keywords_list\": \"protein families, Evolutionary Relationships, high-throughput analysis\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Virus\", \"organism_list\": \"Homo sapiens, Zea mays, Mus musculus\", \"theme_list\": null, \"zindex\": 344.75, \"first_publication_year\": 2005, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:18\", \"ess\": 1}, {\"page_id\": \"33_home\", \"is_success\": true, \"error_count\": 7.0, \"label_missing\": null, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": 5.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 33, \"page_type\": \"home\", \"source_id\": \"dc_37\", \"short_name\": \"COG\", \"accession\": \"DBC000037\", \"full_name\": \"Clusters of Orthologous Genes\", \"description\": \"COG  contains phylogenetic classification of proteins encoded in complete genomes.\", \"founded_year\": 1997, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"Bethesda, MD 2094, USA\", \"city\": \"Bethesda\", \"province\": \"MD\", \"country\": \"United States\", \"current_version\": \"v1.0\", \"contact_name\": \"Eugene V. Koonin\", \"contact_email\": \"koonin@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2014\", \"age\": 26, \"citation\": 8914, \"token\": \"$2a$10$R/TOVmuvA6ZDVOHmPwDizucAwgceuuK/uBh3EKqEnuvAvGBh2LX1W\", \"expire_date\": 1543898860144.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:40\", \"data_type_list\": \"Protein\", \"category_list\": \"Phylogeny and homology\", \"keywords_list\": \"gene orthology\", \"data_object_list\": \"Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Caulobacter vibrioides, Chlamydia pneumoniae, Chlorobaculum tepidum, Clostridioides difficile, Klebsiella variicola, Mesorhizobium japonicum, Methanothrix soehngenii\", \"theme_list\": null, \"zindex\": 342.846, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:03\", \"ess\": 0}, {\"page_id\": \"34_home\", \"is_success\": true, \"error_count\": 0.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 34, \"page_type\": \"home\", \"source_id\": \"dc_256\", \"short_name\": \"RefSeq\", \"accession\": \"DBC000256\", \"full_name\": \"reference sequence database\", \"description\": \"The National Center for Biotechnology Information (NCBI) Reference Sequence (RefSeq) database is a collection of annotated genomic, transcript and protein sequence records derived from data in public sequence archives and from computation, curation and collaboration.\", \"founded_year\": 2000, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, 45 Center Drive, Bethesda, MD 20892-6511, USA.\", \"city\": \"Bethesda\", \"province\": \"MD\", \"country\": \"United States\", \"current_version\": \"Release 220\", \"contact_name\": \"Fran\\u00e7oise Thibaud-Nissen\", \"contact_email\": \"thibaudf@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2023-09-11\", \"age\": 23, \"citation\": 7604, \"token\": \"$2a$10$XsipWUIFeVIkhvykvE1NTOOXELtt1/iG2RHROIHopsg16DGM1iRV2\", \"expire_date\": 1543899010604.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-09-13 22:42:36\", \"data_type_list\": \"DNA, Protein, RNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"reference sequence, protein-coding gene, pseudogene, lncRNA\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": \"lncRNA\", \"zindex\": 330.609, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:09\", \"ess\": 0}, {\"page_id\": \"35_home\", \"is_success\": true, \"error_count\": 0.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 35, \"page_type\": \"home\", \"source_id\": \"dc_7367\", \"short_name\": \"NCBI\", \"accession\": \"DBC007367\", \"full_name\": \"National Center for Biotechnology Information\", \"description\": \"The National Center for Biotechnology Information (NCBI) provides a large suite of online resources for biological information and data, including the GenBank\\u00ae nucleic acid sequence database and the PubMed\\u00ae database of citations and abstracts published in life science journals. The Entrez system provides search and retrieval operations for most of these data from 34 distinct databases. The E-utilities serve as the programming interface for the Entrez system. Custom implementations of the BLAST program provide sequence-based searching of many specialized datasets. New resources released in the past year include a new PubMed interface and NCBI datasets. Additional resources that were updated in the past year include PMC, Bookshelf, Genome Data Viewer, SRA, ClinVar, dbSNP, dbVar, Pathogen Detection, BLAST, Primer-BLAST, IgBLAST, iCn3D and PubChem. \", \"founded_year\": 2000, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Building 38A, 8600 Rockville Pike, Bethesda, MD 20894, USA.\", \"city\": \"Bethesda\", \"province\": null, \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Eric W Sayers\", \"contact_email\": \"sayers@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2020-10-23\", \"age\": 23, \"citation\": 7360, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1605149733000, \"last_modified\": \"2023-04-22 15:46:42\", \"data_type_list\": \"DNA, RNA, Protein\", \"category_list\": \"Raw bio-data, Gene genome and annotation, Genotype phenotype and variation, Literature, Metadata\", \"keywords_list\": \"deposition, integration, Big data\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Virus\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 320.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:47\", \"ess\": 0}, {\"page_id\": \"36_home\", \"is_success\": true, \"error_count\": 10.0, \"label_missing\": 2.0, \"language_missing\": null, \"button_empty\": 1.0, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": 4.0, \"alt_link_missing\": 2.0, \"th_empty\": 1.0, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 36, \"page_type\": \"home\", \"source_id\": \"dc_1334\", \"short_name\": \"SWISS-MODEL\", \"accession\": \"DBC001334\", \"full_name\": \"The SWISS-MODEL Repository\", \"description\": \"The SWISS-MODEL Repository is a database of annotated 3D protein structure models generated by the SWISS-MODEL homology-modelling pipeline.\", \"founded_year\": 2004, \"host_institution\": \"Swiss Institute of Bioinformatics\", \"address\": \"Biozentrum, University of Basel and SIB Swiss Institute of Bioinformatics, Basel, Switzerland\", \"city\": \"Basel\", \"province\": null, \"country\": \"Switzerland\", \"current_version\": \"v1.0\", \"contact_name\": \"Torsten Schwede\", \"contact_email\": \"Torsten.Schwede@unibas.ch\", \"available_protocol\": 1.0, \"last_update\": \"2017-01-14\", \"age\": 19, \"citation\": 6042, \"token\": \"$2a$10$UAu.zDDeFOJQiNxBo3oZEurwt13xY65s4PmjJk/XV9kNFhuaqGSC2\", \"expire_date\": 1543993242762.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1437614368000, \"last_modified\": \"2023-03-03 10:11:18\", \"data_type_list\": \"Protein\", \"category_list\": \"Structure\", \"keywords_list\": \"3D protein structure model\", \"data_object_list\": null, \"organism_list\": null, \"theme_list\": null, \"zindex\": 318.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:37\", \"ess\": 0}, {\"page_id\": \"37_home\", \"is_success\": true, \"error_count\": 30.0, \"label_missing\": 1.0, \"language_missing\": 1.0, \"button_empty\": 3.0, \"alt_missing\": 6.0, \"link_empty\": 5.0, \"aria_menu_broken\": 2.0, \"alt_link_missing\": 5.0, \"th_empty\": null, \"heading_empty\": 4.0, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": 3.0, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 37, \"page_type\": \"home\", \"source_id\": \"dc_4842\", \"short_name\": \"EzTaxon-e\", \"accession\": \"DBC004842\", \"full_name\": \"EzBioCloud Database\", \"description\": \"EzTaxon-e database provides a useful taxonomic backbone for the identification of cultured and uncultured prokaryotes and offers a valuable means of communication among microbiologists who routinely encounter taxonomically novel isolates.\", \"founded_year\": 2012, \"host_institution\": \"Seoul National University\", \"address\": \"Interdisciplinary Program in Bioinformatics, Seoul National University, 56-1 Shillim-dong, Kwanak-gu, Seoul 151-742, Republic of Korea\\n\", \"city\": \"Seoul\", \"province\": null, \"country\": \"Korea, Republic of\", \"current_version\": null, \"contact_name\": \"Jongsik Chun\", \"contact_email\": \"jchun@snu.ac.kr\", \"available_protocol\": 1.0, \"last_update\": \"2017\", \"age\": 11, \"citation\": 3485, \"token\": \"$2a$10$xXFf9.vKRqWpOA6QceOIMeV1D1w2Q5Cqon46L2xx9yrwilOHoow0C\", \"expire_date\": 1544146779481.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1517160985000, \"last_modified\": \"2023-03-03 10:14:35\", \"data_type_list\": \"DNA, RNA\", \"category_list\": \"Phylogeny and homology\", \"keywords_list\": \"16\\u2009S rRNA\", \"data_object_list\": \"Bacteria\", \"organism_list\": null, \"theme_list\": null, \"zindex\": 316.818, \"first_publication_year\": 2012, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:01:51\", \"ess\": 1}, {\"page_id\": \"38_home\", \"is_success\": true, \"error_count\": 1.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 38, \"page_type\": \"home\", \"source_id\": \"dc_672\", \"short_name\": \"FANTOM\", \"accession\": \"DBC000672\", \"full_name\": \"functional annotation of the mammalian genome\", \"description\": \"FANTOM1-3 focused on identifying the transcribed components of mammalian cells. In FANTOM4, the focus has changed to understanding how these components work together in the context of a biological network. FANTOM5 provides atlases of mammalian promoters, enhancers, lncRNAs and miRNAs. FANTOM6 focuses on functional analysis of non-coding RNAs.\", \"founded_year\": 2009, \"host_institution\": \"RIKEN\", \"address\": \"RIKEN Omics Science Center,RIKEN Yokohama Institute,1-7-22 Suehiro-cho,Tsurumi-ku\", \"city\": \"Yokohama\", \"province\": \"Kanagawa\", \"country\": \"Japan\", \"current_version\": \"v6.0\", \"contact_name\": \"FANTOM help\", \"contact_email\": \"fantom-help@riken.jp\", \"available_protocol\": 1.0, \"last_update\": \"2022\", \"age\": 14, \"citation\": 4416, \"token\": \"$2a$10$ofx9LcQGKdVqAvAhHCg/keIkbReDHJ9Xvo/aHvIAWkf8GD14Df6qa\", \"expire_date\": 1543906854762.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-09-13 22:26:25\", \"data_type_list\": \"DNA, RNA\", \"category_list\": \"Expression, Gene genome and annotation\", \"keywords_list\": \"promoter, transcribed component, transcription start site, transcriptional network, lncRNA\", \"data_object_list\": \"Animal\", \"organism_list\": \"Mus musculus, Homo sapiens\", \"theme_list\": \"lncRNA\", \"zindex\": 315.429, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:22\", \"ess\": 0}, {\"page_id\": \"39_home\", \"is_success\": true, \"error_count\": 15.0, \"label_missing\": 2.0, \"language_missing\": null, \"button_empty\": 4.0, \"alt_missing\": 1.0, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": 7.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 39, \"page_type\": \"home\", \"source_id\": \"dc_6926\", \"short_name\": \"GISAID\", \"accession\": \"DBC006926\", \"full_name\": \"Global Initiative on Sharing All Influenza Data\", \"description\": \"The GISAID Initiative promotes the international sharing of all influenza virus sequences, related clinical and epidemiological data associated with human viruses, and geographical as well as species-specific data associated with avian and other animal viruses, to help researchers understand how the viruses evolve, spread and potentially become pandemics.\\nOn June 2 2020, 36,286 hCoV-19 genomic sequences have been shared via GISAID.\", \"founded_year\": 2008, \"host_institution\": \"Max-Planck-Institut for Informatics Computational Biology and Applied Algorithmics\", \"address\": null, \"city\": null, \"province\": null, \"country\": \"Germany\", \"current_version\": null, \"contact_name\": \"GISAID Group\", \"contact_email\": \"curator@gisaid.org\", \"available_protocol\": 1.0, \"last_update\": \"2020.02.13\", \"age\": 6, \"citation\": 1854, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1581601898000, \"last_modified\": \"2023-09-26 09:59:30\", \"data_type_list\": \"DNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"H5N1, SARS-CoV-2, hCoV-19, 2019-nCoV, pneumonia\", \"data_object_list\": \"Virus\", \"organism_list\": \"Influenza A virus, Influenza B virus, Influenza C virus, Influenza D virus, Severe acute respiratory syndrome coronavirus 2\", \"theme_list\": null, \"zindex\": 309.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:37\", \"ess\": 0}, {\"page_id\": \"40_home\", \"is_success\": true, \"error_count\": 1.0, \"label_missing\": null, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 40, \"page_type\": \"home\", \"source_id\": \"dc_3120\", \"short_name\": \"Greengenes\", \"accession\": \"DBC003120\", \"full_name\": \"a chimera-checked 16S rRNA gene database \", \"description\": \"A 16S rRNA gene database addresses limitations of public repositories by providing chimera screening, standard alignment, and taxonomic classification using multiple published taxonomies.\", \"founded_year\": 2006, \"host_institution\": \"Lawrence Berkeley National Laboratory\", \"address\": \"Center for Environmental Biotechnology, Lawrence Berkeley National Laboratory, 1 Cyclotron Road, Mail Stop 70A-3317, Berkeley, CA 94720.\", \"city\": \"Berkeley\", \"province\": null, \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"G. L. Andersen\", \"contact_email\": \"GLAndersen@lbl.gov\", \"available_protocol\": 1.0, \"last_update\": \"2017-07-03\", \"age\": 17, \"citation\": 5239, \"token\": \"$2a$10$bBY6tqoll5wcIuLK4BLEweajcNiiAq7FWyykfG9J.84LW8W52lZEC\", \"expire_date\": 1543997717521.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1516962597000, \"last_modified\": \"2023-03-03 10:12:25\", \"data_type_list\": \"RNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"16S rRNA, taxonomic classification\", \"data_object_list\": \"Bacteria, Archaea\", \"organism_list\": \"Escherichia coli\", \"theme_list\": null, \"zindex\": 308.176, \"first_publication_year\": 2006, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:01:02\", \"ess\": 0}, {\"page_id\": \"41_home\", \"is_success\": true, \"error_count\": 3.0, \"label_missing\": 1.0, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 41, \"page_type\": \"home\", \"source_id\": \"dc_402\", \"short_name\": \"HMDB\", \"accession\": \"DBC000402\", \"full_name\": \"Human Metabolome Database\", \"description\": \"The Human Metabolome Database (HMDB) is a freely available electronic database containing detailed information about small molecule metabolites found in the human body.\", \"founded_year\": 2007, \"host_institution\": \"University of Alberta\", \"address\": \"Edmonton, AB, Canada T6G 2E8\", \"city\": \"Edmonton\", \"province\": \"AB\", \"country\": \"Canada\", \"current_version\": \"v4.0\", \"contact_name\": \"David S. Wishart\", \"contact_email\": \"david.wishart@ualberta.ca\", \"available_protocol\": 1.0, \"last_update\": \"2018-06-03\", \"age\": 16, \"citation\": 4912, \"token\": \"$2a$10$POAoFc5GmC.mpv35EEsgyOyWlf.ZfFE98KqJY2GuDXtVRxzYXQonW\", \"expire_date\": 1543899102912.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:10:11\", \"data_type_list\": \"Protein, Other\", \"category_list\": \"Pathway\", \"keywords_list\": \"chemical data, clinical data, molecular biology/biochemistry data, metabolomic data, Spectral data, chemical classification, chemical ontology\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 307.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:14\", \"ess\": 0}, {\"page_id\": \"42_home\", \"is_success\": false, \"error_count\": null, \"label_missing\": 1.0, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 42, \"page_type\": \"home\", \"source_id\": \"dc_237\", \"short_name\": \"RDP\", \"accession\": \"DBC000237\", \"full_name\": \"Ribosomal Database Project\", \"description\": \"Ribosomal Database Project (RDP; http://rdp.cme.msu.edu/) provides the research community with aligned and annotated rRNA gene sequence data, along with tools to allow researchers to analyze their own rRNA gene sequences in the RDP framework.\", \"founded_year\": 1991, \"host_institution\": \"Michigan State University\", \"address\": \"East Lansing, MI 48824, USA\", \"city\": \"East Lansing\", \"province\": \"MI\", \"country\": \"United States\", \"current_version\": \"v11.4\", \"contact_name\": \"James R. Cole\", \"contact_email\": \"colej@msu.edu\", \"available_protocol\": 1.0, \"last_update\": \"2016-09-30\", \"age\": 32, \"citation\": 9106, \"token\": \"$2a$10$Chaq7CO7nfW/FOgrH4rSI.zp3RQVtkjn8Wd0YHtcl6TnypA5YwtFC\", \"expire_date\": 1543898997584.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:57\", \"data_type_list\": \"RNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"16S rRNA, 28S rRNA, ribosomal\", \"data_object_list\": \"Fungi, Bacteria, Archaea\", \"organism_list\": null, \"theme_list\": null, \"zindex\": 284.562, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:08\", \"ess\": 0}, {\"page_id\": \"43_home\", \"is_success\": true, \"error_count\": 0.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 43, \"page_type\": \"home\", \"source_id\": \"dc_1773\", \"short_name\": \"PubChem \", \"accession\": \"DBC001773\", \"full_name\": null, \"description\": \"PubChem is a public repository for information on chemical substances and their biological activities, launched in 2004 as a component of the Molecular Libraries Roadmap Initiatives of the US National Institutes of Health (NIH).\\n\", \"founded_year\": 2009, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"Bethesda, Maryland 20894, USA\", \"city\": \"Bethesda\", \"province\": \"Maryland\", \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Yanli Wang\", \"contact_email\": \"ywang@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2017-01-01\", \"age\": 14, \"citation\": 3971, \"token\": \"$2a$10$etEN8.GF.a0e76BM5rYyoO.8VjMRBQdmdeacRsrp7iL0KmEcoSfCS\", \"expire_date\": 1543995359190.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1514980558000, \"last_modified\": \"2023-08-22 14:56:42\", \"data_type_list\": \"Other\", \"category_list\": \"Health and medicine, Structure\", \"keywords_list\": \"chemical substance, bioassay, compound\", \"data_object_list\": \"Animal\", \"organism_list\": null, \"theme_list\": null, \"zindex\": 283.643, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:51\", \"ess\": 0}, {\"page_id\": \"44_home\", \"is_success\": true, \"error_count\": 3.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 2.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 44, \"page_type\": \"home\", \"source_id\": \"dc_7106\", \"short_name\": \"CARD\", \"accession\": \"DBC007106\", \"full_name\": \"The Comprehensive Antibiotic Resistance Database\", \"description\": \"The Comprehensive Antibiotic Resistance Database (CARD; card.mcmaster.ca) combines the Antibiotic Resistance Ontology (ARO) with curated AMR gene (ARG) sequences and resistance-conferring mutations to provide an informatics framework for annotation and interpretation of resistomes. As of version 3.2.4, CARD encompasses 6627 ontology terms, 5010 reference sequences, 1933 mutations, 3004 publications, and 5057 AMR detection models that can be used by the accompanying Resistance Gene Identifier (RGI) software to annotate genomic or metagenomic sequences. New features include the CARD:Live collection of community submitted isolate resistome data and the introduction of standardized 15 character CARD Short Names for ARGs to support machine learning efforts.\", \"founded_year\": 2013, \"host_institution\": \"McMaster University\", \"address\": \"McMaster University\\nc/o McMaster Industry Liaison Office\\n305 \\u2013 175 Longwood Rd South\\nHamilton, Ontario L8P 0A1\", \"city\": \"Hamilton\", \"province\": \"Ontario\", \"country\": \"Canada\", \"current_version\": null, \"contact_name\": \"Andrew G. McArthur\", \"contact_email\": \"mcarthua@mcmaster.ca\", \"available_protocol\": 1.0, \"last_update\": \"2019\", \"age\": 10, \"citation\": 2815, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1604798602000, \"last_modified\": \"2023-08-28 16:16:12\", \"data_type_list\": \"DNA, Protein\", \"category_list\": \"Standard ontology and nomenclature, Literature, Health and medicine\", \"keywords_list\": \"reference DNA and protein, molecular sequences, software, curation, ontological structure\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 281.5, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:41\", \"ess\": 0}, {\"page_id\": \"45_home\", \"is_success\": true, \"error_count\": 3.0, \"label_missing\": 3.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 45, \"page_type\": \"home\", \"source_id\": \"dc_66\", \"short_name\": \"BioGRID\", \"accession\": \"DBC000066\", \"full_name\": \"Biological General Repository for Interaction Datasets\", \"description\": \"BioGRID is an online interaction repository with data compiled through comprehensive curation efforts.The present version contains 1,585,828 protein and genetic interactions, 27,785 chemical associations and 39,028 post translational modifications from major model organism species.\", \"founded_year\": 2006, \"host_institution\": \"University of Montreal\", \"address\": \"Montr\\u00e9al, Quebec H3C 3J7, Canada\", \"city\": \"Mike Tyers\", \"province\": \"Quebec\", \"country\": \"Canada\", \"current_version\": \"v3.4.145\", \"contact_name\": \"biogridadmin\", \"contact_email\": \"biogridadmin@gmail.com\", \"available_protocol\": 1.0, \"last_update\": \"2017-02-01\", \"age\": 17, \"citation\": 4692, \"token\": \"$2a$10$KCGIMnRPzHxFzr9q1POCWOq3jANWbQkSR2M4p1MZLC8coujg2Su/6\", \"expire_date\": 1543898882242.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:42\", \"data_type_list\": \"Protein, DNA\", \"category_list\": \"Modification, Interaction, Literature\", \"keywords_list\": \"protein and genetic interaction, translational modification\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens, Escherichia coli, Mus musculus, Saccharomyces cerevisiae, Arabidopsis thaliana\", \"theme_list\": null, \"zindex\": 276.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:04\", \"ess\": 0}, {\"page_id\": \"46_home\", \"is_success\": true, \"error_count\": 19.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 2.0, \"link_empty\": 15.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": 1.0, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 46, \"page_type\": \"home\", \"source_id\": \"dc_217\", \"short_name\": \"Reactome\", \"accession\": \"DBC000217\", \"full_name\": \"Reactome Pathway Knowledgebase\", \"description\": \"Reactome is a free, open-source, curated and peer-reviewed pathway database. Our goal is to provide intuitive bioinformatics tools for the visualization, interpretation, and analysis of pathway knowledge to support basic research, genome analysis, modeling, systems biology, and education. \\n\\n\", \"founded_year\": 2005, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"661 University Ave, Suite 510, Toronto, ON, M5G 0A3\", \"city\": null, \"province\": null, \"country\": \"United Kingdom\", \"current_version\": \"v81\", \"contact_name\": \"Reactome Helpdesk\", \"contact_email\": \"help@reactome.org\", \"available_protocol\": 1.0, \"last_update\": \"2022-6-16\", \"age\": 18, \"citation\": 4932, \"token\": \"$2a$10$Vb7Y3w1DnZKjF5Irtbe4k.5gyT6utABrXxpaVxHv4y5sbazFYqhT6\", \"expire_date\": 1543898986809.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-10-09 14:54:03\", \"data_type_list\": \"Other, DNA, RNA, Protein\", \"category_list\": \"Pathway\", \"keywords_list\": \"pathway, interaction, SARS-CoV-2\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria\", \"organism_list\": \"Arabidopsis thaliana, Bos taurus, Caenorhabditis elegans, Danio rerio, Drosophila melanogaster, Homo sapiens, Mus musculus, Oryza sativa, Rattus norvegicus, Saccharomyces cerevisiae\", \"theme_list\": null, \"zindex\": 274.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:08\", \"ess\": 0}, {\"page_id\": \"47_home\", \"is_success\": true, \"error_count\": 2.0, \"label_missing\": 1.0, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 47, \"page_type\": \"home\", \"source_id\": \"dc_3990\", \"short_name\": \"Brain RNA-Seq\", \"accession\": \"DBC003990\", \"full_name\": null, \"description\": \"The major cell classes of the brain differ in their developmental processes, metabolism, signaling, and function. To better understand the functions and interactions of the cell types that comprise these classes, we acutely purified representative populations of neurons, astrocytes, oligodendrocyte precursor cells, newly formed oligodendrocytes, myelinating oligodendrocytes, microglia, endothelial cells, and pericytes from mouse cerebral cortex. We generated a transcriptome database for these eight cell types by RNA sequencing and used a sensitive algorithm to detect alternative splicing events in each cell type. \", \"founded_year\": 2014, \"host_institution\": \"University of Texas Medical School at Houston\", \"address\": \"The Vivian L. Smith Department of Neurosurgery, University of Texas Medical School at Houston, Houston, Texas 77057, Center for Stem Cell and Regenerative Medicine, University of Texas Brown Institute of Molecular Medicine, Houston, Texas 77057.\", \"city\": null, \"province\": null, \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Wu JQ\", \"contact_email\": \"jiaqian.wu@uth.tmc.edu\", \"available_protocol\": 1.0, \"last_update\": null, \"age\": 9, \"citation\": 2351, \"token\": \"$2a$10$h5PVwyYKVT7SQJoLq18LoeOMtvsrrvo.UT7UTd6.m/sy3VmBLjeXy\", \"expire_date\": 1544082524946.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1517084197000, \"last_modified\": \"2023-03-03 10:13:32\", \"data_type_list\": \"RNA\", \"category_list\": \"Expression\", \"keywords_list\": \"RNA-Seq, alternative splice, brain development\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 261.222, \"first_publication_year\": 2014, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:01:26\", \"ess\": 1}, {\"page_id\": \"48_home\", \"is_success\": true, \"error_count\": 10.0, \"label_missing\": 2.0, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": 3.0, \"link_empty\": 3.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": 1.0, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 48, \"page_type\": \"home\", \"source_id\": \"dc_167\", \"short_name\": \"miRTarBase\", \"accession\": \"DBC000167\", \"full_name\": \"microRNA-target interactions database\", \"description\": \"miRTarBase has accumulated more than three hundred and sixty thousand miRNA-target interactions (MTIs), which are collected by manually surveying pertinent literature after NLP of the text systematically to filter research articles related to functional studies of miRNAs. Generally, the collected MTIs are validated experimentally by reporter assay, western blot, microarray and next-generation sequencing experiments. While containing the largest amount of validated MTIs, the miRTarBase provides the most updated collection by comparing with other similar, previously developed databases.\", \"founded_year\": 2011, \"host_institution\": \"Chinese University of Hong Kong\", \"address\": \"School of Life and Health Sciences, The Chinese University of Hong Kong, Shenzhen, Longgang District, Shenzhen, Guangdong Province 518172, China.\", \"city\": \"HongKong\", \"province\": \"HongKong\", \"country\": \"China\", \"current_version\": \"Release 9.0 beta\", \"contact_name\": \"Hsien-Da Huang\", \"contact_email\": \"huanghsiyuan@cuhk.edu.cn\", \"available_protocol\": 1.0, \"last_update\": \"2021-09\", \"age\": 12, \"citation\": 3126, \"token\": \"$2a$10$v12HaPlA26wUtjPtOfomRO4zEXXEBGsib0gfo7Bu1dVFJrZfLsii.\", \"expire_date\": 1543898953010.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:51\", \"data_type_list\": \"RNA\", \"category_list\": \"Interaction, Literature\", \"keywords_list\": \"miRNA-target interaction, miRNA\", \"data_object_list\": \"Animal, Plant, Virus\", \"organism_list\": \"Arabidopsis thaliana, Caenorhabditis elegans, Drosophila melanogaster, Homo sapiens, Mus musculus, Rattus norvegicus, Bombyx mori, Bos taurus, Taeniopygia guttata, Xenopus laevis\", \"theme_list\": null, \"zindex\": 260.5, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:07\", \"ess\": 0}, {\"page_id\": \"49_home\", \"is_success\": true, \"error_count\": 27.0, \"label_missing\": 3.0, \"language_missing\": 1.0, \"button_empty\": 1.0, \"alt_missing\": 16.0, \"link_empty\": 2.0, \"aria_menu_broken\": null, \"alt_link_missing\": 4.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 49, \"page_type\": \"home\", \"source_id\": \"dc_295\", \"short_name\": \"ChEMBL\", \"accession\": \"DBC000295\", \"full_name\": null, \"description\": \"ChEMBL is a manually curated database of bioactive molecules with drug-like properties. It brings together chemical, bioactivity and genomic data to aid the translation of genomic information into effective new drugs. \", \"founded_year\": 2011, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"Hinxton, Cambridge CB10 1SD, UK\", \"city\": \"Hinxton\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"v25\", \"contact_name\": \"Andrew R. Leach\", \"contact_email\": \"arl@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2018-12-10\", \"age\": 12, \"citation\": 3030, \"token\": \"$2a$10$/yamR44PS4ERv0pHaX9FzuhrRee4TvZcxpe4a2kJPAknEGRmEGzq.\", \"expire_date\": 1543899035077.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:10:01\", \"data_type_list\": \"Protein\", \"category_list\": \"Health and medicine, Interaction, Literature, Structure\", \"keywords_list\": \"drug target, drug-like small molecule\", \"data_object_list\": \"Animal\", \"organism_list\": null, \"theme_list\": null, \"zindex\": 252.5, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:10\", \"ess\": 0}, {\"page_id\": \"50_home\", \"is_success\": true, \"error_count\": 1.0, \"label_missing\": null, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 50, \"page_type\": \"home\", \"source_id\": \"dc_176\", \"short_name\": \"JASPAR\", \"accession\": \"DBC000176\", \"full_name\": null, \"description\": \"JASPAR is the largest open-access database of matrix-based nucleotide profiles describing the binding preference of transcription factors from multiple species.\", \"founded_year\": 2004, \"host_institution\": \"University of British Columbia\", \"address\": \"Department of Medical Genetics,Centre for Molecular Medicine and Therapeutics at the Child and Family Research Institute,University of British Columbia,Vancouver,BC,Canada\", \"city\": \"Vancouver\", \"province\": \"BC\", \"country\": \"Canada\", \"current_version\": \"v5.0\", \"contact_name\": \"Boris Lenhard\", \"contact_email\": \"b.lenhard@csc.mrc.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2016-03-18\", \"age\": 19, \"citation\": 4481, \"token\": \"$2a$10$Ui0EK2W5ecDRwczesHsiMuBo2GukKNm9m.R9qBHo4G.9sb/bRPoqa\", \"expire_date\": 1543898958412.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:52\", \"data_type_list\": \"DNA, Protein\", \"category_list\": \"Interaction\", \"keywords_list\": \"transcription factor binding preference\", \"data_object_list\": \"Animal\", \"organism_list\": \"Caenorhabditis elegans, Drosophila melanogaster, Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 235.842, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:07\", \"ess\": 0}, {\"page_id\": \"51_home\", \"is_success\": true, \"error_count\": 2.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 1.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 51, \"page_type\": \"home\", \"source_id\": \"dc_169\", \"short_name\": \"ENCORI/starBase\", \"accession\": \"DBC000169\", \"full_name\": \"Encyclopedia of RNA Interactomes\", \"description\": \"ENCORI/starBase is an openly licensed and state-of-the-art platform to facilitate the integrative, interactive and versatile display of, as well as the comprehensive annotation and discovery of, RNA-RNA and protein-RNA interactions by developing new software and pipelines to deeply mining thousands of high-throughput sequencing data of RNA-RNA interactome, CLIP-seq and degradome sequencing (degradome-seq).\", \"founded_year\": 2011, \"host_institution\": \"Sun Yat-sen University\", \"address\": \"Guangzhou 510275,PR China\", \"city\": \"Zhongshan\", \"province\": \"Guangdong\", \"country\": \"China\", \"current_version\": \"v2.0\", \"contact_name\": \"Jian-Hua Yang\", \"contact_email\": \"yangjh7@mail.sysu.edu.cn\", \"available_protocol\": 1.0, \"last_update\": \"2023-05-21\", \"age\": 12, \"citation\": 2744, \"token\": \"$2a$10$46EwcFPXb64nkoWw7ySLa.yIXeeMdOlNjOY12M8oHiG5pVU1TQxvq\", \"expire_date\": 1543898954267.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-09-13 23:14:56\", \"data_type_list\": \"RNA\", \"category_list\": \"Interaction\", \"keywords_list\": \"lncRNA, RNA-binding protein, RBP-RNA interaction, RNA-RNA interaction, miRNA, circRNA, pseudogene, sncRNA, ceRNA\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": \"lncRNA\", \"zindex\": 228.667, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:07\", \"ess\": 0}, {\"page_id\": \"52_home\", \"is_success\": true, \"error_count\": 0.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 52, \"page_type\": \"home\", \"source_id\": \"dc_1619\", \"short_name\": \"GenBank\", \"accession\": \"DBC001619\", \"full_name\": \"GenBank\", \"description\": \"GenBank is a comprehensive database that contains publicly available DNA sequences for more than 165\\u2009000 named organisms, obtained primarily through submissions from individual laboratories and batch submissions from large-scale sequencing projects.\\n\", \"founded_year\": 1985, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"8600 Rockville Pike, Bethesda, MD 20894, USA\", \"city\": \"Bethesda\", \"province\": \"MD\", \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"GenBank team\", \"contact_email\": \"genomes@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2020-10-20\", \"age\": 38, \"citation\": 8118, \"token\": \"$2a$10$YorLifqtkghGstw/JUmDYOP9qXDg0eKDkMmBTIhr7hY7QUsJWlwym\", \"expire_date\": 1543993384944.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1450663065000, \"last_modified\": \"2023-03-03 10:11:38\", \"data_type_list\": \"DNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"DNA sequence submission\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens, Mus musculus, Arabidopsis thaliana\", \"theme_list\": null, \"zindex\": 213.632, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:47\", \"ess\": 0}, {\"page_id\": \"53_home\", \"is_success\": true, \"error_count\": 11.0, \"label_missing\": null, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": 2.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 7.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": 1.0, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 53, \"page_type\": \"home\", \"source_id\": \"dc_6849\", \"short_name\": \"MR-Base\", \"accession\": \"DBC006849\", \"full_name\": null, \"description\": \"The database currently comprises 11 billion single nucleotide polymorphism-trait associations from 1673 GWAS and is updated on a regular basis. Integrating data with software ensures more rigorous application of hypothesis-driven analyses and allows millions of potential causal relationships to be efficiently evaluated in phenome-wide association studies.\", \"founded_year\": 2018, \"host_institution\": \"University of Bristol\", \"address\": \"Medical Research Council (MRC) Integrative Epidemiology Unit, Population Health Sciences, Bristol Medical School, University of Bristol, Bristol, United Kingdom\", \"city\": null, \"province\": null, \"country\": \"United Kingdom\", \"current_version\": null, \"contact_name\": \"Gibran Hemani\", \"contact_email\": \"g.hemani@bristol.ac.uk\", \"available_protocol\": 1.0, \"last_update\": null, \"age\": 5, \"citation\": 1048, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1572226039000, \"last_modified\": \"2023-03-03 10:16:51\", \"data_type_list\": \"DNA\", \"category_list\": \"Genotype phenotype and variation, Health and medicine\", \"keywords_list\": \"Mendelian randomization, single nucleotide polymorphism-trait associations\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 209.6, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:36\", \"ess\": 0}, {\"page_id\": \"54_home\", \"is_success\": true, \"error_count\": 8.0, \"label_missing\": 2.0, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 2.0, \"aria_menu_broken\": null, \"alt_link_missing\": 2.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": 1.0, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 54, \"page_type\": \"home\", \"source_id\": \"dc_1281\", \"short_name\": \"TRANSFAC\", \"accession\": \"DBC001281\", \"full_name\": \"a database on transcription factors and their DNA binding sites\", \"description\": \"TRANSFAC\\u00ae is a unique knowledge-base containing published data on eukaryotic transcription factors and miRNAs, their experimentally-proven binding sites, and regulated genes.\", \"founded_year\": 1996, \"host_institution\": \"GeneXplain GmbH\", \"address\": \"Halchtersche Strasse 33, D-38304 Wolfenb\\u00fcttel, Germany\", \"city\": \"Wolfenb\\u00fcttel\", \"province\": null, \"country\": \"Germany\", \"current_version\": \"7.0\", \"contact_name\": \"Edgar Wingender\", \"contact_email\": \"info@biobase-international.com\", \"available_protocol\": 1.0, \"last_update\": \"2015-04-30\", \"age\": 27, \"citation\": 5541, \"token\": \"$2a$10$yy/Xod8MxwW/JgftwuwDROQH3gc4gK1qbMbsFmil9ISB7qbvNs1MK\", \"expire_date\": 1543989940811.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1437123877000, \"last_modified\": \"2023-03-03 10:11:13\", \"data_type_list\": \"DNA, Protein, RNA\", \"category_list\": \"Interaction\", \"keywords_list\": \"ChIP-seq, miRNA, transcription factor\", \"data_object_list\": \"Animal, Plant, Fungi\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 205.222, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:36\", \"ess\": 0}, {\"page_id\": \"55_home\", \"is_success\": true, \"error_count\": 2.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 1.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 55, \"page_type\": \"home\", \"source_id\": \"dc_3941\", \"short_name\": \"miRWalk\", \"accession\": \"DBC003941\", \"full_name\": null, \"description\": \"a freely accessible, regularly updated comprehensive archive supplying the largest available collection of predicted and experimentally verified miRNA-target interactions, with various novel and unique features to assist the scientific community.\", \"founded_year\": 2011, \"host_institution\": \"University of Heidelberg\", \"address\": \"Present affiliation: Division of Bioinformatics and Biostatistics, National Center for Toxicological Research (NCTR), Food and Drug Administration (FDA), Jefferson, Arizona.\\n\", \"city\": null, \"province\": null, \"country\": \"Germany\", \"current_version\": \"v.2.0\", \"contact_name\": \"Carsten Sticht\", \"contact_email\": \"mirwalkteam@medma.uni-heidelberg.de\", \"available_protocol\": 1.0, \"last_update\": \"2018\", \"age\": 12, \"citation\": 2461, \"token\": \"$2a$10$oxjlXiJV1GXLd0og.VMRteci97SgFwd.2fiDA459kE1nLcMxrlwhW\", \"expire_date\": 1544076379011.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1517081551000, \"last_modified\": \"2023-03-03 10:13:28\", \"data_type_list\": \"RNA\", \"category_list\": \"Interaction\", \"keywords_list\": \"gene ontology, micro RNA, target prediction\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 205.083, \"first_publication_year\": 2016, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:01:24\", \"ess\": 1}, {\"page_id\": \"56_home\", \"is_success\": true, \"error_count\": 3.0, \"label_missing\": 2.0, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 56, \"page_type\": \"home\", \"source_id\": \"dc_17\", \"short_name\": \"Rfam\", \"accession\": \"DBC000017\", \"full_name\": \"RNA Families\", \"description\": \"The Rfam database is a collection of non-coding RNA families represented by manually curated sequence alignments, consensus secondary structures and annotation gathered from corresponding Wikipedia, taxonomy and ontology resources.\", \"founded_year\": 2003, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"Wellcome Genome Campus, Hinxton, Cambridge, UK\", \"city\": \"Cambridge\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"v14.9\", \"contact_name\": \"Alex Bateman\", \"contact_email\": \"agb@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2022-11\", \"age\": 20, \"citation\": 4084, \"token\": \"$2a$10$thJXTWwFFA4Bcto2HKpqbO.s4VwHolBuAIZuY4p5TJjfGUeyrZeIO\", \"expire_date\": 1543898843283.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-06 11:06:27\", \"data_type_list\": \"RNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"covariance model, ncRNA, RNA family, RNA motif, secondary structure, sequence alignment\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens, Escherichia coli, Drosophila melanogaster, Mus musculus, Caenorhabditis elegans, Bacillus subtilis\", \"theme_list\": null, \"zindex\": 204.2, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:02\", \"ess\": 0}, {\"page_id\": \"57_home\", \"is_success\": true, \"error_count\": 4.0, \"label_missing\": 1.0, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 2.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 57, \"page_type\": \"home\", \"source_id\": \"dc_71\", \"short_name\": \"PhosphoSitePlus\", \"accession\": \"DBC000071\", \"full_name\": null, \"description\": \"PhosphoSitePlus(\\u00ae) (PSP), a knowledgebase dedicated to mammalian post-translational modifications (PTMs), contains over 330,000 non-redundant PTMs, including phospho, acetyl, ubiquityl and methyl groups.\", \"founded_year\": 2012, \"host_institution\": \"Cell Signaling Technology\", \"address\": \"Cell Signaling Technology,3 Trask Lane,Danvers,MA 01923,USA\", \"city\": \"Danvers\", \"province\": \"MA\", \"country\": \"United States\", \"current_version\": \"v1.0\", \"contact_name\": \"EditorPhosphoSite\", \"contact_email\": \"EditorPhosphoSite@cellsignal.com\", \"available_protocol\": 1.0, \"last_update\": \"2014-12\", \"age\": 11, \"citation\": 2244, \"token\": \"$2a$10$MYHS.0iaIuxOeZpCKWKy9eYp6Pmq9KT27hilryN39UeTMBqmByE6a\", \"expire_date\": 1543898885234.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:43\", \"data_type_list\": \"Protein\", \"category_list\": \"Modification\", \"keywords_list\": \"mass spectrometry experiment, post-translational modification\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 204.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:04\", \"ess\": 0}, {\"page_id\": \"58_home\", \"is_success\": true, \"error_count\": 7.0, \"label_missing\": null, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": 3.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": 2.0, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 58, \"page_type\": \"home\", \"source_id\": \"dc_1077\", \"short_name\": \"MSigDB\", \"accession\": \"DBC001077\", \"full_name\": \"Molecular Signatures Database\", \"description\": \"The Molecular Signatures Database (MSigDB) is a collection of annotated gene sets for use with GSEA software.The Molecular(MSigDB) is a collection of annotated gene sets for use with GSEA software.\", \"founded_year\": 2011, \"host_institution\": \"Broad Institute\", \"address\": \"320 Charles Street, Cambridge, MA\", \"city\": \"Cambridge\", \"province\": \"MA\", \"country\": \"United States\", \"current_version\": \"5.1\", \"contact_name\": \"Mesirov JP\", \"contact_email\": \"gsea@broadinstitute.org\", \"available_protocol\": 1.0, \"last_update\": \"2016-01-01\", \"age\": 12, \"citation\": 2375, \"token\": \"$2a$10$1R84HHbv9wMtdEZCSkzyWulsOC7I2SGgy5X9xLBqxPX5pSgl9JNR2\", \"expire_date\": 1543975626640.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1435653470000, \"last_modified\": \"2023-03-03 10:10:57\", \"data_type_list\": \"DNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"GSEA, gene annotation\", \"data_object_list\": null, \"organism_list\": null, \"theme_list\": null, \"zindex\": 197.917, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:31\", \"ess\": 0}, {\"page_id\": \"59_home\", \"is_success\": true, \"error_count\": 2.0, \"label_missing\": null, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 59, \"page_type\": \"home\", \"source_id\": \"dc_7903\", \"short_name\": \"CellChat\", \"accession\": \"DBC007903\", \"full_name\": null, \"description\": \"CellChat Explorer contains two major components: (a) Ligand-Receptor Interaction Explorer that allows easy exploration of our ligand-receptor interaction database, and (b) Cell- Cell Communication Atlas Explorer that allows easy exploration of the cell-cell communications for any given scRNA-seq dataset that has been processed by the R toolkit CellChat.\", \"founded_year\": 2021, \"host_institution\": \"University of California Irvine\", \"address\": null, \"city\": null, \"province\": null, \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Qing Nie\", \"contact_email\": \"qnie@uci.edu\", \"available_protocol\": 1.0, \"last_update\": null, \"age\": 2, \"citation\": 393, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1650615744000, \"last_modified\": \"2023-10-08 23:06:13\", \"data_type_list\": \"RNA, Protein\", \"category_list\": \"Interaction, Pathway, Expression\", \"keywords_list\": \"scRNA-seq, ligand-receptor, cell- cell communication\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 196.5, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:58\", \"ess\": 0}, {\"page_id\": \"60_home\", \"is_success\": true, \"error_count\": 1.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 60, \"page_type\": \"home\", \"source_id\": \"dc_36\", \"short_name\": \"SMART\", \"accession\": \"DBC000036\", \"full_name\": \"Simple Modular Architecture Research Tool\", \"description\": \"A web resource providing simple identification and extensive annotation of protein domains and the exploration of protein domain architectures\", \"founded_year\": 1998, \"host_institution\": \"European Molecular Biology Laboratory\", \"address\": \"Bothestr 142,69126 Heidelberg,Germany\", \"city\": \"Heidelberg\", \"province\": null, \"country\": \"Germany\", \"current_version\": \"v8.0\", \"contact_name\": \"Peer Bork\", \"contact_email\": \"bork@embl.de\", \"available_protocol\": 1.0, \"last_update\": \"2017\", \"age\": 25, \"citation\": 4893, \"token\": \"$2a$10$t5ywnrRp/KNgAGjUZb0PrOqH1MFYfkgJASFLjOFy.MxoByOMkGOU2\", \"expire_date\": 1543898859394.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:40\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation, Structure\", \"keywords_list\": \"protein domain\", \"data_object_list\": \"Animal, Plant\", \"organism_list\": null, \"theme_list\": null, \"zindex\": 195.72, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:03\", \"ess\": 0}, {\"page_id\": \"1_search\", \"is_success\": true, \"error_count\": 165.0, \"label_missing\": 162.0, \"language_missing\": 1.0, \"button_empty\": 1.0, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 1, \"page_type\": \"search\", \"source_id\": \"dc_3061\", \"short_name\": \"DAVID\", \"accession\": \"DBC003061\", \"full_name\": \"Database for Annotation, Visualization and Integrated Discovery\", \"description\": \"The Database for Annotation, Visualization and Integrated Discovery (DAVID) comprises a full knowledgebase update to the sixth version of our original web-accessible programs. DAVID now provides a comprehensive set of functional annotation tools for investigators to understand biological meaning behind large list of genes. \", \"founded_year\": 2003, \"host_institution\": \"National Cancer Institute\", \"address\": \" Laboratory of Immunopathogenesis and Bioinformatics, National Cancer Institute, Frederick, MD 21702, USA\", \"city\": \"Frederick\", \"province\": \"Maryland\", \"country\": \"United States\", \"current_version\": \"version6.8\", \"contact_name\": \"DAVID Bioinformatic Team\", \"contact_email\": \"weizhong.chang@nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2022-2-19\", \"age\": 20, \"citation\": 36956, \"token\": \"$2a$10$zwt9PsSE9s4OW93qMVcFwe3j0lOsvk1m.vZgQ/V4/plC7WQ30GO0G\", \"expire_date\": 1543995762746.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1516919168000, \"last_modified\": \"2023-03-03 10:12:16\", \"data_type_list\": \"DNA\", \"category_list\": \"Pathway, Standard ontology and nomenclature\", \"keywords_list\": \"functional annotation\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 1847.8, \"first_publication_year\": 2007, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:01:00\", \"ess\": 0}, {\"page_id\": \"1_search_result\", \"is_success\": true, \"error_count\": 165.0, \"label_missing\": 162.0, \"language_missing\": 1.0, \"button_empty\": 1.0, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 1, \"page_type\": \"search_result\", \"source_id\": \"dc_3061\", \"short_name\": \"DAVID\", \"accession\": \"DBC003061\", \"full_name\": \"Database for Annotation, Visualization and Integrated Discovery\", \"description\": \"The Database for Annotation, Visualization and Integrated Discovery (DAVID) comprises a full knowledgebase update to the sixth version of our original web-accessible programs. DAVID now provides a comprehensive set of functional annotation tools for investigators to understand biological meaning behind large list of genes. \", \"founded_year\": 2003, \"host_institution\": \"National Cancer Institute\", \"address\": \" Laboratory of Immunopathogenesis and Bioinformatics, National Cancer Institute, Frederick, MD 21702, USA\", \"city\": \"Frederick\", \"province\": \"Maryland\", \"country\": \"United States\", \"current_version\": \"version6.8\", \"contact_name\": \"DAVID Bioinformatic Team\", \"contact_email\": \"weizhong.chang@nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2022-2-19\", \"age\": 20, \"citation\": 36956, \"token\": \"$2a$10$zwt9PsSE9s4OW93qMVcFwe3j0lOsvk1m.vZgQ/V4/plC7WQ30GO0G\", \"expire_date\": 1543995762746.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1516919168000, \"last_modified\": \"2023-03-03 10:12:16\", \"data_type_list\": \"DNA\", \"category_list\": \"Pathway, Standard ontology and nomenclature\", \"keywords_list\": \"functional annotation\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 1847.8, \"first_publication_year\": 2007, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:01:00\", \"ess\": 0}, {\"page_id\": \"1_data_entity\", \"is_success\": true, \"error_count\": 2.0, \"label_missing\": 1.0, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 1, \"page_type\": \"data_entity\", \"source_id\": \"dc_3061\", \"short_name\": \"DAVID\", \"accession\": \"DBC003061\", \"full_name\": \"Database for Annotation, Visualization and Integrated Discovery\", \"description\": \"The Database for Annotation, Visualization and Integrated Discovery (DAVID) comprises a full knowledgebase update to the sixth version of our original web-accessible programs. DAVID now provides a comprehensive set of functional annotation tools for investigators to understand biological meaning behind large list of genes. \", \"founded_year\": 2003, \"host_institution\": \"National Cancer Institute\", \"address\": \" Laboratory of Immunopathogenesis and Bioinformatics, National Cancer Institute, Frederick, MD 21702, USA\", \"city\": \"Frederick\", \"province\": \"Maryland\", \"country\": \"United States\", \"current_version\": \"version6.8\", \"contact_name\": \"DAVID Bioinformatic Team\", \"contact_email\": \"weizhong.chang@nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2022-2-19\", \"age\": 20, \"citation\": 36956, \"token\": \"$2a$10$zwt9PsSE9s4OW93qMVcFwe3j0lOsvk1m.vZgQ/V4/plC7WQ30GO0G\", \"expire_date\": 1543995762746.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1516919168000, \"last_modified\": \"2023-03-03 10:12:16\", \"data_type_list\": \"DNA\", \"category_list\": \"Pathway, Standard ontology and nomenclature\", \"keywords_list\": \"functional annotation\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 1847.8, \"first_publication_year\": 2007, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:01:00\", \"ess\": 0}, {\"page_id\": \"1_documentation\", \"is_success\": true, \"error_count\": 3.0, \"label_missing\": 1.0, \"language_missing\": 1.0, \"button_empty\": 1.0, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 1, \"page_type\": \"documentation\", \"source_id\": \"dc_3061\", \"short_name\": \"DAVID\", \"accession\": \"DBC003061\", \"full_name\": \"Database for Annotation, Visualization and Integrated Discovery\", \"description\": \"The Database for Annotation, Visualization and Integrated Discovery (DAVID) comprises a full knowledgebase update to the sixth version of our original web-accessible programs. DAVID now provides a comprehensive set of functional annotation tools for investigators to understand biological meaning behind large list of genes. \", \"founded_year\": 2003, \"host_institution\": \"National Cancer Institute\", \"address\": \" Laboratory of Immunopathogenesis and Bioinformatics, National Cancer Institute, Frederick, MD 21702, USA\", \"city\": \"Frederick\", \"province\": \"Maryland\", \"country\": \"United States\", \"current_version\": \"version6.8\", \"contact_name\": \"DAVID Bioinformatic Team\", \"contact_email\": \"weizhong.chang@nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2022-2-19\", \"age\": 20, \"citation\": 36956, \"token\": \"$2a$10$zwt9PsSE9s4OW93qMVcFwe3j0lOsvk1m.vZgQ/V4/plC7WQ30GO0G\", \"expire_date\": 1543995762746.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1516919168000, \"last_modified\": \"2023-03-03 10:12:16\", \"data_type_list\": \"DNA\", \"category_list\": \"Pathway, Standard ontology and nomenclature\", \"keywords_list\": \"functional annotation\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 1847.8, \"first_publication_year\": 2007, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:01:00\", \"ess\": 0}, {\"page_id\": \"2_search\", \"is_success\": true, \"error_count\": 3.0, \"label_missing\": 3.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 2, \"page_type\": \"search\", \"source_id\": \"dc_184\", \"short_name\": \"KEGG\", \"accession\": \"DBC000184\", \"full_name\": \"Kyoto Encyclopedia of Genes and Genomes\", \"description\": \"KEGG is a database resource for understanding high-level functions and utilities of the biological system.\", \"founded_year\": 1997, \"host_institution\": \"Kyoto University\", \"address\": \"Uji,Kyoto 611-0011,Japan\", \"city\": \"Kyoto\", \"province\": null, \"country\": \"Japan\", \"current_version\": \"v86.1\", \"contact_name\": \"Minoru Kanehisa\", \"contact_email\": \"kanehisa@kuicr.kyoto-u.ac.jp\", \"available_protocol\": 1.0, \"last_update\": \"2018-05-10\", \"age\": 26, \"citation\": 36897, \"token\": \"$2a$10$PoC/2ktUbVoHS7Jjdb1R9.1vMe93zm3TG7TsKiN9MvpCeZ1aiLro.\", \"expire_date\": 1543898963375.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:53\", \"data_type_list\": \"Other\", \"category_list\": \"Pathway, Standard ontology and nomenclature\", \"keywords_list\": \"functional annotation\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Escherichia coli, Homo sapiens\", \"theme_list\": null, \"zindex\": 1419.12, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:07\", \"ess\": 0}, {\"page_id\": \"2_search_result\", \"is_success\": true, \"error_count\": 2.0, \"label_missing\": null, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": 1.0, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 2, \"page_type\": \"search_result\", \"source_id\": \"dc_184\", \"short_name\": \"KEGG\", \"accession\": \"DBC000184\", \"full_name\": \"Kyoto Encyclopedia of Genes and Genomes\", \"description\": \"KEGG is a database resource for understanding high-level functions and utilities of the biological system.\", \"founded_year\": 1997, \"host_institution\": \"Kyoto University\", \"address\": \"Uji,Kyoto 611-0011,Japan\", \"city\": \"Kyoto\", \"province\": null, \"country\": \"Japan\", \"current_version\": \"v86.1\", \"contact_name\": \"Minoru Kanehisa\", \"contact_email\": \"kanehisa@kuicr.kyoto-u.ac.jp\", \"available_protocol\": 1.0, \"last_update\": \"2018-05-10\", \"age\": 26, \"citation\": 36897, \"token\": \"$2a$10$PoC/2ktUbVoHS7Jjdb1R9.1vMe93zm3TG7TsKiN9MvpCeZ1aiLro.\", \"expire_date\": 1543898963375.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:53\", \"data_type_list\": \"Other\", \"category_list\": \"Pathway, Standard ontology and nomenclature\", \"keywords_list\": \"functional annotation\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Escherichia coli, Homo sapiens\", \"theme_list\": null, \"zindex\": 1419.12, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:07\", \"ess\": 0}, {\"page_id\": \"2_data_entity\", \"is_success\": true, \"error_count\": 3.0, \"label_missing\": 2.0, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 2, \"page_type\": \"data_entity\", \"source_id\": \"dc_184\", \"short_name\": \"KEGG\", \"accession\": \"DBC000184\", \"full_name\": \"Kyoto Encyclopedia of Genes and Genomes\", \"description\": \"KEGG is a database resource for understanding high-level functions and utilities of the biological system.\", \"founded_year\": 1997, \"host_institution\": \"Kyoto University\", \"address\": \"Uji,Kyoto 611-0011,Japan\", \"city\": \"Kyoto\", \"province\": null, \"country\": \"Japan\", \"current_version\": \"v86.1\", \"contact_name\": \"Minoru Kanehisa\", \"contact_email\": \"kanehisa@kuicr.kyoto-u.ac.jp\", \"available_protocol\": 1.0, \"last_update\": \"2018-05-10\", \"age\": 26, \"citation\": 36897, \"token\": \"$2a$10$PoC/2ktUbVoHS7Jjdb1R9.1vMe93zm3TG7TsKiN9MvpCeZ1aiLro.\", \"expire_date\": 1543898963375.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:53\", \"data_type_list\": \"Other\", \"category_list\": \"Pathway, Standard ontology and nomenclature\", \"keywords_list\": \"functional annotation\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Escherichia coli, Homo sapiens\", \"theme_list\": null, \"zindex\": 1419.12, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:07\", \"ess\": 0}, {\"page_id\": \"2_documentation\", \"is_success\": true, \"error_count\": 1.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 2, \"page_type\": \"documentation\", \"source_id\": \"dc_184\", \"short_name\": \"KEGG\", \"accession\": \"DBC000184\", \"full_name\": \"Kyoto Encyclopedia of Genes and Genomes\", \"description\": \"KEGG is a database resource for understanding high-level functions and utilities of the biological system.\", \"founded_year\": 1997, \"host_institution\": \"Kyoto University\", \"address\": \"Uji,Kyoto 611-0011,Japan\", \"city\": \"Kyoto\", \"province\": null, \"country\": \"Japan\", \"current_version\": \"v86.1\", \"contact_name\": \"Minoru Kanehisa\", \"contact_email\": \"kanehisa@kuicr.kyoto-u.ac.jp\", \"available_protocol\": 1.0, \"last_update\": \"2018-05-10\", \"age\": 26, \"citation\": 36897, \"token\": \"$2a$10$PoC/2ktUbVoHS7Jjdb1R9.1vMe93zm3TG7TsKiN9MvpCeZ1aiLro.\", \"expire_date\": 1543898963375.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:53\", \"data_type_list\": \"Other\", \"category_list\": \"Pathway, Standard ontology and nomenclature\", \"keywords_list\": \"functional annotation\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Escherichia coli, Homo sapiens\", \"theme_list\": null, \"zindex\": 1419.12, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:07\", \"ess\": 0}, {\"page_id\": \"3_search\", \"is_success\": true, \"error_count\": 835.0, \"label_missing\": 1.0, \"language_missing\": 1.0, \"button_empty\": 1.0, \"alt_missing\": 39.0, \"link_empty\": 792.0, \"aria_menu_broken\": 1.0, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 3, \"page_type\": \"search\", \"source_id\": \"dc_6569\", \"short_name\": \"cBioPortal\", \"accession\": \"DBC006569\", \"full_name\": \"cBio Cancer Genomics Portal\", \"description\": \"cBio Cancer Genomics Portal is an open-access resource for interactive exploration of multidimensional cancer genomics data sets, currently providing access to data from more than 5,000 tumor samples from 20 cancer studies.\", \"founded_year\": 2012, \"host_institution\": \"Memorial Sloan Kettering Cancer Center\", \"address\": \"Computational Biology Center, Memorial Sloan-Kettering Cancer Center, 1275 York Avenue, New York, NY 10021.\", \"city\": \"New York\", \"province\": null, \"country\": \"United States\", \"current_version\": \"3.7.9\", \"contact_name\": \"Ethan Cerami\", \"contact_email\": \"cancergenomics@cbio.mskcc.org\", \"available_protocol\": 1.0, \"last_update\": \"2021-10-05\", \"age\": 11, \"citation\": 14522, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1571795759000, \"last_modified\": \"2023-03-03 10:16:31\", \"data_type_list\": \"DNA, RNA, Protein\", \"category_list\": \"Genotype phenotype and variation, Expression, Modification, Structure, Health and medicine\", \"keywords_list\": \"tumor, cancer genome, TCGA\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 1320.18, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:30\", \"ess\": 0}, {\"page_id\": \"3_search_result\", \"is_success\": true, \"error_count\": 835.0, \"label_missing\": 1.0, \"language_missing\": 1.0, \"button_empty\": 1.0, \"alt_missing\": 39.0, \"link_empty\": 792.0, \"aria_menu_broken\": 1.0, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 3, \"page_type\": \"search_result\", \"source_id\": \"dc_6569\", \"short_name\": \"cBioPortal\", \"accession\": \"DBC006569\", \"full_name\": \"cBio Cancer Genomics Portal\", \"description\": \"cBio Cancer Genomics Portal is an open-access resource for interactive exploration of multidimensional cancer genomics data sets, currently providing access to data from more than 5,000 tumor samples from 20 cancer studies.\", \"founded_year\": 2012, \"host_institution\": \"Memorial Sloan Kettering Cancer Center\", \"address\": \"Computational Biology Center, Memorial Sloan-Kettering Cancer Center, 1275 York Avenue, New York, NY 10021.\", \"city\": \"New York\", \"province\": null, \"country\": \"United States\", \"current_version\": \"3.7.9\", \"contact_name\": \"Ethan Cerami\", \"contact_email\": \"cancergenomics@cbio.mskcc.org\", \"available_protocol\": 1.0, \"last_update\": \"2021-10-05\", \"age\": 11, \"citation\": 14522, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1571795759000, \"last_modified\": \"2023-03-03 10:16:31\", \"data_type_list\": \"DNA, RNA, Protein\", \"category_list\": \"Genotype phenotype and variation, Expression, Modification, Structure, Health and medicine\", \"keywords_list\": \"tumor, cancer genome, TCGA\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 1320.18, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:30\", \"ess\": 0}, {\"page_id\": \"3_data_entity\", \"is_success\": true, \"error_count\": 25.0, \"label_missing\": 7.0, \"language_missing\": 1.0, \"button_empty\": 4.0, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": 12.0, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 3, \"page_type\": \"data_entity\", \"source_id\": \"dc_6569\", \"short_name\": \"cBioPortal\", \"accession\": \"DBC006569\", \"full_name\": \"cBio Cancer Genomics Portal\", \"description\": \"cBio Cancer Genomics Portal is an open-access resource for interactive exploration of multidimensional cancer genomics data sets, currently providing access to data from more than 5,000 tumor samples from 20 cancer studies.\", \"founded_year\": 2012, \"host_institution\": \"Memorial Sloan Kettering Cancer Center\", \"address\": \"Computational Biology Center, Memorial Sloan-Kettering Cancer Center, 1275 York Avenue, New York, NY 10021.\", \"city\": \"New York\", \"province\": null, \"country\": \"United States\", \"current_version\": \"3.7.9\", \"contact_name\": \"Ethan Cerami\", \"contact_email\": \"cancergenomics@cbio.mskcc.org\", \"available_protocol\": 1.0, \"last_update\": \"2021-10-05\", \"age\": 11, \"citation\": 14522, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1571795759000, \"last_modified\": \"2023-03-03 10:16:31\", \"data_type_list\": \"DNA, RNA, Protein\", \"category_list\": \"Genotype phenotype and variation, Expression, Modification, Structure, Health and medicine\", \"keywords_list\": \"tumor, cancer genome, TCGA\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 1320.18, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:30\", \"ess\": 0}, {\"page_id\": \"3_documentation\", \"is_success\": true, \"error_count\": 31.0, \"label_missing\": 4.0, \"language_missing\": null, \"button_empty\": 21.0, \"alt_missing\": 2.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": 4.0, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 3, \"page_type\": \"documentation\", \"source_id\": \"dc_6569\", \"short_name\": \"cBioPortal\", \"accession\": \"DBC006569\", \"full_name\": \"cBio Cancer Genomics Portal\", \"description\": \"cBio Cancer Genomics Portal is an open-access resource for interactive exploration of multidimensional cancer genomics data sets, currently providing access to data from more than 5,000 tumor samples from 20 cancer studies.\", \"founded_year\": 2012, \"host_institution\": \"Memorial Sloan Kettering Cancer Center\", \"address\": \"Computational Biology Center, Memorial Sloan-Kettering Cancer Center, 1275 York Avenue, New York, NY 10021.\", \"city\": \"New York\", \"province\": null, \"country\": \"United States\", \"current_version\": \"3.7.9\", \"contact_name\": \"Ethan Cerami\", \"contact_email\": \"cancergenomics@cbio.mskcc.org\", \"available_protocol\": 1.0, \"last_update\": \"2021-10-05\", \"age\": 11, \"citation\": 14522, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1571795759000, \"last_modified\": \"2023-03-03 10:16:31\", \"data_type_list\": \"DNA, RNA, Protein\", \"category_list\": \"Genotype phenotype and variation, Expression, Modification, Structure, Health and medicine\", \"keywords_list\": \"tumor, cancer genome, TCGA\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 1320.18, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:30\", \"ess\": 0}, {\"page_id\": \"4_search\", \"is_success\": true, \"error_count\": 48.0, \"label_missing\": 46.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 2.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 4, \"page_type\": \"search\", \"source_id\": \"dc_62\", \"short_name\": \"STRING\", \"accession\": \"DBC000062\", \"full_name\": \"Known and Predicted Protein-Protein Interactions\", \"description\": \"STRING is a database of known and predicted protein interactions. The interactions include direct (physical) and indirect (functional) associations\", \"founded_year\": 2000, \"host_institution\": \"Swiss Institute of Bioinformatics\", \"address\": \"Quartier Sorge - Batiment Amphipole\\n1015 Lausanne / Switzerland\", \"city\": null, \"province\": null, \"country\": \"Switzerland\", \"current_version\": \"v11.5\", \"contact_name\": \"Peer Bork\", \"contact_email\": \"bork@embl.de\", \"available_protocol\": 1.0, \"last_update\": null, \"age\": 23, \"citation\": 22950, \"token\": \"$2a$10$qCP054uAASyeYX0FoxplH.cHAIM4TlBWV1lqo/nPXM69GAAGR1HyS\", \"expire_date\": 1543898876141.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-08-23 10:25:04\", \"data_type_list\": \"Protein\", \"category_list\": \"Interaction\", \"keywords_list\": \"protein-protein interaction (PPI)\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea\", \"organism_list\": \"Homo sapiens, Methanococcus voltae, Arabidopsis thaliana\", \"theme_list\": null, \"zindex\": 997.826, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:04\", \"ess\": 0}, {\"page_id\": \"4_search_result\", \"is_success\": true, \"error_count\": 1.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 1.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 4, \"page_type\": \"search_result\", \"source_id\": \"dc_62\", \"short_name\": \"STRING\", \"accession\": \"DBC000062\", \"full_name\": \"Known and Predicted Protein-Protein Interactions\", \"description\": \"STRING is a database of known and predicted protein interactions. The interactions include direct (physical) and indirect (functional) associations\", \"founded_year\": 2000, \"host_institution\": \"Swiss Institute of Bioinformatics\", \"address\": \"Quartier Sorge - Batiment Amphipole\\n1015 Lausanne / Switzerland\", \"city\": null, \"province\": null, \"country\": \"Switzerland\", \"current_version\": \"v11.5\", \"contact_name\": \"Peer Bork\", \"contact_email\": \"bork@embl.de\", \"available_protocol\": 1.0, \"last_update\": null, \"age\": 23, \"citation\": 22950, \"token\": \"$2a$10$qCP054uAASyeYX0FoxplH.cHAIM4TlBWV1lqo/nPXM69GAAGR1HyS\", \"expire_date\": 1543898876141.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-08-23 10:25:04\", \"data_type_list\": \"Protein\", \"category_list\": \"Interaction\", \"keywords_list\": \"protein-protein interaction (PPI)\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea\", \"organism_list\": \"Homo sapiens, Methanococcus voltae, Arabidopsis thaliana\", \"theme_list\": null, \"zindex\": 997.826, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:04\", \"ess\": 0}, {\"page_id\": \"4_documentation\", \"is_success\": true, \"error_count\": 1.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 4, \"page_type\": \"documentation\", \"source_id\": \"dc_62\", \"short_name\": \"STRING\", \"accession\": \"DBC000062\", \"full_name\": \"Known and Predicted Protein-Protein Interactions\", \"description\": \"STRING is a database of known and predicted protein interactions. The interactions include direct (physical) and indirect (functional) associations\", \"founded_year\": 2000, \"host_institution\": \"Swiss Institute of Bioinformatics\", \"address\": \"Quartier Sorge - Batiment Amphipole\\n1015 Lausanne / Switzerland\", \"city\": null, \"province\": null, \"country\": \"Switzerland\", \"current_version\": \"v11.5\", \"contact_name\": \"Peer Bork\", \"contact_email\": \"bork@embl.de\", \"available_protocol\": 1.0, \"last_update\": null, \"age\": 23, \"citation\": 22950, \"token\": \"$2a$10$qCP054uAASyeYX0FoxplH.cHAIM4TlBWV1lqo/nPXM69GAAGR1HyS\", \"expire_date\": 1543898876141.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-08-23 10:25:04\", \"data_type_list\": \"Protein\", \"category_list\": \"Interaction\", \"keywords_list\": \"protein-protein interaction (PPI)\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea\", \"organism_list\": \"Homo sapiens, Methanococcus voltae, Arabidopsis thaliana\", \"theme_list\": null, \"zindex\": 997.826, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:04\", \"ess\": 0}, {\"page_id\": \"5_search\", \"is_success\": true, \"error_count\": 4.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": 2.0, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 5, \"page_type\": \"search\", \"source_id\": \"dc_317\", \"short_name\": \"ENCODE\", \"accession\": \"DBC000317\", \"full_name\": \"Encyclopedia of DNA Elements\", \"description\": \"The ENCODE Consortium is an international collaboration of research groups funded by the National Human Genome Research Institute (NHGRI). The goal of ENCODE is to build a comprehensive parts list of functional elements in the human genome, including elements that act at the protein and RNA levels,and regulatory elements that control cells and circumstances in which a gene is active.\", \"founded_year\": 2004, \"host_institution\": \"Stanford University\", \"address\": \"Center for Biomolecular Science and Engineering, School of Engineering, University of California Santa Cruz (UCSC), Santa Cruz, CA 95064, USA.\", \"city\": \"Stanford\", \"province\": \"CA\", \"country\": \"United States\", \"current_version\": \"v128\", \"contact_name\": \"ENCODE team\", \"contact_email\": \"encode-help@lists.stanford.edu\", \"available_protocol\": 1.0, \"last_update\": null, \"age\": 19, \"citation\": 17908, \"token\": \"$2a$10$jHL5wkABrxhdzzz065t9I..ybMHhGl2yVaf2jFfx.bmUX8d4z5euu\", \"expire_date\": 1543899046887.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-24 22:11:42\", \"data_type_list\": \"DNA, Protein, RNA\", \"category_list\": \"Gene genome and annotation, Raw bio-data\", \"keywords_list\": \"functional element, regulatory element\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 942.526, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:11\", \"ess\": 0}, {\"page_id\": \"5_search_result\", \"is_success\": true, \"error_count\": 4.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": 2.0, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 5, \"page_type\": \"search_result\", \"source_id\": \"dc_317\", \"short_name\": \"ENCODE\", \"accession\": \"DBC000317\", \"full_name\": \"Encyclopedia of DNA Elements\", \"description\": \"The ENCODE Consortium is an international collaboration of research groups funded by the National Human Genome Research Institute (NHGRI). The goal of ENCODE is to build a comprehensive parts list of functional elements in the human genome, including elements that act at the protein and RNA levels,and regulatory elements that control cells and circumstances in which a gene is active.\", \"founded_year\": 2004, \"host_institution\": \"Stanford University\", \"address\": \"Center for Biomolecular Science and Engineering, School of Engineering, University of California Santa Cruz (UCSC), Santa Cruz, CA 95064, USA.\", \"city\": \"Stanford\", \"province\": \"CA\", \"country\": \"United States\", \"current_version\": \"v128\", \"contact_name\": \"ENCODE team\", \"contact_email\": \"encode-help@lists.stanford.edu\", \"available_protocol\": 1.0, \"last_update\": null, \"age\": 19, \"citation\": 17908, \"token\": \"$2a$10$jHL5wkABrxhdzzz065t9I..ybMHhGl2yVaf2jFfx.bmUX8d4z5euu\", \"expire_date\": 1543899046887.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-24 22:11:42\", \"data_type_list\": \"DNA, Protein, RNA\", \"category_list\": \"Gene genome and annotation, Raw bio-data\", \"keywords_list\": \"functional element, regulatory element\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 942.526, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:11\", \"ess\": 0}, {\"page_id\": \"5_data_entity\", \"is_success\": true, \"error_count\": 4.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": 2.0, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 5, \"page_type\": \"data_entity\", \"source_id\": \"dc_317\", \"short_name\": \"ENCODE\", \"accession\": \"DBC000317\", \"full_name\": \"Encyclopedia of DNA Elements\", \"description\": \"The ENCODE Consortium is an international collaboration of research groups funded by the National Human Genome Research Institute (NHGRI). The goal of ENCODE is to build a comprehensive parts list of functional elements in the human genome, including elements that act at the protein and RNA levels,and regulatory elements that control cells and circumstances in which a gene is active.\", \"founded_year\": 2004, \"host_institution\": \"Stanford University\", \"address\": \"Center for Biomolecular Science and Engineering, School of Engineering, University of California Santa Cruz (UCSC), Santa Cruz, CA 95064, USA.\", \"city\": \"Stanford\", \"province\": \"CA\", \"country\": \"United States\", \"current_version\": \"v128\", \"contact_name\": \"ENCODE team\", \"contact_email\": \"encode-help@lists.stanford.edu\", \"available_protocol\": 1.0, \"last_update\": null, \"age\": 19, \"citation\": 17908, \"token\": \"$2a$10$jHL5wkABrxhdzzz065t9I..ybMHhGl2yVaf2jFfx.bmUX8d4z5euu\", \"expire_date\": 1543899046887.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-24 22:11:42\", \"data_type_list\": \"DNA, Protein, RNA\", \"category_list\": \"Gene genome and annotation, Raw bio-data\", \"keywords_list\": \"functional element, regulatory element\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 942.526, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:11\", \"ess\": 0}, {\"page_id\": \"5_documentation\", \"is_success\": true, \"error_count\": 4.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": 2.0, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 5, \"page_type\": \"documentation\", \"source_id\": \"dc_317\", \"short_name\": \"ENCODE\", \"accession\": \"DBC000317\", \"full_name\": \"Encyclopedia of DNA Elements\", \"description\": \"The ENCODE Consortium is an international collaboration of research groups funded by the National Human Genome Research Institute (NHGRI). The goal of ENCODE is to build a comprehensive parts list of functional elements in the human genome, including elements that act at the protein and RNA levels,and regulatory elements that control cells and circumstances in which a gene is active.\", \"founded_year\": 2004, \"host_institution\": \"Stanford University\", \"address\": \"Center for Biomolecular Science and Engineering, School of Engineering, University of California Santa Cruz (UCSC), Santa Cruz, CA 95064, USA.\", \"city\": \"Stanford\", \"province\": \"CA\", \"country\": \"United States\", \"current_version\": \"v128\", \"contact_name\": \"ENCODE team\", \"contact_email\": \"encode-help@lists.stanford.edu\", \"available_protocol\": 1.0, \"last_update\": null, \"age\": 19, \"citation\": 17908, \"token\": \"$2a$10$jHL5wkABrxhdzzz065t9I..ybMHhGl2yVaf2jFfx.bmUX8d4z5euu\", \"expire_date\": 1543899046887.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-24 22:11:42\", \"data_type_list\": \"DNA, Protein, RNA\", \"category_list\": \"Gene genome and annotation, Raw bio-data\", \"keywords_list\": \"functional element, regulatory element\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 942.526, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:11\", \"ess\": 0}, {\"page_id\": \"6_search\", \"is_success\": true, \"error_count\": 11.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": 10.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 6, \"page_type\": \"search\", \"source_id\": \"dc_29\", \"short_name\": \"UniProt\", \"accession\": \"DBC000029\", \"full_name\": \"Universal Protein Resource\", \"description\": \"UniProt provides the scientific community with a comprehensive,high-quality and freely accessible resource of protein sequence and functional information.\", \"founded_year\": 2004, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SD, UK\", \"city\": \"Cambridge\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"2022_05\", \"contact_name\": \"Alex Bateman\", \"contact_email\": \"agb@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2022-12-14\", \"age\": 19, \"citation\": 17081, \"token\": \"$2a$10$KXuraWjdOou1ypsILkffj.prKtTSD8gKiDnv6u5FxZcsBrvur0oxC\", \"expire_date\": 1543898855446.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-06 11:06:30\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation, Structure, Literature\", \"keywords_list\": \"protein sequence, protein structure, protein function\", \"data_object_list\": \"Fungi, Bacteria, Archaea, Virus, Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 899.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:03\", \"ess\": 0}, {\"page_id\": \"6_search_result\", \"is_success\": true, \"error_count\": 10.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": 7.0, \"th_empty\": 2.0, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 6, \"page_type\": \"search_result\", \"source_id\": \"dc_29\", \"short_name\": \"UniProt\", \"accession\": \"DBC000029\", \"full_name\": \"Universal Protein Resource\", \"description\": \"UniProt provides the scientific community with a comprehensive,high-quality and freely accessible resource of protein sequence and functional information.\", \"founded_year\": 2004, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SD, UK\", \"city\": \"Cambridge\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"2022_05\", \"contact_name\": \"Alex Bateman\", \"contact_email\": \"agb@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2022-12-14\", \"age\": 19, \"citation\": 17081, \"token\": \"$2a$10$KXuraWjdOou1ypsILkffj.prKtTSD8gKiDnv6u5FxZcsBrvur0oxC\", \"expire_date\": 1543898855446.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-06 11:06:30\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation, Structure, Literature\", \"keywords_list\": \"protein sequence, protein structure, protein function\", \"data_object_list\": \"Fungi, Bacteria, Archaea, Virus, Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 899.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:03\", \"ess\": 0}, {\"page_id\": \"6_data_entity\", \"is_success\": true, \"error_count\": 335.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": 303.0, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": 7.0, \"th_empty\": 24.0, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 6, \"page_type\": \"data_entity\", \"source_id\": \"dc_29\", \"short_name\": \"UniProt\", \"accession\": \"DBC000029\", \"full_name\": \"Universal Protein Resource\", \"description\": \"UniProt provides the scientific community with a comprehensive,high-quality and freely accessible resource of protein sequence and functional information.\", \"founded_year\": 2004, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SD, UK\", \"city\": \"Cambridge\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"2022_05\", \"contact_name\": \"Alex Bateman\", \"contact_email\": \"agb@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2022-12-14\", \"age\": 19, \"citation\": 17081, \"token\": \"$2a$10$KXuraWjdOou1ypsILkffj.prKtTSD8gKiDnv6u5FxZcsBrvur0oxC\", \"expire_date\": 1543898855446.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-06 11:06:30\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation, Structure, Literature\", \"keywords_list\": \"protein sequence, protein structure, protein function\", \"data_object_list\": \"Fungi, Bacteria, Archaea, Virus, Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 899.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:03\", \"ess\": 0}, {\"page_id\": \"6_documentation\", \"is_success\": true, \"error_count\": 8.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": 7.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 6, \"page_type\": \"documentation\", \"source_id\": \"dc_29\", \"short_name\": \"UniProt\", \"accession\": \"DBC000029\", \"full_name\": \"Universal Protein Resource\", \"description\": \"UniProt provides the scientific community with a comprehensive,high-quality and freely accessible resource of protein sequence and functional information.\", \"founded_year\": 2004, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SD, UK\", \"city\": \"Cambridge\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"2022_05\", \"contact_name\": \"Alex Bateman\", \"contact_email\": \"agb@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2022-12-14\", \"age\": 19, \"citation\": 17081, \"token\": \"$2a$10$KXuraWjdOou1ypsILkffj.prKtTSD8gKiDnv6u5FxZcsBrvur0oxC\", \"expire_date\": 1543898855446.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-06 11:06:30\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation, Structure, Literature\", \"keywords_list\": \"protein sequence, protein structure, protein function\", \"data_object_list\": \"Fungi, Bacteria, Archaea, Virus, Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 899.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:03\", \"ess\": 0}, {\"page_id\": \"7_search\", \"is_success\": true, \"error_count\": 3.0, \"label_missing\": 2.0, \"language_missing\": null, \"button_empty\": 1.0, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 7, \"page_type\": \"search\", \"source_id\": \"dc_1861\", \"short_name\": \"IGSR\", \"accession\": \"DBC001861\", \"full_name\": \"International Genome Sample Resource\", \"description\": \"IGSR expands in data type and population diversity the resources from the 1000 Genomes Project. IGSR represents the largest open collection of human variation data and provides easy access to these resources.\", \"founded_year\": 2015, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"Wellcome Genome Campus, Hinxton, Cambridge CB10 1SD\", \"city\": \"Hinxton\", \"province\": \"Cambridge\", \"country\": \"United Kingdom\", \"current_version\": null, \"contact_name\": \"IGSR team\", \"contact_email\": \"info@1000genomes.org\", \"available_protocol\": 1.0, \"last_update\": \"2021-09-30\", \"age\": 8, \"citation\": 6498, \"token\": \"$2a$10$cMNRhQ8RWxiMGUSGneIVD.n73MeMYb.kPThr8uI9mU5CAhA0k6NAq\", \"expire_date\": 1543995395760.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1487560853000, \"last_modified\": \"2023-03-03 10:11:56\", \"data_type_list\": \"DNA\", \"category_list\": \"Genotype phenotype and variation\", \"keywords_list\": \"1000 Genome\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 812.25, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:53\", \"ess\": 0}, {\"page_id\": \"7_search_result\", \"is_success\": true, \"error_count\": 3.0, \"label_missing\": 2.0, \"language_missing\": null, \"button_empty\": 1.0, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 7, \"page_type\": \"search_result\", \"source_id\": \"dc_1861\", \"short_name\": \"IGSR\", \"accession\": \"DBC001861\", \"full_name\": \"International Genome Sample Resource\", \"description\": \"IGSR expands in data type and population diversity the resources from the 1000 Genomes Project. IGSR represents the largest open collection of human variation data and provides easy access to these resources.\", \"founded_year\": 2015, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"Wellcome Genome Campus, Hinxton, Cambridge CB10 1SD\", \"city\": \"Hinxton\", \"province\": \"Cambridge\", \"country\": \"United Kingdom\", \"current_version\": null, \"contact_name\": \"IGSR team\", \"contact_email\": \"info@1000genomes.org\", \"available_protocol\": 1.0, \"last_update\": \"2021-09-30\", \"age\": 8, \"citation\": 6498, \"token\": \"$2a$10$cMNRhQ8RWxiMGUSGneIVD.n73MeMYb.kPThr8uI9mU5CAhA0k6NAq\", \"expire_date\": 1543995395760.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1487560853000, \"last_modified\": \"2023-03-03 10:11:56\", \"data_type_list\": \"DNA\", \"category_list\": \"Genotype phenotype and variation\", \"keywords_list\": \"1000 Genome\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 812.25, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:53\", \"ess\": 0}, {\"page_id\": \"7_data_entity\", \"is_success\": true, \"error_count\": 23.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": 1.0, \"alt_missing\": null, \"link_empty\": 20.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": 1.0, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 7, \"page_type\": \"data_entity\", \"source_id\": \"dc_1861\", \"short_name\": \"IGSR\", \"accession\": \"DBC001861\", \"full_name\": \"International Genome Sample Resource\", \"description\": \"IGSR expands in data type and population diversity the resources from the 1000 Genomes Project. IGSR represents the largest open collection of human variation data and provides easy access to these resources.\", \"founded_year\": 2015, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"Wellcome Genome Campus, Hinxton, Cambridge CB10 1SD\", \"city\": \"Hinxton\", \"province\": \"Cambridge\", \"country\": \"United Kingdom\", \"current_version\": null, \"contact_name\": \"IGSR team\", \"contact_email\": \"info@1000genomes.org\", \"available_protocol\": 1.0, \"last_update\": \"2021-09-30\", \"age\": 8, \"citation\": 6498, \"token\": \"$2a$10$cMNRhQ8RWxiMGUSGneIVD.n73MeMYb.kPThr8uI9mU5CAhA0k6NAq\", \"expire_date\": 1543995395760.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1487560853000, \"last_modified\": \"2023-03-03 10:11:56\", \"data_type_list\": \"DNA\", \"category_list\": \"Genotype phenotype and variation\", \"keywords_list\": \"1000 Genome\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 812.25, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:53\", \"ess\": 0}, {\"page_id\": \"7_documentation\", \"is_success\": true, \"error_count\": 2.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": 1.0, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 7, \"page_type\": \"documentation\", \"source_id\": \"dc_1861\", \"short_name\": \"IGSR\", \"accession\": \"DBC001861\", \"full_name\": \"International Genome Sample Resource\", \"description\": \"IGSR expands in data type and population diversity the resources from the 1000 Genomes Project. IGSR represents the largest open collection of human variation data and provides easy access to these resources.\", \"founded_year\": 2015, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"Wellcome Genome Campus, Hinxton, Cambridge CB10 1SD\", \"city\": \"Hinxton\", \"province\": \"Cambridge\", \"country\": \"United Kingdom\", \"current_version\": null, \"contact_name\": \"IGSR team\", \"contact_email\": \"info@1000genomes.org\", \"available_protocol\": 1.0, \"last_update\": \"2021-09-30\", \"age\": 8, \"citation\": 6498, \"token\": \"$2a$10$cMNRhQ8RWxiMGUSGneIVD.n73MeMYb.kPThr8uI9mU5CAhA0k6NAq\", \"expire_date\": 1543995395760.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1487560853000, \"last_modified\": \"2023-03-03 10:11:56\", \"data_type_list\": \"DNA\", \"category_list\": \"Genotype phenotype and variation\", \"keywords_list\": \"1000 Genome\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 812.25, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:53\", \"ess\": 0}, {\"page_id\": \"8_search\", \"is_success\": true, \"error_count\": 4.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": 1.0, \"alt_missing\": null, \"link_empty\": 2.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 8, \"page_type\": \"search\", \"source_id\": \"dc_186\", \"short_name\": \"Pfam\", \"accession\": \"DBC000186\", \"full_name\": \"Protein Families Database\", \"description\": \" A database of conserved protein families and domains. Pfam is a member database of InterPro. Pfam data is now accessible via the InterPro website.\", \"founded_year\": 1997, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI), Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SD, UK\", \"city\": \"Cambridge\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"v35\", \"contact_name\": \"Pfam team\", \"contact_email\": \"pfam-help@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2021-11-01\", \"age\": 26, \"citation\": 20861, \"token\": \"$2a$10$iaZh/Uo3kRz3BQkSgni3z.K4IIWtGzwDS9ZSCCzIhb6Ex13Rc3PLC\", \"expire_date\": 1543898964393.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-10-09 11:21:36\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation, Raw bio-data, Structure\", \"keywords_list\": \"protein domain, protein family, SARS-CoV-2\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 802.346, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:07\", \"ess\": 0}, {\"page_id\": \"8_search_result\", \"is_success\": true, \"error_count\": 4.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": 1.0, \"alt_missing\": null, \"link_empty\": 2.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 8, \"page_type\": \"search_result\", \"source_id\": \"dc_186\", \"short_name\": \"Pfam\", \"accession\": \"DBC000186\", \"full_name\": \"Protein Families Database\", \"description\": \" A database of conserved protein families and domains. Pfam is a member database of InterPro. Pfam data is now accessible via the InterPro website.\", \"founded_year\": 1997, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI), Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SD, UK\", \"city\": \"Cambridge\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"v35\", \"contact_name\": \"Pfam team\", \"contact_email\": \"pfam-help@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2021-11-01\", \"age\": 26, \"citation\": 20861, \"token\": \"$2a$10$iaZh/Uo3kRz3BQkSgni3z.K4IIWtGzwDS9ZSCCzIhb6Ex13Rc3PLC\", \"expire_date\": 1543898964393.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-10-09 11:21:36\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation, Raw bio-data, Structure\", \"keywords_list\": \"protein domain, protein family, SARS-CoV-2\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 802.346, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:07\", \"ess\": 0}, {\"page_id\": \"8_data_entity\", \"is_success\": true, \"error_count\": 3.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": 1.0, \"alt_missing\": null, \"link_empty\": 2.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 8, \"page_type\": \"data_entity\", \"source_id\": \"dc_186\", \"short_name\": \"Pfam\", \"accession\": \"DBC000186\", \"full_name\": \"Protein Families Database\", \"description\": \" A database of conserved protein families and domains. Pfam is a member database of InterPro. Pfam data is now accessible via the InterPro website.\", \"founded_year\": 1997, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI), Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SD, UK\", \"city\": \"Cambridge\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"v35\", \"contact_name\": \"Pfam team\", \"contact_email\": \"pfam-help@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2021-11-01\", \"age\": 26, \"citation\": 20861, \"token\": \"$2a$10$iaZh/Uo3kRz3BQkSgni3z.K4IIWtGzwDS9ZSCCzIhb6Ex13Rc3PLC\", \"expire_date\": 1543898964393.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-10-09 11:21:36\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation, Raw bio-data, Structure\", \"keywords_list\": \"protein domain, protein family, SARS-CoV-2\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 802.346, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:07\", \"ess\": 0}, {\"page_id\": \"8_documentation\", \"is_success\": true, \"error_count\": 1.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 8, \"page_type\": \"documentation\", \"source_id\": \"dc_186\", \"short_name\": \"Pfam\", \"accession\": \"DBC000186\", \"full_name\": \"Protein Families Database\", \"description\": \" A database of conserved protein families and domains. Pfam is a member database of InterPro. Pfam data is now accessible via the InterPro website.\", \"founded_year\": 1997, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI), Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SD, UK\", \"city\": \"Cambridge\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"v35\", \"contact_name\": \"Pfam team\", \"contact_email\": \"pfam-help@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2021-11-01\", \"age\": 26, \"citation\": 20861, \"token\": \"$2a$10$iaZh/Uo3kRz3BQkSgni3z.K4IIWtGzwDS9ZSCCzIhb6Ex13Rc3PLC\", \"expire_date\": 1543898964393.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-10-09 11:21:36\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation, Raw bio-data, Structure\", \"keywords_list\": \"protein domain, protein family, SARS-CoV-2\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 802.346, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:07\", \"ess\": 0}, {\"page_id\": \"9_search\", \"is_success\": true, \"error_count\": 15.0, \"label_missing\": 15.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 9, \"page_type\": \"search\", \"source_id\": \"dc_238\", \"short_name\": \"SILVA\", \"accession\": \"DBC000238\", \"full_name\": \"High quality ribosomal RNA databases\", \"description\": \"SILVA is a comprehensive, quality-controlled web resource for up-to-date aligned ribosomal RNA (rRNA) gene sequences from the Bacteria, Archaea and Eukaryota domains alongside supplementary online services. In addition to data products, SILVA provides various online tools such as alignment and classification, phylogenetic tree calculation and viewer, probe/primer matching, and an amplicon analysis pipeline. With every full release a curated guide tree is provided that contains the latest taxonomy and nomenclature based on multiple references. SILVA is an ELIXIR Core Data Resource.\", \"founded_year\": 2007, \"host_institution\": \"Max Planck Institute for Marine Microbiology\", \"address\": \"D-28359 Bremen, Germany\", \"city\": \"Bremen\", \"province\": null, \"country\": \"Germany\", \"current_version\": \"v138.1\", \"contact_name\": \"Frank Oliver Gl\\u00f6ckner\", \"contact_email\": \"contact@arb-silva.de\", \"available_protocol\": 1.0, \"last_update\": \"2020-08-27\", \"age\": 16, \"citation\": 12280, \"token\": \"$2a$10$s2ECbbpHh9YQkSCW0xlMjeAxCT03qE6IHk3l35SVoX1nuf669nvte\", \"expire_date\": 1543898998134.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:57\", \"data_type_list\": \"RNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"rRNA\", \"data_object_list\": \"Animal, Bacteria, Archaea\", \"organism_list\": null, \"theme_list\": null, \"zindex\": 767.5, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:08\", \"ess\": 0}, {\"page_id\": \"9_documentation\", \"is_success\": true, \"error_count\": 0.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 9, \"page_type\": \"documentation\", \"source_id\": \"dc_238\", \"short_name\": \"SILVA\", \"accession\": \"DBC000238\", \"full_name\": \"High quality ribosomal RNA databases\", \"description\": \"SILVA is a comprehensive, quality-controlled web resource for up-to-date aligned ribosomal RNA (rRNA) gene sequences from the Bacteria, Archaea and Eukaryota domains alongside supplementary online services. In addition to data products, SILVA provides various online tools such as alignment and classification, phylogenetic tree calculation and viewer, probe/primer matching, and an amplicon analysis pipeline. With every full release a curated guide tree is provided that contains the latest taxonomy and nomenclature based on multiple references. SILVA is an ELIXIR Core Data Resource.\", \"founded_year\": 2007, \"host_institution\": \"Max Planck Institute for Marine Microbiology\", \"address\": \"D-28359 Bremen, Germany\", \"city\": \"Bremen\", \"province\": null, \"country\": \"Germany\", \"current_version\": \"v138.1\", \"contact_name\": \"Frank Oliver Gl\\u00f6ckner\", \"contact_email\": \"contact@arb-silva.de\", \"available_protocol\": 1.0, \"last_update\": \"2020-08-27\", \"age\": 16, \"citation\": 12280, \"token\": \"$2a$10$s2ECbbpHh9YQkSCW0xlMjeAxCT03qE6IHk3l35SVoX1nuf669nvte\", \"expire_date\": 1543898998134.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:57\", \"data_type_list\": \"RNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"rRNA\", \"data_object_list\": \"Animal, Bacteria, Archaea\", \"organism_list\": null, \"theme_list\": null, \"zindex\": 767.5, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:08\", \"ess\": 0}, {\"page_id\": \"10_search_result\", \"is_success\": true, \"error_count\": 32.0, \"label_missing\": 2.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 30.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 10, \"page_type\": \"search_result\", \"source_id\": \"dc_6934\", \"short_name\": \"gnomAD\", \"accession\": \"DBC006934\", \"full_name\": \"Genome Aggregation Database\", \"description\": \"The Genome Aggregation Database (gnomAD) is a resource developed by an international coalition of investigators, with the goal of aggregating and harmonizing both exome and genome sequencing data from a wide variety of large-scale sequencing projects, and making summary data available for the wider scientific community.\", \"founded_year\": 2020, \"host_institution\": \"Broad Institute\", \"address\": null, \"city\": null, \"province\": null, \"country\": \"United States\", \"current_version\": \"v2.1.1 & v3\", \"contact_name\": \"Genome Aggregation Database Consortium\", \"contact_email\": \"exomeconsortium@gmail.com\", \"available_protocol\": 1.0, \"last_update\": null, \"age\": 3, \"citation\": 2300, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1594014179000, \"last_modified\": \"2023-03-03 10:16:54\", \"data_type_list\": \"DNA\", \"category_list\": \"Raw bio-data, Gene genome and annotation, Genotype phenotype and variation\", \"keywords_list\": \"human, genome, variation\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 766.667, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:37\", \"ess\": 0}, {\"page_id\": \"10_data_entity\", \"is_success\": true, \"error_count\": 32.0, \"label_missing\": 2.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 30.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 10, \"page_type\": \"data_entity\", \"source_id\": \"dc_6934\", \"short_name\": \"gnomAD\", \"accession\": \"DBC006934\", \"full_name\": \"Genome Aggregation Database\", \"description\": \"The Genome Aggregation Database (gnomAD) is a resource developed by an international coalition of investigators, with the goal of aggregating and harmonizing both exome and genome sequencing data from a wide variety of large-scale sequencing projects, and making summary data available for the wider scientific community.\", \"founded_year\": 2020, \"host_institution\": \"Broad Institute\", \"address\": null, \"city\": null, \"province\": null, \"country\": \"United States\", \"current_version\": \"v2.1.1 & v3\", \"contact_name\": \"Genome Aggregation Database Consortium\", \"contact_email\": \"exomeconsortium@gmail.com\", \"available_protocol\": 1.0, \"last_update\": null, \"age\": 3, \"citation\": 2300, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1594014179000, \"last_modified\": \"2023-03-03 10:16:54\", \"data_type_list\": \"DNA\", \"category_list\": \"Raw bio-data, Gene genome and annotation, Genotype phenotype and variation\", \"keywords_list\": \"human, genome, variation\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 766.667, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:37\", \"ess\": 0}, {\"page_id\": \"10_documentation\", \"is_success\": true, \"error_count\": 8.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 7.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 10, \"page_type\": \"documentation\", \"source_id\": \"dc_6934\", \"short_name\": \"gnomAD\", \"accession\": \"DBC006934\", \"full_name\": \"Genome Aggregation Database\", \"description\": \"The Genome Aggregation Database (gnomAD) is a resource developed by an international coalition of investigators, with the goal of aggregating and harmonizing both exome and genome sequencing data from a wide variety of large-scale sequencing projects, and making summary data available for the wider scientific community.\", \"founded_year\": 2020, \"host_institution\": \"Broad Institute\", \"address\": null, \"city\": null, \"province\": null, \"country\": \"United States\", \"current_version\": \"v2.1.1 & v3\", \"contact_name\": \"Genome Aggregation Database Consortium\", \"contact_email\": \"exomeconsortium@gmail.com\", \"available_protocol\": 1.0, \"last_update\": null, \"age\": 3, \"citation\": 2300, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1594014179000, \"last_modified\": \"2023-03-03 10:16:54\", \"data_type_list\": \"DNA\", \"category_list\": \"Raw bio-data, Gene genome and annotation, Genotype phenotype and variation\", \"keywords_list\": \"human, genome, variation\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 766.667, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:37\", \"ess\": 0}, {\"page_id\": \"11_search\", \"is_success\": true, \"error_count\": 7.0, \"label_missing\": 5.0, \"language_missing\": null, \"button_empty\": 2.0, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 11, \"page_type\": \"search\", \"source_id\": \"dc_165\", \"short_name\": \"miRBase\", \"accession\": \"DBC000165\", \"full_name\": \"microRNA database\", \"description\": \"miRBase is the central repository for microRNA (miRNA) sequence information. miRBase has a role in defining the nomenclature for miRNA genes and assigning names to novel miRNAs for publication in peer-reviewed journals. The online miRBase database is a resource containing all published miRNA sequences, together with textual annotation and links to the primary literature and to other secondary databases. The database provides a variety of methods to query the data, by specific searches of sequences and associated text and literature.\", \"founded_year\": 2004, \"host_institution\": \"University of Manchester\", \"address\": \"Manchester, M13 9PT, UK\", \"city\": \"Manchester\", \"province\": null, \"country\": \"United Kingdom\", \"current_version\": \"v22.1\", \"contact_name\": \"Sam Griffiths-Jones\", \"contact_email\": \"sam.griffiths-jones@manchester.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2018-10\", \"age\": 19, \"citation\": 13219, \"token\": \"$2a$10$RXHT0aNg5L9bRBUSCAtcP.Mr.74LjE/7M2IHyx.dWBgnGYQmhBvjG\", \"expire_date\": 1543898952074.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:51\", \"data_type_list\": \"RNA\", \"category_list\": \"Standard ontology and nomenclature, Gene genome and annotation, Literature\", \"keywords_list\": \"miRNA, microRNA\", \"data_object_list\": \"Animal, Plant, Virus\", \"organism_list\": \"Arabidopsis thaliana, Drosophila melanogaster, Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 695.737, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:07\", \"ess\": 0}, {\"page_id\": \"11_search_result\", \"is_success\": true, \"error_count\": 4.0, \"label_missing\": 2.0, \"language_missing\": null, \"button_empty\": 2.0, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 11, \"page_type\": \"search_result\", \"source_id\": \"dc_165\", \"short_name\": \"miRBase\", \"accession\": \"DBC000165\", \"full_name\": \"microRNA database\", \"description\": \"miRBase is the central repository for microRNA (miRNA) sequence information. miRBase has a role in defining the nomenclature for miRNA genes and assigning names to novel miRNAs for publication in peer-reviewed journals. The online miRBase database is a resource containing all published miRNA sequences, together with textual annotation and links to the primary literature and to other secondary databases. The database provides a variety of methods to query the data, by specific searches of sequences and associated text and literature.\", \"founded_year\": 2004, \"host_institution\": \"University of Manchester\", \"address\": \"Manchester, M13 9PT, UK\", \"city\": \"Manchester\", \"province\": null, \"country\": \"United Kingdom\", \"current_version\": \"v22.1\", \"contact_name\": \"Sam Griffiths-Jones\", \"contact_email\": \"sam.griffiths-jones@manchester.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2018-10\", \"age\": 19, \"citation\": 13219, \"token\": \"$2a$10$RXHT0aNg5L9bRBUSCAtcP.Mr.74LjE/7M2IHyx.dWBgnGYQmhBvjG\", \"expire_date\": 1543898952074.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:51\", \"data_type_list\": \"RNA\", \"category_list\": \"Standard ontology and nomenclature, Gene genome and annotation, Literature\", \"keywords_list\": \"miRNA, microRNA\", \"data_object_list\": \"Animal, Plant, Virus\", \"organism_list\": \"Arabidopsis thaliana, Drosophila melanogaster, Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 695.737, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:07\", \"ess\": 0}, {\"page_id\": \"11_data_entity\", \"is_success\": true, \"error_count\": 11.0, \"label_missing\": 3.0, \"language_missing\": null, \"button_empty\": 2.0, \"alt_missing\": 3.0, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": 2.0, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 11, \"page_type\": \"data_entity\", \"source_id\": \"dc_165\", \"short_name\": \"miRBase\", \"accession\": \"DBC000165\", \"full_name\": \"microRNA database\", \"description\": \"miRBase is the central repository for microRNA (miRNA) sequence information. miRBase has a role in defining the nomenclature for miRNA genes and assigning names to novel miRNAs for publication in peer-reviewed journals. The online miRBase database is a resource containing all published miRNA sequences, together with textual annotation and links to the primary literature and to other secondary databases. The database provides a variety of methods to query the data, by specific searches of sequences and associated text and literature.\", \"founded_year\": 2004, \"host_institution\": \"University of Manchester\", \"address\": \"Manchester, M13 9PT, UK\", \"city\": \"Manchester\", \"province\": null, \"country\": \"United Kingdom\", \"current_version\": \"v22.1\", \"contact_name\": \"Sam Griffiths-Jones\", \"contact_email\": \"sam.griffiths-jones@manchester.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2018-10\", \"age\": 19, \"citation\": 13219, \"token\": \"$2a$10$RXHT0aNg5L9bRBUSCAtcP.Mr.74LjE/7M2IHyx.dWBgnGYQmhBvjG\", \"expire_date\": 1543898952074.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:51\", \"data_type_list\": \"RNA\", \"category_list\": \"Standard ontology and nomenclature, Gene genome and annotation, Literature\", \"keywords_list\": \"miRNA, microRNA\", \"data_object_list\": \"Animal, Plant, Virus\", \"organism_list\": \"Arabidopsis thaliana, Drosophila melanogaster, Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 695.737, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:07\", \"ess\": 0}, {\"page_id\": \"11_documentation\", \"is_success\": true, \"error_count\": 4.0, \"label_missing\": 2.0, \"language_missing\": null, \"button_empty\": 2.0, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 11, \"page_type\": \"documentation\", \"source_id\": \"dc_165\", \"short_name\": \"miRBase\", \"accession\": \"DBC000165\", \"full_name\": \"microRNA database\", \"description\": \"miRBase is the central repository for microRNA (miRNA) sequence information. miRBase has a role in defining the nomenclature for miRNA genes and assigning names to novel miRNAs for publication in peer-reviewed journals. The online miRBase database is a resource containing all published miRNA sequences, together with textual annotation and links to the primary literature and to other secondary databases. The database provides a variety of methods to query the data, by specific searches of sequences and associated text and literature.\", \"founded_year\": 2004, \"host_institution\": \"University of Manchester\", \"address\": \"Manchester, M13 9PT, UK\", \"city\": \"Manchester\", \"province\": null, \"country\": \"United Kingdom\", \"current_version\": \"v22.1\", \"contact_name\": \"Sam Griffiths-Jones\", \"contact_email\": \"sam.griffiths-jones@manchester.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2018-10\", \"age\": 19, \"citation\": 13219, \"token\": \"$2a$10$RXHT0aNg5L9bRBUSCAtcP.Mr.74LjE/7M2IHyx.dWBgnGYQmhBvjG\", \"expire_date\": 1543898952074.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:51\", \"data_type_list\": \"RNA\", \"category_list\": \"Standard ontology and nomenclature, Gene genome and annotation, Literature\", \"keywords_list\": \"miRNA, microRNA\", \"data_object_list\": \"Animal, Plant, Virus\", \"organism_list\": \"Arabidopsis thaliana, Drosophila melanogaster, Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 695.737, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:07\", \"ess\": 0}, {\"page_id\": \"12_search\", \"is_success\": true, \"error_count\": 97.0, \"label_missing\": 4.0, \"language_missing\": 1.0, \"button_empty\": 1.0, \"alt_missing\": 15.0, \"link_empty\": 14.0, \"aria_menu_broken\": null, \"alt_link_missing\": 9.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": 1.0, \"alt_area_missing\": 26.0, \"alt_map_missing\": 26.0, \"label_empty\": null, \"label_multiple\": null, \"id\": 12, \"page_type\": \"search\", \"source_id\": \"dc_93\", \"short_name\": \"UCSC Genome Browser\", \"accession\": \"DBC000093\", \"full_name\": \"University of California Santa Cruz Genome Browser Database\", \"description\": \"Launched in 2001 to showcase the draft human genome assembly, the UCSC Genome Browser database (http://genome.ucsc.edu) and associated tools continue to grow, providing a comprehensive resource of genome assemblies and annotations to scientists and students worldwide.\", \"founded_year\": 2002, \"host_institution\": \"University of California Santa Cruz\", \"address\": \"Santa Cruz, CA 95064, USA\", \"city\": \"Santa Cruz\", \"province\": \"CA\", \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Cath Tyner\", \"contact_email\": \"cath@ucsc.edu\", \"available_protocol\": 1.0, \"last_update\": \"2022-04-02\", \"age\": 21, \"citation\": 14446, \"token\": \"$2a$10$lzBoRmzKBCSq1H2T9dZXgucqlni3xTPk7bUs5ckBWSdn16W3Ae5ya\", \"expire_date\": 1543898901943.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-10-09 14:43:15\", \"data_type_list\": \"DNA, Protein, RNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"genome assembly, genome browser, genome alignment, SARS-CoV-2\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 687.905, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:04\", \"ess\": 0}, {\"page_id\": \"12_search_result\", \"is_success\": true, \"error_count\": 28.0, \"label_missing\": 3.0, \"language_missing\": 1.0, \"button_empty\": 1.0, \"alt_missing\": 18.0, \"link_empty\": 3.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": 2.0, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 12, \"page_type\": \"search_result\", \"source_id\": \"dc_93\", \"short_name\": \"UCSC Genome Browser\", \"accession\": \"DBC000093\", \"full_name\": \"University of California Santa Cruz Genome Browser Database\", \"description\": \"Launched in 2001 to showcase the draft human genome assembly, the UCSC Genome Browser database (http://genome.ucsc.edu) and associated tools continue to grow, providing a comprehensive resource of genome assemblies and annotations to scientists and students worldwide.\", \"founded_year\": 2002, \"host_institution\": \"University of California Santa Cruz\", \"address\": \"Santa Cruz, CA 95064, USA\", \"city\": \"Santa Cruz\", \"province\": \"CA\", \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Cath Tyner\", \"contact_email\": \"cath@ucsc.edu\", \"available_protocol\": 1.0, \"last_update\": \"2022-04-02\", \"age\": 21, \"citation\": 14446, \"token\": \"$2a$10$lzBoRmzKBCSq1H2T9dZXgucqlni3xTPk7bUs5ckBWSdn16W3Ae5ya\", \"expire_date\": 1543898901943.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-10-09 14:43:15\", \"data_type_list\": \"DNA, Protein, RNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"genome assembly, genome browser, genome alignment, SARS-CoV-2\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 687.905, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:04\", \"ess\": 0}, {\"page_id\": \"12_data_entity\", \"is_success\": true, \"error_count\": 97.0, \"label_missing\": 4.0, \"language_missing\": 1.0, \"button_empty\": 1.0, \"alt_missing\": 15.0, \"link_empty\": 14.0, \"aria_menu_broken\": null, \"alt_link_missing\": 9.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": 1.0, \"alt_area_missing\": 26.0, \"alt_map_missing\": 26.0, \"label_empty\": null, \"label_multiple\": null, \"id\": 12, \"page_type\": \"data_entity\", \"source_id\": \"dc_93\", \"short_name\": \"UCSC Genome Browser\", \"accession\": \"DBC000093\", \"full_name\": \"University of California Santa Cruz Genome Browser Database\", \"description\": \"Launched in 2001 to showcase the draft human genome assembly, the UCSC Genome Browser database (http://genome.ucsc.edu) and associated tools continue to grow, providing a comprehensive resource of genome assemblies and annotations to scientists and students worldwide.\", \"founded_year\": 2002, \"host_institution\": \"University of California Santa Cruz\", \"address\": \"Santa Cruz, CA 95064, USA\", \"city\": \"Santa Cruz\", \"province\": \"CA\", \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Cath Tyner\", \"contact_email\": \"cath@ucsc.edu\", \"available_protocol\": 1.0, \"last_update\": \"2022-04-02\", \"age\": 21, \"citation\": 14446, \"token\": \"$2a$10$lzBoRmzKBCSq1H2T9dZXgucqlni3xTPk7bUs5ckBWSdn16W3Ae5ya\", \"expire_date\": 1543898901943.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-10-09 14:43:15\", \"data_type_list\": \"DNA, Protein, RNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"genome assembly, genome browser, genome alignment, SARS-CoV-2\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 687.905, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:04\", \"ess\": 0}, {\"page_id\": \"12_documentation\", \"is_success\": true, \"error_count\": 8.0, \"label_missing\": 4.0, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 3.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 12, \"page_type\": \"documentation\", \"source_id\": \"dc_93\", \"short_name\": \"UCSC Genome Browser\", \"accession\": \"DBC000093\", \"full_name\": \"University of California Santa Cruz Genome Browser Database\", \"description\": \"Launched in 2001 to showcase the draft human genome assembly, the UCSC Genome Browser database (http://genome.ucsc.edu) and associated tools continue to grow, providing a comprehensive resource of genome assemblies and annotations to scientists and students worldwide.\", \"founded_year\": 2002, \"host_institution\": \"University of California Santa Cruz\", \"address\": \"Santa Cruz, CA 95064, USA\", \"city\": \"Santa Cruz\", \"province\": \"CA\", \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Cath Tyner\", \"contact_email\": \"cath@ucsc.edu\", \"available_protocol\": 1.0, \"last_update\": \"2022-04-02\", \"age\": 21, \"citation\": 14446, \"token\": \"$2a$10$lzBoRmzKBCSq1H2T9dZXgucqlni3xTPk7bUs5ckBWSdn16W3Ae5ya\", \"expire_date\": 1543898901943.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-10-09 14:43:15\", \"data_type_list\": \"DNA, Protein, RNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"genome assembly, genome browser, genome alignment, SARS-CoV-2\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 687.905, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:04\", \"ess\": 0}, {\"page_id\": \"13_search\", \"is_success\": true, \"error_count\": 0.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 13, \"page_type\": \"search\", \"source_id\": \"dc_424\", \"short_name\": \"GEO\", \"accession\": \"DBC000424\", \"full_name\": \"Gene Expression Omnibus\", \"description\": \"The Gene Expression Omnibus is an international public repository for high-throughput microarray and next-generation sequence functional genomic data sets submitted by the research community.\", \"founded_year\": 2002, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"Bethesda,MD 20892,USA\", \"city\": \"Bethesda\", \"province\": \"MD\", \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Tanya Barrett\", \"contact_email\": \"barrett@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2015-12-30\", \"age\": 21, \"citation\": 14444, \"token\": \"$2a$10$CTgiIN2tFI9mB6cj8NXWNuLXdbqZ6TUnJ3PpmV/6mCmuKG04tmWju\", \"expire_date\": 1543906755125.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:10:13\", \"data_type_list\": \"RNA, DNA, Protein\", \"category_list\": \"Expression, Raw bio-data, Metadata\", \"keywords_list\": \"NGS, microarray, RNA-Seq, omics data\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 687.81, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:14\", \"ess\": 0}, {\"page_id\": \"13_search_result\", \"is_success\": true, \"error_count\": 2.0, \"label_missing\": 2.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 13, \"page_type\": \"search_result\", \"source_id\": \"dc_424\", \"short_name\": \"GEO\", \"accession\": \"DBC000424\", \"full_name\": \"Gene Expression Omnibus\", \"description\": \"The Gene Expression Omnibus is an international public repository for high-throughput microarray and next-generation sequence functional genomic data sets submitted by the research community.\", \"founded_year\": 2002, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"Bethesda,MD 20892,USA\", \"city\": \"Bethesda\", \"province\": \"MD\", \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Tanya Barrett\", \"contact_email\": \"barrett@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2015-12-30\", \"age\": 21, \"citation\": 14444, \"token\": \"$2a$10$CTgiIN2tFI9mB6cj8NXWNuLXdbqZ6TUnJ3PpmV/6mCmuKG04tmWju\", \"expire_date\": 1543906755125.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:10:13\", \"data_type_list\": \"RNA, DNA, Protein\", \"category_list\": \"Expression, Raw bio-data, Metadata\", \"keywords_list\": \"NGS, microarray, RNA-Seq, omics data\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 687.81, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:14\", \"ess\": 0}, {\"page_id\": \"13_data_entity\", \"is_success\": true, \"error_count\": 1.0, \"label_missing\": null, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 13, \"page_type\": \"data_entity\", \"source_id\": \"dc_424\", \"short_name\": \"GEO\", \"accession\": \"DBC000424\", \"full_name\": \"Gene Expression Omnibus\", \"description\": \"The Gene Expression Omnibus is an international public repository for high-throughput microarray and next-generation sequence functional genomic data sets submitted by the research community.\", \"founded_year\": 2002, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"Bethesda,MD 20892,USA\", \"city\": \"Bethesda\", \"province\": \"MD\", \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Tanya Barrett\", \"contact_email\": \"barrett@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2015-12-30\", \"age\": 21, \"citation\": 14444, \"token\": \"$2a$10$CTgiIN2tFI9mB6cj8NXWNuLXdbqZ6TUnJ3PpmV/6mCmuKG04tmWju\", \"expire_date\": 1543906755125.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:10:13\", \"data_type_list\": \"RNA, DNA, Protein\", \"category_list\": \"Expression, Raw bio-data, Metadata\", \"keywords_list\": \"NGS, microarray, RNA-Seq, omics data\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 687.81, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:14\", \"ess\": 0}, {\"page_id\": \"13_documentation\", \"is_success\": true, \"error_count\": 0.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 13, \"page_type\": \"documentation\", \"source_id\": \"dc_424\", \"short_name\": \"GEO\", \"accession\": \"DBC000424\", \"full_name\": \"Gene Expression Omnibus\", \"description\": \"The Gene Expression Omnibus is an international public repository for high-throughput microarray and next-generation sequence functional genomic data sets submitted by the research community.\", \"founded_year\": 2002, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"Bethesda,MD 20892,USA\", \"city\": \"Bethesda\", \"province\": \"MD\", \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Tanya Barrett\", \"contact_email\": \"barrett@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2015-12-30\", \"age\": 21, \"citation\": 14444, \"token\": \"$2a$10$CTgiIN2tFI9mB6cj8NXWNuLXdbqZ6TUnJ3PpmV/6mCmuKG04tmWju\", \"expire_date\": 1543906755125.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:10:13\", \"data_type_list\": \"RNA, DNA, Protein\", \"category_list\": \"Expression, Raw bio-data, Metadata\", \"keywords_list\": \"NGS, microarray, RNA-Seq, omics data\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 687.81, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:14\", \"ess\": 0}, {\"page_id\": \"14_search_result\", \"is_success\": true, \"error_count\": 9.0, \"label_missing\": 2.0, \"language_missing\": null, \"button_empty\": 4.0, \"alt_missing\": 1.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": 1.0, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 14, \"page_type\": \"search_result\", \"source_id\": \"dc_6329\", \"short_name\": \"GTEx\", \"accession\": \"DBC006329\", \"full_name\": \"Genotype-Tissue Expression \", \"description\": \"GTEx established a data resource and tissue bank to study the relationship between genetic variation and gene expression in multiple human tissues. This release includes genotype data from approximately 714 donors and approximately 11688 RNA-seq samples across 53 tissue sites and 2 cell lines, with adequate power to detect Expression Quantitative Trait Loci in 48 tissues.\\n\", \"founded_year\": 2013, \"host_institution\": \"Broad Institute\", \"address\": \"9000 Rockville Pike, Bethesda, Maryland 20892\", \"city\": \"Bethesda\", \"province\": \"Maryland\", \"country\": \"United States\", \"current_version\": \"v8\", \"contact_name\": \"GTEx consortium\", \"contact_email\": \"volpis@mail.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2019-7-24\", \"age\": 10, \"citation\": 6832, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1564463149000, \"last_modified\": \"2023-03-03 10:16:14\", \"data_type_list\": \"DNA, RNA\", \"category_list\": \"Genotype phenotype and variation, Expression\", \"keywords_list\": \"normal tissue, tissue site, eQTL, RNA-seq\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 683.2, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:25\", \"ess\": 0}, {\"page_id\": \"14_data_entity\", \"is_success\": true, \"error_count\": 28.0, \"label_missing\": 16.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 11.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 14, \"page_type\": \"data_entity\", \"source_id\": \"dc_6329\", \"short_name\": \"GTEx\", \"accession\": \"DBC006329\", \"full_name\": \"Genotype-Tissue Expression \", \"description\": \"GTEx established a data resource and tissue bank to study the relationship between genetic variation and gene expression in multiple human tissues. This release includes genotype data from approximately 714 donors and approximately 11688 RNA-seq samples across 53 tissue sites and 2 cell lines, with adequate power to detect Expression Quantitative Trait Loci in 48 tissues.\\n\", \"founded_year\": 2013, \"host_institution\": \"Broad Institute\", \"address\": \"9000 Rockville Pike, Bethesda, Maryland 20892\", \"city\": \"Bethesda\", \"province\": \"Maryland\", \"country\": \"United States\", \"current_version\": \"v8\", \"contact_name\": \"GTEx consortium\", \"contact_email\": \"volpis@mail.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2019-7-24\", \"age\": 10, \"citation\": 6832, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1564463149000, \"last_modified\": \"2023-03-03 10:16:14\", \"data_type_list\": \"DNA, RNA\", \"category_list\": \"Genotype phenotype and variation, Expression\", \"keywords_list\": \"normal tissue, tissue site, eQTL, RNA-seq\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 683.2, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:25\", \"ess\": 0}, {\"page_id\": \"14_documentation\", \"is_success\": true, \"error_count\": 4.0, \"label_missing\": 2.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 1.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 14, \"page_type\": \"documentation\", \"source_id\": \"dc_6329\", \"short_name\": \"GTEx\", \"accession\": \"DBC006329\", \"full_name\": \"Genotype-Tissue Expression \", \"description\": \"GTEx established a data resource and tissue bank to study the relationship between genetic variation and gene expression in multiple human tissues. This release includes genotype data from approximately 714 donors and approximately 11688 RNA-seq samples across 53 tissue sites and 2 cell lines, with adequate power to detect Expression Quantitative Trait Loci in 48 tissues.\\n\", \"founded_year\": 2013, \"host_institution\": \"Broad Institute\", \"address\": \"9000 Rockville Pike, Bethesda, Maryland 20892\", \"city\": \"Bethesda\", \"province\": \"Maryland\", \"country\": \"United States\", \"current_version\": \"v8\", \"contact_name\": \"GTEx consortium\", \"contact_email\": \"volpis@mail.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2019-7-24\", \"age\": 10, \"citation\": 6832, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1564463149000, \"last_modified\": \"2023-03-03 10:16:14\", \"data_type_list\": \"DNA, RNA\", \"category_list\": \"Genotype phenotype and variation, Expression\", \"keywords_list\": \"normal tissue, tissue site, eQTL, RNA-seq\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 683.2, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:25\", \"ess\": 0}, {\"page_id\": \"15_search\", \"is_success\": true, \"error_count\": 11.0, \"label_missing\": 3.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 6.0, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": 1.0, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 15, \"page_type\": \"search\", \"source_id\": \"dc_92\", \"short_name\": \"Ensembl\", \"accession\": \"DBC000092\", \"full_name\": \"genome browser for vertebrate genomes\", \"description\": \"Ensembl is a genome browser for vertebrate genomes that supports research in comparative genomics, evolution, sequence variation and transcriptional regulation. Ensembl annotate genes, computes multiple alignments, predicts regulatory function and collects disease data. Ensembl tools include BLAST, BLAT, BioMart and the Variant Effect Predictor (VEP) for all supported species.\", \"founded_year\": 2002, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"European Molecular Biology Laboratory, European Bioinformatics Institute, Wellcome Genome Campus, Hinxton, CB10 1SD, Cambridge, UK.\", \"city\": \"Cambridge\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"release 110\", \"contact_name\": \"Fergal J Martin\", \"contact_email\": \"fergal@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2023-07\", \"age\": 21, \"citation\": 11979, \"token\": \"$2a$10$vQ..9kDsX7/gRQh/fnLlT.JxiBicHQFIusL22pepdl4VvfvmoWfnW\", \"expire_date\": 1543898901295.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-09-13 22:33:18\", \"data_type_list\": \"DNA, Protein, RNA\", \"category_list\": \"Gene genome and annotation, Genotype phenotype and variation, Phylogeny and homology, Interaction\", \"keywords_list\": \"protein-coding gene, pseudogene, lncRNA\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus, Danio rerio, Sus scrofa\", \"theme_list\": \"lncRNA\", \"zindex\": 570.429, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:04\", \"ess\": 0}, {\"page_id\": \"15_search_result\", \"is_success\": true, \"error_count\": 6.0, \"label_missing\": 4.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 1.0, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 15, \"page_type\": \"search_result\", \"source_id\": \"dc_92\", \"short_name\": \"Ensembl\", \"accession\": \"DBC000092\", \"full_name\": \"genome browser for vertebrate genomes\", \"description\": \"Ensembl is a genome browser for vertebrate genomes that supports research in comparative genomics, evolution, sequence variation and transcriptional regulation. Ensembl annotate genes, computes multiple alignments, predicts regulatory function and collects disease data. Ensembl tools include BLAST, BLAT, BioMart and the Variant Effect Predictor (VEP) for all supported species.\", \"founded_year\": 2002, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"European Molecular Biology Laboratory, European Bioinformatics Institute, Wellcome Genome Campus, Hinxton, CB10 1SD, Cambridge, UK.\", \"city\": \"Cambridge\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"release 110\", \"contact_name\": \"Fergal J Martin\", \"contact_email\": \"fergal@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2023-07\", \"age\": 21, \"citation\": 11979, \"token\": \"$2a$10$vQ..9kDsX7/gRQh/fnLlT.JxiBicHQFIusL22pepdl4VvfvmoWfnW\", \"expire_date\": 1543898901295.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-09-13 22:33:18\", \"data_type_list\": \"DNA, Protein, RNA\", \"category_list\": \"Gene genome and annotation, Genotype phenotype and variation, Phylogeny and homology, Interaction\", \"keywords_list\": \"protein-coding gene, pseudogene, lncRNA\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus, Danio rerio, Sus scrofa\", \"theme_list\": \"lncRNA\", \"zindex\": 570.429, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:04\", \"ess\": 0}, {\"page_id\": \"15_data_entity\", \"is_success\": true, \"error_count\": 54.0, \"label_missing\": 16.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 25.0, \"link_empty\": 13.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 15, \"page_type\": \"data_entity\", \"source_id\": \"dc_92\", \"short_name\": \"Ensembl\", \"accession\": \"DBC000092\", \"full_name\": \"genome browser for vertebrate genomes\", \"description\": \"Ensembl is a genome browser for vertebrate genomes that supports research in comparative genomics, evolution, sequence variation and transcriptional regulation. Ensembl annotate genes, computes multiple alignments, predicts regulatory function and collects disease data. Ensembl tools include BLAST, BLAT, BioMart and the Variant Effect Predictor (VEP) for all supported species.\", \"founded_year\": 2002, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"European Molecular Biology Laboratory, European Bioinformatics Institute, Wellcome Genome Campus, Hinxton, CB10 1SD, Cambridge, UK.\", \"city\": \"Cambridge\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"release 110\", \"contact_name\": \"Fergal J Martin\", \"contact_email\": \"fergal@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2023-07\", \"age\": 21, \"citation\": 11979, \"token\": \"$2a$10$vQ..9kDsX7/gRQh/fnLlT.JxiBicHQFIusL22pepdl4VvfvmoWfnW\", \"expire_date\": 1543898901295.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-09-13 22:33:18\", \"data_type_list\": \"DNA, Protein, RNA\", \"category_list\": \"Gene genome and annotation, Genotype phenotype and variation, Phylogeny and homology, Interaction\", \"keywords_list\": \"protein-coding gene, pseudogene, lncRNA\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus, Danio rerio, Sus scrofa\", \"theme_list\": \"lncRNA\", \"zindex\": 570.429, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:04\", \"ess\": 0}, {\"page_id\": \"15_documentation\", \"is_success\": true, \"error_count\": 12.0, \"label_missing\": 4.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 7.0, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 15, \"page_type\": \"documentation\", \"source_id\": \"dc_92\", \"short_name\": \"Ensembl\", \"accession\": \"DBC000092\", \"full_name\": \"genome browser for vertebrate genomes\", \"description\": \"Ensembl is a genome browser for vertebrate genomes that supports research in comparative genomics, evolution, sequence variation and transcriptional regulation. Ensembl annotate genes, computes multiple alignments, predicts regulatory function and collects disease data. Ensembl tools include BLAST, BLAT, BioMart and the Variant Effect Predictor (VEP) for all supported species.\", \"founded_year\": 2002, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"European Molecular Biology Laboratory, European Bioinformatics Institute, Wellcome Genome Campus, Hinxton, CB10 1SD, Cambridge, UK.\", \"city\": \"Cambridge\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"release 110\", \"contact_name\": \"Fergal J Martin\", \"contact_email\": \"fergal@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2023-07\", \"age\": 21, \"citation\": 11979, \"token\": \"$2a$10$vQ..9kDsX7/gRQh/fnLlT.JxiBicHQFIusL22pepdl4VvfvmoWfnW\", \"expire_date\": 1543898901295.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-09-13 22:33:18\", \"data_type_list\": \"DNA, Protein, RNA\", \"category_list\": \"Gene genome and annotation, Genotype phenotype and variation, Phylogeny and homology, Interaction\", \"keywords_list\": \"protein-coding gene, pseudogene, lncRNA\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus, Danio rerio, Sus scrofa\", \"theme_list\": \"lncRNA\", \"zindex\": 570.429, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:04\", \"ess\": 0}, {\"page_id\": \"16_search_result\", \"is_success\": true, \"error_count\": 0.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 16, \"page_type\": \"search_result\", \"source_id\": \"dc_6331\", \"short_name\": \"TCGA\", \"accession\": \"DBC006331\", \"full_name\": \"The Cancer Genome Atlas\", \"description\": \"A landmark cancer genomics program, molecularly characterized over 20,000 primary cancer and matched normal samples spanning 33 cancer types. \", \"founded_year\": 2013, \"host_institution\": \"National Cancer Institute\", \"address\": \"Center for Cancer Genomics\\n31 Center Drive, Room 10A11\\nBethesda, MD 20892\", \"city\": \"Bethesda\", \"province\": null, \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Harold Varmus\", \"contact_email\": \"tcga@mail.nih.gov\", \"available_protocol\": 1.0, \"last_update\": null, \"age\": 10, \"citation\": 5085, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1564636398000, \"last_modified\": \"2023-03-03 10:16:14\", \"data_type_list\": \"DNA, RNA, Protein\", \"category_list\": \"Genotype phenotype and variation, Expression, Structure, Health and medicine, Modification\", \"keywords_list\": \"cancer, human\", \"data_object_list\": null, \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 508.5, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:25\", \"ess\": 0}, {\"page_id\": \"16_data_entity\", \"is_success\": true, \"error_count\": 0.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 16, \"page_type\": \"data_entity\", \"source_id\": \"dc_6331\", \"short_name\": \"TCGA\", \"accession\": \"DBC006331\", \"full_name\": \"The Cancer Genome Atlas\", \"description\": \"A landmark cancer genomics program, molecularly characterized over 20,000 primary cancer and matched normal samples spanning 33 cancer types. \", \"founded_year\": 2013, \"host_institution\": \"National Cancer Institute\", \"address\": \"Center for Cancer Genomics\\n31 Center Drive, Room 10A11\\nBethesda, MD 20892\", \"city\": \"Bethesda\", \"province\": null, \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Harold Varmus\", \"contact_email\": \"tcga@mail.nih.gov\", \"available_protocol\": 1.0, \"last_update\": null, \"age\": 10, \"citation\": 5085, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1564636398000, \"last_modified\": \"2023-03-03 10:16:14\", \"data_type_list\": \"DNA, RNA, Protein\", \"category_list\": \"Genotype phenotype and variation, Expression, Structure, Health and medicine, Modification\", \"keywords_list\": \"cancer, human\", \"data_object_list\": null, \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 508.5, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:25\", \"ess\": 0}, {\"page_id\": \"16_documentation\", \"is_success\": true, \"error_count\": 4.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 1.0, \"link_empty\": null, \"aria_menu_broken\": 1.0, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": 1.0, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": 1.0, \"label_multiple\": null, \"id\": 16, \"page_type\": \"documentation\", \"source_id\": \"dc_6331\", \"short_name\": \"TCGA\", \"accession\": \"DBC006331\", \"full_name\": \"The Cancer Genome Atlas\", \"description\": \"A landmark cancer genomics program, molecularly characterized over 20,000 primary cancer and matched normal samples spanning 33 cancer types. \", \"founded_year\": 2013, \"host_institution\": \"National Cancer Institute\", \"address\": \"Center for Cancer Genomics\\n31 Center Drive, Room 10A11\\nBethesda, MD 20892\", \"city\": \"Bethesda\", \"province\": null, \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Harold Varmus\", \"contact_email\": \"tcga@mail.nih.gov\", \"available_protocol\": 1.0, \"last_update\": null, \"age\": 10, \"citation\": 5085, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1564636398000, \"last_modified\": \"2023-03-03 10:16:14\", \"data_type_list\": \"DNA, RNA, Protein\", \"category_list\": \"Genotype phenotype and variation, Expression, Structure, Health and medicine, Modification\", \"keywords_list\": \"cancer, human\", \"data_object_list\": null, \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 508.5, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:25\", \"ess\": 0}, {\"page_id\": \"17_search_result\", \"is_success\": true, \"error_count\": 34.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 32.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 2.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 17, \"page_type\": \"search_result\", \"source_id\": \"dc_7433\", \"short_name\": \"AlphaFold DB\", \"accession\": \"DBC007433\", \"full_name\": \"AlphaFold Protein Structure Database\", \"description\": \"AlphaFold DB provides open access to AlphaFold protein structure predictions for the human proteome and other key organisms to accelerate scientific research.\", \"founded_year\": 2022, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"EMBL-EBI, Wellcome Genome Campus, Hinxton, Cambridgeshire, CB10 1SD, UK. +44 (0)1223 49 44 44\", \"city\": \"Hinxton\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": null, \"contact_name\": \"Sameer Velankar\", \"contact_email\": \"afdbhelp@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2021\", \"age\": 1, \"citation\": 476, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1634611017000, \"last_modified\": \"2023-03-03 10:17:30\", \"data_type_list\": \"Protein\", \"category_list\": \"Structure\", \"keywords_list\": \"AI, DeepMind, 3D structure\", \"data_object_list\": \"Animal, Plant, Bacteria, Archaea\", \"organism_list\": \"Homo sapiens, Arabidopsis thaliana, Caenorhabditis elegans, Candida albicans\", \"theme_list\": null, \"zindex\": 476.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:48\", \"ess\": 0}, {\"page_id\": \"17_data_entity\", \"is_success\": true, \"error_count\": 32.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 29.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 2.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 17, \"page_type\": \"data_entity\", \"source_id\": \"dc_7433\", \"short_name\": \"AlphaFold DB\", \"accession\": \"DBC007433\", \"full_name\": \"AlphaFold Protein Structure Database\", \"description\": \"AlphaFold DB provides open access to AlphaFold protein structure predictions for the human proteome and other key organisms to accelerate scientific research.\", \"founded_year\": 2022, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"EMBL-EBI, Wellcome Genome Campus, Hinxton, Cambridgeshire, CB10 1SD, UK. +44 (0)1223 49 44 44\", \"city\": \"Hinxton\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": null, \"contact_name\": \"Sameer Velankar\", \"contact_email\": \"afdbhelp@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2021\", \"age\": 1, \"citation\": 476, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1634611017000, \"last_modified\": \"2023-03-03 10:17:30\", \"data_type_list\": \"Protein\", \"category_list\": \"Structure\", \"keywords_list\": \"AI, DeepMind, 3D structure\", \"data_object_list\": \"Animal, Plant, Bacteria, Archaea\", \"organism_list\": \"Homo sapiens, Arabidopsis thaliana, Caenorhabditis elegans, Candida albicans\", \"theme_list\": null, \"zindex\": 476.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:48\", \"ess\": 0}, {\"page_id\": \"17_documentation\", \"is_success\": true, \"error_count\": 0.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 17, \"page_type\": \"documentation\", \"source_id\": \"dc_7433\", \"short_name\": \"AlphaFold DB\", \"accession\": \"DBC007433\", \"full_name\": \"AlphaFold Protein Structure Database\", \"description\": \"AlphaFold DB provides open access to AlphaFold protein structure predictions for the human proteome and other key organisms to accelerate scientific research.\", \"founded_year\": 2022, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"EMBL-EBI, Wellcome Genome Campus, Hinxton, Cambridgeshire, CB10 1SD, UK. +44 (0)1223 49 44 44\", \"city\": \"Hinxton\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": null, \"contact_name\": \"Sameer Velankar\", \"contact_email\": \"afdbhelp@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2021\", \"age\": 1, \"citation\": 476, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1634611017000, \"last_modified\": \"2023-03-03 10:17:30\", \"data_type_list\": \"Protein\", \"category_list\": \"Structure\", \"keywords_list\": \"AI, DeepMind, 3D structure\", \"data_object_list\": \"Animal, Plant, Bacteria, Archaea\", \"organism_list\": \"Homo sapiens, Arabidopsis thaliana, Caenorhabditis elegans, Candida albicans\", \"theme_list\": null, \"zindex\": 476.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:48\", \"ess\": 0}, {\"page_id\": \"18_search_result\", \"is_success\": true, \"error_count\": 23.0, \"label_missing\": 2.0, \"language_missing\": null, \"button_empty\": 2.0, \"alt_missing\": 1.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": 17.0, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 18, \"page_type\": \"search_result\", \"source_id\": \"dc_1865\", \"short_name\": \"GWAS Catalog\", \"accession\": \"DBC001865\", \"full_name\": null, \"description\": \"The GWAS Catalog is a publicly available, manually curated resource of all published GWAS and association results, collaboratively produced and developed by the NHGRI and EMBL-EBI.\", \"founded_year\": 2009, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"Wellcome Genome Campus, Hinxton, Cambridge, CB10 1SD\", \"city\": \"Hinxton\", \"province\": \"Cambridge\", \"country\": \"United Kingdom\", \"current_version\": null, \"contact_name\": \"GWAS Catalog team\", \"contact_email\": \"gwas-info@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2017-01-01\", \"age\": 14, \"citation\": 6367, \"token\": \"$2a$10$aGMaTxqVlB0PDuY5ROMgK.4EG9PhdGTLnvWHj4zcp6FFJHm4IiII6\", \"expire_date\": 1543995396646.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1487571954000, \"last_modified\": \"2023-03-03 10:11:56\", \"data_type_list\": \"DNA\", \"category_list\": \"Genotype phenotype and variation\", \"keywords_list\": \"Genome-wide association studies (GWAS), SNP\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 454.786, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:54\", \"ess\": 0}, {\"page_id\": \"18_data_entity\", \"is_success\": true, \"error_count\": 154.0, \"label_missing\": 78.0, \"language_missing\": null, \"button_empty\": 1.0, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": 74.0, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": 1.0, \"label_multiple\": null, \"id\": 18, \"page_type\": \"data_entity\", \"source_id\": \"dc_1865\", \"short_name\": \"GWAS Catalog\", \"accession\": \"DBC001865\", \"full_name\": null, \"description\": \"The GWAS Catalog is a publicly available, manually curated resource of all published GWAS and association results, collaboratively produced and developed by the NHGRI and EMBL-EBI.\", \"founded_year\": 2009, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"Wellcome Genome Campus, Hinxton, Cambridge, CB10 1SD\", \"city\": \"Hinxton\", \"province\": \"Cambridge\", \"country\": \"United Kingdom\", \"current_version\": null, \"contact_name\": \"GWAS Catalog team\", \"contact_email\": \"gwas-info@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2017-01-01\", \"age\": 14, \"citation\": 6367, \"token\": \"$2a$10$aGMaTxqVlB0PDuY5ROMgK.4EG9PhdGTLnvWHj4zcp6FFJHm4IiII6\", \"expire_date\": 1543995396646.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1487571954000, \"last_modified\": \"2023-03-03 10:11:56\", \"data_type_list\": \"DNA\", \"category_list\": \"Genotype phenotype and variation\", \"keywords_list\": \"Genome-wide association studies (GWAS), SNP\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 454.786, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:54\", \"ess\": 0}, {\"page_id\": \"18_documentation\", \"is_success\": true, \"error_count\": 6.0, \"label_missing\": 2.0, \"language_missing\": null, \"button_empty\": 2.0, \"alt_missing\": 1.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 18, \"page_type\": \"documentation\", \"source_id\": \"dc_1865\", \"short_name\": \"GWAS Catalog\", \"accession\": \"DBC001865\", \"full_name\": null, \"description\": \"The GWAS Catalog is a publicly available, manually curated resource of all published GWAS and association results, collaboratively produced and developed by the NHGRI and EMBL-EBI.\", \"founded_year\": 2009, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"Wellcome Genome Campus, Hinxton, Cambridge, CB10 1SD\", \"city\": \"Hinxton\", \"province\": \"Cambridge\", \"country\": \"United Kingdom\", \"current_version\": null, \"contact_name\": \"GWAS Catalog team\", \"contact_email\": \"gwas-info@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2017-01-01\", \"age\": 14, \"citation\": 6367, \"token\": \"$2a$10$aGMaTxqVlB0PDuY5ROMgK.4EG9PhdGTLnvWHj4zcp6FFJHm4IiII6\", \"expire_date\": 1543995396646.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1487571954000, \"last_modified\": \"2023-03-03 10:11:56\", \"data_type_list\": \"DNA\", \"category_list\": \"Genotype phenotype and variation\", \"keywords_list\": \"Genome-wide association studies (GWAS), SNP\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 454.786, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:54\", \"ess\": 0}, {\"page_id\": \"19_search\", \"is_success\": true, \"error_count\": 7.0, \"label_missing\": 4.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 3.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 19, \"page_type\": \"search\", \"source_id\": \"dc_5009\", \"short_name\": \"GtoPdb\", \"accession\": \"DBC005009\", \"full_name\": \"IUPHAR/BPS Guide to PHARMACOLOGY\", \"description\": \"GtoPdb provides a searchable database with quantitative information on drug targets and the prescription medicines and experimental drugs that act on them.\", \"founded_year\": 2011, \"host_institution\": \"University of Edinburgh\", \"address\": \"Centre for Integrative Physiology, University of Edinburgh, Edinburgh, UK\", \"city\": null, \"province\": null, \"country\": \"United Kingdom\", \"current_version\": \"Release 2019.4\", \"contact_name\": \"Jamie A Davies\", \"contact_email\": \"j.davies@trinitycollegebristol.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2019-09-18\", \"age\": 12, \"citation\": 5208, \"token\": \"$2a$10$BRpnffTZzR9ulTOcpQeqoeTGqE73D2ZZ9e4TlbAkI4V/WhMgchRFq\", \"expire_date\": 1544148156020.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1517168584000, \"last_modified\": \"2023-10-10 10:17:41\", \"data_type_list\": \"Protein\", \"category_list\": \"Interaction, Health and medicine\", \"keywords_list\": \"pharmacology, ligand and target, SARS-CoV-2\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 434.0, \"first_publication_year\": 2015, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:01:55\", \"ess\": 1}, {\"page_id\": \"19_search_result\", \"is_success\": true, \"error_count\": 1.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 19, \"page_type\": \"search_result\", \"source_id\": \"dc_5009\", \"short_name\": \"GtoPdb\", \"accession\": \"DBC005009\", \"full_name\": \"IUPHAR/BPS Guide to PHARMACOLOGY\", \"description\": \"GtoPdb provides a searchable database with quantitative information on drug targets and the prescription medicines and experimental drugs that act on them.\", \"founded_year\": 2011, \"host_institution\": \"University of Edinburgh\", \"address\": \"Centre for Integrative Physiology, University of Edinburgh, Edinburgh, UK\", \"city\": null, \"province\": null, \"country\": \"United Kingdom\", \"current_version\": \"Release 2019.4\", \"contact_name\": \"Jamie A Davies\", \"contact_email\": \"j.davies@trinitycollegebristol.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2019-09-18\", \"age\": 12, \"citation\": 5208, \"token\": \"$2a$10$BRpnffTZzR9ulTOcpQeqoeTGqE73D2ZZ9e4TlbAkI4V/WhMgchRFq\", \"expire_date\": 1544148156020.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1517168584000, \"last_modified\": \"2023-10-10 10:17:41\", \"data_type_list\": \"Protein\", \"category_list\": \"Interaction, Health and medicine\", \"keywords_list\": \"pharmacology, ligand and target, SARS-CoV-2\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 434.0, \"first_publication_year\": 2015, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:01:55\", \"ess\": 1}, {\"page_id\": \"19_data_entity\", \"is_success\": true, \"error_count\": 1.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 19, \"page_type\": \"data_entity\", \"source_id\": \"dc_5009\", \"short_name\": \"GtoPdb\", \"accession\": \"DBC005009\", \"full_name\": \"IUPHAR/BPS Guide to PHARMACOLOGY\", \"description\": \"GtoPdb provides a searchable database with quantitative information on drug targets and the prescription medicines and experimental drugs that act on them.\", \"founded_year\": 2011, \"host_institution\": \"University of Edinburgh\", \"address\": \"Centre for Integrative Physiology, University of Edinburgh, Edinburgh, UK\", \"city\": null, \"province\": null, \"country\": \"United Kingdom\", \"current_version\": \"Release 2019.4\", \"contact_name\": \"Jamie A Davies\", \"contact_email\": \"j.davies@trinitycollegebristol.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2019-09-18\", \"age\": 12, \"citation\": 5208, \"token\": \"$2a$10$BRpnffTZzR9ulTOcpQeqoeTGqE73D2ZZ9e4TlbAkI4V/WhMgchRFq\", \"expire_date\": 1544148156020.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1517168584000, \"last_modified\": \"2023-10-10 10:17:41\", \"data_type_list\": \"Protein\", \"category_list\": \"Interaction, Health and medicine\", \"keywords_list\": \"pharmacology, ligand and target, SARS-CoV-2\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 434.0, \"first_publication_year\": 2015, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:01:55\", \"ess\": 1}, {\"page_id\": \"19_documentation\", \"is_success\": true, \"error_count\": 1.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 19, \"page_type\": \"documentation\", \"source_id\": \"dc_5009\", \"short_name\": \"GtoPdb\", \"accession\": \"DBC005009\", \"full_name\": \"IUPHAR/BPS Guide to PHARMACOLOGY\", \"description\": \"GtoPdb provides a searchable database with quantitative information on drug targets and the prescription medicines and experimental drugs that act on them.\", \"founded_year\": 2011, \"host_institution\": \"University of Edinburgh\", \"address\": \"Centre for Integrative Physiology, University of Edinburgh, Edinburgh, UK\", \"city\": null, \"province\": null, \"country\": \"United Kingdom\", \"current_version\": \"Release 2019.4\", \"contact_name\": \"Jamie A Davies\", \"contact_email\": \"j.davies@trinitycollegebristol.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2019-09-18\", \"age\": 12, \"citation\": 5208, \"token\": \"$2a$10$BRpnffTZzR9ulTOcpQeqoeTGqE73D2ZZ9e4TlbAkI4V/WhMgchRFq\", \"expire_date\": 1544148156020.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1517168584000, \"last_modified\": \"2023-10-10 10:17:41\", \"data_type_list\": \"Protein\", \"category_list\": \"Interaction, Health and medicine\", \"keywords_list\": \"pharmacology, ligand and target, SARS-CoV-2\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 434.0, \"first_publication_year\": 2015, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:01:55\", \"ess\": 1}, {\"page_id\": \"20_documentation\", \"is_success\": true, \"error_count\": 3.0, \"label_missing\": null, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 20, \"page_type\": \"documentation\", \"source_id\": \"dc_6158\", \"short_name\": \"GENCODE\", \"accession\": \"DBC006158\", \"full_name\": \"reference annotation for the human and mouse genomes\", \"description\": \"The goal of the GENCODE project is to identify and classify all gene features in the human and mouse genomes with high accuracy based on biological evidence, and to release these annotations for the benefit of biomedical research and genome interpretation.\", \"founded_year\": 2006, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"European Bioinformatics Institute, Wellcome Genome Campus, Hinxton, Cambridge CB10 1SD, UK\", \"city\": \"Cambridge\", \"province\": null, \"country\": \"United Kingdom\", \"current_version\": \"Release 44\", \"contact_name\": \"Paul Flicek\", \"contact_email\": \"flicek@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2023-07-17\", \"age\": 17, \"citation\": 7278, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1546534020000, \"last_modified\": \"2023-09-13 20:27:38\", \"data_type_list\": \"DNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"lncRNA, pseudogene, alternative splicing\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": \"lncRNA\", \"zindex\": 428.118, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:23\", \"ess\": 0}, {\"page_id\": \"21_search\", \"is_success\": true, \"error_count\": 4.0, \"label_missing\": 2.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": 1.0, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 21, \"page_type\": \"search\", \"source_id\": \"dc_433\", \"short_name\": \"PRIDE\", \"accession\": \"DBC000433\", \"full_name\": \"PRoteomics IDEntifications\", \"description\": \"An archive of protein expression data determined by mass spectrometry. The PRIDE database is a centralized, standards compliant, public data repository for proteomics data, including protein and peptide identifications, post-translational modifications and supporting spectral evidence.\", \"founded_year\": 2005, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"Wellcome Trust Genome Campus, Hinxton, Cambridge, UK\", \"city\": \"Cambridge\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"v1.0\", \"contact_name\": \"PRIDE team\", \"contact_email\": \"pride-support@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2018-12\", \"age\": 18, \"citation\": 7425, \"token\": \"$2a$10$.guYHfwqkY.yZteP02bDC.2Esa/0uB.uyKxgGlSGYdVp13BBSCAh.\", \"expire_date\": 1543906757995.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:10:13\", \"data_type_list\": \"Protein\", \"category_list\": \"Raw bio-data, Expression\", \"keywords_list\": \"protein and peptide identification, post-translational modification, mass spectrometry\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 412.5, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:15\", \"ess\": 0}, {\"page_id\": \"21_search_result\", \"is_success\": true, \"error_count\": 5.0, \"label_missing\": 3.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": 1.0, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 21, \"page_type\": \"search_result\", \"source_id\": \"dc_433\", \"short_name\": \"PRIDE\", \"accession\": \"DBC000433\", \"full_name\": \"PRoteomics IDEntifications\", \"description\": \"An archive of protein expression data determined by mass spectrometry. The PRIDE database is a centralized, standards compliant, public data repository for proteomics data, including protein and peptide identifications, post-translational modifications and supporting spectral evidence.\", \"founded_year\": 2005, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"Wellcome Trust Genome Campus, Hinxton, Cambridge, UK\", \"city\": \"Cambridge\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"v1.0\", \"contact_name\": \"PRIDE team\", \"contact_email\": \"pride-support@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2018-12\", \"age\": 18, \"citation\": 7425, \"token\": \"$2a$10$.guYHfwqkY.yZteP02bDC.2Esa/0uB.uyKxgGlSGYdVp13BBSCAh.\", \"expire_date\": 1543906757995.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:10:13\", \"data_type_list\": \"Protein\", \"category_list\": \"Raw bio-data, Expression\", \"keywords_list\": \"protein and peptide identification, post-translational modification, mass spectrometry\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 412.5, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:15\", \"ess\": 0}, {\"page_id\": \"21_data_entity\", \"is_success\": true, \"error_count\": 7.0, \"label_missing\": 2.0, \"language_missing\": null, \"button_empty\": 1.0, \"alt_missing\": null, \"link_empty\": 2.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": 1.0, \"heading_empty\": null, \"link_skip_broken\": 1.0, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 21, \"page_type\": \"data_entity\", \"source_id\": \"dc_433\", \"short_name\": \"PRIDE\", \"accession\": \"DBC000433\", \"full_name\": \"PRoteomics IDEntifications\", \"description\": \"An archive of protein expression data determined by mass spectrometry. The PRIDE database is a centralized, standards compliant, public data repository for proteomics data, including protein and peptide identifications, post-translational modifications and supporting spectral evidence.\", \"founded_year\": 2005, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"Wellcome Trust Genome Campus, Hinxton, Cambridge, UK\", \"city\": \"Cambridge\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"v1.0\", \"contact_name\": \"PRIDE team\", \"contact_email\": \"pride-support@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2018-12\", \"age\": 18, \"citation\": 7425, \"token\": \"$2a$10$.guYHfwqkY.yZteP02bDC.2Esa/0uB.uyKxgGlSGYdVp13BBSCAh.\", \"expire_date\": 1543906757995.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:10:13\", \"data_type_list\": \"Protein\", \"category_list\": \"Raw bio-data, Expression\", \"keywords_list\": \"protein and peptide identification, post-translational modification, mass spectrometry\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 412.5, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:15\", \"ess\": 0}, {\"page_id\": \"21_documentation\", \"is_success\": true, \"error_count\": 4.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 1.0, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": 1.0, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 21, \"page_type\": \"documentation\", \"source_id\": \"dc_433\", \"short_name\": \"PRIDE\", \"accession\": \"DBC000433\", \"full_name\": \"PRoteomics IDEntifications\", \"description\": \"An archive of protein expression data determined by mass spectrometry. The PRIDE database is a centralized, standards compliant, public data repository for proteomics data, including protein and peptide identifications, post-translational modifications and supporting spectral evidence.\", \"founded_year\": 2005, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"Wellcome Trust Genome Campus, Hinxton, Cambridge, UK\", \"city\": \"Cambridge\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"v1.0\", \"contact_name\": \"PRIDE team\", \"contact_email\": \"pride-support@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2018-12\", \"age\": 18, \"citation\": 7425, \"token\": \"$2a$10$.guYHfwqkY.yZteP02bDC.2Esa/0uB.uyKxgGlSGYdVp13BBSCAh.\", \"expire_date\": 1543906757995.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:10:13\", \"data_type_list\": \"Protein\", \"category_list\": \"Raw bio-data, Expression\", \"keywords_list\": \"protein and peptide identification, post-translational modification, mass spectrometry\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 412.5, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:15\", \"ess\": 0}, {\"page_id\": \"22_search_result\", \"is_success\": true, \"error_count\": 1.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": 1.0, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 22, \"page_type\": \"search_result\", \"source_id\": \"dc_148\", \"short_name\": \"GO\", \"accession\": \"DBC000148\", \"full_name\": \"Gene Ontology Consortium\", \"description\": \"GO is a community-based bioinformatics resource that supplies information about gene product function using ontologies to represent biological knowledge.\", \"founded_year\": 2001, \"host_institution\": \"na\", \"address\": \"Geneva,Switzerland\", \"city\": \"Geneva\", \"province\": null, \"country\": \"Switzerland\", \"current_version\": null, \"contact_name\": \"The Gene Ontology Consortium\", \"contact_email\": \"help@geneontology.org\", \"available_protocol\": 1.0, \"last_update\": \"2021-2-1\", \"age\": 22, \"citation\": 8684, \"token\": \"$2a$10$5GPfIdKhDVdOyJmfT77CeeiDIpTWb/9mHvLxN8YlbisjFmJ/jFWf.\", \"expire_date\": 1543898942953.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-08-28 20:20:52\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation, Standard ontology and nomenclature\", \"keywords_list\": \"biological process, cellular component, molecular function, functional annotation\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria\", \"organism_list\": \"Escherichia coli, Homo sapiens, Oryza sativa, Saccharomyces cerevisiae, Schizosaccharomyces pombe, Mus musculus\", \"theme_list\": null, \"zindex\": 394.727, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:06\", \"ess\": 0}, {\"page_id\": \"22_data_entity\", \"is_success\": true, \"error_count\": 6.0, \"label_missing\": 3.0, \"language_missing\": null, \"button_empty\": 1.0, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": 2.0, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 22, \"page_type\": \"data_entity\", \"source_id\": \"dc_148\", \"short_name\": \"GO\", \"accession\": \"DBC000148\", \"full_name\": \"Gene Ontology Consortium\", \"description\": \"GO is a community-based bioinformatics resource that supplies information about gene product function using ontologies to represent biological knowledge.\", \"founded_year\": 2001, \"host_institution\": \"na\", \"address\": \"Geneva,Switzerland\", \"city\": \"Geneva\", \"province\": null, \"country\": \"Switzerland\", \"current_version\": null, \"contact_name\": \"The Gene Ontology Consortium\", \"contact_email\": \"help@geneontology.org\", \"available_protocol\": 1.0, \"last_update\": \"2021-2-1\", \"age\": 22, \"citation\": 8684, \"token\": \"$2a$10$5GPfIdKhDVdOyJmfT77CeeiDIpTWb/9mHvLxN8YlbisjFmJ/jFWf.\", \"expire_date\": 1543898942953.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-08-28 20:20:52\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation, Standard ontology and nomenclature\", \"keywords_list\": \"biological process, cellular component, molecular function, functional annotation\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria\", \"organism_list\": \"Escherichia coli, Homo sapiens, Oryza sativa, Saccharomyces cerevisiae, Schizosaccharomyces pombe, Mus musculus\", \"theme_list\": null, \"zindex\": 394.727, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:06\", \"ess\": 0}, {\"page_id\": \"22_documentation\", \"is_success\": true, \"error_count\": 5.0, \"label_missing\": null, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 3.0, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 22, \"page_type\": \"documentation\", \"source_id\": \"dc_148\", \"short_name\": \"GO\", \"accession\": \"DBC000148\", \"full_name\": \"Gene Ontology Consortium\", \"description\": \"GO is a community-based bioinformatics resource that supplies information about gene product function using ontologies to represent biological knowledge.\", \"founded_year\": 2001, \"host_institution\": \"na\", \"address\": \"Geneva,Switzerland\", \"city\": \"Geneva\", \"province\": null, \"country\": \"Switzerland\", \"current_version\": null, \"contact_name\": \"The Gene Ontology Consortium\", \"contact_email\": \"help@geneontology.org\", \"available_protocol\": 1.0, \"last_update\": \"2021-2-1\", \"age\": 22, \"citation\": 8684, \"token\": \"$2a$10$5GPfIdKhDVdOyJmfT77CeeiDIpTWb/9mHvLxN8YlbisjFmJ/jFWf.\", \"expire_date\": 1543898942953.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-08-28 20:20:52\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation, Standard ontology and nomenclature\", \"keywords_list\": \"biological process, cellular component, molecular function, functional annotation\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria\", \"organism_list\": \"Escherichia coli, Homo sapiens, Oryza sativa, Saccharomyces cerevisiae, Schizosaccharomyces pombe, Mus musculus\", \"theme_list\": null, \"zindex\": 394.727, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:06\", \"ess\": 0}, {\"page_id\": \"23_search\", \"is_success\": true, \"error_count\": 0.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 23, \"page_type\": \"search\", \"source_id\": \"dc_282\", \"short_name\": \"ClinVar\", \"accession\": \"DBC000282\", \"full_name\": \"ClinVar\", \"description\": \"ClinVar aggregates information about genomic variation and its relationship to human health.\", \"founded_year\": 2014, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"8600 Rockville Pike,Bethesda,MD 20894,USA\", \"city\": \"Bethesda\", \"province\": \"MD\", \"country\": \"United States\", \"current_version\": \"v1.0\", \"contact_name\": \"ClinVar team\", \"contact_email\": \"clinvar@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": null, \"age\": 9, \"citation\": 3539, \"token\": \"$2a$10$RicS5H80EqfuJxKAuyc5f.bmIRlzXa2CAzRPv6RAdl.HD2S8vYp96\", \"expire_date\": 1543899027028.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:10:00\", \"data_type_list\": \"DNA\", \"category_list\": \"Genotype phenotype and variation, Health and medicine\", \"keywords_list\": \"genomic variation\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 393.222, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:10\", \"ess\": 0}, {\"page_id\": \"23_search_result\", \"is_success\": true, \"error_count\": 2.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": 2.0, \"id\": 23, \"page_type\": \"search_result\", \"source_id\": \"dc_282\", \"short_name\": \"ClinVar\", \"accession\": \"DBC000282\", \"full_name\": \"ClinVar\", \"description\": \"ClinVar aggregates information about genomic variation and its relationship to human health.\", \"founded_year\": 2014, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"8600 Rockville Pike,Bethesda,MD 20894,USA\", \"city\": \"Bethesda\", \"province\": \"MD\", \"country\": \"United States\", \"current_version\": \"v1.0\", \"contact_name\": \"ClinVar team\", \"contact_email\": \"clinvar@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": null, \"age\": 9, \"citation\": 3539, \"token\": \"$2a$10$RicS5H80EqfuJxKAuyc5f.bmIRlzXa2CAzRPv6RAdl.HD2S8vYp96\", \"expire_date\": 1543899027028.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:10:00\", \"data_type_list\": \"DNA\", \"category_list\": \"Genotype phenotype and variation, Health and medicine\", \"keywords_list\": \"genomic variation\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 393.222, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:10\", \"ess\": 0}, {\"page_id\": \"23_data_entity\", \"is_success\": true, \"error_count\": 20.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 1.0, \"link_empty\": 17.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": 1.0, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 23, \"page_type\": \"data_entity\", \"source_id\": \"dc_282\", \"short_name\": \"ClinVar\", \"accession\": \"DBC000282\", \"full_name\": \"ClinVar\", \"description\": \"ClinVar aggregates information about genomic variation and its relationship to human health.\", \"founded_year\": 2014, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"8600 Rockville Pike,Bethesda,MD 20894,USA\", \"city\": \"Bethesda\", \"province\": \"MD\", \"country\": \"United States\", \"current_version\": \"v1.0\", \"contact_name\": \"ClinVar team\", \"contact_email\": \"clinvar@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": null, \"age\": 9, \"citation\": 3539, \"token\": \"$2a$10$RicS5H80EqfuJxKAuyc5f.bmIRlzXa2CAzRPv6RAdl.HD2S8vYp96\", \"expire_date\": 1543899027028.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:10:00\", \"data_type_list\": \"DNA\", \"category_list\": \"Genotype phenotype and variation, Health and medicine\", \"keywords_list\": \"genomic variation\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 393.222, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:10\", \"ess\": 0}, {\"page_id\": \"23_documentation\", \"is_success\": true, \"error_count\": 0.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 23, \"page_type\": \"documentation\", \"source_id\": \"dc_282\", \"short_name\": \"ClinVar\", \"accession\": \"DBC000282\", \"full_name\": \"ClinVar\", \"description\": \"ClinVar aggregates information about genomic variation and its relationship to human health.\", \"founded_year\": 2014, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"8600 Rockville Pike,Bethesda,MD 20894,USA\", \"city\": \"Bethesda\", \"province\": \"MD\", \"country\": \"United States\", \"current_version\": \"v1.0\", \"contact_name\": \"ClinVar team\", \"contact_email\": \"clinvar@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": null, \"age\": 9, \"citation\": 3539, \"token\": \"$2a$10$RicS5H80EqfuJxKAuyc5f.bmIRlzXa2CAzRPv6RAdl.HD2S8vYp96\", \"expire_date\": 1543899027028.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:10:00\", \"data_type_list\": \"DNA\", \"category_list\": \"Genotype phenotype and variation, Health and medicine\", \"keywords_list\": \"genomic variation\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 393.222, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:10\", \"ess\": 0}, {\"page_id\": \"25_search\", \"is_success\": true, \"error_count\": 3.0, \"label_missing\": null, \"language_missing\": 1.0, \"button_empty\": 2.0, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 25, \"page_type\": \"search\", \"source_id\": \"dc_7039\", \"short_name\": \"ALLEN BRAIN MAP\", \"accession\": \"DBC007039\", \"full_name\": null, \"description\": \"The Allen Institute for Brain Science uses a unique approach to generate data, tools and knowledge for researchers to explore the biological complexity of the mammalian brain. This portal provides access to high quality data and web-based applications created for the benefit of the global research community.\", \"founded_year\": 2007, \"host_institution\": \"Allen Institute for Brain Science\", \"address\": \"Seattle, WA, USA\", \"city\": \"Seattle\", \"province\": null, \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Edward S. Lein\", \"contact_email\": \"edl@alleninstitute.org\", \"available_protocol\": 1.0, \"last_update\": \"2020-06-25\", \"age\": 16, \"citation\": 6216, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1604676810000, \"last_modified\": \"2023-03-03 10:17:02\", \"data_type_list\": \"DNA, RNA, Other\", \"category_list\": \"Raw bio-data, Expression, Interaction, Health and medicine, Metadata, Literature\", \"keywords_list\": \"brain\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 388.5, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:40\", \"ess\": 0}, {\"page_id\": \"25_search_result\", \"is_success\": true, \"error_count\": 3.0, \"label_missing\": null, \"language_missing\": 1.0, \"button_empty\": 2.0, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 25, \"page_type\": \"search_result\", \"source_id\": \"dc_7039\", \"short_name\": \"ALLEN BRAIN MAP\", \"accession\": \"DBC007039\", \"full_name\": null, \"description\": \"The Allen Institute for Brain Science uses a unique approach to generate data, tools and knowledge for researchers to explore the biological complexity of the mammalian brain. This portal provides access to high quality data and web-based applications created for the benefit of the global research community.\", \"founded_year\": 2007, \"host_institution\": \"Allen Institute for Brain Science\", \"address\": \"Seattle, WA, USA\", \"city\": \"Seattle\", \"province\": null, \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Edward S. Lein\", \"contact_email\": \"edl@alleninstitute.org\", \"available_protocol\": 1.0, \"last_update\": \"2020-06-25\", \"age\": 16, \"citation\": 6216, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1604676810000, \"last_modified\": \"2023-03-03 10:17:02\", \"data_type_list\": \"DNA, RNA, Other\", \"category_list\": \"Raw bio-data, Expression, Interaction, Health and medicine, Metadata, Literature\", \"keywords_list\": \"brain\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 388.5, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:40\", \"ess\": 0}, {\"page_id\": \"25_data_entity\", \"is_success\": true, \"error_count\": 3.0, \"label_missing\": null, \"language_missing\": 1.0, \"button_empty\": 2.0, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 25, \"page_type\": \"data_entity\", \"source_id\": \"dc_7039\", \"short_name\": \"ALLEN BRAIN MAP\", \"accession\": \"DBC007039\", \"full_name\": null, \"description\": \"The Allen Institute for Brain Science uses a unique approach to generate data, tools and knowledge for researchers to explore the biological complexity of the mammalian brain. This portal provides access to high quality data and web-based applications created for the benefit of the global research community.\", \"founded_year\": 2007, \"host_institution\": \"Allen Institute for Brain Science\", \"address\": \"Seattle, WA, USA\", \"city\": \"Seattle\", \"province\": null, \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Edward S. Lein\", \"contact_email\": \"edl@alleninstitute.org\", \"available_protocol\": 1.0, \"last_update\": \"2020-06-25\", \"age\": 16, \"citation\": 6216, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1604676810000, \"last_modified\": \"2023-03-03 10:17:02\", \"data_type_list\": \"DNA, RNA, Other\", \"category_list\": \"Raw bio-data, Expression, Interaction, Health and medicine, Metadata, Literature\", \"keywords_list\": \"brain\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 388.5, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:40\", \"ess\": 0}, {\"page_id\": \"25_documentation\", \"is_success\": true, \"error_count\": 6.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 4.0, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 25, \"page_type\": \"documentation\", \"source_id\": \"dc_7039\", \"short_name\": \"ALLEN BRAIN MAP\", \"accession\": \"DBC007039\", \"full_name\": null, \"description\": \"The Allen Institute for Brain Science uses a unique approach to generate data, tools and knowledge for researchers to explore the biological complexity of the mammalian brain. This portal provides access to high quality data and web-based applications created for the benefit of the global research community.\", \"founded_year\": 2007, \"host_institution\": \"Allen Institute for Brain Science\", \"address\": \"Seattle, WA, USA\", \"city\": \"Seattle\", \"province\": null, \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Edward S. Lein\", \"contact_email\": \"edl@alleninstitute.org\", \"available_protocol\": 1.0, \"last_update\": \"2020-06-25\", \"age\": 16, \"citation\": 6216, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1604676810000, \"last_modified\": \"2023-03-03 10:17:02\", \"data_type_list\": \"DNA, RNA, Other\", \"category_list\": \"Raw bio-data, Expression, Interaction, Health and medicine, Metadata, Literature\", \"keywords_list\": \"brain\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 388.5, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:40\", \"ess\": 0}, {\"page_id\": \"26_search_result\", \"is_success\": true, \"error_count\": 5.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": 3.0, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 26, \"page_type\": \"search_result\", \"source_id\": \"dc_220\", \"short_name\": \"CAZy\", \"accession\": \"DBC000220\", \"full_name\": \"Carbohydrate-active Enzymes Database\", \"description\": \"The CAZy database describes the families of structurally-related catalytic and carbohydrate-binding modules (or functional domains) of enzymes that degrade,modify,or create glycosidic bonds.\", \"founded_year\": 2009, \"host_institution\": \"Architecture et Fonction des Macromol\\u00e9cules Biologiques\", \"address\": \"13288 Marseille,France\", \"city\": \"Marseille\", \"province\": null, \"country\": \"France\", \"current_version\": \"v1.0\", \"contact_name\": \"Bernard Henrissat\", \"contact_email\": \"Bernard.Henrissat@afmb.univ-mrs.fr\", \"available_protocol\": 1.0, \"last_update\": \"5/22/2015\", \"age\": 14, \"citation\": 5353, \"token\": \"$2a$10$QMR8FYYqeWecfwNuZ6IiouV8VjkYhxCVjwnVlGCu.HV3oWA37yfHy\", \"expire_date\": 1543898988405.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:56\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"carbohydrate-active enzyme, glycoside hydrolase, glycosyltransferase, polysaccharide lyase, carbohydrate esterase\", \"data_object_list\": null, \"organism_list\": null, \"theme_list\": null, \"zindex\": 382.357, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:08\", \"ess\": 0}, {\"page_id\": \"26_data_entity\", \"is_success\": true, \"error_count\": 4.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": 3.0, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 26, \"page_type\": \"data_entity\", \"source_id\": \"dc_220\", \"short_name\": \"CAZy\", \"accession\": \"DBC000220\", \"full_name\": \"Carbohydrate-active Enzymes Database\", \"description\": \"The CAZy database describes the families of structurally-related catalytic and carbohydrate-binding modules (or functional domains) of enzymes that degrade,modify,or create glycosidic bonds.\", \"founded_year\": 2009, \"host_institution\": \"Architecture et Fonction des Macromol\\u00e9cules Biologiques\", \"address\": \"13288 Marseille,France\", \"city\": \"Marseille\", \"province\": null, \"country\": \"France\", \"current_version\": \"v1.0\", \"contact_name\": \"Bernard Henrissat\", \"contact_email\": \"Bernard.Henrissat@afmb.univ-mrs.fr\", \"available_protocol\": 1.0, \"last_update\": \"5/22/2015\", \"age\": 14, \"citation\": 5353, \"token\": \"$2a$10$QMR8FYYqeWecfwNuZ6IiouV8VjkYhxCVjwnVlGCu.HV3oWA37yfHy\", \"expire_date\": 1543898988405.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:56\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"carbohydrate-active enzyme, glycoside hydrolase, glycosyltransferase, polysaccharide lyase, carbohydrate esterase\", \"data_object_list\": null, \"organism_list\": null, \"theme_list\": null, \"zindex\": 382.357, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:08\", \"ess\": 0}, {\"page_id\": \"26_documentation\", \"is_success\": true, \"error_count\": 5.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": 3.0, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 26, \"page_type\": \"documentation\", \"source_id\": \"dc_220\", \"short_name\": \"CAZy\", \"accession\": \"DBC000220\", \"full_name\": \"Carbohydrate-active Enzymes Database\", \"description\": \"The CAZy database describes the families of structurally-related catalytic and carbohydrate-binding modules (or functional domains) of enzymes that degrade,modify,or create glycosidic bonds.\", \"founded_year\": 2009, \"host_institution\": \"Architecture et Fonction des Macromol\\u00e9cules Biologiques\", \"address\": \"13288 Marseille,France\", \"city\": \"Marseille\", \"province\": null, \"country\": \"France\", \"current_version\": \"v1.0\", \"contact_name\": \"Bernard Henrissat\", \"contact_email\": \"Bernard.Henrissat@afmb.univ-mrs.fr\", \"available_protocol\": 1.0, \"last_update\": \"5/22/2015\", \"age\": 14, \"citation\": 5353, \"token\": \"$2a$10$QMR8FYYqeWecfwNuZ6IiouV8VjkYhxCVjwnVlGCu.HV3oWA37yfHy\", \"expire_date\": 1543898988405.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:56\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"carbohydrate-active enzyme, glycoside hydrolase, glycosyltransferase, polysaccharide lyase, carbohydrate esterase\", \"data_object_list\": null, \"organism_list\": null, \"theme_list\": null, \"zindex\": 382.357, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:08\", \"ess\": 0}, {\"page_id\": \"27_search\", \"is_success\": true, \"error_count\": 4.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": 1.0, \"alt_missing\": null, \"link_empty\": 2.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 27, \"page_type\": \"search\", \"source_id\": \"dc_30\", \"short_name\": \"InterPro\", \"accession\": \"DBC000030\", \"full_name\": \"Protein sequence analysis & classification\", \"description\": \"InterPro provides functional analysis of proteins by classifying them into families and predicting domains and important sites.\", \"founded_year\": 2000, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI), Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SD, UK\", \"city\": \"Cambridge\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"v92\", \"contact_name\": \"Alex Bateman\", \"contact_email\": \"agb@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2022-12-15\", \"age\": 23, \"citation\": 8643, \"token\": \"$2a$10$XXRDMgyw57XbhaL1X2YaweUs0ywar53fpZagpvFzpbDOtCIG8TGd6\", \"expire_date\": 1543898855889.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-08-23 10:41:57\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation, Standard ontology and nomenclature, Literature\", \"keywords_list\": \"protein classification, protein domain, protein function, protein family\", \"data_object_list\": \"Bacteria, Archaea, Animal, Plant, Fungi, Virus\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 375.783, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:03\", \"ess\": 0}, {\"page_id\": \"27_search_result\", \"is_success\": true, \"error_count\": 4.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": 1.0, \"alt_missing\": null, \"link_empty\": 2.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 27, \"page_type\": \"search_result\", \"source_id\": \"dc_30\", \"short_name\": \"InterPro\", \"accession\": \"DBC000030\", \"full_name\": \"Protein sequence analysis & classification\", \"description\": \"InterPro provides functional analysis of proteins by classifying them into families and predicting domains and important sites.\", \"founded_year\": 2000, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI), Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SD, UK\", \"city\": \"Cambridge\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"v92\", \"contact_name\": \"Alex Bateman\", \"contact_email\": \"agb@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2022-12-15\", \"age\": 23, \"citation\": 8643, \"token\": \"$2a$10$XXRDMgyw57XbhaL1X2YaweUs0ywar53fpZagpvFzpbDOtCIG8TGd6\", \"expire_date\": 1543898855889.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-08-23 10:41:57\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation, Standard ontology and nomenclature, Literature\", \"keywords_list\": \"protein classification, protein domain, protein function, protein family\", \"data_object_list\": \"Bacteria, Archaea, Animal, Plant, Fungi, Virus\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 375.783, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:03\", \"ess\": 0}, {\"page_id\": \"27_data_entity\", \"is_success\": true, \"error_count\": 3.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": 1.0, \"alt_missing\": null, \"link_empty\": 2.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 27, \"page_type\": \"data_entity\", \"source_id\": \"dc_30\", \"short_name\": \"InterPro\", \"accession\": \"DBC000030\", \"full_name\": \"Protein sequence analysis & classification\", \"description\": \"InterPro provides functional analysis of proteins by classifying them into families and predicting domains and important sites.\", \"founded_year\": 2000, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI), Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SD, UK\", \"city\": \"Cambridge\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"v92\", \"contact_name\": \"Alex Bateman\", \"contact_email\": \"agb@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2022-12-15\", \"age\": 23, \"citation\": 8643, \"token\": \"$2a$10$XXRDMgyw57XbhaL1X2YaweUs0ywar53fpZagpvFzpbDOtCIG8TGd6\", \"expire_date\": 1543898855889.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-08-23 10:41:57\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation, Standard ontology and nomenclature, Literature\", \"keywords_list\": \"protein classification, protein domain, protein function, protein family\", \"data_object_list\": \"Bacteria, Archaea, Animal, Plant, Fungi, Virus\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 375.783, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:03\", \"ess\": 0}, {\"page_id\": \"27_documentation\", \"is_success\": true, \"error_count\": 2.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 2.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 27, \"page_type\": \"documentation\", \"source_id\": \"dc_30\", \"short_name\": \"InterPro\", \"accession\": \"DBC000030\", \"full_name\": \"Protein sequence analysis & classification\", \"description\": \"InterPro provides functional analysis of proteins by classifying them into families and predicting domains and important sites.\", \"founded_year\": 2000, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI), Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SD, UK\", \"city\": \"Cambridge\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"v92\", \"contact_name\": \"Alex Bateman\", \"contact_email\": \"agb@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2022-12-15\", \"age\": 23, \"citation\": 8643, \"token\": \"$2a$10$XXRDMgyw57XbhaL1X2YaweUs0ywar53fpZagpvFzpbDOtCIG8TGd6\", \"expire_date\": 1543898855889.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-08-23 10:41:57\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation, Standard ontology and nomenclature, Literature\", \"keywords_list\": \"protein classification, protein domain, protein function, protein family\", \"data_object_list\": \"Bacteria, Archaea, Animal, Plant, Fungi, Virus\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 375.783, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:03\", \"ess\": 0}, {\"page_id\": \"28_search\", \"is_success\": true, \"error_count\": 2.0, \"label_missing\": null, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 28, \"page_type\": \"search\", \"source_id\": \"dc_31\", \"short_name\": \"CDD\", \"accession\": \"DBC000031\", \"full_name\": \"Conserved Domain Database\", \"description\": \"CDD is a protein annotation resource that consists of a collection of well-annotated multiple sequence alignment models for ancient domains and full-length proteins. CDD content includes NCBI-curated domains, which use 3D-structure information to explicitly define domain boundaries and provide insights into sequence/structure/function relationships, as well as domain models imported from a number of external source databases (Pfam, SMART, COG, PRK, TIGRFAMs).\", \"founded_year\": 2002, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"Room 8N805, 8600 Rockville Pike, Bethesda, MD 20894, USA\", \"city\": \"Bethesda\", \"province\": \"MD\", \"country\": \"United States\", \"current_version\": \"v3.20\", \"contact_name\": \"Aron Marchler-Bauer\", \"contact_email\": \"bauer@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2022-09-20\", \"age\": 21, \"citation\": 7729, \"token\": \"$2a$10$xohVuj0TgShEJjb8arf36uD0BbPPMQuvT3brFGjBp.aA96PeOtv4m\", \"expire_date\": 1543898856440.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-09-06 20:43:28\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation, Phylogeny and homology, Structure\", \"keywords_list\": \"conserved domain, Protein Family Database\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 368.048, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:03\", \"ess\": 0}, {\"page_id\": \"28_search_result\", \"is_success\": true, \"error_count\": 2.0, \"label_missing\": 2.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 28, \"page_type\": \"search_result\", \"source_id\": \"dc_31\", \"short_name\": \"CDD\", \"accession\": \"DBC000031\", \"full_name\": \"Conserved Domain Database\", \"description\": \"CDD is a protein annotation resource that consists of a collection of well-annotated multiple sequence alignment models for ancient domains and full-length proteins. CDD content includes NCBI-curated domains, which use 3D-structure information to explicitly define domain boundaries and provide insights into sequence/structure/function relationships, as well as domain models imported from a number of external source databases (Pfam, SMART, COG, PRK, TIGRFAMs).\", \"founded_year\": 2002, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"Room 8N805, 8600 Rockville Pike, Bethesda, MD 20894, USA\", \"city\": \"Bethesda\", \"province\": \"MD\", \"country\": \"United States\", \"current_version\": \"v3.20\", \"contact_name\": \"Aron Marchler-Bauer\", \"contact_email\": \"bauer@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2022-09-20\", \"age\": 21, \"citation\": 7729, \"token\": \"$2a$10$xohVuj0TgShEJjb8arf36uD0BbPPMQuvT3brFGjBp.aA96PeOtv4m\", \"expire_date\": 1543898856440.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-09-06 20:43:28\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation, Phylogeny and homology, Structure\", \"keywords_list\": \"conserved domain, Protein Family Database\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 368.048, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:03\", \"ess\": 0}, {\"page_id\": \"28_data_entity\", \"is_success\": true, \"error_count\": 4.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 2.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": 1.0, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 28, \"page_type\": \"data_entity\", \"source_id\": \"dc_31\", \"short_name\": \"CDD\", \"accession\": \"DBC000031\", \"full_name\": \"Conserved Domain Database\", \"description\": \"CDD is a protein annotation resource that consists of a collection of well-annotated multiple sequence alignment models for ancient domains and full-length proteins. CDD content includes NCBI-curated domains, which use 3D-structure information to explicitly define domain boundaries and provide insights into sequence/structure/function relationships, as well as domain models imported from a number of external source databases (Pfam, SMART, COG, PRK, TIGRFAMs).\", \"founded_year\": 2002, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"Room 8N805, 8600 Rockville Pike, Bethesda, MD 20894, USA\", \"city\": \"Bethesda\", \"province\": \"MD\", \"country\": \"United States\", \"current_version\": \"v3.20\", \"contact_name\": \"Aron Marchler-Bauer\", \"contact_email\": \"bauer@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2022-09-20\", \"age\": 21, \"citation\": 7729, \"token\": \"$2a$10$xohVuj0TgShEJjb8arf36uD0BbPPMQuvT3brFGjBp.aA96PeOtv4m\", \"expire_date\": 1543898856440.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-09-06 20:43:28\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation, Phylogeny and homology, Structure\", \"keywords_list\": \"conserved domain, Protein Family Database\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 368.048, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:03\", \"ess\": 0}, {\"page_id\": \"28_documentation\", \"is_success\": true, \"error_count\": 305.0, \"label_missing\": 1.0, \"language_missing\": 1.0, \"button_empty\": 1.0, \"alt_missing\": null, \"link_empty\": 4.0, \"aria_menu_broken\": null, \"alt_link_missing\": 4.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": 294.0, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 28, \"page_type\": \"documentation\", \"source_id\": \"dc_31\", \"short_name\": \"CDD\", \"accession\": \"DBC000031\", \"full_name\": \"Conserved Domain Database\", \"description\": \"CDD is a protein annotation resource that consists of a collection of well-annotated multiple sequence alignment models for ancient domains and full-length proteins. CDD content includes NCBI-curated domains, which use 3D-structure information to explicitly define domain boundaries and provide insights into sequence/structure/function relationships, as well as domain models imported from a number of external source databases (Pfam, SMART, COG, PRK, TIGRFAMs).\", \"founded_year\": 2002, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"Room 8N805, 8600 Rockville Pike, Bethesda, MD 20894, USA\", \"city\": \"Bethesda\", \"province\": \"MD\", \"country\": \"United States\", \"current_version\": \"v3.20\", \"contact_name\": \"Aron Marchler-Bauer\", \"contact_email\": \"bauer@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2022-09-20\", \"age\": 21, \"citation\": 7729, \"token\": \"$2a$10$xohVuj0TgShEJjb8arf36uD0BbPPMQuvT3brFGjBp.aA96PeOtv4m\", \"expire_date\": 1543898856440.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-09-06 20:43:28\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation, Phylogeny and homology, Structure\", \"keywords_list\": \"conserved domain, Protein Family Database\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 368.048, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:03\", \"ess\": 0}, {\"page_id\": \"29_search\", \"is_success\": true, \"error_count\": 2.0, \"label_missing\": 2.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 29, \"page_type\": \"search\", \"source_id\": \"dc_296\", \"short_name\": \"DrugBank\", \"accession\": \"DBC000296\", \"full_name\": \"DrugBank - Drug and Drug Target Database\", \"description\": \"DrugBank is a comprehensive online database containing extensive biochemical and pharmacological information about drugs,their mechanisms and their targets.\", \"founded_year\": 2006, \"host_institution\": \"University of Alberta\", \"address\": null, \"city\": \"Edmonton\", \"province\": \"AB\", \"country\": \"Canada\", \"current_version\": \"v5.1.1\", \"contact_name\": \"DrugBank Team\", \"contact_email\": \"info@drugbank.ca\", \"available_protocol\": 1.0, \"last_update\": \"2018-07-03\", \"age\": 17, \"citation\": 6237, \"token\": \"$2a$10$Q1/nA.V/xDJkWQFRWhUSJuDF/9GUMSpiiXs7ksVPeqSSRrrgrAqb.\", \"expire_date\": 1543899035439.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:10:01\", \"data_type_list\": \"Protein\", \"category_list\": \"Health and medicine\", \"keywords_list\": \"biochemical and pharmacological information, drug mechanism, drug target, pathway, drug metabolism, drug repurposing, drug interactions, drug products\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 366.882, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:10\", \"ess\": 0}, {\"page_id\": \"29_search_result\", \"is_success\": true, \"error_count\": 2.0, \"label_missing\": 2.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 29, \"page_type\": \"search_result\", \"source_id\": \"dc_296\", \"short_name\": \"DrugBank\", \"accession\": \"DBC000296\", \"full_name\": \"DrugBank - Drug and Drug Target Database\", \"description\": \"DrugBank is a comprehensive online database containing extensive biochemical and pharmacological information about drugs,their mechanisms and their targets.\", \"founded_year\": 2006, \"host_institution\": \"University of Alberta\", \"address\": null, \"city\": \"Edmonton\", \"province\": \"AB\", \"country\": \"Canada\", \"current_version\": \"v5.1.1\", \"contact_name\": \"DrugBank Team\", \"contact_email\": \"info@drugbank.ca\", \"available_protocol\": 1.0, \"last_update\": \"2018-07-03\", \"age\": 17, \"citation\": 6237, \"token\": \"$2a$10$Q1/nA.V/xDJkWQFRWhUSJuDF/9GUMSpiiXs7ksVPeqSSRrrgrAqb.\", \"expire_date\": 1543899035439.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:10:01\", \"data_type_list\": \"Protein\", \"category_list\": \"Health and medicine\", \"keywords_list\": \"biochemical and pharmacological information, drug mechanism, drug target, pathway, drug metabolism, drug repurposing, drug interactions, drug products\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 366.882, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:10\", \"ess\": 0}, {\"page_id\": \"29_data_entity\", \"is_success\": true, \"error_count\": 13.0, \"label_missing\": 3.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 7.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": 2.0, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 29, \"page_type\": \"data_entity\", \"source_id\": \"dc_296\", \"short_name\": \"DrugBank\", \"accession\": \"DBC000296\", \"full_name\": \"DrugBank - Drug and Drug Target Database\", \"description\": \"DrugBank is a comprehensive online database containing extensive biochemical and pharmacological information about drugs,their mechanisms and their targets.\", \"founded_year\": 2006, \"host_institution\": \"University of Alberta\", \"address\": null, \"city\": \"Edmonton\", \"province\": \"AB\", \"country\": \"Canada\", \"current_version\": \"v5.1.1\", \"contact_name\": \"DrugBank Team\", \"contact_email\": \"info@drugbank.ca\", \"available_protocol\": 1.0, \"last_update\": \"2018-07-03\", \"age\": 17, \"citation\": 6237, \"token\": \"$2a$10$Q1/nA.V/xDJkWQFRWhUSJuDF/9GUMSpiiXs7ksVPeqSSRrrgrAqb.\", \"expire_date\": 1543899035439.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:10:01\", \"data_type_list\": \"Protein\", \"category_list\": \"Health and medicine\", \"keywords_list\": \"biochemical and pharmacological information, drug mechanism, drug target, pathway, drug metabolism, drug repurposing, drug interactions, drug products\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 366.882, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:10\", \"ess\": 0}, {\"page_id\": \"29_documentation\", \"is_success\": true, \"error_count\": 12.0, \"label_missing\": 2.0, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": 9.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 29, \"page_type\": \"documentation\", \"source_id\": \"dc_296\", \"short_name\": \"DrugBank\", \"accession\": \"DBC000296\", \"full_name\": \"DrugBank - Drug and Drug Target Database\", \"description\": \"DrugBank is a comprehensive online database containing extensive biochemical and pharmacological information about drugs,their mechanisms and their targets.\", \"founded_year\": 2006, \"host_institution\": \"University of Alberta\", \"address\": null, \"city\": \"Edmonton\", \"province\": \"AB\", \"country\": \"Canada\", \"current_version\": \"v5.1.1\", \"contact_name\": \"DrugBank Team\", \"contact_email\": \"info@drugbank.ca\", \"available_protocol\": 1.0, \"last_update\": \"2018-07-03\", \"age\": 17, \"citation\": 6237, \"token\": \"$2a$10$Q1/nA.V/xDJkWQFRWhUSJuDF/9GUMSpiiXs7ksVPeqSSRrrgrAqb.\", \"expire_date\": 1543899035439.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:10:01\", \"data_type_list\": \"Protein\", \"category_list\": \"Health and medicine\", \"keywords_list\": \"biochemical and pharmacological information, drug mechanism, drug target, pathway, drug metabolism, drug repurposing, drug interactions, drug products\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 366.882, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:10\", \"ess\": 0}, {\"page_id\": \"30_search_result\", \"is_success\": true, \"error_count\": 13.0, \"label_missing\": 6.0, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 3.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": 3.0, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 30, \"page_type\": \"search_result\", \"source_id\": \"dc_111\", \"short_name\": \"COSMIC\", \"accession\": \"DBC000111\", \"full_name\": \"Catalogue of Somatic Mutations in Cancer\", \"description\": \"COSMIC explores the world's knowledge of somatic mutations in human cancer.\", \"founded_year\": 2004, \"host_institution\": \"Wellcome Sanger Institute\", \"address\": \"Hinxton,Cambridge,UK,CB10 1SA\", \"city\": \"Cambridge\", \"province\": null, \"country\": \"United Kingdom\", \"current_version\": \"v80.0\", \"contact_name\": \"Simon A. Forbes\", \"contact_email\": \"saf@sanger.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2017-02-13\", \"age\": 19, \"citation\": 6827, \"token\": \"$2a$10$YV0UmxomkHVLEBwnfCg5A./VPad00LzCIyG6A2d4kEw2OVY0aLtJq\", \"expire_date\": 1543898917337.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:45\", \"data_type_list\": \"DNA\", \"category_list\": \"Health and medicine, Genotype phenotype and variation\", \"keywords_list\": \"cancer, somatic mutation\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 359.316, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:05\", \"ess\": 0}, {\"page_id\": \"30_data_entity\", \"is_success\": true, \"error_count\": 13.0, \"label_missing\": 6.0, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 3.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": 3.0, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 30, \"page_type\": \"data_entity\", \"source_id\": \"dc_111\", \"short_name\": \"COSMIC\", \"accession\": \"DBC000111\", \"full_name\": \"Catalogue of Somatic Mutations in Cancer\", \"description\": \"COSMIC explores the world's knowledge of somatic mutations in human cancer.\", \"founded_year\": 2004, \"host_institution\": \"Wellcome Sanger Institute\", \"address\": \"Hinxton,Cambridge,UK,CB10 1SA\", \"city\": \"Cambridge\", \"province\": null, \"country\": \"United Kingdom\", \"current_version\": \"v80.0\", \"contact_name\": \"Simon A. Forbes\", \"contact_email\": \"saf@sanger.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2017-02-13\", \"age\": 19, \"citation\": 6827, \"token\": \"$2a$10$YV0UmxomkHVLEBwnfCg5A./VPad00LzCIyG6A2d4kEw2OVY0aLtJq\", \"expire_date\": 1543898917337.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:45\", \"data_type_list\": \"DNA\", \"category_list\": \"Health and medicine, Genotype phenotype and variation\", \"keywords_list\": \"cancer, somatic mutation\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 359.316, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:05\", \"ess\": 0}, {\"page_id\": \"30_documentation\", \"is_success\": true, \"error_count\": 16.0, \"label_missing\": 6.0, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 3.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": 6.0, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 30, \"page_type\": \"documentation\", \"source_id\": \"dc_111\", \"short_name\": \"COSMIC\", \"accession\": \"DBC000111\", \"full_name\": \"Catalogue of Somatic Mutations in Cancer\", \"description\": \"COSMIC explores the world's knowledge of somatic mutations in human cancer.\", \"founded_year\": 2004, \"host_institution\": \"Wellcome Sanger Institute\", \"address\": \"Hinxton,Cambridge,UK,CB10 1SA\", \"city\": \"Cambridge\", \"province\": null, \"country\": \"United Kingdom\", \"current_version\": \"v80.0\", \"contact_name\": \"Simon A. Forbes\", \"contact_email\": \"saf@sanger.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2017-02-13\", \"age\": 19, \"citation\": 6827, \"token\": \"$2a$10$YV0UmxomkHVLEBwnfCg5A./VPad00LzCIyG6A2d4kEw2OVY0aLtJq\", \"expire_date\": 1543898917337.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:45\", \"data_type_list\": \"DNA\", \"category_list\": \"Health and medicine, Genotype phenotype and variation\", \"keywords_list\": \"cancer, somatic mutation\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 359.316, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:05\", \"ess\": 0}, {\"page_id\": \"31_search\", \"is_success\": true, \"error_count\": 23.0, \"label_missing\": 2.0, \"language_missing\": null, \"button_empty\": 5.0, \"alt_missing\": null, \"link_empty\": 16.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 31, \"page_type\": \"search\", \"source_id\": \"dc_6779\", \"short_name\": \"GTDB\", \"accession\": \"DBC006779\", \"full_name\": \"The Genome Taxonomy Database\", \"description\": \"GTDB is an initiative to establish a standardised microbial taxonomy based on genome phylogeny.\", \"founded_year\": 2018, \"host_institution\": \"University of Queensland\", \"address\": \"Australian Centre for Ecogenomics, School of Chemistry and Molecular Biosciences, University of Queensland, Queensland, Australia.\", \"city\": \"Queensland\", \"province\": null, \"country\": \"Australia\", \"current_version\": \"Release 04-RS89\", \"contact_name\": \"Philip Hugenholtz\", \"contact_email\": \"p.hugenholtz@uq.edu.au\", \"available_protocol\": 1.0, \"last_update\": \"2019-6-19\", \"age\": 5, \"citation\": 1730, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1572149519000, \"last_modified\": \"2023-03-03 10:16:47\", \"data_type_list\": \"DNA\", \"category_list\": \"Phylogeny and homology, Standard ontology and nomenclature\", \"keywords_list\": \"multiple sequence alignment, species cluster, prokaryote nomenclature, genome-based taxonomy\", \"data_object_list\": \"Bacteria, Archaea\", \"organism_list\": null, \"theme_list\": null, \"zindex\": 346.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:35\", \"ess\": 0}, {\"page_id\": \"31_search_result\", \"is_success\": true, \"error_count\": 7.0, \"label_missing\": 2.0, \"language_missing\": null, \"button_empty\": 3.0, \"alt_missing\": null, \"link_empty\": 2.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 31, \"page_type\": \"search_result\", \"source_id\": \"dc_6779\", \"short_name\": \"GTDB\", \"accession\": \"DBC006779\", \"full_name\": \"The Genome Taxonomy Database\", \"description\": \"GTDB is an initiative to establish a standardised microbial taxonomy based on genome phylogeny.\", \"founded_year\": 2018, \"host_institution\": \"University of Queensland\", \"address\": \"Australian Centre for Ecogenomics, School of Chemistry and Molecular Biosciences, University of Queensland, Queensland, Australia.\", \"city\": \"Queensland\", \"province\": null, \"country\": \"Australia\", \"current_version\": \"Release 04-RS89\", \"contact_name\": \"Philip Hugenholtz\", \"contact_email\": \"p.hugenholtz@uq.edu.au\", \"available_protocol\": 1.0, \"last_update\": \"2019-6-19\", \"age\": 5, \"citation\": 1730, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1572149519000, \"last_modified\": \"2023-03-03 10:16:47\", \"data_type_list\": \"DNA\", \"category_list\": \"Phylogeny and homology, Standard ontology and nomenclature\", \"keywords_list\": \"multiple sequence alignment, species cluster, prokaryote nomenclature, genome-based taxonomy\", \"data_object_list\": \"Bacteria, Archaea\", \"organism_list\": null, \"theme_list\": null, \"zindex\": 346.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:35\", \"ess\": 0}, {\"page_id\": \"31_data_entity\", \"is_success\": true, \"error_count\": 7.0, \"label_missing\": 2.0, \"language_missing\": null, \"button_empty\": 3.0, \"alt_missing\": null, \"link_empty\": 2.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 31, \"page_type\": \"data_entity\", \"source_id\": \"dc_6779\", \"short_name\": \"GTDB\", \"accession\": \"DBC006779\", \"full_name\": \"The Genome Taxonomy Database\", \"description\": \"GTDB is an initiative to establish a standardised microbial taxonomy based on genome phylogeny.\", \"founded_year\": 2018, \"host_institution\": \"University of Queensland\", \"address\": \"Australian Centre for Ecogenomics, School of Chemistry and Molecular Biosciences, University of Queensland, Queensland, Australia.\", \"city\": \"Queensland\", \"province\": null, \"country\": \"Australia\", \"current_version\": \"Release 04-RS89\", \"contact_name\": \"Philip Hugenholtz\", \"contact_email\": \"p.hugenholtz@uq.edu.au\", \"available_protocol\": 1.0, \"last_update\": \"2019-6-19\", \"age\": 5, \"citation\": 1730, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1572149519000, \"last_modified\": \"2023-03-03 10:16:47\", \"data_type_list\": \"DNA\", \"category_list\": \"Phylogeny and homology, Standard ontology and nomenclature\", \"keywords_list\": \"multiple sequence alignment, species cluster, prokaryote nomenclature, genome-based taxonomy\", \"data_object_list\": \"Bacteria, Archaea\", \"organism_list\": null, \"theme_list\": null, \"zindex\": 346.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:35\", \"ess\": 0}, {\"page_id\": \"31_documentation\", \"is_success\": true, \"error_count\": 8.0, \"label_missing\": 2.0, \"language_missing\": null, \"button_empty\": 4.0, \"alt_missing\": null, \"link_empty\": 2.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 31, \"page_type\": \"documentation\", \"source_id\": \"dc_6779\", \"short_name\": \"GTDB\", \"accession\": \"DBC006779\", \"full_name\": \"The Genome Taxonomy Database\", \"description\": \"GTDB is an initiative to establish a standardised microbial taxonomy based on genome phylogeny.\", \"founded_year\": 2018, \"host_institution\": \"University of Queensland\", \"address\": \"Australian Centre for Ecogenomics, School of Chemistry and Molecular Biosciences, University of Queensland, Queensland, Australia.\", \"city\": \"Queensland\", \"province\": null, \"country\": \"Australia\", \"current_version\": \"Release 04-RS89\", \"contact_name\": \"Philip Hugenholtz\", \"contact_email\": \"p.hugenholtz@uq.edu.au\", \"available_protocol\": 1.0, \"last_update\": \"2019-6-19\", \"age\": 5, \"citation\": 1730, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1572149519000, \"last_modified\": \"2023-03-03 10:16:47\", \"data_type_list\": \"DNA\", \"category_list\": \"Phylogeny and homology, Standard ontology and nomenclature\", \"keywords_list\": \"multiple sequence alignment, species cluster, prokaryote nomenclature, genome-based taxonomy\", \"data_object_list\": \"Bacteria, Archaea\", \"organism_list\": null, \"theme_list\": null, \"zindex\": 346.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:35\", \"ess\": 0}, {\"page_id\": \"32_search_result\", \"is_success\": true, \"error_count\": 0.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 32, \"page_type\": \"search_result\", \"source_id\": \"dc_5935\", \"short_name\": \"PANTHER\", \"accession\": \"DBC005935\", \"full_name\": \"Protein ANalysis THrough Evolutionary Relationships\", \"description\": \"The PANTHER (Protein ANalysis THrough Evolutionary Relationships) Classification System was designed to classify proteins (and their genes) in order to facilitate high-throughput analysis.\", \"founded_year\": 2003, \"host_institution\": \"University of Southern California\", \"address\": \"Computational Biology, Applied Biosystems, 850 Lincoln Center Drive, Foster City, CA 94404, USA.\", \"city\": \"Foster City\", \"province\": null, \"country\": \"United States\", \"current_version\": \"v14\", \"contact_name\": \"Paul D. Thomas\", \"contact_email\": \"paul.thomas@appliedbiosystems.com\", \"available_protocol\": 1.0, \"last_update\": \"2021\", \"age\": 20, \"citation\": 6895, \"token\": \"$2a$10$lIm/OeUCy6/fPydtIhGUuuru/uUq8.Kym.we2W2DgNZC/OAjXa0MW\", \"expire_date\": 1544267110867.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1518205751000, \"last_modified\": \"2023-03-03 10:15:51\", \"data_type_list\": \"Protein\", \"category_list\": \"Phylogeny and homology\", \"keywords_list\": \"protein families, Evolutionary Relationships, high-throughput analysis\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Virus\", \"organism_list\": \"Homo sapiens, Zea mays, Mus musculus\", \"theme_list\": null, \"zindex\": 344.75, \"first_publication_year\": 2005, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:18\", \"ess\": 1}, {\"page_id\": \"32_data_entity\", \"is_success\": true, \"error_count\": 0.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 32, \"page_type\": \"data_entity\", \"source_id\": \"dc_5935\", \"short_name\": \"PANTHER\", \"accession\": \"DBC005935\", \"full_name\": \"Protein ANalysis THrough Evolutionary Relationships\", \"description\": \"The PANTHER (Protein ANalysis THrough Evolutionary Relationships) Classification System was designed to classify proteins (and their genes) in order to facilitate high-throughput analysis.\", \"founded_year\": 2003, \"host_institution\": \"University of Southern California\", \"address\": \"Computational Biology, Applied Biosystems, 850 Lincoln Center Drive, Foster City, CA 94404, USA.\", \"city\": \"Foster City\", \"province\": null, \"country\": \"United States\", \"current_version\": \"v14\", \"contact_name\": \"Paul D. Thomas\", \"contact_email\": \"paul.thomas@appliedbiosystems.com\", \"available_protocol\": 1.0, \"last_update\": \"2021\", \"age\": 20, \"citation\": 6895, \"token\": \"$2a$10$lIm/OeUCy6/fPydtIhGUuuru/uUq8.Kym.we2W2DgNZC/OAjXa0MW\", \"expire_date\": 1544267110867.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1518205751000, \"last_modified\": \"2023-03-03 10:15:51\", \"data_type_list\": \"Protein\", \"category_list\": \"Phylogeny and homology\", \"keywords_list\": \"protein families, Evolutionary Relationships, high-throughput analysis\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Virus\", \"organism_list\": \"Homo sapiens, Zea mays, Mus musculus\", \"theme_list\": null, \"zindex\": 344.75, \"first_publication_year\": 2005, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:18\", \"ess\": 1}, {\"page_id\": \"32_documentation\", \"is_success\": true, \"error_count\": 26.0, \"label_missing\": 1.0, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 4.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": 19.0, \"title_invalid\": 1.0, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 32, \"page_type\": \"documentation\", \"source_id\": \"dc_5935\", \"short_name\": \"PANTHER\", \"accession\": \"DBC005935\", \"full_name\": \"Protein ANalysis THrough Evolutionary Relationships\", \"description\": \"The PANTHER (Protein ANalysis THrough Evolutionary Relationships) Classification System was designed to classify proteins (and their genes) in order to facilitate high-throughput analysis.\", \"founded_year\": 2003, \"host_institution\": \"University of Southern California\", \"address\": \"Computational Biology, Applied Biosystems, 850 Lincoln Center Drive, Foster City, CA 94404, USA.\", \"city\": \"Foster City\", \"province\": null, \"country\": \"United States\", \"current_version\": \"v14\", \"contact_name\": \"Paul D. Thomas\", \"contact_email\": \"paul.thomas@appliedbiosystems.com\", \"available_protocol\": 1.0, \"last_update\": \"2021\", \"age\": 20, \"citation\": 6895, \"token\": \"$2a$10$lIm/OeUCy6/fPydtIhGUuuru/uUq8.Kym.we2W2DgNZC/OAjXa0MW\", \"expire_date\": 1544267110867.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1518205751000, \"last_modified\": \"2023-03-03 10:15:51\", \"data_type_list\": \"Protein\", \"category_list\": \"Phylogeny and homology\", \"keywords_list\": \"protein families, Evolutionary Relationships, high-throughput analysis\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Virus\", \"organism_list\": \"Homo sapiens, Zea mays, Mus musculus\", \"theme_list\": null, \"zindex\": 344.75, \"first_publication_year\": 2005, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:18\", \"ess\": 1}, {\"page_id\": \"34_search_result\", \"is_success\": true, \"error_count\": 3.0, \"label_missing\": 3.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 34, \"page_type\": \"search_result\", \"source_id\": \"dc_256\", \"short_name\": \"RefSeq\", \"accession\": \"DBC000256\", \"full_name\": \"reference sequence database\", \"description\": \"The National Center for Biotechnology Information (NCBI) Reference Sequence (RefSeq) database is a collection of annotated genomic, transcript and protein sequence records derived from data in public sequence archives and from computation, curation and collaboration.\", \"founded_year\": 2000, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, 45 Center Drive, Bethesda, MD 20892-6511, USA.\", \"city\": \"Bethesda\", \"province\": \"MD\", \"country\": \"United States\", \"current_version\": \"Release 220\", \"contact_name\": \"Fran\\u00e7oise Thibaud-Nissen\", \"contact_email\": \"thibaudf@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2023-09-11\", \"age\": 23, \"citation\": 7604, \"token\": \"$2a$10$XsipWUIFeVIkhvykvE1NTOOXELtt1/iG2RHROIHopsg16DGM1iRV2\", \"expire_date\": 1543899010604.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-09-13 22:42:36\", \"data_type_list\": \"DNA, Protein, RNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"reference sequence, protein-coding gene, pseudogene, lncRNA\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": \"lncRNA\", \"zindex\": 330.609, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:09\", \"ess\": 0}, {\"page_id\": \"34_data_entity\", \"is_success\": true, \"error_count\": 3.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": 2.0, \"label_multiple\": 1.0, \"id\": 34, \"page_type\": \"data_entity\", \"source_id\": \"dc_256\", \"short_name\": \"RefSeq\", \"accession\": \"DBC000256\", \"full_name\": \"reference sequence database\", \"description\": \"The National Center for Biotechnology Information (NCBI) Reference Sequence (RefSeq) database is a collection of annotated genomic, transcript and protein sequence records derived from data in public sequence archives and from computation, curation and collaboration.\", \"founded_year\": 2000, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, 45 Center Drive, Bethesda, MD 20892-6511, USA.\", \"city\": \"Bethesda\", \"province\": \"MD\", \"country\": \"United States\", \"current_version\": \"Release 220\", \"contact_name\": \"Fran\\u00e7oise Thibaud-Nissen\", \"contact_email\": \"thibaudf@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2023-09-11\", \"age\": 23, \"citation\": 7604, \"token\": \"$2a$10$XsipWUIFeVIkhvykvE1NTOOXELtt1/iG2RHROIHopsg16DGM1iRV2\", \"expire_date\": 1543899010604.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-09-13 22:42:36\", \"data_type_list\": \"DNA, Protein, RNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"reference sequence, protein-coding gene, pseudogene, lncRNA\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": \"lncRNA\", \"zindex\": 330.609, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:09\", \"ess\": 0}, {\"page_id\": \"34_documentation\", \"is_success\": true, \"error_count\": 2.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": 2.0, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 34, \"page_type\": \"documentation\", \"source_id\": \"dc_256\", \"short_name\": \"RefSeq\", \"accession\": \"DBC000256\", \"full_name\": \"reference sequence database\", \"description\": \"The National Center for Biotechnology Information (NCBI) Reference Sequence (RefSeq) database is a collection of annotated genomic, transcript and protein sequence records derived from data in public sequence archives and from computation, curation and collaboration.\", \"founded_year\": 2000, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, 45 Center Drive, Bethesda, MD 20892-6511, USA.\", \"city\": \"Bethesda\", \"province\": \"MD\", \"country\": \"United States\", \"current_version\": \"Release 220\", \"contact_name\": \"Fran\\u00e7oise Thibaud-Nissen\", \"contact_email\": \"thibaudf@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2023-09-11\", \"age\": 23, \"citation\": 7604, \"token\": \"$2a$10$XsipWUIFeVIkhvykvE1NTOOXELtt1/iG2RHROIHopsg16DGM1iRV2\", \"expire_date\": 1543899010604.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-09-13 22:42:36\", \"data_type_list\": \"DNA, Protein, RNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"reference sequence, protein-coding gene, pseudogene, lncRNA\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": \"lncRNA\", \"zindex\": 330.609, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:09\", \"ess\": 0}, {\"page_id\": \"35_search_result\", \"is_success\": true, \"error_count\": 2.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 35, \"page_type\": \"search_result\", \"source_id\": \"dc_7367\", \"short_name\": \"NCBI\", \"accession\": \"DBC007367\", \"full_name\": \"National Center for Biotechnology Information\", \"description\": \"The National Center for Biotechnology Information (NCBI) provides a large suite of online resources for biological information and data, including the GenBank\\u00ae nucleic acid sequence database and the PubMed\\u00ae database of citations and abstracts published in life science journals. The Entrez system provides search and retrieval operations for most of these data from 34 distinct databases. The E-utilities serve as the programming interface for the Entrez system. Custom implementations of the BLAST program provide sequence-based searching of many specialized datasets. New resources released in the past year include a new PubMed interface and NCBI datasets. Additional resources that were updated in the past year include PMC, Bookshelf, Genome Data Viewer, SRA, ClinVar, dbSNP, dbVar, Pathogen Detection, BLAST, Primer-BLAST, IgBLAST, iCn3D and PubChem. \", \"founded_year\": 2000, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Building 38A, 8600 Rockville Pike, Bethesda, MD 20894, USA.\", \"city\": \"Bethesda\", \"province\": null, \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Eric W Sayers\", \"contact_email\": \"sayers@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2020-10-23\", \"age\": 23, \"citation\": 7360, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1605149733000, \"last_modified\": \"2023-04-22 15:46:42\", \"data_type_list\": \"DNA, RNA, Protein\", \"category_list\": \"Raw bio-data, Gene genome and annotation, Genotype phenotype and variation, Literature, Metadata\", \"keywords_list\": \"deposition, integration, Big data\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Virus\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 320.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:47\", \"ess\": 0}, {\"page_id\": \"35_data_entity\", \"is_success\": true, \"error_count\": 5.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": 1.0, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": 3.0, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 35, \"page_type\": \"data_entity\", \"source_id\": \"dc_7367\", \"short_name\": \"NCBI\", \"accession\": \"DBC007367\", \"full_name\": \"National Center for Biotechnology Information\", \"description\": \"The National Center for Biotechnology Information (NCBI) provides a large suite of online resources for biological information and data, including the GenBank\\u00ae nucleic acid sequence database and the PubMed\\u00ae database of citations and abstracts published in life science journals. The Entrez system provides search and retrieval operations for most of these data from 34 distinct databases. The E-utilities serve as the programming interface for the Entrez system. Custom implementations of the BLAST program provide sequence-based searching of many specialized datasets. New resources released in the past year include a new PubMed interface and NCBI datasets. Additional resources that were updated in the past year include PMC, Bookshelf, Genome Data Viewer, SRA, ClinVar, dbSNP, dbVar, Pathogen Detection, BLAST, Primer-BLAST, IgBLAST, iCn3D and PubChem. \", \"founded_year\": 2000, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Building 38A, 8600 Rockville Pike, Bethesda, MD 20894, USA.\", \"city\": \"Bethesda\", \"province\": null, \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Eric W Sayers\", \"contact_email\": \"sayers@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2020-10-23\", \"age\": 23, \"citation\": 7360, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1605149733000, \"last_modified\": \"2023-04-22 15:46:42\", \"data_type_list\": \"DNA, RNA, Protein\", \"category_list\": \"Raw bio-data, Gene genome and annotation, Genotype phenotype and variation, Literature, Metadata\", \"keywords_list\": \"deposition, integration, Big data\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Virus\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 320.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:47\", \"ess\": 0}, {\"page_id\": \"35_documentation\", \"is_success\": true, \"error_count\": 2.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": 2.0, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 35, \"page_type\": \"documentation\", \"source_id\": \"dc_7367\", \"short_name\": \"NCBI\", \"accession\": \"DBC007367\", \"full_name\": \"National Center for Biotechnology Information\", \"description\": \"The National Center for Biotechnology Information (NCBI) provides a large suite of online resources for biological information and data, including the GenBank\\u00ae nucleic acid sequence database and the PubMed\\u00ae database of citations and abstracts published in life science journals. The Entrez system provides search and retrieval operations for most of these data from 34 distinct databases. The E-utilities serve as the programming interface for the Entrez system. Custom implementations of the BLAST program provide sequence-based searching of many specialized datasets. New resources released in the past year include a new PubMed interface and NCBI datasets. Additional resources that were updated in the past year include PMC, Bookshelf, Genome Data Viewer, SRA, ClinVar, dbSNP, dbVar, Pathogen Detection, BLAST, Primer-BLAST, IgBLAST, iCn3D and PubChem. \", \"founded_year\": 2000, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Building 38A, 8600 Rockville Pike, Bethesda, MD 20894, USA.\", \"city\": \"Bethesda\", \"province\": null, \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Eric W Sayers\", \"contact_email\": \"sayers@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2020-10-23\", \"age\": 23, \"citation\": 7360, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1605149733000, \"last_modified\": \"2023-04-22 15:46:42\", \"data_type_list\": \"DNA, RNA, Protein\", \"category_list\": \"Raw bio-data, Gene genome and annotation, Genotype phenotype and variation, Literature, Metadata\", \"keywords_list\": \"deposition, integration, Big data\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Virus\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 320.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:47\", \"ess\": 0}, {\"page_id\": \"36_search_result\", \"is_success\": true, \"error_count\": 10.0, \"label_missing\": 2.0, \"language_missing\": null, \"button_empty\": 1.0, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": 4.0, \"alt_link_missing\": 2.0, \"th_empty\": 1.0, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 36, \"page_type\": \"search_result\", \"source_id\": \"dc_1334\", \"short_name\": \"SWISS-MODEL\", \"accession\": \"DBC001334\", \"full_name\": \"The SWISS-MODEL Repository\", \"description\": \"The SWISS-MODEL Repository is a database of annotated 3D protein structure models generated by the SWISS-MODEL homology-modelling pipeline.\", \"founded_year\": 2004, \"host_institution\": \"Swiss Institute of Bioinformatics\", \"address\": \"Biozentrum, University of Basel and SIB Swiss Institute of Bioinformatics, Basel, Switzerland\", \"city\": \"Basel\", \"province\": null, \"country\": \"Switzerland\", \"current_version\": \"v1.0\", \"contact_name\": \"Torsten Schwede\", \"contact_email\": \"Torsten.Schwede@unibas.ch\", \"available_protocol\": 1.0, \"last_update\": \"2017-01-14\", \"age\": 19, \"citation\": 6042, \"token\": \"$2a$10$UAu.zDDeFOJQiNxBo3oZEurwt13xY65s4PmjJk/XV9kNFhuaqGSC2\", \"expire_date\": 1543993242762.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1437614368000, \"last_modified\": \"2023-03-03 10:11:18\", \"data_type_list\": \"Protein\", \"category_list\": \"Structure\", \"keywords_list\": \"3D protein structure model\", \"data_object_list\": null, \"organism_list\": null, \"theme_list\": null, \"zindex\": 318.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:37\", \"ess\": 0}, {\"page_id\": \"36_data_entity\", \"is_success\": true, \"error_count\": 8.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": 7.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 36, \"page_type\": \"data_entity\", \"source_id\": \"dc_1334\", \"short_name\": \"SWISS-MODEL\", \"accession\": \"DBC001334\", \"full_name\": \"The SWISS-MODEL Repository\", \"description\": \"The SWISS-MODEL Repository is a database of annotated 3D protein structure models generated by the SWISS-MODEL homology-modelling pipeline.\", \"founded_year\": 2004, \"host_institution\": \"Swiss Institute of Bioinformatics\", \"address\": \"Biozentrum, University of Basel and SIB Swiss Institute of Bioinformatics, Basel, Switzerland\", \"city\": \"Basel\", \"province\": null, \"country\": \"Switzerland\", \"current_version\": \"v1.0\", \"contact_name\": \"Torsten Schwede\", \"contact_email\": \"Torsten.Schwede@unibas.ch\", \"available_protocol\": 1.0, \"last_update\": \"2017-01-14\", \"age\": 19, \"citation\": 6042, \"token\": \"$2a$10$UAu.zDDeFOJQiNxBo3oZEurwt13xY65s4PmjJk/XV9kNFhuaqGSC2\", \"expire_date\": 1543993242762.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1437614368000, \"last_modified\": \"2023-03-03 10:11:18\", \"data_type_list\": \"Protein\", \"category_list\": \"Structure\", \"keywords_list\": \"3D protein structure model\", \"data_object_list\": null, \"organism_list\": null, \"theme_list\": null, \"zindex\": 318.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:37\", \"ess\": 0}, {\"page_id\": \"36_documentation\", \"is_success\": true, \"error_count\": 6.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 1.0, \"link_empty\": null, \"aria_menu_broken\": 4.0, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 36, \"page_type\": \"documentation\", \"source_id\": \"dc_1334\", \"short_name\": \"SWISS-MODEL\", \"accession\": \"DBC001334\", \"full_name\": \"The SWISS-MODEL Repository\", \"description\": \"The SWISS-MODEL Repository is a database of annotated 3D protein structure models generated by the SWISS-MODEL homology-modelling pipeline.\", \"founded_year\": 2004, \"host_institution\": \"Swiss Institute of Bioinformatics\", \"address\": \"Biozentrum, University of Basel and SIB Swiss Institute of Bioinformatics, Basel, Switzerland\", \"city\": \"Basel\", \"province\": null, \"country\": \"Switzerland\", \"current_version\": \"v1.0\", \"contact_name\": \"Torsten Schwede\", \"contact_email\": \"Torsten.Schwede@unibas.ch\", \"available_protocol\": 1.0, \"last_update\": \"2017-01-14\", \"age\": 19, \"citation\": 6042, \"token\": \"$2a$10$UAu.zDDeFOJQiNxBo3oZEurwt13xY65s4PmjJk/XV9kNFhuaqGSC2\", \"expire_date\": 1543993242762.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1437614368000, \"last_modified\": \"2023-03-03 10:11:18\", \"data_type_list\": \"Protein\", \"category_list\": \"Structure\", \"keywords_list\": \"3D protein structure model\", \"data_object_list\": null, \"organism_list\": null, \"theme_list\": null, \"zindex\": 318.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:37\", \"ess\": 0}, {\"page_id\": \"37_search_result\", \"is_success\": true, \"error_count\": 33.0, \"label_missing\": 1.0, \"language_missing\": 1.0, \"button_empty\": 3.0, \"alt_missing\": 6.0, \"link_empty\": 7.0, \"aria_menu_broken\": 2.0, \"alt_link_missing\": 5.0, \"th_empty\": 1.0, \"heading_empty\": 4.0, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": 3.0, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 37, \"page_type\": \"search_result\", \"source_id\": \"dc_4842\", \"short_name\": \"EzTaxon-e\", \"accession\": \"DBC004842\", \"full_name\": \"EzBioCloud Database\", \"description\": \"EzTaxon-e database provides a useful taxonomic backbone for the identification of cultured and uncultured prokaryotes and offers a valuable means of communication among microbiologists who routinely encounter taxonomically novel isolates.\", \"founded_year\": 2012, \"host_institution\": \"Seoul National University\", \"address\": \"Interdisciplinary Program in Bioinformatics, Seoul National University, 56-1 Shillim-dong, Kwanak-gu, Seoul 151-742, Republic of Korea\\n\", \"city\": \"Seoul\", \"province\": null, \"country\": \"Korea, Republic of\", \"current_version\": null, \"contact_name\": \"Jongsik Chun\", \"contact_email\": \"jchun@snu.ac.kr\", \"available_protocol\": 1.0, \"last_update\": \"2017\", \"age\": 11, \"citation\": 3485, \"token\": \"$2a$10$xXFf9.vKRqWpOA6QceOIMeV1D1w2Q5Cqon46L2xx9yrwilOHoow0C\", \"expire_date\": 1544146779481.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1517160985000, \"last_modified\": \"2023-03-03 10:14:35\", \"data_type_list\": \"DNA, RNA\", \"category_list\": \"Phylogeny and homology\", \"keywords_list\": \"16\\u2009S rRNA\", \"data_object_list\": \"Bacteria\", \"organism_list\": null, \"theme_list\": null, \"zindex\": 316.818, \"first_publication_year\": 2012, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:01:51\", \"ess\": 1}, {\"page_id\": \"37_data_entity\", \"is_success\": true, \"error_count\": 27.0, \"label_missing\": null, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": 5.0, \"link_empty\": 6.0, \"aria_menu_broken\": 2.0, \"alt_link_missing\": 7.0, \"th_empty\": null, \"heading_empty\": 3.0, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": 3.0, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 37, \"page_type\": \"data_entity\", \"source_id\": \"dc_4842\", \"short_name\": \"EzTaxon-e\", \"accession\": \"DBC004842\", \"full_name\": \"EzBioCloud Database\", \"description\": \"EzTaxon-e database provides a useful taxonomic backbone for the identification of cultured and uncultured prokaryotes and offers a valuable means of communication among microbiologists who routinely encounter taxonomically novel isolates.\", \"founded_year\": 2012, \"host_institution\": \"Seoul National University\", \"address\": \"Interdisciplinary Program in Bioinformatics, Seoul National University, 56-1 Shillim-dong, Kwanak-gu, Seoul 151-742, Republic of Korea\\n\", \"city\": \"Seoul\", \"province\": null, \"country\": \"Korea, Republic of\", \"current_version\": null, \"contact_name\": \"Jongsik Chun\", \"contact_email\": \"jchun@snu.ac.kr\", \"available_protocol\": 1.0, \"last_update\": \"2017\", \"age\": 11, \"citation\": 3485, \"token\": \"$2a$10$xXFf9.vKRqWpOA6QceOIMeV1D1w2Q5Cqon46L2xx9yrwilOHoow0C\", \"expire_date\": 1544146779481.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1517160985000, \"last_modified\": \"2023-03-03 10:14:35\", \"data_type_list\": \"DNA, RNA\", \"category_list\": \"Phylogeny and homology\", \"keywords_list\": \"16\\u2009S rRNA\", \"data_object_list\": \"Bacteria\", \"organism_list\": null, \"theme_list\": null, \"zindex\": 316.818, \"first_publication_year\": 2012, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:01:51\", \"ess\": 1}, {\"page_id\": \"37_documentation\", \"is_success\": true, \"error_count\": 5.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": 2.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": 2.0, \"id\": 37, \"page_type\": \"documentation\", \"source_id\": \"dc_4842\", \"short_name\": \"EzTaxon-e\", \"accession\": \"DBC004842\", \"full_name\": \"EzBioCloud Database\", \"description\": \"EzTaxon-e database provides a useful taxonomic backbone for the identification of cultured and uncultured prokaryotes and offers a valuable means of communication among microbiologists who routinely encounter taxonomically novel isolates.\", \"founded_year\": 2012, \"host_institution\": \"Seoul National University\", \"address\": \"Interdisciplinary Program in Bioinformatics, Seoul National University, 56-1 Shillim-dong, Kwanak-gu, Seoul 151-742, Republic of Korea\\n\", \"city\": \"Seoul\", \"province\": null, \"country\": \"Korea, Republic of\", \"current_version\": null, \"contact_name\": \"Jongsik Chun\", \"contact_email\": \"jchun@snu.ac.kr\", \"available_protocol\": 1.0, \"last_update\": \"2017\", \"age\": 11, \"citation\": 3485, \"token\": \"$2a$10$xXFf9.vKRqWpOA6QceOIMeV1D1w2Q5Cqon46L2xx9yrwilOHoow0C\", \"expire_date\": 1544146779481.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1517160985000, \"last_modified\": \"2023-03-03 10:14:35\", \"data_type_list\": \"DNA, RNA\", \"category_list\": \"Phylogeny and homology\", \"keywords_list\": \"16\\u2009S rRNA\", \"data_object_list\": \"Bacteria\", \"organism_list\": null, \"theme_list\": null, \"zindex\": 316.818, \"first_publication_year\": 2012, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:01:51\", \"ess\": 1}, {\"page_id\": \"38_search_result\", \"is_success\": true, \"error_count\": 2.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": 1.0, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 38, \"page_type\": \"search_result\", \"source_id\": \"dc_672\", \"short_name\": \"FANTOM\", \"accession\": \"DBC000672\", \"full_name\": \"functional annotation of the mammalian genome\", \"description\": \"FANTOM1-3 focused on identifying the transcribed components of mammalian cells. In FANTOM4, the focus has changed to understanding how these components work together in the context of a biological network. FANTOM5 provides atlases of mammalian promoters, enhancers, lncRNAs and miRNAs. FANTOM6 focuses on functional analysis of non-coding RNAs.\", \"founded_year\": 2009, \"host_institution\": \"RIKEN\", \"address\": \"RIKEN Omics Science Center,RIKEN Yokohama Institute,1-7-22 Suehiro-cho,Tsurumi-ku\", \"city\": \"Yokohama\", \"province\": \"Kanagawa\", \"country\": \"Japan\", \"current_version\": \"v6.0\", \"contact_name\": \"FANTOM help\", \"contact_email\": \"fantom-help@riken.jp\", \"available_protocol\": 1.0, \"last_update\": \"2022\", \"age\": 14, \"citation\": 4416, \"token\": \"$2a$10$ofx9LcQGKdVqAvAhHCg/keIkbReDHJ9Xvo/aHvIAWkf8GD14Df6qa\", \"expire_date\": 1543906854762.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-09-13 22:26:25\", \"data_type_list\": \"DNA, RNA\", \"category_list\": \"Expression, Gene genome and annotation\", \"keywords_list\": \"promoter, transcribed component, transcription start site, transcriptional network, lncRNA\", \"data_object_list\": \"Animal\", \"organism_list\": \"Mus musculus, Homo sapiens\", \"theme_list\": \"lncRNA\", \"zindex\": 315.429, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:22\", \"ess\": 0}, {\"page_id\": \"38_data_entity\", \"is_success\": true, \"error_count\": 1.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 38, \"page_type\": \"data_entity\", \"source_id\": \"dc_672\", \"short_name\": \"FANTOM\", \"accession\": \"DBC000672\", \"full_name\": \"functional annotation of the mammalian genome\", \"description\": \"FANTOM1-3 focused on identifying the transcribed components of mammalian cells. In FANTOM4, the focus has changed to understanding how these components work together in the context of a biological network. FANTOM5 provides atlases of mammalian promoters, enhancers, lncRNAs and miRNAs. FANTOM6 focuses on functional analysis of non-coding RNAs.\", \"founded_year\": 2009, \"host_institution\": \"RIKEN\", \"address\": \"RIKEN Omics Science Center,RIKEN Yokohama Institute,1-7-22 Suehiro-cho,Tsurumi-ku\", \"city\": \"Yokohama\", \"province\": \"Kanagawa\", \"country\": \"Japan\", \"current_version\": \"v6.0\", \"contact_name\": \"FANTOM help\", \"contact_email\": \"fantom-help@riken.jp\", \"available_protocol\": 1.0, \"last_update\": \"2022\", \"age\": 14, \"citation\": 4416, \"token\": \"$2a$10$ofx9LcQGKdVqAvAhHCg/keIkbReDHJ9Xvo/aHvIAWkf8GD14Df6qa\", \"expire_date\": 1543906854762.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-09-13 22:26:25\", \"data_type_list\": \"DNA, RNA\", \"category_list\": \"Expression, Gene genome and annotation\", \"keywords_list\": \"promoter, transcribed component, transcription start site, transcriptional network, lncRNA\", \"data_object_list\": \"Animal\", \"organism_list\": \"Mus musculus, Homo sapiens\", \"theme_list\": \"lncRNA\", \"zindex\": 315.429, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:22\", \"ess\": 0}, {\"page_id\": \"38_documentation\", \"is_success\": true, \"error_count\": 1.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 38, \"page_type\": \"documentation\", \"source_id\": \"dc_672\", \"short_name\": \"FANTOM\", \"accession\": \"DBC000672\", \"full_name\": \"functional annotation of the mammalian genome\", \"description\": \"FANTOM1-3 focused on identifying the transcribed components of mammalian cells. In FANTOM4, the focus has changed to understanding how these components work together in the context of a biological network. FANTOM5 provides atlases of mammalian promoters, enhancers, lncRNAs and miRNAs. FANTOM6 focuses on functional analysis of non-coding RNAs.\", \"founded_year\": 2009, \"host_institution\": \"RIKEN\", \"address\": \"RIKEN Omics Science Center,RIKEN Yokohama Institute,1-7-22 Suehiro-cho,Tsurumi-ku\", \"city\": \"Yokohama\", \"province\": \"Kanagawa\", \"country\": \"Japan\", \"current_version\": \"v6.0\", \"contact_name\": \"FANTOM help\", \"contact_email\": \"fantom-help@riken.jp\", \"available_protocol\": 1.0, \"last_update\": \"2022\", \"age\": 14, \"citation\": 4416, \"token\": \"$2a$10$ofx9LcQGKdVqAvAhHCg/keIkbReDHJ9Xvo/aHvIAWkf8GD14Df6qa\", \"expire_date\": 1543906854762.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-09-13 22:26:25\", \"data_type_list\": \"DNA, RNA\", \"category_list\": \"Expression, Gene genome and annotation\", \"keywords_list\": \"promoter, transcribed component, transcription start site, transcriptional network, lncRNA\", \"data_object_list\": \"Animal\", \"organism_list\": \"Mus musculus, Homo sapiens\", \"theme_list\": \"lncRNA\", \"zindex\": 315.429, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:22\", \"ess\": 0}, {\"page_id\": \"39_data_entity\", \"is_success\": true, \"error_count\": 6.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": 3.0, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 39, \"page_type\": \"data_entity\", \"source_id\": \"dc_6926\", \"short_name\": \"GISAID\", \"accession\": \"DBC006926\", \"full_name\": \"Global Initiative on Sharing All Influenza Data\", \"description\": \"The GISAID Initiative promotes the international sharing of all influenza virus sequences, related clinical and epidemiological data associated with human viruses, and geographical as well as species-specific data associated with avian and other animal viruses, to help researchers understand how the viruses evolve, spread and potentially become pandemics.\\nOn June 2 2020, 36,286 hCoV-19 genomic sequences have been shared via GISAID.\", \"founded_year\": 2008, \"host_institution\": \"Max-Planck-Institut for Informatics Computational Biology and Applied Algorithmics\", \"address\": null, \"city\": null, \"province\": null, \"country\": \"Germany\", \"current_version\": null, \"contact_name\": \"GISAID Group\", \"contact_email\": \"curator@gisaid.org\", \"available_protocol\": 1.0, \"last_update\": \"2020.02.13\", \"age\": 6, \"citation\": 1854, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1581601898000, \"last_modified\": \"2023-09-26 09:59:30\", \"data_type_list\": \"DNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"H5N1, SARS-CoV-2, hCoV-19, 2019-nCoV, pneumonia\", \"data_object_list\": \"Virus\", \"organism_list\": \"Influenza A virus, Influenza B virus, Influenza C virus, Influenza D virus, Severe acute respiratory syndrome coronavirus 2\", \"theme_list\": null, \"zindex\": 309.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:37\", \"ess\": 0}, {\"page_id\": \"39_documentation\", \"is_success\": true, \"error_count\": 6.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": 3.0, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 39, \"page_type\": \"documentation\", \"source_id\": \"dc_6926\", \"short_name\": \"GISAID\", \"accession\": \"DBC006926\", \"full_name\": \"Global Initiative on Sharing All Influenza Data\", \"description\": \"The GISAID Initiative promotes the international sharing of all influenza virus sequences, related clinical and epidemiological data associated with human viruses, and geographical as well as species-specific data associated with avian and other animal viruses, to help researchers understand how the viruses evolve, spread and potentially become pandemics.\\nOn June 2 2020, 36,286 hCoV-19 genomic sequences have been shared via GISAID.\", \"founded_year\": 2008, \"host_institution\": \"Max-Planck-Institut for Informatics Computational Biology and Applied Algorithmics\", \"address\": null, \"city\": null, \"province\": null, \"country\": \"Germany\", \"current_version\": null, \"contact_name\": \"GISAID Group\", \"contact_email\": \"curator@gisaid.org\", \"available_protocol\": 1.0, \"last_update\": \"2020.02.13\", \"age\": 6, \"citation\": 1854, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1581601898000, \"last_modified\": \"2023-09-26 09:59:30\", \"data_type_list\": \"DNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"H5N1, SARS-CoV-2, hCoV-19, 2019-nCoV, pneumonia\", \"data_object_list\": \"Virus\", \"organism_list\": \"Influenza A virus, Influenza B virus, Influenza C virus, Influenza D virus, Severe acute respiratory syndrome coronavirus 2\", \"theme_list\": null, \"zindex\": 309.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:37\", \"ess\": 0}, {\"page_id\": \"40_documentation\", \"is_success\": true, \"error_count\": 1.0, \"label_missing\": null, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 40, \"page_type\": \"documentation\", \"source_id\": \"dc_3120\", \"short_name\": \"Greengenes\", \"accession\": \"DBC003120\", \"full_name\": \"a chimera-checked 16S rRNA gene database \", \"description\": \"A 16S rRNA gene database addresses limitations of public repositories by providing chimera screening, standard alignment, and taxonomic classification using multiple published taxonomies.\", \"founded_year\": 2006, \"host_institution\": \"Lawrence Berkeley National Laboratory\", \"address\": \"Center for Environmental Biotechnology, Lawrence Berkeley National Laboratory, 1 Cyclotron Road, Mail Stop 70A-3317, Berkeley, CA 94720.\", \"city\": \"Berkeley\", \"province\": null, \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"G. L. Andersen\", \"contact_email\": \"GLAndersen@lbl.gov\", \"available_protocol\": 1.0, \"last_update\": \"2017-07-03\", \"age\": 17, \"citation\": 5239, \"token\": \"$2a$10$bBY6tqoll5wcIuLK4BLEweajcNiiAq7FWyykfG9J.84LW8W52lZEC\", \"expire_date\": 1543997717521.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1516962597000, \"last_modified\": \"2023-03-03 10:12:25\", \"data_type_list\": \"RNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"16S rRNA, taxonomic classification\", \"data_object_list\": \"Bacteria, Archaea\", \"organism_list\": \"Escherichia coli\", \"theme_list\": null, \"zindex\": 308.176, \"first_publication_year\": 2006, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:01:02\", \"ess\": 0}, {\"page_id\": \"41_search_result\", \"is_success\": true, \"error_count\": 3.0, \"label_missing\": 1.0, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 41, \"page_type\": \"search_result\", \"source_id\": \"dc_402\", \"short_name\": \"HMDB\", \"accession\": \"DBC000402\", \"full_name\": \"Human Metabolome Database\", \"description\": \"The Human Metabolome Database (HMDB) is a freely available electronic database containing detailed information about small molecule metabolites found in the human body.\", \"founded_year\": 2007, \"host_institution\": \"University of Alberta\", \"address\": \"Edmonton, AB, Canada T6G 2E8\", \"city\": \"Edmonton\", \"province\": \"AB\", \"country\": \"Canada\", \"current_version\": \"v4.0\", \"contact_name\": \"David S. Wishart\", \"contact_email\": \"david.wishart@ualberta.ca\", \"available_protocol\": 1.0, \"last_update\": \"2018-06-03\", \"age\": 16, \"citation\": 4912, \"token\": \"$2a$10$POAoFc5GmC.mpv35EEsgyOyWlf.ZfFE98KqJY2GuDXtVRxzYXQonW\", \"expire_date\": 1543899102912.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:10:11\", \"data_type_list\": \"Protein, Other\", \"category_list\": \"Pathway\", \"keywords_list\": \"chemical data, clinical data, molecular biology/biochemistry data, metabolomic data, Spectral data, chemical classification, chemical ontology\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 307.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:14\", \"ess\": 0}, {\"page_id\": \"41_data_entity\", \"is_success\": true, \"error_count\": 8.0, \"label_missing\": 2.0, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 2.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": 3.0, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 41, \"page_type\": \"data_entity\", \"source_id\": \"dc_402\", \"short_name\": \"HMDB\", \"accession\": \"DBC000402\", \"full_name\": \"Human Metabolome Database\", \"description\": \"The Human Metabolome Database (HMDB) is a freely available electronic database containing detailed information about small molecule metabolites found in the human body.\", \"founded_year\": 2007, \"host_institution\": \"University of Alberta\", \"address\": \"Edmonton, AB, Canada T6G 2E8\", \"city\": \"Edmonton\", \"province\": \"AB\", \"country\": \"Canada\", \"current_version\": \"v4.0\", \"contact_name\": \"David S. Wishart\", \"contact_email\": \"david.wishart@ualberta.ca\", \"available_protocol\": 1.0, \"last_update\": \"2018-06-03\", \"age\": 16, \"citation\": 4912, \"token\": \"$2a$10$POAoFc5GmC.mpv35EEsgyOyWlf.ZfFE98KqJY2GuDXtVRxzYXQonW\", \"expire_date\": 1543899102912.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:10:11\", \"data_type_list\": \"Protein, Other\", \"category_list\": \"Pathway\", \"keywords_list\": \"chemical data, clinical data, molecular biology/biochemistry data, metabolomic data, Spectral data, chemical classification, chemical ontology\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 307.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:14\", \"ess\": 0}, {\"page_id\": \"41_documentation\", \"is_success\": true, \"error_count\": 3.0, \"label_missing\": 1.0, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 41, \"page_type\": \"documentation\", \"source_id\": \"dc_402\", \"short_name\": \"HMDB\", \"accession\": \"DBC000402\", \"full_name\": \"Human Metabolome Database\", \"description\": \"The Human Metabolome Database (HMDB) is a freely available electronic database containing detailed information about small molecule metabolites found in the human body.\", \"founded_year\": 2007, \"host_institution\": \"University of Alberta\", \"address\": \"Edmonton, AB, Canada T6G 2E8\", \"city\": \"Edmonton\", \"province\": \"AB\", \"country\": \"Canada\", \"current_version\": \"v4.0\", \"contact_name\": \"David S. Wishart\", \"contact_email\": \"david.wishart@ualberta.ca\", \"available_protocol\": 1.0, \"last_update\": \"2018-06-03\", \"age\": 16, \"citation\": 4912, \"token\": \"$2a$10$POAoFc5GmC.mpv35EEsgyOyWlf.ZfFE98KqJY2GuDXtVRxzYXQonW\", \"expire_date\": 1543899102912.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:10:11\", \"data_type_list\": \"Protein, Other\", \"category_list\": \"Pathway\", \"keywords_list\": \"chemical data, clinical data, molecular biology/biochemistry data, metabolomic data, Spectral data, chemical classification, chemical ontology\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 307.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:14\", \"ess\": 0}, {\"page_id\": \"43_search_result\", \"is_success\": true, \"error_count\": 0.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 43, \"page_type\": \"search_result\", \"source_id\": \"dc_1773\", \"short_name\": \"PubChem \", \"accession\": \"DBC001773\", \"full_name\": null, \"description\": \"PubChem is a public repository for information on chemical substances and their biological activities, launched in 2004 as a component of the Molecular Libraries Roadmap Initiatives of the US National Institutes of Health (NIH).\\n\", \"founded_year\": 2009, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"Bethesda, Maryland 20894, USA\", \"city\": \"Bethesda\", \"province\": \"Maryland\", \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Yanli Wang\", \"contact_email\": \"ywang@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2017-01-01\", \"age\": 14, \"citation\": 3971, \"token\": \"$2a$10$etEN8.GF.a0e76BM5rYyoO.8VjMRBQdmdeacRsrp7iL0KmEcoSfCS\", \"expire_date\": 1543995359190.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1514980558000, \"last_modified\": \"2023-08-22 14:56:42\", \"data_type_list\": \"Other\", \"category_list\": \"Health and medicine, Structure\", \"keywords_list\": \"chemical substance, bioassay, compound\", \"data_object_list\": \"Animal\", \"organism_list\": null, \"theme_list\": null, \"zindex\": 283.643, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:51\", \"ess\": 0}, {\"page_id\": \"43_data_entity\", \"is_success\": true, \"error_count\": 1.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": 1.0, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 43, \"page_type\": \"data_entity\", \"source_id\": \"dc_1773\", \"short_name\": \"PubChem \", \"accession\": \"DBC001773\", \"full_name\": null, \"description\": \"PubChem is a public repository for information on chemical substances and their biological activities, launched in 2004 as a component of the Molecular Libraries Roadmap Initiatives of the US National Institutes of Health (NIH).\\n\", \"founded_year\": 2009, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"Bethesda, Maryland 20894, USA\", \"city\": \"Bethesda\", \"province\": \"Maryland\", \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Yanli Wang\", \"contact_email\": \"ywang@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2017-01-01\", \"age\": 14, \"citation\": 3971, \"token\": \"$2a$10$etEN8.GF.a0e76BM5rYyoO.8VjMRBQdmdeacRsrp7iL0KmEcoSfCS\", \"expire_date\": 1543995359190.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1514980558000, \"last_modified\": \"2023-08-22 14:56:42\", \"data_type_list\": \"Other\", \"category_list\": \"Health and medicine, Structure\", \"keywords_list\": \"chemical substance, bioassay, compound\", \"data_object_list\": \"Animal\", \"organism_list\": null, \"theme_list\": null, \"zindex\": 283.643, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:51\", \"ess\": 0}, {\"page_id\": \"43_documentation\", \"is_success\": true, \"error_count\": 1.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": 1.0, \"label_multiple\": null, \"id\": 43, \"page_type\": \"documentation\", \"source_id\": \"dc_1773\", \"short_name\": \"PubChem \", \"accession\": \"DBC001773\", \"full_name\": null, \"description\": \"PubChem is a public repository for information on chemical substances and their biological activities, launched in 2004 as a component of the Molecular Libraries Roadmap Initiatives of the US National Institutes of Health (NIH).\\n\", \"founded_year\": 2009, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"Bethesda, Maryland 20894, USA\", \"city\": \"Bethesda\", \"province\": \"Maryland\", \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Yanli Wang\", \"contact_email\": \"ywang@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2017-01-01\", \"age\": 14, \"citation\": 3971, \"token\": \"$2a$10$etEN8.GF.a0e76BM5rYyoO.8VjMRBQdmdeacRsrp7iL0KmEcoSfCS\", \"expire_date\": 1543995359190.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1514980558000, \"last_modified\": \"2023-08-22 14:56:42\", \"data_type_list\": \"Other\", \"category_list\": \"Health and medicine, Structure\", \"keywords_list\": \"chemical substance, bioassay, compound\", \"data_object_list\": \"Animal\", \"organism_list\": null, \"theme_list\": null, \"zindex\": 283.643, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:51\", \"ess\": 0}, {\"page_id\": \"44_search_result\", \"is_success\": true, \"error_count\": 6.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 2.0, \"link_empty\": 2.0, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 44, \"page_type\": \"search_result\", \"source_id\": \"dc_7106\", \"short_name\": \"CARD\", \"accession\": \"DBC007106\", \"full_name\": \"The Comprehensive Antibiotic Resistance Database\", \"description\": \"The Comprehensive Antibiotic Resistance Database (CARD; card.mcmaster.ca) combines the Antibiotic Resistance Ontology (ARO) with curated AMR gene (ARG) sequences and resistance-conferring mutations to provide an informatics framework for annotation and interpretation of resistomes. As of version 3.2.4, CARD encompasses 6627 ontology terms, 5010 reference sequences, 1933 mutations, 3004 publications, and 5057 AMR detection models that can be used by the accompanying Resistance Gene Identifier (RGI) software to annotate genomic or metagenomic sequences. New features include the CARD:Live collection of community submitted isolate resistome data and the introduction of standardized 15 character CARD Short Names for ARGs to support machine learning efforts.\", \"founded_year\": 2013, \"host_institution\": \"McMaster University\", \"address\": \"McMaster University\\nc/o McMaster Industry Liaison Office\\n305 \\u2013 175 Longwood Rd South\\nHamilton, Ontario L8P 0A1\", \"city\": \"Hamilton\", \"province\": \"Ontario\", \"country\": \"Canada\", \"current_version\": null, \"contact_name\": \"Andrew G. McArthur\", \"contact_email\": \"mcarthua@mcmaster.ca\", \"available_protocol\": 1.0, \"last_update\": \"2019\", \"age\": 10, \"citation\": 2815, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1604798602000, \"last_modified\": \"2023-08-28 16:16:12\", \"data_type_list\": \"DNA, Protein\", \"category_list\": \"Standard ontology and nomenclature, Literature, Health and medicine\", \"keywords_list\": \"reference DNA and protein, molecular sequences, software, curation, ontological structure\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 281.5, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:41\", \"ess\": 0}, {\"page_id\": \"44_data_entity\", \"is_success\": true, \"error_count\": 3.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 2.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 44, \"page_type\": \"data_entity\", \"source_id\": \"dc_7106\", \"short_name\": \"CARD\", \"accession\": \"DBC007106\", \"full_name\": \"The Comprehensive Antibiotic Resistance Database\", \"description\": \"The Comprehensive Antibiotic Resistance Database (CARD; card.mcmaster.ca) combines the Antibiotic Resistance Ontology (ARO) with curated AMR gene (ARG) sequences and resistance-conferring mutations to provide an informatics framework for annotation and interpretation of resistomes. As of version 3.2.4, CARD encompasses 6627 ontology terms, 5010 reference sequences, 1933 mutations, 3004 publications, and 5057 AMR detection models that can be used by the accompanying Resistance Gene Identifier (RGI) software to annotate genomic or metagenomic sequences. New features include the CARD:Live collection of community submitted isolate resistome data and the introduction of standardized 15 character CARD Short Names for ARGs to support machine learning efforts.\", \"founded_year\": 2013, \"host_institution\": \"McMaster University\", \"address\": \"McMaster University\\nc/o McMaster Industry Liaison Office\\n305 \\u2013 175 Longwood Rd South\\nHamilton, Ontario L8P 0A1\", \"city\": \"Hamilton\", \"province\": \"Ontario\", \"country\": \"Canada\", \"current_version\": null, \"contact_name\": \"Andrew G. McArthur\", \"contact_email\": \"mcarthua@mcmaster.ca\", \"available_protocol\": 1.0, \"last_update\": \"2019\", \"age\": 10, \"citation\": 2815, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1604798602000, \"last_modified\": \"2023-08-28 16:16:12\", \"data_type_list\": \"DNA, Protein\", \"category_list\": \"Standard ontology and nomenclature, Literature, Health and medicine\", \"keywords_list\": \"reference DNA and protein, molecular sequences, software, curation, ontological structure\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 281.5, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:41\", \"ess\": 0}, {\"page_id\": \"44_documentation\", \"is_success\": true, \"error_count\": 5.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 4.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 44, \"page_type\": \"documentation\", \"source_id\": \"dc_7106\", \"short_name\": \"CARD\", \"accession\": \"DBC007106\", \"full_name\": \"The Comprehensive Antibiotic Resistance Database\", \"description\": \"The Comprehensive Antibiotic Resistance Database (CARD; card.mcmaster.ca) combines the Antibiotic Resistance Ontology (ARO) with curated AMR gene (ARG) sequences and resistance-conferring mutations to provide an informatics framework for annotation and interpretation of resistomes. As of version 3.2.4, CARD encompasses 6627 ontology terms, 5010 reference sequences, 1933 mutations, 3004 publications, and 5057 AMR detection models that can be used by the accompanying Resistance Gene Identifier (RGI) software to annotate genomic or metagenomic sequences. New features include the CARD:Live collection of community submitted isolate resistome data and the introduction of standardized 15 character CARD Short Names for ARGs to support machine learning efforts.\", \"founded_year\": 2013, \"host_institution\": \"McMaster University\", \"address\": \"McMaster University\\nc/o McMaster Industry Liaison Office\\n305 \\u2013 175 Longwood Rd South\\nHamilton, Ontario L8P 0A1\", \"city\": \"Hamilton\", \"province\": \"Ontario\", \"country\": \"Canada\", \"current_version\": null, \"contact_name\": \"Andrew G. McArthur\", \"contact_email\": \"mcarthua@mcmaster.ca\", \"available_protocol\": 1.0, \"last_update\": \"2019\", \"age\": 10, \"citation\": 2815, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1604798602000, \"last_modified\": \"2023-08-28 16:16:12\", \"data_type_list\": \"DNA, Protein\", \"category_list\": \"Standard ontology and nomenclature, Literature, Health and medicine\", \"keywords_list\": \"reference DNA and protein, molecular sequences, software, curation, ontological structure\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens\", \"theme_list\": null, \"zindex\": 281.5, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:41\", \"ess\": 0}, {\"page_id\": \"45_search_result\", \"is_success\": true, \"error_count\": 1.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 45, \"page_type\": \"search_result\", \"source_id\": \"dc_66\", \"short_name\": \"BioGRID\", \"accession\": \"DBC000066\", \"full_name\": \"Biological General Repository for Interaction Datasets\", \"description\": \"BioGRID is an online interaction repository with data compiled through comprehensive curation efforts.The present version contains 1,585,828 protein and genetic interactions, 27,785 chemical associations and 39,028 post translational modifications from major model organism species.\", \"founded_year\": 2006, \"host_institution\": \"University of Montreal\", \"address\": \"Montr\\u00e9al, Quebec H3C 3J7, Canada\", \"city\": \"Mike Tyers\", \"province\": \"Quebec\", \"country\": \"Canada\", \"current_version\": \"v3.4.145\", \"contact_name\": \"biogridadmin\", \"contact_email\": \"biogridadmin@gmail.com\", \"available_protocol\": 1.0, \"last_update\": \"2017-02-01\", \"age\": 17, \"citation\": 4692, \"token\": \"$2a$10$KCGIMnRPzHxFzr9q1POCWOq3jANWbQkSR2M4p1MZLC8coujg2Su/6\", \"expire_date\": 1543898882242.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:42\", \"data_type_list\": \"Protein, DNA\", \"category_list\": \"Modification, Interaction, Literature\", \"keywords_list\": \"protein and genetic interaction, translational modification\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens, Escherichia coli, Mus musculus, Saccharomyces cerevisiae, Arabidopsis thaliana\", \"theme_list\": null, \"zindex\": 276.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:04\", \"ess\": 0}, {\"page_id\": \"45_data_entity\", \"is_success\": true, \"error_count\": 9.0, \"label_missing\": 7.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 2.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 45, \"page_type\": \"data_entity\", \"source_id\": \"dc_66\", \"short_name\": \"BioGRID\", \"accession\": \"DBC000066\", \"full_name\": \"Biological General Repository for Interaction Datasets\", \"description\": \"BioGRID is an online interaction repository with data compiled through comprehensive curation efforts.The present version contains 1,585,828 protein and genetic interactions, 27,785 chemical associations and 39,028 post translational modifications from major model organism species.\", \"founded_year\": 2006, \"host_institution\": \"University of Montreal\", \"address\": \"Montr\\u00e9al, Quebec H3C 3J7, Canada\", \"city\": \"Mike Tyers\", \"province\": \"Quebec\", \"country\": \"Canada\", \"current_version\": \"v3.4.145\", \"contact_name\": \"biogridadmin\", \"contact_email\": \"biogridadmin@gmail.com\", \"available_protocol\": 1.0, \"last_update\": \"2017-02-01\", \"age\": 17, \"citation\": 4692, \"token\": \"$2a$10$KCGIMnRPzHxFzr9q1POCWOq3jANWbQkSR2M4p1MZLC8coujg2Su/6\", \"expire_date\": 1543898882242.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:42\", \"data_type_list\": \"Protein, DNA\", \"category_list\": \"Modification, Interaction, Literature\", \"keywords_list\": \"protein and genetic interaction, translational modification\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens, Escherichia coli, Mus musculus, Saccharomyces cerevisiae, Arabidopsis thaliana\", \"theme_list\": null, \"zindex\": 276.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:04\", \"ess\": 0}, {\"page_id\": \"45_documentation\", \"is_success\": true, \"error_count\": 1.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 45, \"page_type\": \"documentation\", \"source_id\": \"dc_66\", \"short_name\": \"BioGRID\", \"accession\": \"DBC000066\", \"full_name\": \"Biological General Repository for Interaction Datasets\", \"description\": \"BioGRID is an online interaction repository with data compiled through comprehensive curation efforts.The present version contains 1,585,828 protein and genetic interactions, 27,785 chemical associations and 39,028 post translational modifications from major model organism species.\", \"founded_year\": 2006, \"host_institution\": \"University of Montreal\", \"address\": \"Montr\\u00e9al, Quebec H3C 3J7, Canada\", \"city\": \"Mike Tyers\", \"province\": \"Quebec\", \"country\": \"Canada\", \"current_version\": \"v3.4.145\", \"contact_name\": \"biogridadmin\", \"contact_email\": \"biogridadmin@gmail.com\", \"available_protocol\": 1.0, \"last_update\": \"2017-02-01\", \"age\": 17, \"citation\": 4692, \"token\": \"$2a$10$KCGIMnRPzHxFzr9q1POCWOq3jANWbQkSR2M4p1MZLC8coujg2Su/6\", \"expire_date\": 1543898882242.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:42\", \"data_type_list\": \"Protein, DNA\", \"category_list\": \"Modification, Interaction, Literature\", \"keywords_list\": \"protein and genetic interaction, translational modification\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens, Escherichia coli, Mus musculus, Saccharomyces cerevisiae, Arabidopsis thaliana\", \"theme_list\": null, \"zindex\": 276.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:04\", \"ess\": 0}, {\"page_id\": \"46_search\", \"is_success\": true, \"error_count\": 9.0, \"label_missing\": 4.0, \"language_missing\": null, \"button_empty\": 1.0, \"alt_missing\": 1.0, \"link_empty\": 3.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 46, \"page_type\": \"search\", \"source_id\": \"dc_217\", \"short_name\": \"Reactome\", \"accession\": \"DBC000217\", \"full_name\": \"Reactome Pathway Knowledgebase\", \"description\": \"Reactome is a free, open-source, curated and peer-reviewed pathway database. Our goal is to provide intuitive bioinformatics tools for the visualization, interpretation, and analysis of pathway knowledge to support basic research, genome analysis, modeling, systems biology, and education. \\n\\n\", \"founded_year\": 2005, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"661 University Ave, Suite 510, Toronto, ON, M5G 0A3\", \"city\": null, \"province\": null, \"country\": \"United Kingdom\", \"current_version\": \"v81\", \"contact_name\": \"Reactome Helpdesk\", \"contact_email\": \"help@reactome.org\", \"available_protocol\": 1.0, \"last_update\": \"2022-6-16\", \"age\": 18, \"citation\": 4932, \"token\": \"$2a$10$Vb7Y3w1DnZKjF5Irtbe4k.5gyT6utABrXxpaVxHv4y5sbazFYqhT6\", \"expire_date\": 1543898986809.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-10-09 14:54:03\", \"data_type_list\": \"Other, DNA, RNA, Protein\", \"category_list\": \"Pathway\", \"keywords_list\": \"pathway, interaction, SARS-CoV-2\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria\", \"organism_list\": \"Arabidopsis thaliana, Bos taurus, Caenorhabditis elegans, Danio rerio, Drosophila melanogaster, Homo sapiens, Mus musculus, Oryza sativa, Rattus norvegicus, Saccharomyces cerevisiae\", \"theme_list\": null, \"zindex\": 274.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:08\", \"ess\": 0}, {\"page_id\": \"46_search_result\", \"is_success\": true, \"error_count\": 5.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 1.0, \"link_empty\": 3.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 46, \"page_type\": \"search_result\", \"source_id\": \"dc_217\", \"short_name\": \"Reactome\", \"accession\": \"DBC000217\", \"full_name\": \"Reactome Pathway Knowledgebase\", \"description\": \"Reactome is a free, open-source, curated and peer-reviewed pathway database. Our goal is to provide intuitive bioinformatics tools for the visualization, interpretation, and analysis of pathway knowledge to support basic research, genome analysis, modeling, systems biology, and education. \\n\\n\", \"founded_year\": 2005, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"661 University Ave, Suite 510, Toronto, ON, M5G 0A3\", \"city\": null, \"province\": null, \"country\": \"United Kingdom\", \"current_version\": \"v81\", \"contact_name\": \"Reactome Helpdesk\", \"contact_email\": \"help@reactome.org\", \"available_protocol\": 1.0, \"last_update\": \"2022-6-16\", \"age\": 18, \"citation\": 4932, \"token\": \"$2a$10$Vb7Y3w1DnZKjF5Irtbe4k.5gyT6utABrXxpaVxHv4y5sbazFYqhT6\", \"expire_date\": 1543898986809.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-10-09 14:54:03\", \"data_type_list\": \"Other, DNA, RNA, Protein\", \"category_list\": \"Pathway\", \"keywords_list\": \"pathway, interaction, SARS-CoV-2\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria\", \"organism_list\": \"Arabidopsis thaliana, Bos taurus, Caenorhabditis elegans, Danio rerio, Drosophila melanogaster, Homo sapiens, Mus musculus, Oryza sativa, Rattus norvegicus, Saccharomyces cerevisiae\", \"theme_list\": null, \"zindex\": 274.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:08\", \"ess\": 0}, {\"page_id\": \"46_data_entity\", \"is_success\": true, \"error_count\": 5.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 1.0, \"link_empty\": 3.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 46, \"page_type\": \"data_entity\", \"source_id\": \"dc_217\", \"short_name\": \"Reactome\", \"accession\": \"DBC000217\", \"full_name\": \"Reactome Pathway Knowledgebase\", \"description\": \"Reactome is a free, open-source, curated and peer-reviewed pathway database. Our goal is to provide intuitive bioinformatics tools for the visualization, interpretation, and analysis of pathway knowledge to support basic research, genome analysis, modeling, systems biology, and education. \\n\\n\", \"founded_year\": 2005, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"661 University Ave, Suite 510, Toronto, ON, M5G 0A3\", \"city\": null, \"province\": null, \"country\": \"United Kingdom\", \"current_version\": \"v81\", \"contact_name\": \"Reactome Helpdesk\", \"contact_email\": \"help@reactome.org\", \"available_protocol\": 1.0, \"last_update\": \"2022-6-16\", \"age\": 18, \"citation\": 4932, \"token\": \"$2a$10$Vb7Y3w1DnZKjF5Irtbe4k.5gyT6utABrXxpaVxHv4y5sbazFYqhT6\", \"expire_date\": 1543898986809.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-10-09 14:54:03\", \"data_type_list\": \"Other, DNA, RNA, Protein\", \"category_list\": \"Pathway\", \"keywords_list\": \"pathway, interaction, SARS-CoV-2\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria\", \"organism_list\": \"Arabidopsis thaliana, Bos taurus, Caenorhabditis elegans, Danio rerio, Drosophila melanogaster, Homo sapiens, Mus musculus, Oryza sativa, Rattus norvegicus, Saccharomyces cerevisiae\", \"theme_list\": null, \"zindex\": 274.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:08\", \"ess\": 0}, {\"page_id\": \"46_documentation\", \"is_success\": true, \"error_count\": 4.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 3.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 46, \"page_type\": \"documentation\", \"source_id\": \"dc_217\", \"short_name\": \"Reactome\", \"accession\": \"DBC000217\", \"full_name\": \"Reactome Pathway Knowledgebase\", \"description\": \"Reactome is a free, open-source, curated and peer-reviewed pathway database. Our goal is to provide intuitive bioinformatics tools for the visualization, interpretation, and analysis of pathway knowledge to support basic research, genome analysis, modeling, systems biology, and education. \\n\\n\", \"founded_year\": 2005, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"661 University Ave, Suite 510, Toronto, ON, M5G 0A3\", \"city\": null, \"province\": null, \"country\": \"United Kingdom\", \"current_version\": \"v81\", \"contact_name\": \"Reactome Helpdesk\", \"contact_email\": \"help@reactome.org\", \"available_protocol\": 1.0, \"last_update\": \"2022-6-16\", \"age\": 18, \"citation\": 4932, \"token\": \"$2a$10$Vb7Y3w1DnZKjF5Irtbe4k.5gyT6utABrXxpaVxHv4y5sbazFYqhT6\", \"expire_date\": 1543898986809.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-10-09 14:54:03\", \"data_type_list\": \"Other, DNA, RNA, Protein\", \"category_list\": \"Pathway\", \"keywords_list\": \"pathway, interaction, SARS-CoV-2\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria\", \"organism_list\": \"Arabidopsis thaliana, Bos taurus, Caenorhabditis elegans, Danio rerio, Drosophila melanogaster, Homo sapiens, Mus musculus, Oryza sativa, Rattus norvegicus, Saccharomyces cerevisiae\", \"theme_list\": null, \"zindex\": 274.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:08\", \"ess\": 0}, {\"page_id\": \"48_search_result\", \"is_success\": true, \"error_count\": 34.0, \"label_missing\": 28.0, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 3.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": 1.0, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": 1.0, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 48, \"page_type\": \"search_result\", \"source_id\": \"dc_167\", \"short_name\": \"miRTarBase\", \"accession\": \"DBC000167\", \"full_name\": \"microRNA-target interactions database\", \"description\": \"miRTarBase has accumulated more than three hundred and sixty thousand miRNA-target interactions (MTIs), which are collected by manually surveying pertinent literature after NLP of the text systematically to filter research articles related to functional studies of miRNAs. Generally, the collected MTIs are validated experimentally by reporter assay, western blot, microarray and next-generation sequencing experiments. While containing the largest amount of validated MTIs, the miRTarBase provides the most updated collection by comparing with other similar, previously developed databases.\", \"founded_year\": 2011, \"host_institution\": \"Chinese University of Hong Kong\", \"address\": \"School of Life and Health Sciences, The Chinese University of Hong Kong, Shenzhen, Longgang District, Shenzhen, Guangdong Province 518172, China.\", \"city\": \"HongKong\", \"province\": \"HongKong\", \"country\": \"China\", \"current_version\": \"Release 9.0 beta\", \"contact_name\": \"Hsien-Da Huang\", \"contact_email\": \"huanghsiyuan@cuhk.edu.cn\", \"available_protocol\": 1.0, \"last_update\": \"2021-09\", \"age\": 12, \"citation\": 3126, \"token\": \"$2a$10$v12HaPlA26wUtjPtOfomRO4zEXXEBGsib0gfo7Bu1dVFJrZfLsii.\", \"expire_date\": 1543898953010.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:51\", \"data_type_list\": \"RNA\", \"category_list\": \"Interaction, Literature\", \"keywords_list\": \"miRNA-target interaction, miRNA\", \"data_object_list\": \"Animal, Plant, Virus\", \"organism_list\": \"Arabidopsis thaliana, Caenorhabditis elegans, Drosophila melanogaster, Homo sapiens, Mus musculus, Rattus norvegicus, Bombyx mori, Bos taurus, Taeniopygia guttata, Xenopus laevis\", \"theme_list\": null, \"zindex\": 260.5, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:07\", \"ess\": 0}, {\"page_id\": \"48_data_entity\", \"is_success\": true, \"error_count\": 2789.0, \"label_missing\": 6.0, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": 2768.0, \"link_empty\": 3.0, \"aria_menu_broken\": null, \"alt_link_missing\": 10.0, \"th_empty\": null, \"heading_empty\": 1.0, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 48, \"page_type\": \"data_entity\", \"source_id\": \"dc_167\", \"short_name\": \"miRTarBase\", \"accession\": \"DBC000167\", \"full_name\": \"microRNA-target interactions database\", \"description\": \"miRTarBase has accumulated more than three hundred and sixty thousand miRNA-target interactions (MTIs), which are collected by manually surveying pertinent literature after NLP of the text systematically to filter research articles related to functional studies of miRNAs. Generally, the collected MTIs are validated experimentally by reporter assay, western blot, microarray and next-generation sequencing experiments. While containing the largest amount of validated MTIs, the miRTarBase provides the most updated collection by comparing with other similar, previously developed databases.\", \"founded_year\": 2011, \"host_institution\": \"Chinese University of Hong Kong\", \"address\": \"School of Life and Health Sciences, The Chinese University of Hong Kong, Shenzhen, Longgang District, Shenzhen, Guangdong Province 518172, China.\", \"city\": \"HongKong\", \"province\": \"HongKong\", \"country\": \"China\", \"current_version\": \"Release 9.0 beta\", \"contact_name\": \"Hsien-Da Huang\", \"contact_email\": \"huanghsiyuan@cuhk.edu.cn\", \"available_protocol\": 1.0, \"last_update\": \"2021-09\", \"age\": 12, \"citation\": 3126, \"token\": \"$2a$10$v12HaPlA26wUtjPtOfomRO4zEXXEBGsib0gfo7Bu1dVFJrZfLsii.\", \"expire_date\": 1543898953010.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:51\", \"data_type_list\": \"RNA\", \"category_list\": \"Interaction, Literature\", \"keywords_list\": \"miRNA-target interaction, miRNA\", \"data_object_list\": \"Animal, Plant, Virus\", \"organism_list\": \"Arabidopsis thaliana, Caenorhabditis elegans, Drosophila melanogaster, Homo sapiens, Mus musculus, Rattus norvegicus, Bombyx mori, Bos taurus, Taeniopygia guttata, Xenopus laevis\", \"theme_list\": null, \"zindex\": 260.5, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:07\", \"ess\": 0}, {\"page_id\": \"49_search_result\", \"is_success\": true, \"error_count\": 10.0, \"label_missing\": 2.0, \"language_missing\": 1.0, \"button_empty\": 1.0, \"alt_missing\": 1.0, \"link_empty\": 2.0, \"aria_menu_broken\": null, \"alt_link_missing\": 3.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 49, \"page_type\": \"search_result\", \"source_id\": \"dc_295\", \"short_name\": \"ChEMBL\", \"accession\": \"DBC000295\", \"full_name\": null, \"description\": \"ChEMBL is a manually curated database of bioactive molecules with drug-like properties. It brings together chemical, bioactivity and genomic data to aid the translation of genomic information into effective new drugs. \", \"founded_year\": 2011, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"Hinxton, Cambridge CB10 1SD, UK\", \"city\": \"Hinxton\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"v25\", \"contact_name\": \"Andrew R. Leach\", \"contact_email\": \"arl@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2018-12-10\", \"age\": 12, \"citation\": 3030, \"token\": \"$2a$10$/yamR44PS4ERv0pHaX9FzuhrRee4TvZcxpe4a2kJPAknEGRmEGzq.\", \"expire_date\": 1543899035077.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:10:01\", \"data_type_list\": \"Protein\", \"category_list\": \"Health and medicine, Interaction, Literature, Structure\", \"keywords_list\": \"drug target, drug-like small molecule\", \"data_object_list\": \"Animal\", \"organism_list\": null, \"theme_list\": null, \"zindex\": 252.5, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:10\", \"ess\": 0}, {\"page_id\": \"49_data_entity\", \"is_success\": true, \"error_count\": 15.0, \"label_missing\": 5.0, \"language_missing\": 1.0, \"button_empty\": 1.0, \"alt_missing\": 1.0, \"link_empty\": 3.0, \"aria_menu_broken\": null, \"alt_link_missing\": 3.0, \"th_empty\": null, \"heading_empty\": 1.0, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 49, \"page_type\": \"data_entity\", \"source_id\": \"dc_295\", \"short_name\": \"ChEMBL\", \"accession\": \"DBC000295\", \"full_name\": null, \"description\": \"ChEMBL is a manually curated database of bioactive molecules with drug-like properties. It brings together chemical, bioactivity and genomic data to aid the translation of genomic information into effective new drugs. \", \"founded_year\": 2011, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"Hinxton, Cambridge CB10 1SD, UK\", \"city\": \"Hinxton\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"v25\", \"contact_name\": \"Andrew R. Leach\", \"contact_email\": \"arl@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2018-12-10\", \"age\": 12, \"citation\": 3030, \"token\": \"$2a$10$/yamR44PS4ERv0pHaX9FzuhrRee4TvZcxpe4a2kJPAknEGRmEGzq.\", \"expire_date\": 1543899035077.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:10:01\", \"data_type_list\": \"Protein\", \"category_list\": \"Health and medicine, Interaction, Literature, Structure\", \"keywords_list\": \"drug target, drug-like small molecule\", \"data_object_list\": \"Animal\", \"organism_list\": null, \"theme_list\": null, \"zindex\": 252.5, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:10\", \"ess\": 0}, {\"page_id\": \"49_documentation\", \"is_success\": true, \"error_count\": 0.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 49, \"page_type\": \"documentation\", \"source_id\": \"dc_295\", \"short_name\": \"ChEMBL\", \"accession\": \"DBC000295\", \"full_name\": null, \"description\": \"ChEMBL is a manually curated database of bioactive molecules with drug-like properties. It brings together chemical, bioactivity and genomic data to aid the translation of genomic information into effective new drugs. \", \"founded_year\": 2011, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"Hinxton, Cambridge CB10 1SD, UK\", \"city\": \"Hinxton\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"v25\", \"contact_name\": \"Andrew R. Leach\", \"contact_email\": \"arl@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2018-12-10\", \"age\": 12, \"citation\": 3030, \"token\": \"$2a$10$/yamR44PS4ERv0pHaX9FzuhrRee4TvZcxpe4a2kJPAknEGRmEGzq.\", \"expire_date\": 1543899035077.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:10:01\", \"data_type_list\": \"Protein\", \"category_list\": \"Health and medicine, Interaction, Literature, Structure\", \"keywords_list\": \"drug target, drug-like small molecule\", \"data_object_list\": \"Animal\", \"organism_list\": null, \"theme_list\": null, \"zindex\": 252.5, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:10\", \"ess\": 0}, {\"page_id\": \"50_search_result\", \"is_success\": true, \"error_count\": 21.0, \"label_missing\": 12.0, \"language_missing\": 1.0, \"button_empty\": 7.0, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": 1.0, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 50, \"page_type\": \"search_result\", \"source_id\": \"dc_176\", \"short_name\": \"JASPAR\", \"accession\": \"DBC000176\", \"full_name\": null, \"description\": \"JASPAR is the largest open-access database of matrix-based nucleotide profiles describing the binding preference of transcription factors from multiple species.\", \"founded_year\": 2004, \"host_institution\": \"University of British Columbia\", \"address\": \"Department of Medical Genetics,Centre for Molecular Medicine and Therapeutics at the Child and Family Research Institute,University of British Columbia,Vancouver,BC,Canada\", \"city\": \"Vancouver\", \"province\": \"BC\", \"country\": \"Canada\", \"current_version\": \"v5.0\", \"contact_name\": \"Boris Lenhard\", \"contact_email\": \"b.lenhard@csc.mrc.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2016-03-18\", \"age\": 19, \"citation\": 4481, \"token\": \"$2a$10$Ui0EK2W5ecDRwczesHsiMuBo2GukKNm9m.R9qBHo4G.9sb/bRPoqa\", \"expire_date\": 1543898958412.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:52\", \"data_type_list\": \"DNA, Protein\", \"category_list\": \"Interaction\", \"keywords_list\": \"transcription factor binding preference\", \"data_object_list\": \"Animal\", \"organism_list\": \"Caenorhabditis elegans, Drosophila melanogaster, Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 235.842, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:07\", \"ess\": 0}, {\"page_id\": \"50_data_entity\", \"is_success\": true, \"error_count\": 13.0, \"label_missing\": null, \"language_missing\": 1.0, \"button_empty\": 2.0, \"alt_missing\": 5.0, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": 4.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 50, \"page_type\": \"data_entity\", \"source_id\": \"dc_176\", \"short_name\": \"JASPAR\", \"accession\": \"DBC000176\", \"full_name\": null, \"description\": \"JASPAR is the largest open-access database of matrix-based nucleotide profiles describing the binding preference of transcription factors from multiple species.\", \"founded_year\": 2004, \"host_institution\": \"University of British Columbia\", \"address\": \"Department of Medical Genetics,Centre for Molecular Medicine and Therapeutics at the Child and Family Research Institute,University of British Columbia,Vancouver,BC,Canada\", \"city\": \"Vancouver\", \"province\": \"BC\", \"country\": \"Canada\", \"current_version\": \"v5.0\", \"contact_name\": \"Boris Lenhard\", \"contact_email\": \"b.lenhard@csc.mrc.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2016-03-18\", \"age\": 19, \"citation\": 4481, \"token\": \"$2a$10$Ui0EK2W5ecDRwczesHsiMuBo2GukKNm9m.R9qBHo4G.9sb/bRPoqa\", \"expire_date\": 1543898958412.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:52\", \"data_type_list\": \"DNA, Protein\", \"category_list\": \"Interaction\", \"keywords_list\": \"transcription factor binding preference\", \"data_object_list\": \"Animal\", \"organism_list\": \"Caenorhabditis elegans, Drosophila melanogaster, Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 235.842, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:07\", \"ess\": 0}, {\"page_id\": \"50_documentation\", \"is_success\": true, \"error_count\": 1.0, \"label_missing\": null, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 50, \"page_type\": \"documentation\", \"source_id\": \"dc_176\", \"short_name\": \"JASPAR\", \"accession\": \"DBC000176\", \"full_name\": null, \"description\": \"JASPAR is the largest open-access database of matrix-based nucleotide profiles describing the binding preference of transcription factors from multiple species.\", \"founded_year\": 2004, \"host_institution\": \"University of British Columbia\", \"address\": \"Department of Medical Genetics,Centre for Molecular Medicine and Therapeutics at the Child and Family Research Institute,University of British Columbia,Vancouver,BC,Canada\", \"city\": \"Vancouver\", \"province\": \"BC\", \"country\": \"Canada\", \"current_version\": \"v5.0\", \"contact_name\": \"Boris Lenhard\", \"contact_email\": \"b.lenhard@csc.mrc.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2016-03-18\", \"age\": 19, \"citation\": 4481, \"token\": \"$2a$10$Ui0EK2W5ecDRwczesHsiMuBo2GukKNm9m.R9qBHo4G.9sb/bRPoqa\", \"expire_date\": 1543898958412.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:52\", \"data_type_list\": \"DNA, Protein\", \"category_list\": \"Interaction\", \"keywords_list\": \"transcription factor binding preference\", \"data_object_list\": \"Animal\", \"organism_list\": \"Caenorhabditis elegans, Drosophila melanogaster, Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 235.842, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:07\", \"ess\": 0}, {\"page_id\": \"52_search\", \"is_success\": true, \"error_count\": 3.0, \"label_missing\": 3.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 52, \"page_type\": \"search\", \"source_id\": \"dc_1619\", \"short_name\": \"GenBank\", \"accession\": \"DBC001619\", \"full_name\": \"GenBank\", \"description\": \"GenBank is a comprehensive database that contains publicly available DNA sequences for more than 165\\u2009000 named organisms, obtained primarily through submissions from individual laboratories and batch submissions from large-scale sequencing projects.\\n\", \"founded_year\": 1985, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"8600 Rockville Pike, Bethesda, MD 20894, USA\", \"city\": \"Bethesda\", \"province\": \"MD\", \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"GenBank team\", \"contact_email\": \"genomes@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2020-10-20\", \"age\": 38, \"citation\": 8118, \"token\": \"$2a$10$YorLifqtkghGstw/JUmDYOP9qXDg0eKDkMmBTIhr7hY7QUsJWlwym\", \"expire_date\": 1543993384944.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1450663065000, \"last_modified\": \"2023-03-03 10:11:38\", \"data_type_list\": \"DNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"DNA sequence submission\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens, Mus musculus, Arabidopsis thaliana\", \"theme_list\": null, \"zindex\": 213.632, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:47\", \"ess\": 0}, {\"page_id\": \"52_search_result\", \"is_success\": true, \"error_count\": 3.0, \"label_missing\": 3.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 52, \"page_type\": \"search_result\", \"source_id\": \"dc_1619\", \"short_name\": \"GenBank\", \"accession\": \"DBC001619\", \"full_name\": \"GenBank\", \"description\": \"GenBank is a comprehensive database that contains publicly available DNA sequences for more than 165\\u2009000 named organisms, obtained primarily through submissions from individual laboratories and batch submissions from large-scale sequencing projects.\\n\", \"founded_year\": 1985, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"8600 Rockville Pike, Bethesda, MD 20894, USA\", \"city\": \"Bethesda\", \"province\": \"MD\", \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"GenBank team\", \"contact_email\": \"genomes@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2020-10-20\", \"age\": 38, \"citation\": 8118, \"token\": \"$2a$10$YorLifqtkghGstw/JUmDYOP9qXDg0eKDkMmBTIhr7hY7QUsJWlwym\", \"expire_date\": 1543993384944.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1450663065000, \"last_modified\": \"2023-03-03 10:11:38\", \"data_type_list\": \"DNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"DNA sequence submission\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens, Mus musculus, Arabidopsis thaliana\", \"theme_list\": null, \"zindex\": 213.632, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:47\", \"ess\": 0}, {\"page_id\": \"52_data_entity\", \"is_success\": true, \"error_count\": 3.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": 2.0, \"label_multiple\": 1.0, \"id\": 52, \"page_type\": \"data_entity\", \"source_id\": \"dc_1619\", \"short_name\": \"GenBank\", \"accession\": \"DBC001619\", \"full_name\": \"GenBank\", \"description\": \"GenBank is a comprehensive database that contains publicly available DNA sequences for more than 165\\u2009000 named organisms, obtained primarily through submissions from individual laboratories and batch submissions from large-scale sequencing projects.\\n\", \"founded_year\": 1985, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"8600 Rockville Pike, Bethesda, MD 20894, USA\", \"city\": \"Bethesda\", \"province\": \"MD\", \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"GenBank team\", \"contact_email\": \"genomes@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2020-10-20\", \"age\": 38, \"citation\": 8118, \"token\": \"$2a$10$YorLifqtkghGstw/JUmDYOP9qXDg0eKDkMmBTIhr7hY7QUsJWlwym\", \"expire_date\": 1543993384944.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1450663065000, \"last_modified\": \"2023-03-03 10:11:38\", \"data_type_list\": \"DNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"DNA sequence submission\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens, Mus musculus, Arabidopsis thaliana\", \"theme_list\": null, \"zindex\": 213.632, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:47\", \"ess\": 0}, {\"page_id\": \"52_documentation\", \"is_success\": false, \"error_count\": null, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": 2.0, \"label_multiple\": 1.0, \"id\": 52, \"page_type\": \"documentation\", \"source_id\": \"dc_1619\", \"short_name\": \"GenBank\", \"accession\": \"DBC001619\", \"full_name\": \"GenBank\", \"description\": \"GenBank is a comprehensive database that contains publicly available DNA sequences for more than 165\\u2009000 named organisms, obtained primarily through submissions from individual laboratories and batch submissions from large-scale sequencing projects.\\n\", \"founded_year\": 1985, \"host_institution\": \"National Center for Biotechnology Information\", \"address\": \"8600 Rockville Pike, Bethesda, MD 20894, USA\", \"city\": \"Bethesda\", \"province\": \"MD\", \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"GenBank team\", \"contact_email\": \"genomes@ncbi.nlm.nih.gov\", \"available_protocol\": 1.0, \"last_update\": \"2020-10-20\", \"age\": 38, \"citation\": 8118, \"token\": \"$2a$10$YorLifqtkghGstw/JUmDYOP9qXDg0eKDkMmBTIhr7hY7QUsJWlwym\", \"expire_date\": 1543993384944.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1450663065000, \"last_modified\": \"2023-03-03 10:11:38\", \"data_type_list\": \"DNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"DNA sequence submission\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens, Mus musculus, Arabidopsis thaliana\", \"theme_list\": null, \"zindex\": 213.632, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:47\", \"ess\": 0}, {\"page_id\": \"55_search_result\", \"is_success\": true, \"error_count\": 2.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 1.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 55, \"page_type\": \"search_result\", \"source_id\": \"dc_3941\", \"short_name\": \"miRWalk\", \"accession\": \"DBC003941\", \"full_name\": null, \"description\": \"a freely accessible, regularly updated comprehensive archive supplying the largest available collection of predicted and experimentally verified miRNA-target interactions, with various novel and unique features to assist the scientific community.\", \"founded_year\": 2011, \"host_institution\": \"University of Heidelberg\", \"address\": \"Present affiliation: Division of Bioinformatics and Biostatistics, National Center for Toxicological Research (NCTR), Food and Drug Administration (FDA), Jefferson, Arizona.\\n\", \"city\": null, \"province\": null, \"country\": \"Germany\", \"current_version\": \"v.2.0\", \"contact_name\": \"Carsten Sticht\", \"contact_email\": \"mirwalkteam@medma.uni-heidelberg.de\", \"available_protocol\": 1.0, \"last_update\": \"2018\", \"age\": 12, \"citation\": 2461, \"token\": \"$2a$10$oxjlXiJV1GXLd0og.VMRteci97SgFwd.2fiDA459kE1nLcMxrlwhW\", \"expire_date\": 1544076379011.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1517081551000, \"last_modified\": \"2023-03-03 10:13:28\", \"data_type_list\": \"RNA\", \"category_list\": \"Interaction\", \"keywords_list\": \"gene ontology, micro RNA, target prediction\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 205.083, \"first_publication_year\": 2016, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:01:24\", \"ess\": 1}, {\"page_id\": \"55_documentation\", \"is_success\": true, \"error_count\": 2.0, \"label_missing\": null, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": 1.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 55, \"page_type\": \"documentation\", \"source_id\": \"dc_3941\", \"short_name\": \"miRWalk\", \"accession\": \"DBC003941\", \"full_name\": null, \"description\": \"a freely accessible, regularly updated comprehensive archive supplying the largest available collection of predicted and experimentally verified miRNA-target interactions, with various novel and unique features to assist the scientific community.\", \"founded_year\": 2011, \"host_institution\": \"University of Heidelberg\", \"address\": \"Present affiliation: Division of Bioinformatics and Biostatistics, National Center for Toxicological Research (NCTR), Food and Drug Administration (FDA), Jefferson, Arizona.\\n\", \"city\": null, \"province\": null, \"country\": \"Germany\", \"current_version\": \"v.2.0\", \"contact_name\": \"Carsten Sticht\", \"contact_email\": \"mirwalkteam@medma.uni-heidelberg.de\", \"available_protocol\": 1.0, \"last_update\": \"2018\", \"age\": 12, \"citation\": 2461, \"token\": \"$2a$10$oxjlXiJV1GXLd0og.VMRteci97SgFwd.2fiDA459kE1nLcMxrlwhW\", \"expire_date\": 1544076379011.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1517081551000, \"last_modified\": \"2023-03-03 10:13:28\", \"data_type_list\": \"RNA\", \"category_list\": \"Interaction\", \"keywords_list\": \"gene ontology, micro RNA, target prediction\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 205.083, \"first_publication_year\": 2016, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:01:24\", \"ess\": 1}, {\"page_id\": \"56_search_result\", \"is_success\": true, \"error_count\": 29.0, \"label_missing\": 3.0, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 23.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": 2.0, \"id\": 56, \"page_type\": \"search_result\", \"source_id\": \"dc_17\", \"short_name\": \"Rfam\", \"accession\": \"DBC000017\", \"full_name\": \"RNA Families\", \"description\": \"The Rfam database is a collection of non-coding RNA families represented by manually curated sequence alignments, consensus secondary structures and annotation gathered from corresponding Wikipedia, taxonomy and ontology resources.\", \"founded_year\": 2003, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"Wellcome Genome Campus, Hinxton, Cambridge, UK\", \"city\": \"Cambridge\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"v14.9\", \"contact_name\": \"Alex Bateman\", \"contact_email\": \"agb@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2022-11\", \"age\": 20, \"citation\": 4084, \"token\": \"$2a$10$thJXTWwFFA4Bcto2HKpqbO.s4VwHolBuAIZuY4p5TJjfGUeyrZeIO\", \"expire_date\": 1543898843283.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-06 11:06:27\", \"data_type_list\": \"RNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"covariance model, ncRNA, RNA family, RNA motif, secondary structure, sequence alignment\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens, Escherichia coli, Drosophila melanogaster, Mus musculus, Caenorhabditis elegans, Bacillus subtilis\", \"theme_list\": null, \"zindex\": 204.2, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:02\", \"ess\": 0}, {\"page_id\": \"56_data_entity\", \"is_success\": true, \"error_count\": 36.0, \"label_missing\": 1.0, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": 1.0, \"link_empty\": 22.0, \"aria_menu_broken\": null, \"alt_link_missing\": 7.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": 2.0, \"alt_map_missing\": 2.0, \"label_empty\": null, \"label_multiple\": null, \"id\": 56, \"page_type\": \"data_entity\", \"source_id\": \"dc_17\", \"short_name\": \"Rfam\", \"accession\": \"DBC000017\", \"full_name\": \"RNA Families\", \"description\": \"The Rfam database is a collection of non-coding RNA families represented by manually curated sequence alignments, consensus secondary structures and annotation gathered from corresponding Wikipedia, taxonomy and ontology resources.\", \"founded_year\": 2003, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"Wellcome Genome Campus, Hinxton, Cambridge, UK\", \"city\": \"Cambridge\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"v14.9\", \"contact_name\": \"Alex Bateman\", \"contact_email\": \"agb@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2022-11\", \"age\": 20, \"citation\": 4084, \"token\": \"$2a$10$thJXTWwFFA4Bcto2HKpqbO.s4VwHolBuAIZuY4p5TJjfGUeyrZeIO\", \"expire_date\": 1543898843283.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-06 11:06:27\", \"data_type_list\": \"RNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"covariance model, ncRNA, RNA family, RNA motif, secondary structure, sequence alignment\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens, Escherichia coli, Drosophila melanogaster, Mus musculus, Caenorhabditis elegans, Bacillus subtilis\", \"theme_list\": null, \"zindex\": 204.2, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:02\", \"ess\": 0}, {\"page_id\": \"56_documentation\", \"is_success\": true, \"error_count\": 2.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": 1.0, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 56, \"page_type\": \"documentation\", \"source_id\": \"dc_17\", \"short_name\": \"Rfam\", \"accession\": \"DBC000017\", \"full_name\": \"RNA Families\", \"description\": \"The Rfam database is a collection of non-coding RNA families represented by manually curated sequence alignments, consensus secondary structures and annotation gathered from corresponding Wikipedia, taxonomy and ontology resources.\", \"founded_year\": 2003, \"host_institution\": \"European Bioinformatics Institute\", \"address\": \"Wellcome Genome Campus, Hinxton, Cambridge, UK\", \"city\": \"Cambridge\", \"province\": \"Cambridgeshire\", \"country\": \"United Kingdom\", \"current_version\": \"v14.9\", \"contact_name\": \"Alex Bateman\", \"contact_email\": \"agb@ebi.ac.uk\", \"available_protocol\": 1.0, \"last_update\": \"2022-11\", \"age\": 20, \"citation\": 4084, \"token\": \"$2a$10$thJXTWwFFA4Bcto2HKpqbO.s4VwHolBuAIZuY4p5TJjfGUeyrZeIO\", \"expire_date\": 1543898843283.0, \"ins\": 0, \"group_email\": \"ccook@ebi.ac.uk\", \"is_partner\": 0, \"funding\": NaN, \"submitter\": \"ccook@ebi.ac.uk\", \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-06 11:06:27\", \"data_type_list\": \"RNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"covariance model, ncRNA, RNA family, RNA motif, secondary structure, sequence alignment\", \"data_object_list\": \"Animal, Plant, Fungi, Bacteria, Archaea, Virus\", \"organism_list\": \"Homo sapiens, Escherichia coli, Drosophila melanogaster, Mus musculus, Caenorhabditis elegans, Bacillus subtilis\", \"theme_list\": null, \"zindex\": 204.2, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:02\", \"ess\": 0}, {\"page_id\": \"57_search\", \"is_success\": true, \"error_count\": 4.0, \"label_missing\": 1.0, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 2.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 57, \"page_type\": \"search\", \"source_id\": \"dc_71\", \"short_name\": \"PhosphoSitePlus\", \"accession\": \"DBC000071\", \"full_name\": null, \"description\": \"PhosphoSitePlus(\\u00ae) (PSP), a knowledgebase dedicated to mammalian post-translational modifications (PTMs), contains over 330,000 non-redundant PTMs, including phospho, acetyl, ubiquityl and methyl groups.\", \"founded_year\": 2012, \"host_institution\": \"Cell Signaling Technology\", \"address\": \"Cell Signaling Technology,3 Trask Lane,Danvers,MA 01923,USA\", \"city\": \"Danvers\", \"province\": \"MA\", \"country\": \"United States\", \"current_version\": \"v1.0\", \"contact_name\": \"EditorPhosphoSite\", \"contact_email\": \"EditorPhosphoSite@cellsignal.com\", \"available_protocol\": 1.0, \"last_update\": \"2014-12\", \"age\": 11, \"citation\": 2244, \"token\": \"$2a$10$MYHS.0iaIuxOeZpCKWKy9eYp6Pmq9KT27hilryN39UeTMBqmByE6a\", \"expire_date\": 1543898885234.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:43\", \"data_type_list\": \"Protein\", \"category_list\": \"Modification\", \"keywords_list\": \"mass spectrometry experiment, post-translational modification\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 204.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:04\", \"ess\": 0}, {\"page_id\": \"57_search_result\", \"is_success\": true, \"error_count\": 1.0, \"label_missing\": null, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 57, \"page_type\": \"search_result\", \"source_id\": \"dc_71\", \"short_name\": \"PhosphoSitePlus\", \"accession\": \"DBC000071\", \"full_name\": null, \"description\": \"PhosphoSitePlus(\\u00ae) (PSP), a knowledgebase dedicated to mammalian post-translational modifications (PTMs), contains over 330,000 non-redundant PTMs, including phospho, acetyl, ubiquityl and methyl groups.\", \"founded_year\": 2012, \"host_institution\": \"Cell Signaling Technology\", \"address\": \"Cell Signaling Technology,3 Trask Lane,Danvers,MA 01923,USA\", \"city\": \"Danvers\", \"province\": \"MA\", \"country\": \"United States\", \"current_version\": \"v1.0\", \"contact_name\": \"EditorPhosphoSite\", \"contact_email\": \"EditorPhosphoSite@cellsignal.com\", \"available_protocol\": 1.0, \"last_update\": \"2014-12\", \"age\": 11, \"citation\": 2244, \"token\": \"$2a$10$MYHS.0iaIuxOeZpCKWKy9eYp6Pmq9KT27hilryN39UeTMBqmByE6a\", \"expire_date\": 1543898885234.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:43\", \"data_type_list\": \"Protein\", \"category_list\": \"Modification\", \"keywords_list\": \"mass spectrometry experiment, post-translational modification\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 204.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:04\", \"ess\": 0}, {\"page_id\": \"57_data_entity\", \"is_success\": true, \"error_count\": 9.0, \"label_missing\": 8.0, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 57, \"page_type\": \"data_entity\", \"source_id\": \"dc_71\", \"short_name\": \"PhosphoSitePlus\", \"accession\": \"DBC000071\", \"full_name\": null, \"description\": \"PhosphoSitePlus(\\u00ae) (PSP), a knowledgebase dedicated to mammalian post-translational modifications (PTMs), contains over 330,000 non-redundant PTMs, including phospho, acetyl, ubiquityl and methyl groups.\", \"founded_year\": 2012, \"host_institution\": \"Cell Signaling Technology\", \"address\": \"Cell Signaling Technology,3 Trask Lane,Danvers,MA 01923,USA\", \"city\": \"Danvers\", \"province\": \"MA\", \"country\": \"United States\", \"current_version\": \"v1.0\", \"contact_name\": \"EditorPhosphoSite\", \"contact_email\": \"EditorPhosphoSite@cellsignal.com\", \"available_protocol\": 1.0, \"last_update\": \"2014-12\", \"age\": 11, \"citation\": 2244, \"token\": \"$2a$10$MYHS.0iaIuxOeZpCKWKy9eYp6Pmq9KT27hilryN39UeTMBqmByE6a\", \"expire_date\": 1543898885234.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:43\", \"data_type_list\": \"Protein\", \"category_list\": \"Modification\", \"keywords_list\": \"mass spectrometry experiment, post-translational modification\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 204.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:04\", \"ess\": 0}, {\"page_id\": \"57_documentation\", \"is_success\": true, \"error_count\": 2.0, \"label_missing\": null, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": 1.0, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 57, \"page_type\": \"documentation\", \"source_id\": \"dc_71\", \"short_name\": \"PhosphoSitePlus\", \"accession\": \"DBC000071\", \"full_name\": null, \"description\": \"PhosphoSitePlus(\\u00ae) (PSP), a knowledgebase dedicated to mammalian post-translational modifications (PTMs), contains over 330,000 non-redundant PTMs, including phospho, acetyl, ubiquityl and methyl groups.\", \"founded_year\": 2012, \"host_institution\": \"Cell Signaling Technology\", \"address\": \"Cell Signaling Technology,3 Trask Lane,Danvers,MA 01923,USA\", \"city\": \"Danvers\", \"province\": \"MA\", \"country\": \"United States\", \"current_version\": \"v1.0\", \"contact_name\": \"EditorPhosphoSite\", \"contact_email\": \"EditorPhosphoSite@cellsignal.com\", \"available_protocol\": 1.0, \"last_update\": \"2014-12\", \"age\": 11, \"citation\": 2244, \"token\": \"$2a$10$MYHS.0iaIuxOeZpCKWKy9eYp6Pmq9KT27hilryN39UeTMBqmByE6a\", \"expire_date\": 1543898885234.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:43\", \"data_type_list\": \"Protein\", \"category_list\": \"Modification\", \"keywords_list\": \"mass spectrometry experiment, post-translational modification\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 204.0, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:04\", \"ess\": 0}, {\"page_id\": \"58_documentation\", \"is_success\": true, \"error_count\": 5.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": 4.0, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 58, \"page_type\": \"documentation\", \"source_id\": \"dc_1077\", \"short_name\": \"MSigDB\", \"accession\": \"DBC001077\", \"full_name\": \"Molecular Signatures Database\", \"description\": \"The Molecular Signatures Database (MSigDB) is a collection of annotated gene sets for use with GSEA software.The Molecular(MSigDB) is a collection of annotated gene sets for use with GSEA software.\", \"founded_year\": 2011, \"host_institution\": \"Broad Institute\", \"address\": \"320 Charles Street, Cambridge, MA\", \"city\": \"Cambridge\", \"province\": \"MA\", \"country\": \"United States\", \"current_version\": \"5.1\", \"contact_name\": \"Mesirov JP\", \"contact_email\": \"gsea@broadinstitute.org\", \"available_protocol\": 1.0, \"last_update\": \"2016-01-01\", \"age\": 12, \"citation\": 2375, \"token\": \"$2a$10$1R84HHbv9wMtdEZCSkzyWulsOC7I2SGgy5X9xLBqxPX5pSgl9JNR2\", \"expire_date\": 1543975626640.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1435653470000, \"last_modified\": \"2023-03-03 10:10:57\", \"data_type_list\": \"DNA\", \"category_list\": \"Gene genome and annotation\", \"keywords_list\": \"GSEA, gene annotation\", \"data_object_list\": null, \"organism_list\": null, \"theme_list\": null, \"zindex\": 197.917, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:31\", \"ess\": 0}, {\"page_id\": \"59_search\", \"is_success\": true, \"error_count\": 4.0, \"label_missing\": null, \"language_missing\": 1.0, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": 1.0, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": 1.0, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 59, \"page_type\": \"search\", \"source_id\": \"dc_7903\", \"short_name\": \"CellChat\", \"accession\": \"DBC007903\", \"full_name\": null, \"description\": \"CellChat Explorer contains two major components: (a) Ligand-Receptor Interaction Explorer that allows easy exploration of our ligand-receptor interaction database, and (b) Cell- Cell Communication Atlas Explorer that allows easy exploration of the cell-cell communications for any given scRNA-seq dataset that has been processed by the R toolkit CellChat.\", \"founded_year\": 2021, \"host_institution\": \"University of California Irvine\", \"address\": null, \"city\": null, \"province\": null, \"country\": \"United States\", \"current_version\": null, \"contact_name\": \"Qing Nie\", \"contact_email\": \"qnie@uci.edu\", \"available_protocol\": 1.0, \"last_update\": null, \"age\": 2, \"citation\": 393, \"token\": null, \"expire_date\": null, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1650615744000, \"last_modified\": \"2023-10-08 23:06:13\", \"data_type_list\": \"RNA, Protein\", \"category_list\": \"Interaction, Pathway, Expression\", \"keywords_list\": \"scRNA-seq, ligand-receptor, cell- cell communication\", \"data_object_list\": \"Animal\", \"organism_list\": \"Homo sapiens, Mus musculus\", \"theme_list\": null, \"zindex\": 196.5, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:02:58\", \"ess\": 0}, {\"page_id\": \"60_search_result\", \"is_success\": true, \"error_count\": 2.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 1.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 60, \"page_type\": \"search_result\", \"source_id\": \"dc_36\", \"short_name\": \"SMART\", \"accession\": \"DBC000036\", \"full_name\": \"Simple Modular Architecture Research Tool\", \"description\": \"A web resource providing simple identification and extensive annotation of protein domains and the exploration of protein domain architectures\", \"founded_year\": 1998, \"host_institution\": \"European Molecular Biology Laboratory\", \"address\": \"Bothestr 142,69126 Heidelberg,Germany\", \"city\": \"Heidelberg\", \"province\": null, \"country\": \"Germany\", \"current_version\": \"v8.0\", \"contact_name\": \"Peer Bork\", \"contact_email\": \"bork@embl.de\", \"available_protocol\": 1.0, \"last_update\": \"2017\", \"age\": 25, \"citation\": 4893, \"token\": \"$2a$10$t5ywnrRp/KNgAGjUZb0PrOqH1MFYfkgJASFLjOFy.MxoByOMkGOU2\", \"expire_date\": 1543898859394.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:40\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation, Structure\", \"keywords_list\": \"protein domain\", \"data_object_list\": \"Animal, Plant\", \"organism_list\": null, \"theme_list\": null, \"zindex\": 195.72, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:03\", \"ess\": 0}, {\"page_id\": \"60_data_entity\", \"is_success\": true, \"error_count\": 3.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": 2.0, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 60, \"page_type\": \"data_entity\", \"source_id\": \"dc_36\", \"short_name\": \"SMART\", \"accession\": \"DBC000036\", \"full_name\": \"Simple Modular Architecture Research Tool\", \"description\": \"A web resource providing simple identification and extensive annotation of protein domains and the exploration of protein domain architectures\", \"founded_year\": 1998, \"host_institution\": \"European Molecular Biology Laboratory\", \"address\": \"Bothestr 142,69126 Heidelberg,Germany\", \"city\": \"Heidelberg\", \"province\": null, \"country\": \"Germany\", \"current_version\": \"v8.0\", \"contact_name\": \"Peer Bork\", \"contact_email\": \"bork@embl.de\", \"available_protocol\": 1.0, \"last_update\": \"2017\", \"age\": 25, \"citation\": 4893, \"token\": \"$2a$10$t5ywnrRp/KNgAGjUZb0PrOqH1MFYfkgJASFLjOFy.MxoByOMkGOU2\", \"expire_date\": 1543898859394.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:40\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation, Structure\", \"keywords_list\": \"protein domain\", \"data_object_list\": \"Animal, Plant\", \"organism_list\": null, \"theme_list\": null, \"zindex\": 195.72, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:03\", \"ess\": 0}, {\"page_id\": \"60_documentation\", \"is_success\": true, \"error_count\": 1.0, \"label_missing\": 1.0, \"language_missing\": null, \"button_empty\": null, \"alt_missing\": null, \"link_empty\": null, \"aria_menu_broken\": null, \"alt_link_missing\": null, \"th_empty\": null, \"heading_empty\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"title_invalid\": null, \"aria_reference_broken\": null, \"alt_input_missing\": null, \"alt_area_missing\": null, \"alt_map_missing\": null, \"label_empty\": null, \"label_multiple\": null, \"id\": 60, \"page_type\": \"documentation\", \"source_id\": \"dc_36\", \"short_name\": \"SMART\", \"accession\": \"DBC000036\", \"full_name\": \"Simple Modular Architecture Research Tool\", \"description\": \"A web resource providing simple identification and extensive annotation of protein domains and the exploration of protein domain architectures\", \"founded_year\": 1998, \"host_institution\": \"European Molecular Biology Laboratory\", \"address\": \"Bothestr 142,69126 Heidelberg,Germany\", \"city\": \"Heidelberg\", \"province\": null, \"country\": \"Germany\", \"current_version\": \"v8.0\", \"contact_name\": \"Peer Bork\", \"contact_email\": \"bork@embl.de\", \"available_protocol\": 1.0, \"last_update\": \"2017\", \"age\": 25, \"citation\": 4893, \"token\": \"$2a$10$t5ywnrRp/KNgAGjUZb0PrOqH1MFYfkgJASFLjOFy.MxoByOMkGOU2\", \"expire_date\": 1543898859394.0, \"ins\": 0, \"group_email\": null, \"is_partner\": 0, \"funding\": NaN, \"submitter\": null, \"is_new\": 0, \"bigsearch_id\": NaN, \"record_created\": 1434784535000, \"last_modified\": \"2023-03-03 10:09:40\", \"data_type_list\": \"Protein\", \"category_list\": \"Gene genome and annotation, Structure\", \"keywords_list\": \"protein domain\", \"data_object_list\": \"Animal, Plant\", \"organism_list\": null, \"theme_list\": null, \"zindex\": 195.72, \"first_publication_year\": 0, \"search_example\": NaN, \"cited_date\": \"2022-12-04 00:00:03\", \"ess\": 0}]}}, {\"mode\": \"vega-lite\"});\n",
       "</script>"
      ],
      "text/plain": [
       "alt.HConcatChart(...)"
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "alt.Chart(df).mark_bar().encode(\n",
    "    alt.Y('sum(error_count):Q', title='A Total of Error Counts'),\n",
    "    alt.X('page_type:N', title='Page Type'),\n",
    ") | alt.Chart(df).mark_point().encode(\n",
    "    alt.Y('error_count'),\n",
    "    alt.X('citation:Q'),\n",
    ") | alt.Chart(df).mark_bar().encode(\n",
    "    alt.Y('mean(error_count):Q', title='The Average of Error Counts'),\n",
    "    alt.X('country:N'),\n",
    ") | alt.Chart(df).mark_bar().encode(\n",
    "    alt.Y('count(error_count):Q', title='The Number of Data Portals'),\n",
    "    alt.X('country:N'),\n",
    ") | alt.Chart(df).mark_bar().encode(\n",
    "    alt.Y('mean(error_count):Q', title='The Average of Error Counts'),\n",
    "    alt.X('founded_year:O'),\n",
    ")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Journals\n",
    "#### Basic Data Wrangling"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [],
   "source": [
    "\"\"\"\n",
    "Files available in GIST.\n",
    "You can find the following files in the GitHub repository as well.\n",
    "\"\"\"\n",
    "# Contains the accessibility issues for each page\n",
    "URL_EVALUATION_RESULTS = 'https://gist.githubusercontent.com/sehilyi/45ec1eb9115f969279d30fd1734b50e8/raw/68e91ff90b66df05ce4396acefb664c0c1272916/journal-portal_evaluation.csv'\n",
    "\n",
    "# Contains the page types and URLs for each resource\n",
    "URL_PAGES = 'https://gist.githubusercontent.com/sehilyi/45ec1eb9115f969279d30fd1734b50e8/raw/68e91ff90b66df05ce4396acefb664c0c1272916/journal-portal_pages.csv'\n",
    "\n",
    "# Contains the metadata for each resource\n",
    "URL_METADATA = 'https://gist.githubusercontent.com/sehilyi/45ec1eb9115f969279d30fd1734b50e8/raw/68e91ff90b66df05ce4396acefb664c0c1272916/journal-portal_metadata.csv'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>page_id</th>\n",
       "      <th>is_success</th>\n",
       "      <th>error_count</th>\n",
       "      <th>aria_reference_broken</th>\n",
       "      <th>label_multiple</th>\n",
       "      <th>alt_missing</th>\n",
       "      <th>alt_link_missing</th>\n",
       "      <th>label_empty</th>\n",
       "      <th>button_empty</th>\n",
       "      <th>link_empty</th>\n",
       "      <th>...</th>\n",
       "      <th>total_cites_3years</th>\n",
       "      <th>citable_docs_3years</th>\n",
       "      <th>cites_per_doc_2years</th>\n",
       "      <th>ref_per_doc</th>\n",
       "      <th>country</th>\n",
       "      <th>region</th>\n",
       "      <th>publisher</th>\n",
       "      <th>coverage</th>\n",
       "      <th>categories</th>\n",
       "      <th>areas</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>12_home</td>\n",
       "      <td>True</td>\n",
       "      <td>0.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>168543</td>\n",
       "      <td>3565</td>\n",
       "      <td>23,95</td>\n",
       "      <td>21,59</td>\n",
       "      <td>United Kingdom</td>\n",
       "      <td>Western Europe</td>\n",
       "      <td>Nature Publishing Group</td>\n",
       "      <td>1869-2022</td>\n",
       "      <td>Multidisciplinary (Q1)</td>\n",
       "      <td>Multidisciplinary</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>12_research_article</td>\n",
       "      <td>True</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>168543</td>\n",
       "      <td>3565</td>\n",
       "      <td>23,95</td>\n",
       "      <td>21,59</td>\n",
       "      <td>United Kingdom</td>\n",
       "      <td>Western Europe</td>\n",
       "      <td>Nature Publishing Group</td>\n",
       "      <td>1869-2022</td>\n",
       "      <td>Multidisciplinary (Q1)</td>\n",
       "      <td>Multidisciplinary</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>12_none_research_article</td>\n",
       "      <td>True</td>\n",
       "      <td>2.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>168543</td>\n",
       "      <td>3565</td>\n",
       "      <td>23,95</td>\n",
       "      <td>21,59</td>\n",
       "      <td>United Kingdom</td>\n",
       "      <td>Western Europe</td>\n",
       "      <td>Nature Publishing Group</td>\n",
       "      <td>1869-2022</td>\n",
       "      <td>Multidisciplinary (Q1)</td>\n",
       "      <td>Multidisciplinary</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>3 rows Ã— 41 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                    page_id  is_success  error_count  aria_reference_broken  \\\n",
       "0                   12_home        True          0.0                    NaN   \n",
       "1       12_research_article        True          1.0                    1.0   \n",
       "2  12_none_research_article        True          2.0                    NaN   \n",
       "\n",
       "   label_multiple  alt_missing  alt_link_missing  label_empty  button_empty  \\\n",
       "0             NaN          NaN               NaN          NaN           NaN   \n",
       "1             NaN          NaN               NaN          NaN           NaN   \n",
       "2             2.0          NaN               NaN          NaN           NaN   \n",
       "\n",
       "   link_empty  ...  total_cites_3years  citable_docs_3years  \\\n",
       "0         NaN  ...              168543                 3565   \n",
       "1         NaN  ...              168543                 3565   \n",
       "2         NaN  ...              168543                 3565   \n",
       "\n",
       "   cites_per_doc_2years  ref_per_doc         country          region  \\\n",
       "0                 23,95        21,59  United Kingdom  Western Europe   \n",
       "1                 23,95        21,59  United Kingdom  Western Europe   \n",
       "2                 23,95        21,59  United Kingdom  Western Europe   \n",
       "\n",
       "                 publisher   coverage              categories  \\\n",
       "0  Nature Publishing Group  1869-2022  Multidisciplinary (Q1)   \n",
       "1  Nature Publishing Group  1869-2022  Multidisciplinary (Q1)   \n",
       "2  Nature Publishing Group  1869-2022  Multidisciplinary (Q1)   \n",
       "\n",
       "               areas  \n",
       "0  Multidisciplinary  \n",
       "1  Multidisciplinary  \n",
       "2  Multidisciplinary  \n",
       "\n",
       "[3 rows x 41 columns]"
      ]
     },
     "execution_count": 29,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\"\"\"\n",
    "Merge data for visualization\n",
    "\"\"\"\n",
    "df = pd.read_csv(URL_EVALUATION_RESULTS)\n",
    "\n",
    "# Can be useful for visualization\n",
    "ERROR_TYPES = df.columns.tolist()\n",
    "ERROR_TYPES.remove('page_id')\n",
    "ERROR_TYPES.remove('is_success')\n",
    "ERROR_TYPES.remove('error_count')\n",
    "\n",
    "# Add `id` of resources and `page_type` of pages\n",
    "df_pages = pd.read_csv(URL_PAGES)\n",
    "df = df.merge(df_pages[['id', 'page_id', 'page_type']], left_on='page_id', right_on='page_id', how='left')\n",
    "\n",
    "# Add metadata of resources\n",
    "df_meta = pd.read_csv(URL_METADATA)\n",
    "df_meta.drop(columns=['url'], inplace=True)\n",
    "df = df.merge(df_meta, left_on='id', right_on='id', how='left')\n",
    "\n",
    "df.head(3)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Let's add some useful columns for visualization\n",
    "df['has_error'] = df.error_count.apply(lambda x: x > 0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Index(['page_id', 'is_success', 'error_count', 'aria_reference_broken',\n",
       "       'label_multiple', 'alt_missing', 'alt_link_missing', 'label_empty',\n",
       "       'button_empty', 'link_empty', 'heading_empty', 'alt_input_missing',\n",
       "       'label_missing', 'aria_menu_broken', 'link_skip_broken',\n",
       "       'alt_spacer_missing', 'th_empty', 'title_invalid', 'id', 'page_type',\n",
       "       'source_id', 'rank', 'title', 'type', 'issn', 'sjr',\n",
       "       'sjr_best_quartile', 'h_index', 'total_docs_2022', 'total_docs_3years',\n",
       "       'total_refs', 'total_cites_3years', 'citable_docs_3years',\n",
       "       'cites_per_doc_2years', 'ref_per_doc', 'country', 'region', 'publisher',\n",
       "       'coverage', 'categories', 'areas', 'has_error'],\n",
       "      dtype='object')"
      ]
     },
     "execution_count": 31,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Available columns\n",
    "df.columns"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Create Altair Plots!\n",
    "Reference: https://altair-viz.github.io/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "<style>\n",
       "  #altair-viz-fed1e56ebd354a4d813733eced9f0fa4.vega-embed {\n",
       "    width: 100%;\n",
       "    display: flex;\n",
       "  }\n",
       "\n",
       "  #altair-viz-fed1e56ebd354a4d813733eced9f0fa4.vega-embed details,\n",
       "  #altair-viz-fed1e56ebd354a4d813733eced9f0fa4.vega-embed details summary {\n",
       "    position: relative;\n",
       "  }\n",
       "</style>\n",
       "<div id=\"altair-viz-fed1e56ebd354a4d813733eced9f0fa4\"></div>\n",
       "<script type=\"text/javascript\">\n",
       "  var VEGA_DEBUG = (typeof VEGA_DEBUG == \"undefined\") ? {} : VEGA_DEBUG;\n",
       "  (function(spec, embedOpt){\n",
       "    let outputDiv = document.currentScript.previousElementSibling;\n",
       "    if (outputDiv.id !== \"altair-viz-fed1e56ebd354a4d813733eced9f0fa4\") {\n",
       "      outputDiv = document.getElementById(\"altair-viz-fed1e56ebd354a4d813733eced9f0fa4\");\n",
       "    }\n",
       "    const paths = {\n",
       "      \"vega\": \"https://cdn.jsdelivr.net/npm/vega@5?noext\",\n",
       "      \"vega-lib\": \"https://cdn.jsdelivr.net/npm/vega-lib?noext\",\n",
       "      \"vega-lite\": \"https://cdn.jsdelivr.net/npm/vega-lite@5.15.1?noext\",\n",
       "      \"vega-embed\": \"https://cdn.jsdelivr.net/npm/vega-embed@6?noext\",\n",
       "    };\n",
       "\n",
       "    function maybeLoadScript(lib, version) {\n",
       "      var key = `${lib.replace(\"-\", \"\")}_version`;\n",
       "      return (VEGA_DEBUG[key] == version) ?\n",
       "        Promise.resolve(paths[lib]) :\n",
       "        new Promise(function(resolve, reject) {\n",
       "          var s = document.createElement('script');\n",
       "          document.getElementsByTagName(\"head\")[0].appendChild(s);\n",
       "          s.async = true;\n",
       "          s.onload = () => {\n",
       "            VEGA_DEBUG[key] = version;\n",
       "            return resolve(paths[lib]);\n",
       "          };\n",
       "          s.onerror = () => reject(`Error loading script: ${paths[lib]}`);\n",
       "          s.src = paths[lib];\n",
       "        });\n",
       "    }\n",
       "\n",
       "    function showError(err) {\n",
       "      outputDiv.innerHTML = `<div class=\"error\" style=\"color:red;\">${err}</div>`;\n",
       "      throw err;\n",
       "    }\n",
       "\n",
       "    function displayChart(vegaEmbed) {\n",
       "      vegaEmbed(outputDiv, spec, embedOpt)\n",
       "        .catch(err => showError(`Javascript Error: ${err.message}<br>This usually means there's a typo in your chart specification. See the javascript console for the full traceback.`));\n",
       "    }\n",
       "\n",
       "    if(typeof define === \"function\" && define.amd) {\n",
       "      requirejs.config({paths});\n",
       "      require([\"vega-embed\"], displayChart, err => showError(`Error loading script: ${err.message}`));\n",
       "    } else {\n",
       "      maybeLoadScript(\"vega\", \"5\")\n",
       "        .then(() => maybeLoadScript(\"vega-lite\", \"5.15.1\"))\n",
       "        .then(() => maybeLoadScript(\"vega-embed\", \"6\"))\n",
       "        .catch(showError)\n",
       "        .then(() => displayChart(vegaEmbed));\n",
       "    }\n",
       "  })({\"config\": {\"view\": {\"continuousWidth\": 300, \"continuousHeight\": 300}}, \"hconcat\": [{\"mark\": {\"type\": \"bar\"}, \"encoding\": {\"x\": {\"field\": \"page_type\", \"title\": \"Page Type\", \"type\": \"nominal\"}, \"y\": {\"aggregate\": \"sum\", \"field\": \"error_count\", \"title\": \"A Total of Error Counts\", \"type\": \"quantitative\"}}}, {\"mark\": {\"type\": \"point\"}, \"encoding\": {\"x\": {\"field\": \"total_cites_3years\", \"type\": \"quantitative\"}, \"y\": {\"field\": \"error_count\", \"type\": \"quantitative\"}}}, {\"mark\": {\"type\": \"bar\"}, \"encoding\": {\"x\": {\"field\": \"country\", \"type\": \"nominal\"}, \"y\": {\"aggregate\": \"mean\", \"field\": \"error_count\", \"title\": \"The Average of Error Counts\", \"type\": \"quantitative\"}}}, {\"mark\": {\"type\": \"bar\"}, \"encoding\": {\"x\": {\"field\": \"areas\", \"type\": \"nominal\"}, \"y\": {\"aggregate\": \"count\", \"field\": \"error_count\", \"title\": \"The Number of Data Portals\", \"type\": \"quantitative\"}}}, {\"mark\": {\"type\": \"bar\"}, \"encoding\": {\"x\": {\"field\": \"publisher\", \"type\": \"nominal\"}, \"y\": {\"aggregate\": \"mean\", \"field\": \"alt_missing\", \"title\": \"The Average of Alt-text Missing\", \"type\": \"quantitative\"}}}], \"data\": {\"name\": \"data-a6e7cc5f9635543a3d663375f1c74cf7\"}, \"$schema\": \"https://vega.github.io/schema/vega-lite/v5.15.1.json\", \"datasets\": {\"data-a6e7cc5f9635543a3d663375f1c74cf7\": [{\"page_id\": \"12_home\", \"is_success\": true, \"error_count\": 0.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 12, \"page_type\": \"home\", \"source_id\": \"sjr_21206\", \"rank\": 12, \"title\": \"Nature\", \"type\": \"journal\", \"issn\": \"14764687, 00280836\", \"sjr\": \"20,957\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 1331, \"total_docs_2022\": 3132, \"total_docs_3years\": 7467, \"total_refs\": 67627, \"total_cites_3years\": 168543, \"citable_docs_3years\": 3565, \"cites_per_doc_2years\": \"23,95\", \"ref_per_doc\": \"21,59\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Nature Publishing Group\", \"coverage\": \"1869-2022\", \"categories\": \"Multidisciplinary (Q1)\", \"areas\": \"Multidisciplinary\", \"has_error\": false}, {\"page_id\": \"12_research_article\", \"is_success\": true, \"error_count\": 1.0, \"aria_reference_broken\": 1.0, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 12, \"page_type\": \"research_article\", \"source_id\": \"sjr_21206\", \"rank\": 12, \"title\": \"Nature\", \"type\": \"journal\", \"issn\": \"14764687, 00280836\", \"sjr\": \"20,957\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 1331, \"total_docs_2022\": 3132, \"total_docs_3years\": 7467, \"total_refs\": 67627, \"total_cites_3years\": 168543, \"citable_docs_3years\": 3565, \"cites_per_doc_2years\": \"23,95\", \"ref_per_doc\": \"21,59\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Nature Publishing Group\", \"coverage\": \"1869-2022\", \"categories\": \"Multidisciplinary (Q1)\", \"areas\": \"Multidisciplinary\", \"has_error\": true}, {\"page_id\": \"12_none_research_article\", \"is_success\": true, \"error_count\": 2.0, \"aria_reference_broken\": null, \"label_multiple\": 2.0, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 12, \"page_type\": \"none_research_article\", \"source_id\": \"sjr_21206\", \"rank\": 12, \"title\": \"Nature\", \"type\": \"journal\", \"issn\": \"14764687, 00280836\", \"sjr\": \"20,957\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 1331, \"total_docs_2022\": 3132, \"total_docs_3years\": 7467, \"total_refs\": 67627, \"total_cites_3years\": 168543, \"citable_docs_3years\": 3565, \"cites_per_doc_2years\": \"23,95\", \"ref_per_doc\": \"21,59\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Nature Publishing Group\", \"coverage\": \"1869-2022\", \"categories\": \"Multidisciplinary (Q1)\", \"areas\": \"Multidisciplinary\", \"has_error\": true}, {\"page_id\": \"12_article_search_result\", \"is_success\": true, \"error_count\": 0.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 12, \"page_type\": \"article_search_result\", \"source_id\": \"sjr_21206\", \"rank\": 12, \"title\": \"Nature\", \"type\": \"journal\", \"issn\": \"14764687, 00280836\", \"sjr\": \"20,957\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 1331, \"total_docs_2022\": 3132, \"total_docs_3years\": 7467, \"total_refs\": 67627, \"total_cites_3years\": 168543, \"citable_docs_3years\": 3565, \"cites_per_doc_2years\": \"23,95\", \"ref_per_doc\": \"21,59\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Nature Publishing Group\", \"coverage\": \"1869-2022\", \"categories\": \"Multidisciplinary (Q1)\", \"areas\": \"Multidisciplinary\", \"has_error\": false}, {\"page_id\": \"12_latest_issue\", \"is_success\": true, \"error_count\": 0.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 12, \"page_type\": \"latest_issue\", \"source_id\": \"sjr_21206\", \"rank\": 12, \"title\": \"Nature\", \"type\": \"journal\", \"issn\": \"14764687, 00280836\", \"sjr\": \"20,957\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 1331, \"total_docs_2022\": 3132, \"total_docs_3years\": 7467, \"total_refs\": 67627, \"total_cites_3years\": 168543, \"citable_docs_3years\": 3565, \"cites_per_doc_2years\": \"23,95\", \"ref_per_doc\": \"21,59\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Nature Publishing Group\", \"coverage\": \"1869-2022\", \"categories\": \"Multidisciplinary (Q1)\", \"areas\": \"Multidisciplinary\", \"has_error\": false}, {\"page_id\": \"41_home\", \"is_success\": true, \"error_count\": 34.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": 12.0, \"alt_link_missing\": 7.0, \"label_empty\": 2.0, \"button_empty\": 10.0, \"link_empty\": 3.0, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 41, \"page_type\": \"home\", \"source_id\": \"sjr_23571\", \"rank\": 41, \"title\": \"Science\", \"type\": \"journal\", \"issn\": \"10959203, 00368075\", \"sjr\": \"13,328\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 1283, \"total_docs_2022\": 2066, \"total_docs_3years\": 9273, \"total_refs\": 40993, \"total_cites_3years\": 130524, \"citable_docs_3years\": 4211, \"cites_per_doc_2years\": \"14,90\", \"ref_per_doc\": \"19,84\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"American Association for the Advancement of Science\", \"coverage\": \"1880-1881, 1883-2022\", \"categories\": \"History and Philosophy of Science (Q1); Multidisciplinary (Q1)\", \"areas\": \"Arts and Humanities; Multidisciplinary\", \"has_error\": true}, {\"page_id\": \"41_research_article\", \"is_success\": true, \"error_count\": 20.0, \"aria_reference_broken\": 2.0, \"label_multiple\": null, \"alt_missing\": 7.0, \"alt_link_missing\": 1.0, \"label_empty\": null, \"button_empty\": 3.0, \"link_empty\": 6.0, \"heading_empty\": 1.0, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 41, \"page_type\": \"research_article\", \"source_id\": \"sjr_23571\", \"rank\": 41, \"title\": \"Science\", \"type\": \"journal\", \"issn\": \"10959203, 00368075\", \"sjr\": \"13,328\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 1283, \"total_docs_2022\": 2066, \"total_docs_3years\": 9273, \"total_refs\": 40993, \"total_cites_3years\": 130524, \"citable_docs_3years\": 4211, \"cites_per_doc_2years\": \"14,90\", \"ref_per_doc\": \"19,84\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"American Association for the Advancement of Science\", \"coverage\": \"1880-1881, 1883-2022\", \"categories\": \"History and Philosophy of Science (Q1); Multidisciplinary (Q1)\", \"areas\": \"Arts and Humanities; Multidisciplinary\", \"has_error\": true}, {\"page_id\": \"41_none_research_article\", \"is_success\": true, \"error_count\": 15.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": 9.0, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": 3.0, \"link_empty\": 3.0, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 41, \"page_type\": \"none_research_article\", \"source_id\": \"sjr_23571\", \"rank\": 41, \"title\": \"Science\", \"type\": \"journal\", \"issn\": \"10959203, 00368075\", \"sjr\": \"13,328\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 1283, \"total_docs_2022\": 2066, \"total_docs_3years\": 9273, \"total_refs\": 40993, \"total_cites_3years\": 130524, \"citable_docs_3years\": 4211, \"cites_per_doc_2years\": \"14,90\", \"ref_per_doc\": \"19,84\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"American Association for the Advancement of Science\", \"coverage\": \"1880-1881, 1883-2022\", \"categories\": \"History and Philosophy of Science (Q1); Multidisciplinary (Q1)\", \"areas\": \"Arts and Humanities; Multidisciplinary\", \"has_error\": true}, {\"page_id\": \"41_article_search_result\", \"is_success\": true, \"error_count\": 14.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": 8.0, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": 3.0, \"link_empty\": 3.0, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 41, \"page_type\": \"article_search_result\", \"source_id\": \"sjr_23571\", \"rank\": 41, \"title\": \"Science\", \"type\": \"journal\", \"issn\": \"10959203, 00368075\", \"sjr\": \"13,328\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 1283, \"total_docs_2022\": 2066, \"total_docs_3years\": 9273, \"total_refs\": 40993, \"total_cites_3years\": 130524, \"citable_docs_3years\": 4211, \"cites_per_doc_2years\": \"14,90\", \"ref_per_doc\": \"19,84\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"American Association for the Advancement of Science\", \"coverage\": \"1880-1881, 1883-2022\", \"categories\": \"History and Philosophy of Science (Q1); Multidisciplinary (Q1)\", \"areas\": \"Arts and Humanities; Multidisciplinary\", \"has_error\": true}, {\"page_id\": \"41_latest_issue\", \"is_success\": true, \"error_count\": 23.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": 12.0, \"alt_link_missing\": 1.0, \"label_empty\": 2.0, \"button_empty\": 5.0, \"link_empty\": 3.0, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 41, \"page_type\": \"latest_issue\", \"source_id\": \"sjr_23571\", \"rank\": 41, \"title\": \"Science\", \"type\": \"journal\", \"issn\": \"10959203, 00368075\", \"sjr\": \"13,328\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 1283, \"total_docs_2022\": 2066, \"total_docs_3years\": 9273, \"total_refs\": 40993, \"total_cites_3years\": 130524, \"citable_docs_3years\": 4211, \"cites_per_doc_2years\": \"14,90\", \"ref_per_doc\": \"19,84\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"American Association for the Advancement of Science\", \"coverage\": \"1880-1881, 1883-2022\", \"categories\": \"History and Philosophy of Science (Q1); Multidisciplinary (Q1)\", \"areas\": \"Arts and Humanities; Multidisciplinary\", \"has_error\": true}, {\"page_id\": \"5_home\", \"is_success\": true, \"error_count\": 50.0, \"aria_reference_broken\": 5.0, \"label_multiple\": null, \"alt_missing\": 25.0, \"alt_link_missing\": 2.0, \"label_empty\": null, \"button_empty\": null, \"link_empty\": 12.0, \"heading_empty\": null, \"alt_input_missing\": 1.0, \"label_missing\": 3.0, \"aria_menu_broken\": 2.0, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 5, \"page_type\": \"home\", \"source_id\": \"sjr_15847\", \"rank\": 5, \"title\": \"New England Journal of Medicine\", \"type\": \"journal\", \"issn\": \"00284793, 15334406\", \"sjr\": \"26,015\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 1130, \"total_docs_2022\": 1410, \"total_docs_3years\": 4561, \"total_refs\": 14396, \"total_cites_3years\": 133956, \"citable_docs_3years\": 1854, \"cites_per_doc_2years\": \"33,93\", \"ref_per_doc\": \"10,21\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Massachussetts Medical Society\", \"coverage\": \"1945-2022\", \"categories\": \"Medicine (miscellaneous) (Q1)\", \"areas\": \"Medicine\", \"has_error\": true}, {\"page_id\": \"5_research_article\", \"is_success\": true, \"error_count\": 33.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": 23.0, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": 5.0, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": 3.0, \"aria_menu_broken\": 2.0, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 5, \"page_type\": \"research_article\", \"source_id\": \"sjr_15847\", \"rank\": 5, \"title\": \"New England Journal of Medicine\", \"type\": \"journal\", \"issn\": \"00284793, 15334406\", \"sjr\": \"26,015\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 1130, \"total_docs_2022\": 1410, \"total_docs_3years\": 4561, \"total_refs\": 14396, \"total_cites_3years\": 133956, \"citable_docs_3years\": 1854, \"cites_per_doc_2years\": \"33,93\", \"ref_per_doc\": \"10,21\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Massachussetts Medical Society\", \"coverage\": \"1945-2022\", \"categories\": \"Medicine (miscellaneous) (Q1)\", \"areas\": \"Medicine\", \"has_error\": true}, {\"page_id\": \"5_none_research_article\", \"is_success\": true, \"error_count\": 32.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": 22.0, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": 5.0, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": 3.0, \"aria_menu_broken\": 2.0, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 5, \"page_type\": \"none_research_article\", \"source_id\": \"sjr_15847\", \"rank\": 5, \"title\": \"New England Journal of Medicine\", \"type\": \"journal\", \"issn\": \"00284793, 15334406\", \"sjr\": \"26,015\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 1130, \"total_docs_2022\": 1410, \"total_docs_3years\": 4561, \"total_refs\": 14396, \"total_cites_3years\": 133956, \"citable_docs_3years\": 1854, \"cites_per_doc_2years\": \"33,93\", \"ref_per_doc\": \"10,21\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Massachussetts Medical Society\", \"coverage\": \"1945-2022\", \"categories\": \"Medicine (miscellaneous) (Q1)\", \"areas\": \"Medicine\", \"has_error\": true}, {\"page_id\": \"5_article_search_result\", \"is_success\": false, \"error_count\": null, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": 22.0, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": 5.0, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": 3.0, \"aria_menu_broken\": 2.0, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 5, \"page_type\": \"article_search_result\", \"source_id\": \"sjr_15847\", \"rank\": 5, \"title\": \"New England Journal of Medicine\", \"type\": \"journal\", \"issn\": \"00284793, 15334406\", \"sjr\": \"26,015\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 1130, \"total_docs_2022\": 1410, \"total_docs_3years\": 4561, \"total_refs\": 14396, \"total_cites_3years\": 133956, \"citable_docs_3years\": 1854, \"cites_per_doc_2years\": \"33,93\", \"ref_per_doc\": \"10,21\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Massachussetts Medical Society\", \"coverage\": \"1945-2022\", \"categories\": \"Medicine (miscellaneous) (Q1)\", \"areas\": \"Medicine\", \"has_error\": false}, {\"page_id\": \"5_latest_issue\", \"is_success\": true, \"error_count\": 89.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": 22.0, \"alt_link_missing\": 1.0, \"label_empty\": null, \"button_empty\": 19.0, \"link_empty\": 42.0, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": 3.0, \"aria_menu_broken\": 2.0, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 5, \"page_type\": \"latest_issue\", \"source_id\": \"sjr_15847\", \"rank\": 5, \"title\": \"New England Journal of Medicine\", \"type\": \"journal\", \"issn\": \"00284793, 15334406\", \"sjr\": \"26,015\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 1130, \"total_docs_2022\": 1410, \"total_docs_3years\": 4561, \"total_refs\": 14396, \"total_cites_3years\": 133956, \"citable_docs_3years\": 1854, \"cites_per_doc_2years\": \"33,93\", \"ref_per_doc\": \"10,21\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Massachussetts Medical Society\", \"coverage\": \"1945-2022\", \"categories\": \"Medicine (miscellaneous) (Q1)\", \"areas\": \"Medicine\", \"has_error\": true}, {\"page_id\": \"4_home\", \"is_success\": true, \"error_count\": 5.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": 1.0, \"label_empty\": null, \"button_empty\": 2.0, \"link_empty\": 1.0, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": 1.0, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 4, \"page_type\": \"home\", \"source_id\": \"sjr_18434\", \"rank\": 4, \"title\": \"Cell\", \"type\": \"journal\", \"issn\": \"00928674, 10974172\", \"sjr\": \"26,494\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 856, \"total_docs_2022\": 420, \"total_docs_3years\": 1637, \"total_refs\": 27609, \"total_cites_3years\": 67791, \"citable_docs_3years\": 1440, \"cites_per_doc_2years\": \"43,80\", \"ref_per_doc\": \"65,74\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Cell Press\", \"coverage\": \"1974-2022\", \"categories\": \"Biochemistry, Genetics and Molecular Biology (miscellaneous) (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology\", \"has_error\": true}, {\"page_id\": \"4_research_article\", \"is_success\": true, \"error_count\": 6.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": 1.0, \"alt_link_missing\": 2.0, \"label_empty\": null, \"button_empty\": 2.0, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": 1.0, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 4, \"page_type\": \"research_article\", \"source_id\": \"sjr_18434\", \"rank\": 4, \"title\": \"Cell\", \"type\": \"journal\", \"issn\": \"00928674, 10974172\", \"sjr\": \"26,494\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 856, \"total_docs_2022\": 420, \"total_docs_3years\": 1637, \"total_refs\": 27609, \"total_cites_3years\": 67791, \"citable_docs_3years\": 1440, \"cites_per_doc_2years\": \"43,80\", \"ref_per_doc\": \"65,74\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Cell Press\", \"coverage\": \"1974-2022\", \"categories\": \"Biochemistry, Genetics and Molecular Biology (miscellaneous) (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology\", \"has_error\": true}, {\"page_id\": \"4_none_research_article\", \"is_success\": true, \"error_count\": 7.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": 2.0, \"alt_link_missing\": 2.0, \"label_empty\": null, \"button_empty\": 2.0, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": 1.0, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 4, \"page_type\": \"none_research_article\", \"source_id\": \"sjr_18434\", \"rank\": 4, \"title\": \"Cell\", \"type\": \"journal\", \"issn\": \"00928674, 10974172\", \"sjr\": \"26,494\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 856, \"total_docs_2022\": 420, \"total_docs_3years\": 1637, \"total_refs\": 27609, \"total_cites_3years\": 67791, \"citable_docs_3years\": 1440, \"cites_per_doc_2years\": \"43,80\", \"ref_per_doc\": \"65,74\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Cell Press\", \"coverage\": \"1974-2022\", \"categories\": \"Biochemistry, Genetics and Molecular Biology (miscellaneous) (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology\", \"has_error\": true}, {\"page_id\": \"4_article_search_result\", \"is_success\": true, \"error_count\": 5.0, \"aria_reference_broken\": 1.0, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": 1.0, \"label_empty\": null, \"button_empty\": 2.0, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": 1.0, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 4, \"page_type\": \"article_search_result\", \"source_id\": \"sjr_18434\", \"rank\": 4, \"title\": \"Cell\", \"type\": \"journal\", \"issn\": \"00928674, 10974172\", \"sjr\": \"26,494\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 856, \"total_docs_2022\": 420, \"total_docs_3years\": 1637, \"total_refs\": 27609, \"total_cites_3years\": 67791, \"citable_docs_3years\": 1440, \"cites_per_doc_2years\": \"43,80\", \"ref_per_doc\": \"65,74\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Cell Press\", \"coverage\": \"1974-2022\", \"categories\": \"Biochemistry, Genetics and Molecular Biology (miscellaneous) (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology\", \"has_error\": true}, {\"page_id\": \"4_latest_issue\", \"is_success\": true, \"error_count\": 20.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": 1.0, \"label_empty\": 15.0, \"button_empty\": 2.0, \"link_empty\": 1.0, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": 1.0, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 4, \"page_type\": \"latest_issue\", \"source_id\": \"sjr_18434\", \"rank\": 4, \"title\": \"Cell\", \"type\": \"journal\", \"issn\": \"00928674, 10974172\", \"sjr\": \"26,494\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 856, \"total_docs_2022\": 420, \"total_docs_3years\": 1637, \"total_refs\": 27609, \"total_cites_3years\": 67791, \"citable_docs_3years\": 1440, \"cites_per_doc_2years\": \"43,80\", \"ref_per_doc\": \"65,74\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Cell Press\", \"coverage\": \"1974-2022\", \"categories\": \"Biochemistry, Genetics and Molecular Biology (miscellaneous) (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology\", \"has_error\": true}, {\"page_id\": \"31_home\", \"is_success\": true, \"error_count\": 3.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": 1.0, \"label_empty\": null, \"button_empty\": 2.0, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 31, \"page_type\": \"home\", \"source_id\": \"sjr_16590\", \"rank\": 31, \"title\": \"Lancet, The\", \"type\": \"journal\", \"issn\": \"01406736, 1474547X\", \"sjr\": \"14,607\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 855, \"total_docs_2022\": 1198, \"total_docs_3years\": 4401, \"total_refs\": 20483, \"total_cites_3years\": 80420, \"citable_docs_3years\": 1373, \"cites_per_doc_2years\": \"21,54\", \"ref_per_doc\": \"17,10\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Elsevier Ltd.\", \"coverage\": \"1823-2022\", \"categories\": \"Medicine (miscellaneous) (Q1)\", \"areas\": \"Medicine\", \"has_error\": true}, {\"page_id\": \"31_research_article\", \"is_success\": true, \"error_count\": 6.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": 1.0, \"alt_link_missing\": 2.0, \"label_empty\": null, \"button_empty\": 2.0, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": 1.0, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 31, \"page_type\": \"research_article\", \"source_id\": \"sjr_16590\", \"rank\": 31, \"title\": \"Lancet, The\", \"type\": \"journal\", \"issn\": \"01406736, 1474547X\", \"sjr\": \"14,607\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 855, \"total_docs_2022\": 1198, \"total_docs_3years\": 4401, \"total_refs\": 20483, \"total_cites_3years\": 80420, \"citable_docs_3years\": 1373, \"cites_per_doc_2years\": \"21,54\", \"ref_per_doc\": \"17,10\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Elsevier Ltd.\", \"coverage\": \"1823-2022\", \"categories\": \"Medicine (miscellaneous) (Q1)\", \"areas\": \"Medicine\", \"has_error\": true}, {\"page_id\": \"31_none_research_article\", \"is_success\": true, \"error_count\": 5.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": 2.0, \"label_empty\": null, \"button_empty\": 2.0, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": 1.0, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 31, \"page_type\": \"none_research_article\", \"source_id\": \"sjr_16590\", \"rank\": 31, \"title\": \"Lancet, The\", \"type\": \"journal\", \"issn\": \"01406736, 1474547X\", \"sjr\": \"14,607\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 855, \"total_docs_2022\": 1198, \"total_docs_3years\": 4401, \"total_refs\": 20483, \"total_cites_3years\": 80420, \"citable_docs_3years\": 1373, \"cites_per_doc_2years\": \"21,54\", \"ref_per_doc\": \"17,10\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Elsevier Ltd.\", \"coverage\": \"1823-2022\", \"categories\": \"Medicine (miscellaneous) (Q1)\", \"areas\": \"Medicine\", \"has_error\": true}, {\"page_id\": \"31_article_search_result\", \"is_success\": true, \"error_count\": 5.0, \"aria_reference_broken\": 1.0, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": 1.0, \"label_empty\": null, \"button_empty\": 2.0, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": 1.0, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 31, \"page_type\": \"article_search_result\", \"source_id\": \"sjr_16590\", \"rank\": 31, \"title\": \"Lancet, The\", \"type\": \"journal\", \"issn\": \"01406736, 1474547X\", \"sjr\": \"14,607\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 855, \"total_docs_2022\": 1198, \"total_docs_3years\": 4401, \"total_refs\": 20483, \"total_cites_3years\": 80420, \"citable_docs_3years\": 1373, \"cites_per_doc_2years\": \"21,54\", \"ref_per_doc\": \"17,10\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Elsevier Ltd.\", \"coverage\": \"1823-2022\", \"categories\": \"Medicine (miscellaneous) (Q1)\", \"areas\": \"Medicine\", \"has_error\": true}, {\"page_id\": \"31_latest_issue\", \"is_success\": true, \"error_count\": 5.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": 1.0, \"label_empty\": null, \"button_empty\": 2.0, \"link_empty\": 1.0, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": 1.0, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 31, \"page_type\": \"latest_issue\", \"source_id\": \"sjr_16590\", \"rank\": 31, \"title\": \"Lancet, The\", \"type\": \"journal\", \"issn\": \"01406736, 1474547X\", \"sjr\": \"14,607\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 855, \"total_docs_2022\": 1198, \"total_docs_3years\": 4401, \"total_refs\": 20483, \"total_cites_3years\": 80420, \"citable_docs_3years\": 1373, \"cites_per_doc_2years\": \"21,54\", \"ref_per_doc\": \"17,10\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Elsevier Ltd.\", \"coverage\": \"1823-2022\", \"categories\": \"Medicine (miscellaneous) (Q1)\", \"areas\": \"Medicine\", \"has_error\": true}, {\"page_id\": \"154_home\", \"is_success\": true, \"error_count\": 21.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": 1.0, \"label_empty\": null, \"button_empty\": null, \"link_empty\": 17.0, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": 1.0, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": 2.0, \"th_empty\": null, \"title_invalid\": null, \"id\": 154, \"page_type\": \"home\", \"source_id\": \"sjr_85291\", \"rank\": 154, \"title\": \"JAMA - Journal of the American Medical Association\", \"type\": \"journal\", \"issn\": \"15383598, 00987484, 00029955\", \"sjr\": \"6,695\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 739, \"total_docs_2022\": 1520, \"total_docs_3years\": 4812, \"total_refs\": 13195, \"total_cites_3years\": 54840, \"citable_docs_3years\": 1927, \"cites_per_doc_2years\": \"13,62\", \"ref_per_doc\": \"8,68\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"American Medical Association\", \"coverage\": \"1883-2022\", \"categories\": \"Medicine (miscellaneous) (Q1)\", \"areas\": \"Medicine\", \"has_error\": true}, {\"page_id\": \"154_research_article\", \"is_success\": true, \"error_count\": 9.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": 9.0, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 154, \"page_type\": \"research_article\", \"source_id\": \"sjr_85291\", \"rank\": 154, \"title\": \"JAMA - Journal of the American Medical Association\", \"type\": \"journal\", \"issn\": \"15383598, 00987484, 00029955\", \"sjr\": \"6,695\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 739, \"total_docs_2022\": 1520, \"total_docs_3years\": 4812, \"total_refs\": 13195, \"total_cites_3years\": 54840, \"citable_docs_3years\": 1927, \"cites_per_doc_2years\": \"13,62\", \"ref_per_doc\": \"8,68\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"American Medical Association\", \"coverage\": \"1883-2022\", \"categories\": \"Medicine (miscellaneous) (Q1)\", \"areas\": \"Medicine\", \"has_error\": true}, {\"page_id\": \"154_none_research_article\", \"is_success\": false, \"error_count\": null, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": 9.0, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 154, \"page_type\": \"none_research_article\", \"source_id\": \"sjr_85291\", \"rank\": 154, \"title\": \"JAMA - Journal of the American Medical Association\", \"type\": \"journal\", \"issn\": \"15383598, 00987484, 00029955\", \"sjr\": \"6,695\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 739, \"total_docs_2022\": 1520, \"total_docs_3years\": 4812, \"total_refs\": 13195, \"total_cites_3years\": 54840, \"citable_docs_3years\": 1927, \"cites_per_doc_2years\": \"13,62\", \"ref_per_doc\": \"8,68\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"American Medical Association\", \"coverage\": \"1883-2022\", \"categories\": \"Medicine (miscellaneous) (Q1)\", \"areas\": \"Medicine\", \"has_error\": false}, {\"page_id\": \"154_article_search_result\", \"is_success\": true, \"error_count\": 72.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": 2.0, \"label_empty\": null, \"button_empty\": null, \"link_empty\": 9.0, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": 59.0, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": 2.0, \"th_empty\": null, \"title_invalid\": null, \"id\": 154, \"page_type\": \"article_search_result\", \"source_id\": \"sjr_85291\", \"rank\": 154, \"title\": \"JAMA - Journal of the American Medical Association\", \"type\": \"journal\", \"issn\": \"15383598, 00987484, 00029955\", \"sjr\": \"6,695\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 739, \"total_docs_2022\": 1520, \"total_docs_3years\": 4812, \"total_refs\": 13195, \"total_cites_3years\": 54840, \"citable_docs_3years\": 1927, \"cites_per_doc_2years\": \"13,62\", \"ref_per_doc\": \"8,68\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"American Medical Association\", \"coverage\": \"1883-2022\", \"categories\": \"Medicine (miscellaneous) (Q1)\", \"areas\": \"Medicine\", \"has_error\": true}, {\"page_id\": \"154_latest_issue\", \"is_success\": true, \"error_count\": 24.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": 2.0, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": 18.0, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": 2.0, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": 2.0, \"th_empty\": null, \"title_invalid\": null, \"id\": 154, \"page_type\": \"latest_issue\", \"source_id\": \"sjr_85291\", \"rank\": 154, \"title\": \"JAMA - Journal of the American Medical Association\", \"type\": \"journal\", \"issn\": \"15383598, 00987484, 00029955\", \"sjr\": \"6,695\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 739, \"total_docs_2022\": 1520, \"total_docs_3years\": 4812, \"total_refs\": 13195, \"total_cites_3years\": 54840, \"citable_docs_3years\": 1927, \"cites_per_doc_2years\": \"13,62\", \"ref_per_doc\": \"8,68\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"American Medical Association\", \"coverage\": \"1883-2022\", \"categories\": \"Medicine (miscellaneous) (Q1)\", \"areas\": \"Medicine\", \"has_error\": true}, {\"page_id\": \"119_home\", \"is_success\": true, \"error_count\": 32.0, \"aria_reference_broken\": null, \"label_multiple\": 3.0, \"alt_missing\": null, \"alt_link_missing\": 11.0, \"label_empty\": 3.0, \"button_empty\": 1.0, \"link_empty\": 8.0, \"heading_empty\": 2.0, \"alt_input_missing\": null, \"label_missing\": 4.0, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 119, \"page_type\": \"home\", \"source_id\": \"sjr_22581\", \"rank\": 119, \"title\": \"Circulation\", \"type\": \"journal\", \"issn\": \"00097322, 15244539\", \"sjr\": \"7,800\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 654, \"total_docs_2022\": 560, \"total_docs_3years\": 2310, \"total_refs\": 21347, \"total_cites_3years\": 31955, \"citable_docs_3years\": 1512, \"cites_per_doc_2years\": \"13,95\", \"ref_per_doc\": \"38,12\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Lippincott Williams and Wilkins Ltd.\", \"coverage\": \"1950-2022\", \"categories\": \"Cardiology and Cardiovascular Medicine (Q1); Physiology (medical) (Q1)\", \"areas\": \"Medicine\", \"has_error\": true}, {\"page_id\": \"119_research_article\", \"is_success\": true, \"error_count\": 22.0, \"aria_reference_broken\": null, \"label_multiple\": 3.0, \"alt_missing\": null, \"alt_link_missing\": 1.0, \"label_empty\": 3.0, \"button_empty\": 2.0, \"link_empty\": 6.0, \"heading_empty\": 2.0, \"alt_input_missing\": null, \"label_missing\": 5.0, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 119, \"page_type\": \"research_article\", \"source_id\": \"sjr_22581\", \"rank\": 119, \"title\": \"Circulation\", \"type\": \"journal\", \"issn\": \"00097322, 15244539\", \"sjr\": \"7,800\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 654, \"total_docs_2022\": 560, \"total_docs_3years\": 2310, \"total_refs\": 21347, \"total_cites_3years\": 31955, \"citable_docs_3years\": 1512, \"cites_per_doc_2years\": \"13,95\", \"ref_per_doc\": \"38,12\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Lippincott Williams and Wilkins Ltd.\", \"coverage\": \"1950-2022\", \"categories\": \"Cardiology and Cardiovascular Medicine (Q1); Physiology (medical) (Q1)\", \"areas\": \"Medicine\", \"has_error\": true}, {\"page_id\": \"119_none_research_article\", \"is_success\": true, \"error_count\": 22.0, \"aria_reference_broken\": null, \"label_multiple\": 3.0, \"alt_missing\": null, \"alt_link_missing\": 1.0, \"label_empty\": 3.0, \"button_empty\": 2.0, \"link_empty\": 6.0, \"heading_empty\": 2.0, \"alt_input_missing\": null, \"label_missing\": 5.0, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 119, \"page_type\": \"none_research_article\", \"source_id\": \"sjr_22581\", \"rank\": 119, \"title\": \"Circulation\", \"type\": \"journal\", \"issn\": \"00097322, 15244539\", \"sjr\": \"7,800\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 654, \"total_docs_2022\": 560, \"total_docs_3years\": 2310, \"total_refs\": 21347, \"total_cites_3years\": 31955, \"citable_docs_3years\": 1512, \"cites_per_doc_2years\": \"13,95\", \"ref_per_doc\": \"38,12\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Lippincott Williams and Wilkins Ltd.\", \"coverage\": \"1950-2022\", \"categories\": \"Cardiology and Cardiovascular Medicine (Q1); Physiology (medical) (Q1)\", \"areas\": \"Medicine\", \"has_error\": true}, {\"page_id\": \"119_article_search_result\", \"is_success\": true, \"error_count\": 66.0, \"aria_reference_broken\": 20.0, \"label_multiple\": 5.0, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": 23.0, \"button_empty\": 3.0, \"link_empty\": 9.0, \"heading_empty\": 2.0, \"alt_input_missing\": null, \"label_missing\": 4.0, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 119, \"page_type\": \"article_search_result\", \"source_id\": \"sjr_22581\", \"rank\": 119, \"title\": \"Circulation\", \"type\": \"journal\", \"issn\": \"00097322, 15244539\", \"sjr\": \"7,800\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 654, \"total_docs_2022\": 560, \"total_docs_3years\": 2310, \"total_refs\": 21347, \"total_cites_3years\": 31955, \"citable_docs_3years\": 1512, \"cites_per_doc_2years\": \"13,95\", \"ref_per_doc\": \"38,12\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Lippincott Williams and Wilkins Ltd.\", \"coverage\": \"1950-2022\", \"categories\": \"Cardiology and Cardiovascular Medicine (Q1); Physiology (medical) (Q1)\", \"areas\": \"Medicine\", \"has_error\": true}, {\"page_id\": \"119_latest_issue\", \"is_success\": true, \"error_count\": 41.0, \"aria_reference_broken\": null, \"label_multiple\": 3.0, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": 13.0, \"button_empty\": 2.0, \"link_empty\": 7.0, \"heading_empty\": 12.0, \"alt_input_missing\": null, \"label_missing\": 4.0, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 119, \"page_type\": \"latest_issue\", \"source_id\": \"sjr_22581\", \"rank\": 119, \"title\": \"Circulation\", \"type\": \"journal\", \"issn\": \"00097322, 15244539\", \"sjr\": \"7,800\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 654, \"total_docs_2022\": 560, \"total_docs_3years\": 2310, \"total_refs\": 21347, \"total_cites_3years\": 31955, \"citable_docs_3years\": 1512, \"cites_per_doc_2years\": \"13,95\", \"ref_per_doc\": \"38,12\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Lippincott Williams and Wilkins Ltd.\", \"coverage\": \"1950-2022\", \"categories\": \"Cardiology and Cardiovascular Medicine (Q1); Physiology (medical) (Q1)\", \"areas\": \"Medicine\", \"has_error\": true}, {\"page_id\": \"21_home\", \"is_success\": true, \"error_count\": 0.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 21, \"page_type\": \"home\", \"source_id\": \"sjr_18990\", \"rank\": 21, \"title\": \"Nature Genetics\", \"type\": \"journal\", \"issn\": \"10614036, 15461718\", \"sjr\": \"16,732\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 621, \"total_docs_2022\": 260, \"total_docs_3years\": 640, \"total_refs\": 12836, \"total_cites_3years\": 15264, \"citable_docs_3years\": 485, \"cites_per_doc_2years\": \"19,56\", \"ref_per_doc\": \"49,37\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Nature Publishing Group\", \"coverage\": \"1992-2022\", \"categories\": \"Genetics (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology\", \"has_error\": false}, {\"page_id\": \"21_research_article\", \"is_success\": true, \"error_count\": 0.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 21, \"page_type\": \"research_article\", \"source_id\": \"sjr_18990\", \"rank\": 21, \"title\": \"Nature Genetics\", \"type\": \"journal\", \"issn\": \"10614036, 15461718\", \"sjr\": \"16,732\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 621, \"total_docs_2022\": 260, \"total_docs_3years\": 640, \"total_refs\": 12836, \"total_cites_3years\": 15264, \"citable_docs_3years\": 485, \"cites_per_doc_2years\": \"19,56\", \"ref_per_doc\": \"49,37\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Nature Publishing Group\", \"coverage\": \"1992-2022\", \"categories\": \"Genetics (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology\", \"has_error\": false}, {\"page_id\": \"21_none_research_article\", \"is_success\": true, \"error_count\": 0.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 21, \"page_type\": \"none_research_article\", \"source_id\": \"sjr_18990\", \"rank\": 21, \"title\": \"Nature Genetics\", \"type\": \"journal\", \"issn\": \"10614036, 15461718\", \"sjr\": \"16,732\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 621, \"total_docs_2022\": 260, \"total_docs_3years\": 640, \"total_refs\": 12836, \"total_cites_3years\": 15264, \"citable_docs_3years\": 485, \"cites_per_doc_2years\": \"19,56\", \"ref_per_doc\": \"49,37\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Nature Publishing Group\", \"coverage\": \"1992-2022\", \"categories\": \"Genetics (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology\", \"has_error\": false}, {\"page_id\": \"21_article_search_result\", \"is_success\": true, \"error_count\": 0.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 21, \"page_type\": \"article_search_result\", \"source_id\": \"sjr_18990\", \"rank\": 21, \"title\": \"Nature Genetics\", \"type\": \"journal\", \"issn\": \"10614036, 15461718\", \"sjr\": \"16,732\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 621, \"total_docs_2022\": 260, \"total_docs_3years\": 640, \"total_refs\": 12836, \"total_cites_3years\": 15264, \"citable_docs_3years\": 485, \"cites_per_doc_2years\": \"19,56\", \"ref_per_doc\": \"49,37\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Nature Publishing Group\", \"coverage\": \"1992-2022\", \"categories\": \"Genetics (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology\", \"has_error\": false}, {\"page_id\": \"21_latest_issue\", \"is_success\": true, \"error_count\": 0.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 21, \"page_type\": \"latest_issue\", \"source_id\": \"sjr_18990\", \"rank\": 21, \"title\": \"Nature Genetics\", \"type\": \"journal\", \"issn\": \"10614036, 15461718\", \"sjr\": \"16,732\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 621, \"total_docs_2022\": 260, \"total_docs_3years\": 640, \"total_refs\": 12836, \"total_cites_3years\": 15264, \"citable_docs_3years\": 485, \"cites_per_doc_2years\": \"19,56\", \"ref_per_doc\": \"49,37\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Nature Publishing Group\", \"coverage\": \"1992-2022\", \"categories\": \"Genetics (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology\", \"has_error\": false}, {\"page_id\": \"107_home\", \"is_success\": true, \"error_count\": 10.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": 9.0, \"alt_link_missing\": 1.0, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 107, \"page_type\": \"home\", \"source_id\": \"sjr_14204\", \"rank\": 107, \"title\": \"Nucleic Acids Research\", \"type\": \"journal\", \"issn\": \"03051048, 13624962\", \"sjr\": \"8,234\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 607, \"total_docs_2022\": 1214, \"total_docs_3years\": 3872, \"total_refs\": 58320, \"total_cites_3years\": 68591, \"citable_docs_3years\": 3866, \"cites_per_doc_2years\": \"14,16\", \"ref_per_doc\": \"48,04\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Oxford University Press\", \"coverage\": \"1974-2022\", \"categories\": \"Genetics (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology\", \"has_error\": true}, {\"page_id\": \"107_research_article\", \"is_success\": true, \"error_count\": 4.0, \"aria_reference_broken\": null, \"label_multiple\": 1.0, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": 1.0, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": 2.0, \"th_empty\": null, \"title_invalid\": null, \"id\": 107, \"page_type\": \"research_article\", \"source_id\": \"sjr_14204\", \"rank\": 107, \"title\": \"Nucleic Acids Research\", \"type\": \"journal\", \"issn\": \"03051048, 13624962\", \"sjr\": \"8,234\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 607, \"total_docs_2022\": 1214, \"total_docs_3years\": 3872, \"total_refs\": 58320, \"total_cites_3years\": 68591, \"citable_docs_3years\": 3866, \"cites_per_doc_2years\": \"14,16\", \"ref_per_doc\": \"48,04\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Oxford University Press\", \"coverage\": \"1974-2022\", \"categories\": \"Genetics (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology\", \"has_error\": true}, {\"page_id\": \"107_article_search_result\", \"is_success\": true, \"error_count\": 8.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": 8.0, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 107, \"page_type\": \"article_search_result\", \"source_id\": \"sjr_14204\", \"rank\": 107, \"title\": \"Nucleic Acids Research\", \"type\": \"journal\", \"issn\": \"03051048, 13624962\", \"sjr\": \"8,234\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 607, \"total_docs_2022\": 1214, \"total_docs_3years\": 3872, \"total_refs\": 58320, \"total_cites_3years\": 68591, \"citable_docs_3years\": 3866, \"cites_per_doc_2years\": \"14,16\", \"ref_per_doc\": \"48,04\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Oxford University Press\", \"coverage\": \"1974-2022\", \"categories\": \"Genetics (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology\", \"has_error\": true}, {\"page_id\": \"107_latest_issue\", \"is_success\": true, \"error_count\": 0.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 107, \"page_type\": \"latest_issue\", \"source_id\": \"sjr_14204\", \"rank\": 107, \"title\": \"Nucleic Acids Research\", \"type\": \"journal\", \"issn\": \"03051048, 13624962\", \"sjr\": \"8,234\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 607, \"total_docs_2022\": 1214, \"total_docs_3years\": 3872, \"total_refs\": 58320, \"total_cites_3years\": 68591, \"citable_docs_3years\": 3866, \"cites_per_doc_2years\": \"14,16\", \"ref_per_doc\": \"48,04\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Oxford University Press\", \"coverage\": \"1974-2022\", \"categories\": \"Genetics (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology\", \"has_error\": false}, {\"page_id\": \"6_home\", \"is_success\": true, \"error_count\": 0.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 6, \"page_type\": \"home\", \"source_id\": \"sjr_15819\", \"rank\": 6, \"title\": \"Nature Medicine\", \"type\": \"journal\", \"issn\": \"1546170X, 10788956\", \"sjr\": \"24,687\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 605, \"total_docs_2022\": 522, \"total_docs_3years\": 1245, \"total_refs\": 15256, \"total_cites_3years\": 41058, \"citable_docs_3years\": 664, \"cites_per_doc_2years\": \"34,67\", \"ref_per_doc\": \"29,23\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Nature Publishing Group\", \"coverage\": \"1995-2022\", \"categories\": \"Biochemistry, Genetics and Molecular Biology (miscellaneous) (Q1); Medicine (miscellaneous) (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology; Medicine\", \"has_error\": false}, {\"page_id\": \"6_research_article\", \"is_success\": true, \"error_count\": 0.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 6, \"page_type\": \"research_article\", \"source_id\": \"sjr_15819\", \"rank\": 6, \"title\": \"Nature Medicine\", \"type\": \"journal\", \"issn\": \"1546170X, 10788956\", \"sjr\": \"24,687\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 605, \"total_docs_2022\": 522, \"total_docs_3years\": 1245, \"total_refs\": 15256, \"total_cites_3years\": 41058, \"citable_docs_3years\": 664, \"cites_per_doc_2years\": \"34,67\", \"ref_per_doc\": \"29,23\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Nature Publishing Group\", \"coverage\": \"1995-2022\", \"categories\": \"Biochemistry, Genetics and Molecular Biology (miscellaneous) (Q1); Medicine (miscellaneous) (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology; Medicine\", \"has_error\": false}, {\"page_id\": \"6_none_research_article\", \"is_success\": true, \"error_count\": 1.0, \"aria_reference_broken\": 1.0, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 6, \"page_type\": \"none_research_article\", \"source_id\": \"sjr_15819\", \"rank\": 6, \"title\": \"Nature Medicine\", \"type\": \"journal\", \"issn\": \"1546170X, 10788956\", \"sjr\": \"24,687\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 605, \"total_docs_2022\": 522, \"total_docs_3years\": 1245, \"total_refs\": 15256, \"total_cites_3years\": 41058, \"citable_docs_3years\": 664, \"cites_per_doc_2years\": \"34,67\", \"ref_per_doc\": \"29,23\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Nature Publishing Group\", \"coverage\": \"1995-2022\", \"categories\": \"Biochemistry, Genetics and Molecular Biology (miscellaneous) (Q1); Medicine (miscellaneous) (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology; Medicine\", \"has_error\": true}, {\"page_id\": \"6_article_search_result\", \"is_success\": true, \"error_count\": 0.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 6, \"page_type\": \"article_search_result\", \"source_id\": \"sjr_15819\", \"rank\": 6, \"title\": \"Nature Medicine\", \"type\": \"journal\", \"issn\": \"1546170X, 10788956\", \"sjr\": \"24,687\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 605, \"total_docs_2022\": 522, \"total_docs_3years\": 1245, \"total_refs\": 15256, \"total_cites_3years\": 41058, \"citable_docs_3years\": 664, \"cites_per_doc_2years\": \"34,67\", \"ref_per_doc\": \"29,23\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Nature Publishing Group\", \"coverage\": \"1995-2022\", \"categories\": \"Biochemistry, Genetics and Molecular Biology (miscellaneous) (Q1); Medicine (miscellaneous) (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology; Medicine\", \"has_error\": false}, {\"page_id\": \"6_latest_issue\", \"is_success\": true, \"error_count\": 0.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 6, \"page_type\": \"latest_issue\", \"source_id\": \"sjr_15819\", \"rank\": 6, \"title\": \"Nature Medicine\", \"type\": \"journal\", \"issn\": \"1546170X, 10788956\", \"sjr\": \"24,687\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 605, \"total_docs_2022\": 522, \"total_docs_3years\": 1245, \"total_refs\": 15256, \"total_cites_3years\": 41058, \"citable_docs_3years\": 664, \"cites_per_doc_2years\": \"34,67\", \"ref_per_doc\": \"29,23\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Nature Publishing Group\", \"coverage\": \"1995-2022\", \"categories\": \"Biochemistry, Genetics and Molecular Biology (miscellaneous) (Q1); Medicine (miscellaneous) (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology; Medicine\", \"has_error\": false}, {\"page_id\": \"76_home\", \"is_success\": true, \"error_count\": 5.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": 1.0, \"link_empty\": null, \"heading_empty\": 2.0, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": 2.0, \"th_empty\": null, \"title_invalid\": null, \"id\": 76, \"page_type\": \"home\", \"source_id\": \"sjr_29949\", \"rank\": 76, \"title\": \"Journal of Clinical Oncology\", \"type\": \"journal\", \"issn\": \"15277755, 0732183X\", \"sjr\": \"10,163\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 600, \"total_docs_2022\": 575, \"total_docs_3years\": 1639, \"total_refs\": 16901, \"total_cites_3years\": 25122, \"citable_docs_3years\": 1168, \"cites_per_doc_2years\": \"16,30\", \"ref_per_doc\": \"29,39\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Lippincott Williams and Wilkins Ltd.\", \"coverage\": \"1983-2022\", \"categories\": \"Cancer Research (Q1); Medicine (miscellaneous) (Q1); Oncology (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology; Medicine\", \"has_error\": true}, {\"page_id\": \"76_research_article\", \"is_success\": true, \"error_count\": 21.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": 2.0, \"alt_link_missing\": 6.0, \"label_empty\": null, \"button_empty\": 1.0, \"link_empty\": 8.0, \"heading_empty\": 2.0, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": 2.0, \"th_empty\": null, \"title_invalid\": null, \"id\": 76, \"page_type\": \"research_article\", \"source_id\": \"sjr_29949\", \"rank\": 76, \"title\": \"Journal of Clinical Oncology\", \"type\": \"journal\", \"issn\": \"15277755, 0732183X\", \"sjr\": \"10,163\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 600, \"total_docs_2022\": 575, \"total_docs_3years\": 1639, \"total_refs\": 16901, \"total_cites_3years\": 25122, \"citable_docs_3years\": 1168, \"cites_per_doc_2years\": \"16,30\", \"ref_per_doc\": \"29,39\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Lippincott Williams and Wilkins Ltd.\", \"coverage\": \"1983-2022\", \"categories\": \"Cancer Research (Q1); Medicine (miscellaneous) (Q1); Oncology (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology; Medicine\", \"has_error\": true}, {\"page_id\": \"76_none_research_article\", \"is_success\": true, \"error_count\": 12.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": 2.0, \"alt_link_missing\": 2.0, \"label_empty\": null, \"button_empty\": 1.0, \"link_empty\": 3.0, \"heading_empty\": 2.0, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": 2.0, \"th_empty\": null, \"title_invalid\": null, \"id\": 76, \"page_type\": \"none_research_article\", \"source_id\": \"sjr_29949\", \"rank\": 76, \"title\": \"Journal of Clinical Oncology\", \"type\": \"journal\", \"issn\": \"15277755, 0732183X\", \"sjr\": \"10,163\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 600, \"total_docs_2022\": 575, \"total_docs_3years\": 1639, \"total_refs\": 16901, \"total_cites_3years\": 25122, \"citable_docs_3years\": 1168, \"cites_per_doc_2years\": \"16,30\", \"ref_per_doc\": \"29,39\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Lippincott Williams and Wilkins Ltd.\", \"coverage\": \"1983-2022\", \"categories\": \"Cancer Research (Q1); Medicine (miscellaneous) (Q1); Oncology (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology; Medicine\", \"has_error\": true}, {\"page_id\": \"76_article_search_result\", \"is_success\": true, \"error_count\": 12.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": 2.0, \"button_empty\": 1.0, \"link_empty\": 1.0, \"heading_empty\": 3.0, \"alt_input_missing\": null, \"label_missing\": 5.0, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 76, \"page_type\": \"article_search_result\", \"source_id\": \"sjr_29949\", \"rank\": 76, \"title\": \"Journal of Clinical Oncology\", \"type\": \"journal\", \"issn\": \"15277755, 0732183X\", \"sjr\": \"10,163\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 600, \"total_docs_2022\": 575, \"total_docs_3years\": 1639, \"total_refs\": 16901, \"total_cites_3years\": 25122, \"citable_docs_3years\": 1168, \"cites_per_doc_2years\": \"16,30\", \"ref_per_doc\": \"29,39\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Lippincott Williams and Wilkins Ltd.\", \"coverage\": \"1983-2022\", \"categories\": \"Cancer Research (Q1); Medicine (miscellaneous) (Q1); Oncology (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology; Medicine\", \"has_error\": true}, {\"page_id\": \"76_latest_issue\", \"is_success\": true, \"error_count\": 5.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": 1.0, \"link_empty\": null, \"heading_empty\": 2.0, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": 2.0, \"th_empty\": null, \"title_invalid\": null, \"id\": 76, \"page_type\": \"latest_issue\", \"source_id\": \"sjr_29949\", \"rank\": 76, \"title\": \"Journal of Clinical Oncology\", \"type\": \"journal\", \"issn\": \"15277755, 0732183X\", \"sjr\": \"10,163\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 600, \"total_docs_2022\": 575, \"total_docs_3years\": 1639, \"total_refs\": 16901, \"total_cites_3years\": 25122, \"citable_docs_3years\": 1168, \"cites_per_doc_2years\": \"16,30\", \"ref_per_doc\": \"29,39\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Lippincott Williams and Wilkins Ltd.\", \"coverage\": \"1983-2022\", \"categories\": \"Cancer Research (Q1); Medicine (miscellaneous) (Q1); Oncology (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology; Medicine\", \"has_error\": true}, {\"page_id\": \"1561_home\", \"is_success\": true, \"error_count\": 5.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": 2.0, \"label_empty\": null, \"button_empty\": 2.0, \"link_empty\": null, \"heading_empty\": 1.0, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 1561, \"page_type\": \"home\", \"source_id\": \"sjr_17592\", \"rank\": 1561, \"title\": \"Journal of Biological Chemistry\", \"type\": \"journal\", \"issn\": \"00219258, 1083351X\", \"sjr\": \"1,805\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 544, \"total_docs_2022\": 1248, \"total_docs_3years\": 4591, \"total_refs\": 77124, \"total_cites_3years\": 21251, \"citable_docs_3years\": 4529, \"cites_per_doc_2years\": \"4,51\", \"ref_per_doc\": \"61,80\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Elsevier Inc.\", \"coverage\": \"1945-2022\", \"categories\": \"Biochemistry (Q1); Cell Biology (Q1); Molecular Biology (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology\", \"has_error\": true}, {\"page_id\": \"1561_research_article\", \"is_success\": true, \"error_count\": 6.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": 1.0, \"alt_link_missing\": 1.0, \"label_empty\": null, \"button_empty\": 2.0, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": 1.0, \"alt_spacer_missing\": null, \"th_empty\": 1.0, \"title_invalid\": null, \"id\": 1561, \"page_type\": \"research_article\", \"source_id\": \"sjr_17592\", \"rank\": 1561, \"title\": \"Journal of Biological Chemistry\", \"type\": \"journal\", \"issn\": \"00219258, 1083351X\", \"sjr\": \"1,805\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 544, \"total_docs_2022\": 1248, \"total_docs_3years\": 4591, \"total_refs\": 77124, \"total_cites_3years\": 21251, \"citable_docs_3years\": 4529, \"cites_per_doc_2years\": \"4,51\", \"ref_per_doc\": \"61,80\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Elsevier Inc.\", \"coverage\": \"1945-2022\", \"categories\": \"Biochemistry (Q1); Cell Biology (Q1); Molecular Biology (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology\", \"has_error\": true}, {\"page_id\": \"1561_none_research_article\", \"is_success\": true, \"error_count\": 5.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": 1.0, \"alt_link_missing\": 1.0, \"label_empty\": null, \"button_empty\": 2.0, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": 1.0, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 1561, \"page_type\": \"none_research_article\", \"source_id\": \"sjr_17592\", \"rank\": 1561, \"title\": \"Journal of Biological Chemistry\", \"type\": \"journal\", \"issn\": \"00219258, 1083351X\", \"sjr\": \"1,805\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 544, \"total_docs_2022\": 1248, \"total_docs_3years\": 4591, \"total_refs\": 77124, \"total_cites_3years\": 21251, \"citable_docs_3years\": 4529, \"cites_per_doc_2years\": \"4,51\", \"ref_per_doc\": \"61,80\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Elsevier Inc.\", \"coverage\": \"1945-2022\", \"categories\": \"Biochemistry (Q1); Cell Biology (Q1); Molecular Biology (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology\", \"has_error\": true}, {\"page_id\": \"1561_article_search_result\", \"is_success\": true, \"error_count\": 4.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": 1.0, \"label_empty\": null, \"button_empty\": 2.0, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": 1.0, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 1561, \"page_type\": \"article_search_result\", \"source_id\": \"sjr_17592\", \"rank\": 1561, \"title\": \"Journal of Biological Chemistry\", \"type\": \"journal\", \"issn\": \"00219258, 1083351X\", \"sjr\": \"1,805\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 544, \"total_docs_2022\": 1248, \"total_docs_3years\": 4591, \"total_refs\": 77124, \"total_cites_3years\": 21251, \"citable_docs_3years\": 4529, \"cites_per_doc_2years\": \"4,51\", \"ref_per_doc\": \"61,80\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Elsevier Inc.\", \"coverage\": \"1945-2022\", \"categories\": \"Biochemistry (Q1); Cell Biology (Q1); Molecular Biology (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology\", \"has_error\": true}, {\"page_id\": \"1561_latest_issue\", \"is_success\": true, \"error_count\": 101.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": 1.0, \"label_empty\": 96.0, \"button_empty\": 2.0, \"link_empty\": 1.0, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": 1.0, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 1561, \"page_type\": \"latest_issue\", \"source_id\": \"sjr_17592\", \"rank\": 1561, \"title\": \"Journal of Biological Chemistry\", \"type\": \"journal\", \"issn\": \"00219258, 1083351X\", \"sjr\": \"1,805\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 544, \"total_docs_2022\": 1248, \"total_docs_3years\": 4591, \"total_refs\": 77124, \"total_cites_3years\": 21251, \"citable_docs_3years\": 4529, \"cites_per_doc_2years\": \"4,51\", \"ref_per_doc\": \"61,80\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Elsevier Inc.\", \"coverage\": \"1945-2022\", \"categories\": \"Biochemistry (Q1); Cell Biology (Q1); Molecular Biology (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology\", \"has_error\": true}, {\"page_id\": \"257_home\", \"is_success\": true, \"error_count\": 31.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": 1.0, \"alt_link_missing\": 23.0, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": 1.0, \"alt_input_missing\": 2.0, \"label_missing\": 4.0, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 257, \"page_type\": \"home\", \"source_id\": \"sjr_15870\", \"rank\": 257, \"title\": \"Journal of Clinical Investigation\", \"type\": \"journal\", \"issn\": \"00219738, 15588238\", \"sjr\": \"5,117\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 527, \"total_docs_2022\": 533, \"total_docs_3years\": 1754, \"total_refs\": 28750, \"total_cites_3years\": 21826, \"citable_docs_3years\": 1711, \"cites_per_doc_2years\": \"11,96\", \"ref_per_doc\": \"53,94\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"The American Society for Clinical Investigation\", \"coverage\": \"1945-2022\", \"categories\": \"Medicine (miscellaneous) (Q1)\", \"areas\": \"Medicine\", \"has_error\": true}, {\"page_id\": \"257_research_article\", \"is_success\": true, \"error_count\": 22.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": 4.0, \"alt_link_missing\": 12.0, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": 1.0, \"alt_input_missing\": 2.0, \"label_missing\": 3.0, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 257, \"page_type\": \"research_article\", \"source_id\": \"sjr_15870\", \"rank\": 257, \"title\": \"Journal of Clinical Investigation\", \"type\": \"journal\", \"issn\": \"00219738, 15588238\", \"sjr\": \"5,117\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 527, \"total_docs_2022\": 533, \"total_docs_3years\": 1754, \"total_refs\": 28750, \"total_cites_3years\": 21826, \"citable_docs_3years\": 1711, \"cites_per_doc_2years\": \"11,96\", \"ref_per_doc\": \"53,94\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"The American Society for Clinical Investigation\", \"coverage\": \"1945-2022\", \"categories\": \"Medicine (miscellaneous) (Q1)\", \"areas\": \"Medicine\", \"has_error\": true}, {\"page_id\": \"257_none_research_article\", \"is_success\": true, \"error_count\": 15.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": 4.0, \"alt_link_missing\": 5.0, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": 1.0, \"alt_input_missing\": 2.0, \"label_missing\": 3.0, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 257, \"page_type\": \"none_research_article\", \"source_id\": \"sjr_15870\", \"rank\": 257, \"title\": \"Journal of Clinical Investigation\", \"type\": \"journal\", \"issn\": \"00219738, 15588238\", \"sjr\": \"5,117\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 527, \"total_docs_2022\": 533, \"total_docs_3years\": 1754, \"total_refs\": 28750, \"total_cites_3years\": 21826, \"citable_docs_3years\": 1711, \"cites_per_doc_2years\": \"11,96\", \"ref_per_doc\": \"53,94\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"The American Society for Clinical Investigation\", \"coverage\": \"1945-2022\", \"categories\": \"Medicine (miscellaneous) (Q1)\", \"areas\": \"Medicine\", \"has_error\": true}, {\"page_id\": \"257_article_search_result\", \"is_success\": true, \"error_count\": 25.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": 7.0, \"label_empty\": null, \"button_empty\": null, \"link_empty\": 11.0, \"heading_empty\": 1.0, \"alt_input_missing\": 2.0, \"label_missing\": 4.0, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 257, \"page_type\": \"article_search_result\", \"source_id\": \"sjr_15870\", \"rank\": 257, \"title\": \"Journal of Clinical Investigation\", \"type\": \"journal\", \"issn\": \"00219738, 15588238\", \"sjr\": \"5,117\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 527, \"total_docs_2022\": 533, \"total_docs_3years\": 1754, \"total_refs\": 28750, \"total_cites_3years\": 21826, \"citable_docs_3years\": 1711, \"cites_per_doc_2years\": \"11,96\", \"ref_per_doc\": \"53,94\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"The American Society for Clinical Investigation\", \"coverage\": \"1945-2022\", \"categories\": \"Medicine (miscellaneous) (Q1)\", \"areas\": \"Medicine\", \"has_error\": true}, {\"page_id\": \"257_latest_issue\", \"is_success\": true, \"error_count\": 29.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": 1.0, \"alt_link_missing\": 22.0, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": 1.0, \"alt_input_missing\": 2.0, \"label_missing\": 3.0, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 257, \"page_type\": \"latest_issue\", \"source_id\": \"sjr_15870\", \"rank\": 257, \"title\": \"Journal of Clinical Investigation\", \"type\": \"journal\", \"issn\": \"00219738, 15588238\", \"sjr\": \"5,117\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 527, \"total_docs_2022\": 533, \"total_docs_3years\": 1754, \"total_refs\": 28750, \"total_cites_3years\": 21826, \"citable_docs_3years\": 1711, \"cites_per_doc_2years\": \"11,96\", \"ref_per_doc\": \"53,94\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"The American Society for Clinical Investigation\", \"coverage\": \"1945-2022\", \"categories\": \"Medicine (miscellaneous) (Q1)\", \"areas\": \"Medicine\", \"has_error\": true}, {\"page_id\": \"120_home\", \"is_success\": true, \"error_count\": 3.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": 1.0, \"label_empty\": null, \"button_empty\": 2.0, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 120, \"page_type\": \"home\", \"source_id\": \"sjr_17978\", \"rank\": 120, \"title\": \"Neuron\", \"type\": \"journal\", \"issn\": \"10974199, 08966273\", \"sjr\": \"7,736\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 511, \"total_docs_2022\": 378, \"total_docs_3years\": 1263, \"total_refs\": 25298, \"total_cites_3years\": 14680, \"citable_docs_3years\": 1120, \"cites_per_doc_2years\": \"10,37\", \"ref_per_doc\": \"66,93\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Cell Press\", \"coverage\": \"1988-2022\", \"categories\": \"Neuroscience (miscellaneous) (Q1)\", \"areas\": \"Neuroscience\", \"has_error\": true}, {\"page_id\": \"120_research_article\", \"is_success\": true, \"error_count\": 6.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": 1.0, \"alt_link_missing\": 2.0, \"label_empty\": null, \"button_empty\": 2.0, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": 1.0, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 120, \"page_type\": \"research_article\", \"source_id\": \"sjr_17978\", \"rank\": 120, \"title\": \"Neuron\", \"type\": \"journal\", \"issn\": \"10974199, 08966273\", \"sjr\": \"7,736\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 511, \"total_docs_2022\": 378, \"total_docs_3years\": 1263, \"total_refs\": 25298, \"total_cites_3years\": 14680, \"citable_docs_3years\": 1120, \"cites_per_doc_2years\": \"10,37\", \"ref_per_doc\": \"66,93\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Cell Press\", \"coverage\": \"1988-2022\", \"categories\": \"Neuroscience (miscellaneous) (Q1)\", \"areas\": \"Neuroscience\", \"has_error\": true}, {\"page_id\": \"120_none_research_article\", \"is_success\": true, \"error_count\": 6.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": 1.0, \"alt_link_missing\": 2.0, \"label_empty\": null, \"button_empty\": 2.0, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": 1.0, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 120, \"page_type\": \"none_research_article\", \"source_id\": \"sjr_17978\", \"rank\": 120, \"title\": \"Neuron\", \"type\": \"journal\", \"issn\": \"10974199, 08966273\", \"sjr\": \"7,736\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 511, \"total_docs_2022\": 378, \"total_docs_3years\": 1263, \"total_refs\": 25298, \"total_cites_3years\": 14680, \"citable_docs_3years\": 1120, \"cites_per_doc_2years\": \"10,37\", \"ref_per_doc\": \"66,93\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Cell Press\", \"coverage\": \"1988-2022\", \"categories\": \"Neuroscience (miscellaneous) (Q1)\", \"areas\": \"Neuroscience\", \"has_error\": true}, {\"page_id\": \"120_article_search_result\", \"is_success\": true, \"error_count\": 5.0, \"aria_reference_broken\": 1.0, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": 1.0, \"label_empty\": null, \"button_empty\": 2.0, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": 1.0, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 120, \"page_type\": \"article_search_result\", \"source_id\": \"sjr_17978\", \"rank\": 120, \"title\": \"Neuron\", \"type\": \"journal\", \"issn\": \"10974199, 08966273\", \"sjr\": \"7,736\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 511, \"total_docs_2022\": 378, \"total_docs_3years\": 1263, \"total_refs\": 25298, \"total_cites_3years\": 14680, \"citable_docs_3years\": 1120, \"cites_per_doc_2years\": \"10,37\", \"ref_per_doc\": \"66,93\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Cell Press\", \"coverage\": \"1988-2022\", \"categories\": \"Neuroscience (miscellaneous) (Q1)\", \"areas\": \"Neuroscience\", \"has_error\": true}, {\"page_id\": \"120_latest_issue\", \"is_success\": true, \"error_count\": 24.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": 1.0, \"label_empty\": 19.0, \"button_empty\": 2.0, \"link_empty\": 1.0, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": 1.0, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 120, \"page_type\": \"latest_issue\", \"source_id\": \"sjr_17978\", \"rank\": 120, \"title\": \"Neuron\", \"type\": \"journal\", \"issn\": \"10974199, 08966273\", \"sjr\": \"7,736\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 511, \"total_docs_2022\": 378, \"total_docs_3years\": 1263, \"total_refs\": 25298, \"total_cites_3years\": 14680, \"citable_docs_3years\": 1120, \"cites_per_doc_2years\": \"10,37\", \"ref_per_doc\": \"66,93\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Cell Press\", \"coverage\": \"1988-2022\", \"categories\": \"Neuroscience (miscellaneous) (Q1)\", \"areas\": \"Neuroscience\", \"has_error\": true}, {\"page_id\": \"272_home\", \"is_success\": true, \"error_count\": 4.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": 4.0, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 272, \"page_type\": \"home\", \"source_id\": \"sjr_25454\", \"rank\": 272, \"title\": \"Blood\", \"type\": \"journal\", \"issn\": \"15280020, 00064971\", \"sjr\": \"4,927\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 506, \"total_docs_2022\": 900, \"total_docs_3years\": 2535, \"total_refs\": 30720, \"total_cites_3years\": 21405, \"citable_docs_3years\": 1757, \"cites_per_doc_2years\": \"8,23\", \"ref_per_doc\": \"34,13\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Elsevier BV\", \"coverage\": \"1946-2022\", \"categories\": \"Biochemistry (Q1); Cell Biology (Q1); Hematology (Q1); Immunology (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology; Immunology and Microbiology; Medicine\", \"has_error\": true}, {\"page_id\": \"272_research_article\", \"is_success\": true, \"error_count\": 5.0, \"aria_reference_broken\": null, \"label_multiple\": 5.0, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 272, \"page_type\": \"research_article\", \"source_id\": \"sjr_25454\", \"rank\": 272, \"title\": \"Blood\", \"type\": \"journal\", \"issn\": \"15280020, 00064971\", \"sjr\": \"4,927\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 506, \"total_docs_2022\": 900, \"total_docs_3years\": 2535, \"total_refs\": 30720, \"total_cites_3years\": 21405, \"citable_docs_3years\": 1757, \"cites_per_doc_2years\": \"8,23\", \"ref_per_doc\": \"34,13\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Elsevier BV\", \"coverage\": \"1946-2022\", \"categories\": \"Biochemistry (Q1); Cell Biology (Q1); Hematology (Q1); Immunology (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology; Immunology and Microbiology; Medicine\", \"has_error\": true}, {\"page_id\": \"272_none_research_article\", \"is_success\": true, \"error_count\": 1.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": 1.0, \"id\": 272, \"page_type\": \"none_research_article\", \"source_id\": \"sjr_25454\", \"rank\": 272, \"title\": \"Blood\", \"type\": \"journal\", \"issn\": \"15280020, 00064971\", \"sjr\": \"4,927\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 506, \"total_docs_2022\": 900, \"total_docs_3years\": 2535, \"total_refs\": 30720, \"total_cites_3years\": 21405, \"citable_docs_3years\": 1757, \"cites_per_doc_2years\": \"8,23\", \"ref_per_doc\": \"34,13\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Elsevier BV\", \"coverage\": \"1946-2022\", \"categories\": \"Biochemistry (Q1); Cell Biology (Q1); Hematology (Q1); Immunology (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology; Immunology and Microbiology; Medicine\", \"has_error\": true}, {\"page_id\": \"272_article_search_result\", \"is_success\": true, \"error_count\": 0.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 272, \"page_type\": \"article_search_result\", \"source_id\": \"sjr_25454\", \"rank\": 272, \"title\": \"Blood\", \"type\": \"journal\", \"issn\": \"15280020, 00064971\", \"sjr\": \"4,927\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 506, \"total_docs_2022\": 900, \"total_docs_3years\": 2535, \"total_refs\": 30720, \"total_cites_3years\": 21405, \"citable_docs_3years\": 1757, \"cites_per_doc_2years\": \"8,23\", \"ref_per_doc\": \"34,13\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Elsevier BV\", \"coverage\": \"1946-2022\", \"categories\": \"Biochemistry (Q1); Cell Biology (Q1); Hematology (Q1); Immunology (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology; Immunology and Microbiology; Medicine\", \"has_error\": false}, {\"page_id\": \"272_latest_issue\", \"is_success\": true, \"error_count\": 8.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": 6.0, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": 2.0, \"th_empty\": null, \"title_invalid\": null, \"id\": 272, \"page_type\": \"latest_issue\", \"source_id\": \"sjr_25454\", \"rank\": 272, \"title\": \"Blood\", \"type\": \"journal\", \"issn\": \"15280020, 00064971\", \"sjr\": \"4,927\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 506, \"total_docs_2022\": 900, \"total_docs_3years\": 2535, \"total_refs\": 30720, \"total_cites_3years\": 21405, \"citable_docs_3years\": 1757, \"cites_per_doc_2years\": \"8,23\", \"ref_per_doc\": \"34,13\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Elsevier BV\", \"coverage\": \"1946-2022\", \"categories\": \"Biochemistry (Q1); Cell Biology (Q1); Hematology (Q1); Immunology (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology; Immunology and Microbiology; Medicine\", \"has_error\": true}, {\"page_id\": \"8_home\", \"is_success\": true, \"error_count\": 0.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 8, \"page_type\": \"home\", \"source_id\": \"sjr_16115\", \"rank\": 8, \"title\": \"Nature Biotechnology\", \"type\": \"journal\", \"issn\": \"15461696, 10870156\", \"sjr\": \"22,781\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 491, \"total_docs_2022\": 423, \"total_docs_3years\": 1038, \"total_refs\": 10852, \"total_cites_3years\": 20441, \"citable_docs_3years\": 426, \"cites_per_doc_2years\": \"16,35\", \"ref_per_doc\": \"25,65\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Nature Publishing Group\", \"coverage\": \"1989-2022\", \"categories\": \"Applied Microbiology and Biotechnology (Q1); Bioengineering (Q1); Biomedical Engineering (Q1); Biotechnology (Q1); Molecular Medicine (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology; Chemical Engineering; Engineering; Immunology and Microbiology\", \"has_error\": false}, {\"page_id\": \"8_research_article\", \"is_success\": true, \"error_count\": 0.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 8, \"page_type\": \"research_article\", \"source_id\": \"sjr_16115\", \"rank\": 8, \"title\": \"Nature Biotechnology\", \"type\": \"journal\", \"issn\": \"15461696, 10870156\", \"sjr\": \"22,781\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 491, \"total_docs_2022\": 423, \"total_docs_3years\": 1038, \"total_refs\": 10852, \"total_cites_3years\": 20441, \"citable_docs_3years\": 426, \"cites_per_doc_2years\": \"16,35\", \"ref_per_doc\": \"25,65\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Nature Publishing Group\", \"coverage\": \"1989-2022\", \"categories\": \"Applied Microbiology and Biotechnology (Q1); Bioengineering (Q1); Biomedical Engineering (Q1); Biotechnology (Q1); Molecular Medicine (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology; Chemical Engineering; Engineering; Immunology and Microbiology\", \"has_error\": false}, {\"page_id\": \"8_none_research_article\", \"is_success\": true, \"error_count\": 2.0, \"aria_reference_broken\": null, \"label_multiple\": 2.0, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 8, \"page_type\": \"none_research_article\", \"source_id\": \"sjr_16115\", \"rank\": 8, \"title\": \"Nature Biotechnology\", \"type\": \"journal\", \"issn\": \"15461696, 10870156\", \"sjr\": \"22,781\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 491, \"total_docs_2022\": 423, \"total_docs_3years\": 1038, \"total_refs\": 10852, \"total_cites_3years\": 20441, \"citable_docs_3years\": 426, \"cites_per_doc_2years\": \"16,35\", \"ref_per_doc\": \"25,65\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Nature Publishing Group\", \"coverage\": \"1989-2022\", \"categories\": \"Applied Microbiology and Biotechnology (Q1); Bioengineering (Q1); Biomedical Engineering (Q1); Biotechnology (Q1); Molecular Medicine (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology; Chemical Engineering; Engineering; Immunology and Microbiology\", \"has_error\": true}, {\"page_id\": \"8_article_search_result\", \"is_success\": true, \"error_count\": 0.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 8, \"page_type\": \"article_search_result\", \"source_id\": \"sjr_16115\", \"rank\": 8, \"title\": \"Nature Biotechnology\", \"type\": \"journal\", \"issn\": \"15461696, 10870156\", \"sjr\": \"22,781\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 491, \"total_docs_2022\": 423, \"total_docs_3years\": 1038, \"total_refs\": 10852, \"total_cites_3years\": 20441, \"citable_docs_3years\": 426, \"cites_per_doc_2years\": \"16,35\", \"ref_per_doc\": \"25,65\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Nature Publishing Group\", \"coverage\": \"1989-2022\", \"categories\": \"Applied Microbiology and Biotechnology (Q1); Bioengineering (Q1); Biomedical Engineering (Q1); Biotechnology (Q1); Molecular Medicine (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology; Chemical Engineering; Engineering; Immunology and Microbiology\", \"has_error\": false}, {\"page_id\": \"8_latest_issue\", \"is_success\": true, \"error_count\": 0.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 8, \"page_type\": \"latest_issue\", \"source_id\": \"sjr_16115\", \"rank\": 8, \"title\": \"Nature Biotechnology\", \"type\": \"journal\", \"issn\": \"15461696, 10870156\", \"sjr\": \"22,781\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 491, \"total_docs_2022\": 423, \"total_docs_3years\": 1038, \"total_refs\": 10852, \"total_cites_3years\": 20441, \"citable_docs_3years\": 426, \"cites_per_doc_2years\": \"16,35\", \"ref_per_doc\": \"25,65\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Nature Publishing Group\", \"coverage\": \"1989-2022\", \"categories\": \"Applied Microbiology and Biotechnology (Q1); Bioengineering (Q1); Biomedical Engineering (Q1); Biotechnology (Q1); Molecular Medicine (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology; Chemical Engineering; Engineering; Immunology and Microbiology\", \"has_error\": false}, {\"page_id\": \"1002_home\", \"is_success\": true, \"error_count\": 9.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": 3.0, \"link_empty\": 6.0, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 1002, \"page_type\": \"home\", \"source_id\": \"sjr_16764\", \"rank\": 1002, \"title\": \"Journal of Neuroscience\", \"type\": \"journal\", \"issn\": \"02706474, 15292401\", \"sjr\": \"2,350\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 488, \"total_docs_2022\": 707, \"total_docs_3years\": 2323, \"total_refs\": 47444, \"total_cites_3years\": 12615, \"citable_docs_3years\": 2270, \"cites_per_doc_2years\": \"4,96\", \"ref_per_doc\": \"67,11\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Society for Neuroscience\", \"coverage\": \"1981-2022\", \"categories\": \"Neuroscience (miscellaneous) (Q1)\", \"areas\": \"Neuroscience\", \"has_error\": true}, {\"page_id\": \"1002_research_article\", \"is_success\": true, \"error_count\": 9.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": 3.0, \"link_empty\": 5.0, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": 1.0, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 1002, \"page_type\": \"research_article\", \"source_id\": \"sjr_16764\", \"rank\": 1002, \"title\": \"Journal of Neuroscience\", \"type\": \"journal\", \"issn\": \"02706474, 15292401\", \"sjr\": \"2,350\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 488, \"total_docs_2022\": 707, \"total_docs_3years\": 2323, \"total_refs\": 47444, \"total_cites_3years\": 12615, \"citable_docs_3years\": 2270, \"cites_per_doc_2years\": \"4,96\", \"ref_per_doc\": \"67,11\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Society for Neuroscience\", \"coverage\": \"1981-2022\", \"categories\": \"Neuroscience (miscellaneous) (Q1)\", \"areas\": \"Neuroscience\", \"has_error\": true}, {\"page_id\": \"1002_article_search_result\", \"is_success\": true, \"error_count\": 8.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": 3.0, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": 5.0, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 1002, \"page_type\": \"article_search_result\", \"source_id\": \"sjr_16764\", \"rank\": 1002, \"title\": \"Journal of Neuroscience\", \"type\": \"journal\", \"issn\": \"02706474, 15292401\", \"sjr\": \"2,350\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 488, \"total_docs_2022\": 707, \"total_docs_3years\": 2323, \"total_refs\": 47444, \"total_cites_3years\": 12615, \"citable_docs_3years\": 2270, \"cites_per_doc_2years\": \"4,96\", \"ref_per_doc\": \"67,11\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Society for Neuroscience\", \"coverage\": \"1981-2022\", \"categories\": \"Neuroscience (miscellaneous) (Q1)\", \"areas\": \"Neuroscience\", \"has_error\": true}, {\"page_id\": \"1002_latest_issue\", \"is_success\": true, \"error_count\": 6.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": 3.0, \"link_empty\": 3.0, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 1002, \"page_type\": \"latest_issue\", \"source_id\": \"sjr_16764\", \"rank\": 1002, \"title\": \"Journal of Neuroscience\", \"type\": \"journal\", \"issn\": \"02706474, 15292401\", \"sjr\": \"2,350\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 488, \"total_docs_2022\": 707, \"total_docs_3years\": 2323, \"total_refs\": 47444, \"total_cites_3years\": 12615, \"citable_docs_3years\": 2270, \"cites_per_doc_2years\": \"4,96\", \"ref_per_doc\": \"67,11\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Society for Neuroscience\", \"coverage\": \"1981-2022\", \"categories\": \"Neuroscience (miscellaneous) (Q1)\", \"areas\": \"Neuroscience\", \"has_error\": true}, {\"page_id\": \"3_home\", \"is_success\": true, \"error_count\": 0.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 3, \"page_type\": \"home\", \"source_id\": \"sjr_20315\", \"rank\": 3, \"title\": \"Nature Reviews Molecular Cell Biology\", \"type\": \"journal\", \"issn\": \"14710072, 14710080\", \"sjr\": \"34,201\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 485, \"total_docs_2022\": 121, \"total_docs_3years\": 328, \"total_refs\": 10804, \"total_cites_3years\": 13331, \"citable_docs_3years\": 156, \"cites_per_doc_2years\": \"35,47\", \"ref_per_doc\": \"89,29\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Nature Publishing Group\", \"coverage\": \"2000-2022\", \"categories\": \"Cell Biology (Q1); Molecular Biology (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology\", \"has_error\": false}, {\"page_id\": \"3_research_article\", \"is_success\": true, \"error_count\": 0.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 3, \"page_type\": \"research_article\", \"source_id\": \"sjr_20315\", \"rank\": 3, \"title\": \"Nature Reviews Molecular Cell Biology\", \"type\": \"journal\", \"issn\": \"14710072, 14710080\", \"sjr\": \"34,201\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 485, \"total_docs_2022\": 121, \"total_docs_3years\": 328, \"total_refs\": 10804, \"total_cites_3years\": 13331, \"citable_docs_3years\": 156, \"cites_per_doc_2years\": \"35,47\", \"ref_per_doc\": \"89,29\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Nature Publishing Group\", \"coverage\": \"2000-2022\", \"categories\": \"Cell Biology (Q1); Molecular Biology (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology\", \"has_error\": false}, {\"page_id\": \"3_article_search_result\", \"is_success\": true, \"error_count\": 0.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 3, \"page_type\": \"article_search_result\", \"source_id\": \"sjr_20315\", \"rank\": 3, \"title\": \"Nature Reviews Molecular Cell Biology\", \"type\": \"journal\", \"issn\": \"14710072, 14710080\", \"sjr\": \"34,201\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 485, \"total_docs_2022\": 121, \"total_docs_3years\": 328, \"total_refs\": 10804, \"total_cites_3years\": 13331, \"citable_docs_3years\": 156, \"cites_per_doc_2years\": \"35,47\", \"ref_per_doc\": \"89,29\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Nature Publishing Group\", \"coverage\": \"2000-2022\", \"categories\": \"Cell Biology (Q1); Molecular Biology (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology\", \"has_error\": false}, {\"page_id\": \"3_latest_issue\", \"is_success\": true, \"error_count\": 0.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 3, \"page_type\": \"latest_issue\", \"source_id\": \"sjr_20315\", \"rank\": 3, \"title\": \"Nature Reviews Molecular Cell Biology\", \"type\": \"journal\", \"issn\": \"14710072, 14710080\", \"sjr\": \"34,201\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 485, \"total_docs_2022\": 121, \"total_docs_3years\": 328, \"total_refs\": 10804, \"total_cites_3years\": 13331, \"citable_docs_3years\": 156, \"cites_per_doc_2years\": \"35,47\", \"ref_per_doc\": \"89,29\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Nature Publishing Group\", \"coverage\": \"2000-2022\", \"categories\": \"Cell Biology (Q1); Molecular Biology (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology\", \"has_error\": false}, {\"page_id\": \"11_home\", \"is_success\": true, \"error_count\": 0.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 11, \"page_type\": \"home\", \"source_id\": \"sjr_12464\", \"rank\": 11, \"title\": \"Nature Reviews Cancer\", \"type\": \"journal\", \"issn\": \"1474175X, 14741768\", \"sjr\": \"20,977\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 484, \"total_docs_2022\": 111, \"total_docs_3years\": 325, \"total_refs\": 8402, \"total_cites_3years\": 10528, \"citable_docs_3years\": 174, \"cites_per_doc_2years\": \"37,34\", \"ref_per_doc\": \"75,69\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Nature Publishing Group\", \"coverage\": \"2001-2022\", \"categories\": \"Cancer Research (Q1); Oncology (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology; Medicine\", \"has_error\": false}, {\"page_id\": \"11_research_article\", \"is_success\": true, \"error_count\": 0.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 11, \"page_type\": \"research_article\", \"source_id\": \"sjr_12464\", \"rank\": 11, \"title\": \"Nature Reviews Cancer\", \"type\": \"journal\", \"issn\": \"1474175X, 14741768\", \"sjr\": \"20,977\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 484, \"total_docs_2022\": 111, \"total_docs_3years\": 325, \"total_refs\": 8402, \"total_cites_3years\": 10528, \"citable_docs_3years\": 174, \"cites_per_doc_2years\": \"37,34\", \"ref_per_doc\": \"75,69\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Nature Publishing Group\", \"coverage\": \"2001-2022\", \"categories\": \"Cancer Research (Q1); Oncology (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology; Medicine\", \"has_error\": false}, {\"page_id\": \"11_article_search_result\", \"is_success\": true, \"error_count\": 0.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 11, \"page_type\": \"article_search_result\", \"source_id\": \"sjr_12464\", \"rank\": 11, \"title\": \"Nature Reviews Cancer\", \"type\": \"journal\", \"issn\": \"1474175X, 14741768\", \"sjr\": \"20,977\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 484, \"total_docs_2022\": 111, \"total_docs_3years\": 325, \"total_refs\": 8402, \"total_cites_3years\": 10528, \"citable_docs_3years\": 174, \"cites_per_doc_2years\": \"37,34\", \"ref_per_doc\": \"75,69\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Nature Publishing Group\", \"coverage\": \"2001-2022\", \"categories\": \"Cancer Research (Q1); Oncology (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology; Medicine\", \"has_error\": false}, {\"page_id\": \"11_latest_issue\", \"is_success\": true, \"error_count\": 0.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 11, \"page_type\": \"latest_issue\", \"source_id\": \"sjr_12464\", \"rank\": 11, \"title\": \"Nature Reviews Cancer\", \"type\": \"journal\", \"issn\": \"1474175X, 14741768\", \"sjr\": \"20,977\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 484, \"total_docs_2022\": 111, \"total_docs_3years\": 325, \"total_refs\": 8402, \"total_cites_3years\": 10528, \"citable_docs_3years\": 174, \"cites_per_doc_2years\": \"37,34\", \"ref_per_doc\": \"75,69\", \"country\": \"United Kingdom\", \"region\": \"Western Europe\", \"publisher\": \"Nature Publishing Group\", \"coverage\": \"2001-2022\", \"categories\": \"Cancer Research (Q1); Oncology (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology; Medicine\", \"has_error\": false}, {\"page_id\": \"621_home\", \"is_success\": true, \"error_count\": 9.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": 2.0, \"label_empty\": null, \"button_empty\": null, \"link_empty\": 5.0, \"heading_empty\": 2.0, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 621, \"page_type\": \"home\", \"source_id\": \"sjr_29183\", \"rank\": 621, \"title\": \"Cancer Research\", \"type\": \"journal\", \"issn\": \"00085472, 15387445\", \"sjr\": \"2,984\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 483, \"total_docs_2022\": 392, \"total_docs_3years\": 1632, \"total_refs\": 17897, \"total_cites_3years\": 14665, \"citable_docs_3years\": 1617, \"cites_per_doc_2years\": \"7,92\", \"ref_per_doc\": \"45,66\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"American Association for Cancer Research Inc.\", \"coverage\": \"1941-2022\", \"categories\": \"Cancer Research (Q1); Oncology (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology; Medicine\", \"has_error\": true}, {\"page_id\": \"621_research_article\", \"is_success\": true, \"error_count\": 0.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 621, \"page_type\": \"research_article\", \"source_id\": \"sjr_29183\", \"rank\": 621, \"title\": \"Cancer Research\", \"type\": \"journal\", \"issn\": \"00085472, 15387445\", \"sjr\": \"2,984\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 483, \"total_docs_2022\": 392, \"total_docs_3years\": 1632, \"total_refs\": 17897, \"total_cites_3years\": 14665, \"citable_docs_3years\": 1617, \"cites_per_doc_2years\": \"7,92\", \"ref_per_doc\": \"45,66\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"American Association for Cancer Research Inc.\", \"coverage\": \"1941-2022\", \"categories\": \"Cancer Research (Q1); Oncology (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology; Medicine\", \"has_error\": false}, {\"page_id\": \"621_article_search_result\", \"is_success\": false, \"error_count\": null, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 621, \"page_type\": \"article_search_result\", \"source_id\": \"sjr_29183\", \"rank\": 621, \"title\": \"Cancer Research\", \"type\": \"journal\", \"issn\": \"00085472, 15387445\", \"sjr\": \"2,984\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 483, \"total_docs_2022\": 392, \"total_docs_3years\": 1632, \"total_refs\": 17897, \"total_cites_3years\": 14665, \"citable_docs_3years\": 1617, \"cites_per_doc_2years\": \"7,92\", \"ref_per_doc\": \"45,66\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"American Association for Cancer Research Inc.\", \"coverage\": \"1941-2022\", \"categories\": \"Cancer Research (Q1); Oncology (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology; Medicine\", \"has_error\": false}, {\"page_id\": \"621_latest_issue\", \"is_success\": true, \"error_count\": 8.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": 1.0, \"label_empty\": null, \"button_empty\": null, \"link_empty\": 5.0, \"heading_empty\": 2.0, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 621, \"page_type\": \"latest_issue\", \"source_id\": \"sjr_29183\", \"rank\": 621, \"title\": \"Cancer Research\", \"type\": \"journal\", \"issn\": \"00085472, 15387445\", \"sjr\": \"2,984\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 483, \"total_docs_2022\": 392, \"total_docs_3years\": 1632, \"total_refs\": 17897, \"total_cites_3years\": 14665, \"citable_docs_3years\": 1617, \"cites_per_doc_2years\": \"7,92\", \"ref_per_doc\": \"45,66\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"American Association for Cancer Research Inc.\", \"coverage\": \"1941-2022\", \"categories\": \"Cancer Research (Q1); Oncology (Q1)\", \"areas\": \"Biochemistry, Genetics and Molecular Biology; Medicine\", \"has_error\": true}, {\"page_id\": \"179_home\", \"is_success\": true, \"error_count\": 3.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": 3.0, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 179, \"page_type\": \"home\", \"source_id\": \"sjr_21272\", \"rank\": 179, \"title\": \"Journal of Experimental Medicine\", \"type\": \"journal\", \"issn\": \"00221007, 15409538\", \"sjr\": \"6,237\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 478, \"total_docs_2022\": 169, \"total_docs_3years\": 858, \"total_refs\": 11168, \"total_cites_3years\": 10764, \"citable_docs_3years\": 828, \"cites_per_doc_2years\": \"12,31\", \"ref_per_doc\": \"66,08\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Rockefeller University Press\", \"coverage\": \"1909-1910, 1929, 1938, 1945-2022\", \"categories\": \"Immunology (Q1); Immunology and Allergy (Q1); Medicine (miscellaneous) (Q1)\", \"areas\": \"Immunology and Microbiology; Medicine\", \"has_error\": true}, {\"page_id\": \"179_research_article\", \"is_success\": false, \"error_count\": null, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": 3.0, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 179, \"page_type\": \"research_article\", \"source_id\": \"sjr_21272\", \"rank\": 179, \"title\": \"Journal of Experimental Medicine\", \"type\": \"journal\", \"issn\": \"00221007, 15409538\", \"sjr\": \"6,237\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 478, \"total_docs_2022\": 169, \"total_docs_3years\": 858, \"total_refs\": 11168, \"total_cites_3years\": 10764, \"citable_docs_3years\": 828, \"cites_per_doc_2years\": \"12,31\", \"ref_per_doc\": \"66,08\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Rockefeller University Press\", \"coverage\": \"1909-1910, 1929, 1938, 1945-2022\", \"categories\": \"Immunology (Q1); Immunology and Allergy (Q1); Medicine (miscellaneous) (Q1)\", \"areas\": \"Immunology and Microbiology; Medicine\", \"has_error\": false}, {\"page_id\": \"179_none_research_article\", \"is_success\": true, \"error_count\": 0.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": null, \"label_empty\": null, \"button_empty\": null, \"link_empty\": null, \"heading_empty\": null, \"alt_input_missing\": null, \"label_missing\": null, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 179, \"page_type\": \"none_research_article\", \"source_id\": \"sjr_21272\", \"rank\": 179, \"title\": \"Journal of Experimental Medicine\", \"type\": \"journal\", \"issn\": \"00221007, 15409538\", \"sjr\": \"6,237\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 478, \"total_docs_2022\": 169, \"total_docs_3years\": 858, \"total_refs\": 11168, \"total_cites_3years\": 10764, \"citable_docs_3years\": 828, \"cites_per_doc_2years\": \"12,31\", \"ref_per_doc\": \"66,08\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Rockefeller University Press\", \"coverage\": \"1909-1910, 1929, 1938, 1945-2022\", \"categories\": \"Immunology (Q1); Immunology and Allergy (Q1); Medicine (miscellaneous) (Q1)\", \"areas\": \"Immunology and Microbiology; Medicine\", \"has_error\": false}, {\"page_id\": \"179_article_search_result\", \"is_success\": true, \"error_count\": 54.0, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": 5.0, \"label_empty\": null, \"button_empty\": null, \"link_empty\": 5.0, \"heading_empty\": 5.0, \"alt_input_missing\": null, \"label_missing\": 39.0, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 179, \"page_type\": \"article_search_result\", \"source_id\": \"sjr_21272\", \"rank\": 179, \"title\": \"Journal of Experimental Medicine\", \"type\": \"journal\", \"issn\": \"00221007, 15409538\", \"sjr\": \"6,237\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 478, \"total_docs_2022\": 169, \"total_docs_3years\": 858, \"total_refs\": 11168, \"total_cites_3years\": 10764, \"citable_docs_3years\": 828, \"cites_per_doc_2years\": \"12,31\", \"ref_per_doc\": \"66,08\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Rockefeller University Press\", \"coverage\": \"1909-1910, 1929, 1938, 1945-2022\", \"categories\": \"Immunology (Q1); Immunology and Allergy (Q1); Medicine (miscellaneous) (Q1)\", \"areas\": \"Immunology and Microbiology; Medicine\", \"has_error\": true}, {\"page_id\": \"179_latest_issue\", \"is_success\": false, \"error_count\": null, \"aria_reference_broken\": null, \"label_multiple\": null, \"alt_missing\": null, \"alt_link_missing\": 5.0, \"label_empty\": null, \"button_empty\": null, \"link_empty\": 5.0, \"heading_empty\": 5.0, \"alt_input_missing\": null, \"label_missing\": 39.0, \"aria_menu_broken\": null, \"link_skip_broken\": null, \"alt_spacer_missing\": null, \"th_empty\": null, \"title_invalid\": null, \"id\": 179, \"page_type\": \"latest_issue\", \"source_id\": \"sjr_21272\", \"rank\": 179, \"title\": \"Journal of Experimental Medicine\", \"type\": \"journal\", \"issn\": \"00221007, 15409538\", \"sjr\": \"6,237\", \"sjr_best_quartile\": \"Q1\", \"h_index\": 478, \"total_docs_2022\": 169, \"total_docs_3years\": 858, \"total_refs\": 11168, \"total_cites_3years\": 10764, \"citable_docs_3years\": 828, \"cites_per_doc_2years\": \"12,31\", \"ref_per_doc\": \"66,08\", \"country\": \"United States\", \"region\": \"Northern America\", \"publisher\": \"Rockefeller University Press\", \"coverage\": \"1909-1910, 1929, 1938, 1945-2022\", \"categories\": \"Immunology (Q1); Immunology and Allergy (Q1); Medicine (miscellaneous) (Q1)\", \"areas\": \"Immunology and Microbiology; Medicine\", \"has_error\": false}]}}, {\"mode\": \"vega-lite\"});\n",
       "</script>"
      ],
      "text/plain": [
       "alt.HConcatChart(...)"
      ]
     },
     "execution_count": 33,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "alt.Chart(df).mark_bar().encode(\n",
    "    alt.Y('sum(error_count):Q', title='A Total of Error Counts'),\n",
    "    alt.X('page_type:N', title='Page Type'),\n",
    ") | alt.Chart(df).mark_point().encode(\n",
    "    alt.Y('error_count'),\n",
    "    alt.X('total_cites_3years:Q'),\n",
    ") | alt.Chart(df).mark_bar().encode(\n",
    "    alt.Y('mean(error_count):Q', title='The Average of Error Counts'),\n",
    "    alt.X('country:N'),\n",
    ") | alt.Chart(df).mark_bar().encode(\n",
    "    alt.Y('count(error_count):Q', title='The Number of Data Portals'),\n",
    "    alt.X('areas:N'),\n",
    ") | alt.Chart(df).mark_bar().encode(\n",
    "    alt.Y('mean(alt_missing):Q', title='The Average of Alt-text Missing'),\n",
    "    alt.X('publisher:N'),\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "life-sciences-a11y-evaluation",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
